{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Bismark practice with C. virginica 2018-04-07 data\n",
    "Bismark paper:  \n",
    "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3102221/  \n",
    "Bismark user guide:  \n",
    "https://rawgit.com/FelixKrueger/Bismark/master/Docs/Bismark_User_Guide.html  \n",
    "\n",
    "First, needed to install bowtie2, samtools, and bismark locally. I already had anaconda3 on my computer, so I just needed to run the following commands:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!conda install -c bioconda bowtie2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!conda install -c bioconda samtools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!conda install -c bioconda bismark "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "FIRST:  connect to Gannett server; this will mount as /Volumes/web\n",
    "SECOND:  connect to OWL server; this will mount as /Volumes/web-1\n",
    "Then make new directories for bismark analysis in the 'metacarcinus' folder on seashell"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
    "pwd "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "copy genome to working directory folder"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "cp /Volumes/web-1/halfshell/genomic-databank/Cvirginica_v300.fa /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "run bismark genome preparation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Path to genome folder specified as: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "Aligner to be used: Bowtie 2 (default)\n",
      "Writing bisulfite genomes out into a single MFA (multi FastA) file\n",
      "\n",
      "Bismark Genome Preparation - Step I: Preparing folders\n",
      "\n",
      "Path to Bowtie 2 specified: /Users/Shelly/anaconda3/bin/\n",
      "Bisulfite Genome Indexer version v0.20.0 (last modified 26 April 2018)\n",
      "Created Bisulfite Genome folder /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/Bisulfite_Genome/\n",
      "Created Bisulfite Genome folder /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/Bisulfite_Genome/CT_conversion/\n",
      "Created Bisulfite Genome folder /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/Bisulfite_Genome/GA_conversion/\n",
      "\n",
      "Step I - Prepare genome folders - completed\n",
      "\n",
      "\n",
      "Bismark Genome Preparation - Step II: Bisulfite converting reference genome\n",
      "\n",
      "conversions performed:\n",
      "chromosome\tC->T\tG->A\n",
      "NC_035780.1\t11492316\t11528162\n",
      "NC_035781.1\t10879823\t10858285\n",
      "NC_035782.1\t13475475\t13496235\n",
      "NC_035783.1\t10484268\t10482193\n",
      "NC_035784.1\t17306783\t17253434\n",
      "NC_035785.1\t8878985\t8876470\n",
      "NC_035786.1\t9961761\t9989843\n",
      "NC_035787.1\t13145205\t13164212\n",
      "NC_035788.1\t17925605\t17932896\n",
      "NC_035789.1\t5676071\t5682065\n",
      "NC_007175.2\t2827\t3882\n",
      "\n",
      "Total number of conversions performed:\n",
      "C->T:\t119229119\n",
      "G->A:\t119267677\n",
      "\n",
      "Step II - Genome bisulfite conversions - completed\n",
      "\n",
      "\n",
      "Bismark Genome Preparation - Step III: Launching the Bowtie 2 indexer\n",
      "Please be aware that this process can - depending on genome size - take several hours!\n",
      "\n",
      "Preparing indexing of CT converted genome in /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/Bisulfite_Genome/CT_conversion/\n",
      "Parent process: Starting to index C->T converted genome with the following command:\n",
      "\n",
      "/Users/Shelly/anaconda3/bin/bowtie2-build -f genome_mfa.CT_conversion.fa BS_CT\n",
      "\n",
      "Preparing indexing of GA converted genome in /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/Bisulfite_Genome/GA_conversion/\n",
      "Child process: Starting to index G->A converted genome with the following command:\n",
      "\n",
      "/Users/Shelly/anaconda3/bin/bowtie2-build -f genome_mfa.GA_conversion.fa BS_GA\n",
      "\n",
      "Settings:\n",
      "  Output files: \"BS_CT.*.bt2\"\n",
      "  Line rate: 6 (line is 64 bytes)\n",
      "  Lines per side: 1 (side is 64 bytes)\n",
      "  Offset rate: 4 (one in 16)\n",
      "  FTable chars: 10\n",
      "  Strings: unpacked\n",
      "  Max bucket size: default\n",
      "  Max bucket size, sqrt multiplier: default\n",
      "  Max bucket size, len divisor: 4\n",
      "  Difference-cover sample period: 1024\n",
      "  Endianness: little\n",
      "  Actual local endianness: little\n",
      "  Sanity checking: disabled\n",
      "  Assertions: disabled\n",
      "  Random seed: 0\n",
      "  Sizeofs: void*:8, int:4, long:8, size_t:8\n",
      "Input files DNA, FASTA:\n",
      "  genome_mfa.CT_conversion.fa\n",
      "Building a SMALL index\n",
      "Reading reference sizes\n",
      "Settings:\n",
      "  Output files: \"BS_GA.*.bt2\"\n",
      "  Line rate: 6 (line is 64 bytes)\n",
      "  Lines per side: 1 (side is 64 bytes)\n",
      "  Offset rate: 4 (one in 16)\n",
      "  FTable chars: 10\n",
      "  Strings: unpacked\n",
      "  Max bucket size: default\n",
      "  Max bucket size, sqrt multiplier: default\n",
      "  Max bucket size, len divisor: 4\n",
      "  Difference-cover sample period: 1024\n",
      "  Endianness: little\n",
      "  Actual local endianness: little\n",
      "  Sanity checking: disabled\n",
      "  Assertions: disabled\n",
      "  Random seed: 0\n",
      "  Sizeofs: void*:8, int:4, long:8, size_t:8\n",
      "Input files DNA, FASTA:\n",
      "  genome_mfa.GA_conversion.fa\n",
      "Building a SMALL index\n",
      "Reading reference sizes\n",
      "  Time reading reference sizes: 00:01:55\n",
      "Calculating joined length\n",
      "Writing header\n",
      "Reserving space for joined string\n",
      "Joining reference sequences\n",
      "  Time reading reference sizes: 00:01:55\n",
      "Calculating joined length\n",
      "Writing header\n",
      "Reserving space for joined string\n",
      "Joining reference sequences\n",
      "  Time to join reference sequences: 00:00:12\n",
      "bmax according to bmaxDivN setting: 171168832\n",
      "Using parameters --bmax 128376624 --dcv 1024\n",
      "  Doing ahead-of-time memory usage test\n",
      "  Passed!  Constructing with these parameters: --bmax 128376624 --dcv 1024\n",
      "Constructing suffix-array element generator\n",
      "Building DifferenceCoverSample\n",
      "  Building sPrime\n",
      "  Time to join reference sequences: 00:00:13\n",
      "bmax according to bmaxDivN setting: 171168832\n",
      "Using parameters --bmax 128376624 --dcv 1024\n",
      "  Doing ahead-of-time memory usage test\n",
      "  Passed!  Constructing with these parameters: --bmax 128376624 --dcv 1024\n",
      "Constructing suffix-array element generator\n",
      "Building DifferenceCoverSample\n",
      "  Building sPrime\n",
      "  Building sPrimeOrder\n",
      "  Building sPrimeOrder\n",
      "  V-Sorting samples\n",
      "  V-Sorting samples\n",
      "  V-Sorting samples time: 00:00:47\n",
      "  Allocating rank array\n",
      "  Ranking v-sort output\n",
      "  V-Sorting samples time: 00:00:47\n",
      "  Allocating rank array\n",
      "  Ranking v-sort output\n",
      "  Ranking v-sort output time: 00:00:11\n",
      "  Invoking Larsson-Sadakane on ranks\n",
      "  Ranking v-sort output time: 00:00:11\n",
      "  Invoking Larsson-Sadakane on ranks\n",
      "  Invoking Larsson-Sadakane on ranks time: 00:00:16\n",
      "  Sanity-checking and returning\n",
      "Building samples\n",
      "Reserving space for 12 sample suffixes\n",
      "Generating random suffixes\n",
      "QSorting 12 sample offsets, eliminating duplicates\n",
      "QSorting sample offsets, eliminating duplicates time: 00:00:00\n",
      "Multikey QSorting 12 samples\n",
      "  (Using difference cover)\n",
      "  Multikey QSorting samples time: 00:00:00\n",
      "Calculating bucket sizes\n",
      "Splitting and merging\n",
      "  Splitting and merging time: 00:00:00\n",
      "Avg bucket size: 6.84675e+08 (target: 128376623)\n",
      "Converting suffix-array elements to index image\n",
      "Allocating ftab, absorbFtab\n",
      "Entering Ebwt loop\n",
      "Getting block 1 of 1\n",
      "  No samples; assembling all-inclusive block\n",
      "  Invoking Larsson-Sadakane on ranks time: 00:00:16\n",
      "  Sanity-checking and returning\n",
      "Building samples\n",
      "Reserving space for 12 sample suffixes\n",
      "Generating random suffixes\n",
      "QSorting 12 sample offsets, eliminating duplicates\n",
      "QSorting sample offsets, eliminating duplicates time: 00:00:00\n",
      "Multikey QSorting 12 samples\n",
      "  (Using difference cover)\n",
      "  Multikey QSorting samples time: 00:00:00\n",
      "Calculating bucket sizes\n",
      "Splitting and merging\n",
      "  Splitting and merging time: 00:00:00\n",
      "Avg bucket size: 6.84675e+08 (target: 128376623)\n",
      "Converting suffix-array elements to index image\n",
      "Allocating ftab, absorbFtab\n",
      "Entering Ebwt loop\n",
      "Getting block 1 of 1\n",
      "  No samples; assembling all-inclusive block\n",
      "  Sorting block of length 684675328 for bucket 1\n",
      "  (Using difference cover)\n",
      "  Sorting block of length 684675328 for bucket 1\n",
      "  (Using difference cover)\n",
      "  Sorting block time: 00:13:58\n",
      "Returning block of 684675329 for bucket 1\n",
      "  Sorting block time: 00:16:06\n",
      "Returning block of 684675329 for bucket 1\n",
      "Exited Ebwt loop\n",
      "fchr[A]: 0\n",
      "fchr[C]: 342402452\n",
      "fchr[G]: 461631571\n",
      "fchr[T]: 461631571\n",
      "fchr[$]: 684675328\n",
      "Exiting Ebwt::buildToDisk()\n",
      "Returning from initFromVector\n",
      "Wrote 232427948 bytes to primary EBWT file: BS_GA.1.bt2\n",
      "Wrote 171168840 bytes to secondary EBWT file: BS_GA.2.bt2\n",
      "Re-opening _in1 and _in2 as input streams\n",
      "Returning from Ebwt constructor\n",
      "Headers:\n",
      "    len: 684675328\n",
      "    bwtLen: 684675329\n",
      "    sz: 171168832\n",
      "    bwtSz: 171168833\n",
      "    lineRate: 6\n",
      "    offRate: 4\n",
      "    offMask: 0xfffffff0\n",
      "    ftabChars: 10\n",
      "    eftabLen: 20\n",
      "    eftabSz: 80\n",
      "    ftabLen: 1048577\n",
      "    ftabSz: 4194308\n",
      "    offsLen: 42792209\n",
      "    offsSz: 171168836\n",
      "    lineSz: 64\n",
      "    sideSz: 64\n",
      "    sideBwtSz: 48\n",
      "    sideBwtLen: 192\n",
      "    numSides: 3566018\n",
      "    numLines: 3566018\n",
      "    ebwtTotLen: 228225152\n",
      "    ebwtTotSz: 228225152\n",
      "    color: 0\n",
      "    reverse: 0\n",
      "Total time for call to driver() for forward index: 00:19:59\n",
      "Reading reference sizes\n",
      "  Time reading reference sizes: 00:00:08\n",
      "Calculating joined length\n",
      "Writing header\n",
      "Reserving space for joined string\n",
      "Joining reference sequences\n",
      "  Time to join reference sequences: 00:00:06\n",
      "  Time to reverse reference sequence: 00:00:01\n",
      "bmax according to bmaxDivN setting: 171168832\n",
      "Using parameters --bmax 128376624 --dcv 1024\n",
      "  Doing ahead-of-time memory usage test\n",
      "  Passed!  Constructing with these parameters: --bmax 128376624 --dcv 1024\n",
      "Constructing suffix-array element generator\n",
      "Building DifferenceCoverSample\n",
      "  Building sPrime\n",
      "  Building sPrimeOrder\n",
      "  V-Sorting samples\n",
      "  V-Sorting samples time: 00:00:25\n",
      "  Allocating rank array\n",
      "  Ranking v-sort output\n",
      "  Ranking v-sort output time: 00:00:07\n",
      "  Invoking Larsson-Sadakane on ranks\n",
      "  Invoking Larsson-Sadakane on ranks time: 00:00:10\n",
      "  Sanity-checking and returning\n",
      "Building samples\n",
      "Reserving space for 12 sample suffixes\n",
      "Generating random suffixes\n",
      "QSorting 12 sample offsets, eliminating duplicates\n",
      "QSorting sample offsets, eliminating duplicates time: 00:00:00\n",
      "Multikey QSorting 12 samples\n",
      "  (Using difference cover)\n",
      "  Multikey QSorting samples time: 00:00:00\n",
      "Calculating bucket sizes\n",
      "Splitting and merging\n",
      "  Splitting and merging time: 00:00:00\n",
      "Avg bucket size: 6.84675e+08 (target: 128376623)\n",
      "Converting suffix-array elements to index image\n",
      "Allocating ftab, absorbFtab\n",
      "Entering Ebwt loop\n",
      "Getting block 1 of 1\n",
      "  No samples; assembling all-inclusive block\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  Sorting block of length 684675328 for bucket 1\n",
      "  (Using difference cover)\n",
      "Exited Ebwt loop\n",
      "fchr[A]: 0\n",
      "fchr[C]: 223134775\n",
      "fchr[G]: 223134775\n",
      "fchr[T]: 342402452\n",
      "fchr[$]: 684675328\n",
      "Exiting Ebwt::buildToDisk()\n",
      "Returning from initFromVector\n",
      "Wrote 232427948 bytes to primary EBWT file: BS_CT.1.bt2\n",
      "Wrote 171168840 bytes to secondary EBWT file: BS_CT.2.bt2\n",
      "Re-opening _in1 and _in2 as input streams\n",
      "Returning from Ebwt constructor\n",
      "Headers:\n",
      "    len: 684675328\n",
      "    bwtLen: 684675329\n",
      "    sz: 171168832\n",
      "    bwtSz: 171168833\n",
      "    lineRate: 6\n",
      "    offRate: 4\n",
      "    offMask: 0xfffffff0\n",
      "    ftabChars: 10\n",
      "    eftabLen: 20\n",
      "    eftabSz: 80\n",
      "    ftabLen: 1048577\n",
      "    ftabSz: 4194308\n",
      "    offsLen: 42792209\n",
      "    offsSz: 171168836\n",
      "    lineSz: 64\n",
      "    sideSz: 64\n",
      "    sideBwtSz: 48\n",
      "    sideBwtLen: 192\n",
      "    numSides: 3566018\n",
      "    numLines: 3566018\n",
      "    ebwtTotLen: 228225152\n",
      "    ebwtTotSz: 228225152\n",
      "    color: 0\n",
      "    reverse: 0\n",
      "Total time for call to driver() for forward index: 00:22:02\n",
      "Reading reference sizes\n",
      "  Time reading reference sizes: 00:00:09\n",
      "Calculating joined length\n",
      "Writing header\n",
      "Reserving space for joined string\n",
      "Joining reference sequences\n",
      "  Time to join reference sequences: 00:00:06\n",
      "  Time to reverse reference sequence: 00:00:00\n",
      "bmax according to bmaxDivN setting: 171168832\n",
      "Using parameters --bmax 128376624 --dcv 1024\n",
      "  Doing ahead-of-time memory usage test\n",
      "  Passed!  Constructing with these parameters: --bmax 128376624 --dcv 1024\n",
      "Constructing suffix-array element generator\n",
      "Building DifferenceCoverSample\n",
      "  Building sPrime\n",
      "  Building sPrimeOrder\n",
      "  V-Sorting samples\n",
      "  V-Sorting samples time: 00:00:25\n",
      "  Allocating rank array\n",
      "  Ranking v-sort output\n",
      "  Ranking v-sort output time: 00:00:08\n",
      "  Invoking Larsson-Sadakane on ranks\n",
      "  Invoking Larsson-Sadakane on ranks time: 00:00:12\n",
      "  Sanity-checking and returning\n",
      "Building samples\n",
      "Reserving space for 12 sample suffixes\n",
      "Generating random suffixes\n",
      "QSorting 12 sample offsets, eliminating duplicates\n",
      "QSorting sample offsets, eliminating duplicates time: 00:00:00\n",
      "Multikey QSorting 12 samples\n",
      "  (Using difference cover)\n",
      "  Multikey QSorting samples time: 00:00:00\n",
      "Calculating bucket sizes\n",
      "Splitting and merging\n",
      "  Splitting and merging time: 00:00:00\n",
      "Avg bucket size: 6.84675e+08 (target: 128376623)\n",
      "Converting suffix-array elements to index image\n",
      "Allocating ftab, absorbFtab\n",
      "Entering Ebwt loop\n",
      "Getting block 1 of 1\n",
      "  No samples; assembling all-inclusive block\n",
      "  Sorting block of length 684675328 for bucket 1\n",
      "  (Using difference cover)\n",
      "  Sorting block time: 00:11:33\n",
      "Returning block of 684675329 for bucket 1\n",
      "Exited Ebwt loop\n",
      "fchr[A]: 0\n",
      "fchr[C]: 342402452\n",
      "fchr[G]: 461631571\n",
      "fchr[T]: 461631571\n",
      "fchr[$]: 684675328\n",
      "Exiting Ebwt::buildToDisk()\n",
      "Returning from initFromVector\n",
      "Wrote 232427948 bytes to primary EBWT file: BS_GA.rev.1.bt2\n",
      "Wrote 171168840 bytes to secondary EBWT file: BS_GA.rev.2.bt2\n",
      "Re-opening _in1 and _in2 as input streams\n",
      "Returning from Ebwt constructor\n",
      "Headers:\n",
      "    len: 684675328\n",
      "    bwtLen: 684675329\n",
      "    sz: 171168832\n",
      "    bwtSz: 171168833\n",
      "    lineRate: 6\n",
      "    offRate: 4\n",
      "    offMask: 0xfffffff0\n",
      "    ftabChars: 10\n",
      "    eftabLen: 20\n",
      "    eftabSz: 80\n",
      "    ftabLen: 1048577\n",
      "    ftabSz: 4194308\n",
      "    offsLen: 42792209\n",
      "    offsSz: 171168836\n",
      "    lineSz: 64\n",
      "    sideSz: 64\n",
      "    sideBwtSz: 48\n",
      "    sideBwtLen: 192\n",
      "    numSides: 3566018\n",
      "    numLines: 3566018\n",
      "    ebwtTotLen: 228225152\n",
      "    ebwtTotSz: 228225152\n",
      "    color: 0\n",
      "    reverse: 1\n",
      "Total time for backward call to driver() for mirror index: 00:15:02\n",
      "  Sorting block time: 00:14:01\n",
      "Returning block of 684675329 for bucket 1\n",
      "Exited Ebwt loop\n",
      "fchr[A]: 0\n",
      "fchr[C]: 223134775\n",
      "fchr[G]: 223134775\n",
      "fchr[T]: 342402452\n",
      "fchr[$]: 684675328\n",
      "Exiting Ebwt::buildToDisk()\n",
      "Returning from initFromVector\n",
      "Wrote 232427948 bytes to primary EBWT file: BS_CT.rev.1.bt2\n",
      "Wrote 171168840 bytes to secondary EBWT file: BS_CT.rev.2.bt2\n",
      "Re-opening _in1 and _in2 as input streams\n",
      "Returning from Ebwt constructor\n",
      "Headers:\n",
      "    len: 684675328\n",
      "    bwtLen: 684675329\n",
      "    sz: 171168832\n",
      "    bwtSz: 171168833\n",
      "    lineRate: 6\n",
      "    offRate: 4\n",
      "    offMask: 0xfffffff0\n",
      "    ftabChars: 10\n",
      "    eftabLen: 20\n",
      "    eftabSz: 80\n",
      "    ftabLen: 1048577\n",
      "    ftabSz: 4194308\n",
      "    offsLen: 42792209\n",
      "    offsSz: 171168836\n",
      "    lineSz: 64\n",
      "    sideSz: 64\n",
      "    sideBwtSz: 48\n",
      "    sideBwtLen: 192\n",
      "    numSides: 3566018\n",
      "    numLines: 3566018\n",
      "    ebwtTotLen: 228225152\n",
      "    ebwtTotSz: 228225152\n",
      "    color: 0\n",
      "    reverse: 1\n",
      "Total time for backward call to driver() for mirror index: 00:17:24\n"
     ]
    }
   ],
   "source": [
    "!bismark_genome_preparation --path_to_bowtie /Users/Shelly/anaconda3/bin/ \\\n",
    "--verbose /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Then I ran bismark subsetting only the first 10K reads from the first two .fastqs, specifying reads are non-directional, and using a minimum score threshold that Steven previously used"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "FastQ format assumed (by default)\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz: uncompress failed\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t99\tNC_035789.1_CT_converted\t4531039\t2\t37M4I59M\t=\t4531139\t200\tGAGTTTTTTTGATTATTTGTTGTTTGTTGTTTGTTTGNNNNTNNNTNNGTTTGTTTGTTTGTTTGTTTGTTTGTAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tAS:i:-22\tXS:i:-31\tXN:i:0\tXM:i:5\tXO:i:1\tXG:i:4\tNM:i:9\tMD:Z:38T0T0G1T0T52\tYS:i:-24\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t147\tNC_035789.1_CT_converted\t4531139\t2\t100M\t=\t4531039\t-200\tGAATTATTGGGTTAATTTTAATTAAATTTGATATAAAGTATTTTTAGGTGAAGGGGATTTAAGTTTGTTTGAATGAAGGGTTTTGTTTTTTTTTAAGGGG\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-24\tXS:i:-12\tXN:i:0\tXM:i:4\tXO:i:0\tXG:i:0\tNM:i:4\tMD:Z:30G14G9A14A29\tYS:i:-22\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTAAAAAAAAATGAAATTTTTTATTTAAATAAATTTAAATTTTTTTTATTTAAAAATATTTTATATTAAATTTAATTAAAATTAATTTAATAATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTAAAAAAAAATGAAATTTTTTATTTAAATAAATTTAAATTTTTTTTATTTAAAAATATTTTATATTAAATTTAATTAAAATTAATTTAATAATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t83\tNC_035780.1_GA_converted\t8090288\t6\t26M4D74M\t=\t8090028\t-364\tAAAAAAAATAAAAATATAAAAAATTTACAAACAAACAAACAAACAAACAAACNNANNNANNNNCAAACAAACAACAAACAACAAATAATCAAAAAAACTC\tFFFFFFFF/FFFFF/B/FFFFFF/</FFFFFF<FFFFFFFFFFFFFFFF<<<##<###<####FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-26\tXS:i:-9\tXN:i:0\tXM:i:9\tXO:i:1\tXG:i:4\tNM:i:13\tMD:Z:26^ACAA26A0A1C0A0A1C0A0A0A37\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t163\tNC_035780.1_GA_converted\t8090028\t6\t100M\t=\t8090288\t364\tCCCCTTAAAAAAAAACAAAACCCTTCATTCAAACAAACTTAAATCCCCTTCACCTAAAAATACTTTATATCAAATTTAATTAAAATTAACCCAATAATTC\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-18\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:29T70\tYS:i:-26\tYT:Z:CP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_1_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    93584 (93.58%) aligned concordantly 0 times\n",
      "    3180 (3.18%) aligned concordantly exactly 1 time\n",
      "    3236 (3.24%) aligned concordantly >1 times\n",
      "6.42% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    93560 (93.56%) aligned concordantly 0 times\n",
      "    3220 (3.22%) aligned concordantly exactly 1 time\n",
      "    3220 (3.22%) aligned concordantly >1 times\n",
      "6.44% overall alignment rate\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    93582 (93.58%) aligned concordantly 0 times\n",
      "    3175 (3.17%) aligned concordantly exactly 1 time\n",
      "    3243 (3.24%) aligned concordantly >1 times\n",
      "6.42% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    93611 (93.61%) aligned concordantly 0 times\n",
      "    3157 (3.16%) aligned concordantly exactly 1 time\n",
      "    3232 (3.23%) aligned concordantly >1 times\n",
      "6.39% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_1_s1_R1.fastq.gz_C_to_T.fastq, zr2096_1_s1_R1.fastq.gz_G_to_A.fastq, zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Number of paired-end alignments with a unique best hit:\t13744\n",
      "Mapping efficiency:\t13.7%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t81302\n",
      "Sequence pairs did not map uniquely:\t4954\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t3402\t((converted) top strand)\n",
      "GA/CT/CT:\t3480\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t3458\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t3404\t((converted) bottom strand)\n",
      "\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t481569\n",
      "\n",
      "Total methylated C's in CpG context:\t47945\n",
      "Total methylated C's in CHG context:\t2774\n",
      "Total methylated C's in CHH context:\t7846\n",
      "Total methylated C's in Unknown context:\t624\n",
      "\n",
      "Total unmethylated C's in CpG context:\t17126\n",
      "Total unmethylated C's in CHG context:\t107925\n",
      "Total unmethylated C's in CHH context:\t297953\n",
      "Total unmethylated C's in Unknown context:\t1993\n",
      "\n",
      "C methylated in CpG context:\t73.7%\n",
      "C methylated in CHG context:\t2.5%\n",
      "C methylated in CHH context:\t2.6%\n",
      "C methylated in unknown context (CN or CHN):\t23.8%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 3m 43s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz: uncompress failed\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz: uncompress failed\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz: uncompress failed\n"
     ]
    }
   ],
   "source": [
    "!bismark -u 100000 --non_directional --score_min L,0,-0.6 \\\n",
    "--genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ \\\n",
    "-1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz \\\n",
    "-2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz \\\n",
    "-o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/ "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "run Bismark Methylation extractor"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      " *** Bismark methylation extractor version v0.20.0 ***\n",
      "\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/\n",
      "Trying to determine the type of mapping from the SAM header line of file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.bam\n",
      "Treating file(s) as paired-end data (as extracted from @PG line)\n",
      "\n",
      "Setting option '--no_overlap' since this is (normally) the right thing to do for paired-end data\n",
      "\n",
      "Setting core usage to single-threaded (default). Consider using --multicore <int> to speed up the extraction process.\n",
      "\n",
      "Summarising Bismark methylation extractor parameters:\n",
      "===============================================================\n",
      "Bismark paired-end SAM format specified (default)\n",
      "Number of cores to be used: 1\n",
      "Output path specified as: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/\n",
      "\n",
      "\n",
      "Summarising bedGraph parameters:\n",
      "===============================================================\n",
      "Generating additional output in bedGraph and coverage format\n",
      "bedGraph format:\t<Chromosome> <Start Position> <End Position> <Methylation Percentage>\n",
      "coverage format:\t<Chromosome> <Start Position> <End Position> <Methylation Percentage> <count methylated> <count non-methylated>\n",
      "\n",
      "Using a cutoff of 1 read(s) to report cytosine positions\n",
      "Reporting and sorting cytosine methylation information in CpG context only (default)\n",
      "The bedGraph UNIX sort command will use the following memory setting:\t'2G'. Temporary directory used for sorting is the output directory\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "Now testing Bismark result file >/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.bam< for positional sorting (which would be bad...)\t...passed!\n",
      "Writing result file containing methylation information for C in CpG context from the original top strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Writing result file containing methylation information for C in CpG context from the complementary to original top strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Writing result file containing methylation information for C in CpG context from the complementary to original bottom strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Writing result file containing methylation information for C in CpG context from the original bottom strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "\n",
      "Writing result file containing methylation information for C in CHG context from the original top strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Writing result file containing methylation information for C in CHG context from the complementary to original top strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Writing result file containing methylation information for C in CHG context from the complementary to original bottom strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Writing result file containing methylation information for C in CHG context from the original bottom strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "\n",
      "Writing result file containing methylation information for C in CHH context from the original top strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Writing result file containing methylation information for C in CHH context from the complementary to original top strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Writing result file containing methylation information for C in CHH context from the complementary to original bottom strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Writing result file containing methylation information for C in CHH context from the original bottom strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "\n",
      "Now reading in Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.bam\n",
      "skipping SAM header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping SAM header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping SAM header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping SAM header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping SAM header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping SAM header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping SAM header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping SAM header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping SAM header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping SAM header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping SAM header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping SAM header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping SAM header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark -u 100000 --non_directional --score_min L,0,-0.6 --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/\"\n",
      "\n",
      "Processed 13744 lines in total\n",
      "Total number of methylation call strings processed: 27488\n",
      "\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t304266\n",
      "\n",
      "Total methylated C's in CpG context:\t29931\n",
      "Total methylated C's in CHG context:\t1873\n",
      "Total methylated C's in CHH context:\t5172\n",
      "\n",
      "Total C to T conversions in CpG context:\t10904\n",
      "Total C to T conversions in CHG context:\t67806\n",
      "Total C to T conversions in CHH context:\t188580\n",
      "\n",
      "C methylated in CpG context:\t73.3%\n",
      "C methylated in CHG context:\t2.7%\n",
      "C methylated in CHH context:\t2.7%\n",
      "\n",
      "\n",
      "Determining maximum read lengths for M-Bias plots\n",
      "Maximum read length of Read 1: 100\n",
      "Maximum read length of Read 2: 100\n",
      "\n",
      "Perl module GD::Graph::lines is not installed, skipping drawing M-bias plots (only writing out M-bias plot table)\n",
      "Determining maximum read lengths for M-Bias plots\n",
      "Maximum read length of Read 1: 100\n",
      "Maximum read length of Read 2: 100\n",
      "\n",
      "Perl module GD::Graph::lines is not installed, skipping drawing M-bias plots (only writing out M-bias plot table)\n",
      "Deleting unused files ...\n",
      "\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "gunzip: error writing to output: Broken pipe\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "\n",
      "\n",
      "Using these input files: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "\n",
      "Summary of parameters for bismark2bedGraph conversion:\n",
      "======================================================\n",
      "bedGraph output:\t\tzr2096_1_s1_R1_bismark_bt2_pe.bedGraph.gz\n",
      "output directory:\t\t>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/<\n",
      "remove whitespaces:\t\tno\n",
      "CX context:\t\t\tno (CpG context only, default)\n",
      "No-header selected:\t\tno\n",
      "Sorting method:\t\t\tUnix sort-based (smaller memory footprint, but slower)\n",
      "Sort buffer size:\t\t2G\n",
      "Coverage threshold:\t\t1\n",
      "=============================================================================\n",
      "Methylation information will now be written into a bedGraph and coverage file\n",
      "=============================================================================\n",
      "\n",
      "Using the following files as Input:\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "\n",
      "Writing bedGraph to file: zr2096_1_s1_R1_bismark_bt2_pe.bedGraph.gz\n",
      "Also writing out a coverage file including counts methylated and unmethylated residues to file: zr2096_1_s1_R1_bismark_bt2_pe.bismark.cov.gz\n",
      "\n",
      "Changed directory to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/\n",
      "Now writing methylation information for file >>CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz<< to individual files for each chromosome\n",
      "Finished writing out individual chromosome files for CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Now writing methylation information for file >>CpG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz<< to individual files for each chromosome\n",
      "Finished writing out individual chromosome files for CpG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Now writing methylation information for file >>CpG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz<< to individual files for each chromosome\n",
      "Finished writing out individual chromosome files for CpG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Now writing methylation information for file >>CpG_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz<< to individual files for each chromosome\n",
      "Finished writing out individual chromosome files for CpG_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "\n",
      "Collecting temporary chromosome file information... Processing the following input file(s):\n",
      "CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035780.1.methXtractor.temp\n",
      "CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035784.1.methXtractor.temp\n",
      "CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035782.1.methXtractor.temp\n",
      "CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035785.1.methXtractor.temp\n",
      "CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035783.1.methXtractor.temp\n",
      "CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035781.1.methXtractor.temp\n",
      "CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035787.1.methXtractor.temp\n",
      "CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035788.1.methXtractor.temp\n",
      "CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035786.1.methXtractor.temp\n",
      "CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035789.1.methXtractor.temp\n",
      "\n",
      "Sorting input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035780.1.methXtractor.temp by positions (using -S of 2G)\n",
      "Successfully deleted the temporary input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035780.1.methXtractor.temp\n",
      "\n",
      "Sorting input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035784.1.methXtractor.temp by positions (using -S of 2G)\n",
      "Successfully deleted the temporary input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035784.1.methXtractor.temp\n",
      "\n",
      "Sorting input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035782.1.methXtractor.temp by positions (using -S of 2G)\n",
      "Successfully deleted the temporary input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035782.1.methXtractor.temp\n",
      "\n",
      "Sorting input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035785.1.methXtractor.temp by positions (using -S of 2G)\n",
      "Successfully deleted the temporary input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035785.1.methXtractor.temp\n",
      "\n",
      "Sorting input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035783.1.methXtractor.temp by positions (using -S of 2G)\n",
      "Successfully deleted the temporary input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035783.1.methXtractor.temp\n",
      "\n",
      "Sorting input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035781.1.methXtractor.temp by positions (using -S of 2G)\n",
      "Successfully deleted the temporary input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035781.1.methXtractor.temp\n",
      "\n",
      "Sorting input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035787.1.methXtractor.temp by positions (using -S of 2G)\n",
      "Successfully deleted the temporary input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035787.1.methXtractor.temp\n",
      "\n",
      "Sorting input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035788.1.methXtractor.temp by positions (using -S of 2G)\n",
      "Successfully deleted the temporary input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035788.1.methXtractor.temp\n",
      "\n",
      "Sorting input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035786.1.methXtractor.temp by positions (using -S of 2G)\n",
      "Successfully deleted the temporary input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035786.1.methXtractor.temp\n",
      "\n",
      "Sorting input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035789.1.methXtractor.temp by positions (using -S of 2G)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Successfully deleted the temporary input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035789.1.methXtractor.temp\n",
      "\n",
      "Finished BedGraph conversion ...\n",
      "\n",
      "samtools view: writing to standard output failed: Broken pipe\n",
      "samtools view: error closing standard output: -1\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n"
     ]
    }
   ],
   "source": [
    "!bismark_methylation_extractor --gzip -bedGraph /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.bam -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/ "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "generate Bismark Nucleotide Coverage report"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Summary of parameters for nucleotide coverage report:\n",
      "=====================================================\n",
      "Output directory:\t\t\t>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/<\n",
      "Parent directory:\t\t\t>/Users/Shelly/Documents/GitHub/C_virginica/<\n",
      "Genome directory:\t\t\t>/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/<\n",
      "Samtools installation:\t\t\t>/Users/Shelly/anaconda3/bin/samtools<\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Stored sequence information of 11 chromosomes/scaffolds in total\n",
      "\n",
      "==================================================================\n",
      "Mono- and di-nucleotide coverage will now be written into a report\n",
      "==================================================================\n",
      "\n",
      "Detected file 'genomic_nucleotide_frequencies.txt' in the genome folder /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ already. Using nucleotide frequencies contained therein ...\n",
      "============================================================================================================================================================================================\n",
      "\n",
      "\n",
      "Calculating read frequencies from file '/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.bam'\n",
      "==========================================================================================\n",
      "Determined the file to be paired-end\n",
      "\n",
      "\n",
      "Printing nucleotide stats to >> /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.nucleotide_stats.txt <<\n",
      "Final Stage: Calculating averages\n",
      "=================================\n",
      "\n",
      "(di-)nucleotide\tcount sample\tpercent sample\tcount genomic\tpercent genomic\tcoverage\n",
      "A\t397765\t27.39\t223134775\t32.59\t0.002\n",
      "C\t290760\t20.02\t119229119\t17.41\t0.002\n",
      "G\t346559\t23.86\t119267677\t17.42\t0.003\n",
      "T\t417399\t28.74\t223043757\t32.58\t0.002\n",
      "AA\t115616\t8.05\t80795037\t11.80\t0.001\n",
      "AC\t77775\t5.42\t37359527\t5.46\t0.002\n",
      "AG\t95546\t6.65\t35548672\t5.19\t0.003\n",
      "AT\t105707\t7.36\t69431301\t10.14\t0.002\n",
      "CA\t96576\t6.73\t45827414\t6.69\t0.002\n",
      "CC\t66568\t4.64\t23444678\t3.42\t0.003\n",
      "CG\t39753\t2.77\t14458703\t2.11\t0.003\n",
      "CT\t84353\t5.87\t35498217\t5.18\t0.002\n",
      "GA\t100972\t7.03\t38810489\t5.67\t0.003\n",
      "GC\t57424\t4.00\t19646670\t2.87\t0.003\n",
      "GG\t86627\t6.03\t23423030\t3.42\t0.004\n",
      "GT\t95958\t6.68\t37387368\t5.46\t0.003\n",
      "TA\t80967\t5.64\t57701608\t8.43\t0.001\n",
      "TC\t85855\t5.98\t38778132\t5.66\t0.002\n",
      "TG\t119602\t8.33\t45837169\t6.69\t0.003\n",
      "TT\t126771\t8.83\t80726644\t11.79\t0.002\n"
     ]
    }
   ],
   "source": [
    "!bam2nuc /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.bam \\\n",
    "--dir /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/ \\\n",
    "--samtools /Users/Shelly/anaconda3/bin/samtools \\\n",
    "--genome_folder /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "generate Bismark report"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Writing Bismark HTML report to >> /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_PE_report.html <<\n",
      "\n",
      "Redundant argument in sprintf at /Users/Shelly/anaconda3/bin/bismark2report line 170.\n",
      "==============================================================================================================\n",
      "Using the following alignment report:\t\t> /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_PE_report.txt <\n",
      "Processing alignment report /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_PE_report.txt ...\n",
      "Complete\n",
      "\n",
      "No deduplication report file specified, skipping this step\n",
      "Using the following splitting report:\t\t> /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe_splitting_report.txt <\n",
      "Processing splitting report /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe_splitting_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following M-bias report:\t\t> /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.M-bias.txt <\n",
      "Processing M-bias report /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.M-bias.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following nucleotide coverage report:\t> /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.nucleotide_stats.txt <\n",
      "Processing nucleotide coverage report '/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.nucleotide_stats.txt' ...\n",
      "Complete\n",
      "\n",
      "==============================================================================================================\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!bismark2report \\\n",
    "--alignment_report /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_PE_report.txt \\\n",
    "--splitting_report /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe_splitting_report.txt \\\n",
    "--mbias_report /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.M-bias.txt \\\n",
    "--nucleotide_report /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.nucleotide_stats.txt \\\n",
    "--dir /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/ \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#Subsampling from each fastq file and aligning with Bismark"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t55304\n",
      "Mapping efficiency:\t55.3%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t27490\n",
      "Sequence pairs did not map uniquely:\t17206\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t13863\t((converted) top strand)\n",
      "GA/CT/CT:\t13771\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t13915\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t13755\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t25623\n",
      "Mapping efficiency:\t25.6%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t66914\n",
      "Sequence pairs did not map uniquely:\t7463\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t6395\t((converted) top strand)\n",
      "GA/CT/CT:\t6412\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t6401\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t6415\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t50484\n",
      "Mapping efficiency:\t50.5%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t33358\n",
      "Sequence pairs did not map uniquely:\t16158\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t12761\t((converted) top strand)\n",
      "GA/CT/CT:\t12571\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t12410\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t12742\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t55171\n",
      "Mapping efficiency:\t55.2%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t26540\n",
      "Sequence pairs did not map uniquely:\t18289\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t13872\t((converted) top strand)\n",
      "GA/CT/CT:\t13727\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t13678\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t13894\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t56412\n",
      "Mapping efficiency:\t56.4%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t25464\n",
      "Sequence pairs did not map uniquely:\t18124\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t14301\t((converted) top strand)\n",
      "GA/CT/CT:\t13881\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t13768\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t14462\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t53779\n",
      "Mapping efficiency:\t53.8%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t29695\n",
      "Sequence pairs did not map uniquely:\t16526\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t13871\t((converted) top strand)\n",
      "GA/CT/CT:\t13053\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t12995\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t13860\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t56844\n",
      "Mapping efficiency:\t56.8%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t25108\n",
      "Sequence pairs did not map uniquely:\t18048\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t14846\t((converted) top strand)\n",
      "GA/CT/CT:\t13445\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t13483\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t15070\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t53463\n",
      "Mapping efficiency:\t53.5%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t30401\n",
      "Sequence pairs did not map uniquely:\t16136\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t13576\t((converted) top strand)\n",
      "GA/CT/CT:\t12963\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t13213\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t13711\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t48884\n",
      "Mapping efficiency:\t48.9%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t36530\n",
      "Sequence pairs did not map uniquely:\t14586\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t12344\t((converted) top strand)\n",
      "GA/CT/CT:\t12125\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t12019\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t12396\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t52449\n",
      "Mapping efficiency:\t52.4%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t30963\n",
      "Sequence pairs did not map uniquely:\t16588\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t14037\t((converted) top strand)\n",
      "GA/CT/CT:\t12205\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t12162\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t14045\t((converted) bottom strand)\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_10_s1_R1.fastq.gz to zr2096_10_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_10_s1_R1.fastq.gz to zr2096_10_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_10_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_10_s1_R2.fastq.gz to zr2096_10_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_10_s1_R2.fastq.gz to zr2096_10_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_10_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tGTTATATATAATTGTATTAATTATAAAAAATATNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATTTATTATTAAAAATATTAAATATATTATTTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNATTTTATATTTTTAATAATAAATATAATTATATATTAAAATAAAATAATATATTTAATATTTTTAATAATAAATACAATTAAATATCATAATAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tACCATATACAATCATATCAACTATAAAAAATACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATCTACTATCAAAAATACTAAACACACTACCTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNGTTTTGTATTTTTGATAGTAGATATGATTGTATATTAGGGTAAGGTAGTGTGTTTAGTATTTTTGATAGTAGATATGATTGAATATTGTGGTAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tACCATATACAATCATATCAACTATAAAAAATACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATCTACTATCAAAAATACTAAACACACTACCTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNGTTTTGTATTTTTGATAGTAGATATGATTGTATATTAGGGTAAGGTAGTGTGTTTAGTATTTTTGATAGTAGATATGATTGAATATTGTGGTAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tGTTATATATAATTGTATTAATTATAAAAAATATNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATTTATTATTAAAAATATTAAATATATTATTTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNATTTTATATTTTTAATAATAAATATAATTATATATTAAAATAAAATAATATATTTAATATTTTTAATAATAAATACAATTAAATATCATAATAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_10_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70695 (70.69%) aligned concordantly 0 times\n",
      "    12336 (12.34%) aligned concordantly exactly 1 time\n",
      "    16969 (16.97%) aligned concordantly >1 times\n",
      "29.30% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70600 (70.60%) aligned concordantly 0 times\n",
      "    12225 (12.22%) aligned concordantly exactly 1 time\n",
      "    17175 (17.18%) aligned concordantly >1 times\n",
      "29.40% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70611 (70.61%) aligned concordantly 0 times\n",
      "    12401 (12.40%) aligned concordantly exactly 1 time\n",
      "    16988 (16.99%) aligned concordantly >1 times\n",
      "29.39% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70498 (70.50%) aligned concordantly 0 times\n",
      "    12497 (12.50%) aligned concordantly exactly 1 time\n",
      "    17005 (17.00%) aligned concordantly >1 times\n",
      "29.50% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_10_s1_R1.fastq.gz_C_to_T.fastq, zr2096_10_s1_R1.fastq.gz_G_to_A.fastq, zr2096_10_s1_R2.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1911136\n",
      "\n",
      "Total methylated C's in CpG context:\t185771\n",
      "Total methylated C's in CHG context:\t13978\n",
      "Total methylated C's in CHH context:\t47702\n",
      "Total methylated C's in Unknown context:\t6403\n",
      "\n",
      "Total unmethylated C's in CpG context:\t84112\n",
      "Total unmethylated C's in CHG context:\t424427\n",
      "Total unmethylated C's in CHH context:\t1155146\n",
      "Total unmethylated C's in Unknown context:\t26070\n",
      "\n",
      "C methylated in CpG context:\t68.8%\n",
      "C methylated in CHG context:\t3.2%\n",
      "C methylated in CHH context:\t4.0%\n",
      "C methylated in unknown context (CN or CHN):\t19.7%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 46s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t99\tNC_035780.1_CT_converted\t65371909\t6\t100M\t=\t65372013\t204\tGAGTTTTTTTGATTATTTGTTGTTTGTTGTTTGTTTGNNNNTNNNTNNGTTTGTTTGTTTGTTTGTTTGTTTGTAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tAS:i:-15\tXS:i:-22\tXN:i:0\tXM:i:10\tXO:i:0\tXG:i:0\tNM:i:10\tMD:Z:37T0T0T0G1T0T0G1T0T10A41\tYS:i:-24\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t147\tNC_035780.1_CT_converted\t65372013\t6\t100M\t=\t65371909\t-204\tGAATTATTGGGTTAATTTTAATTAAATTTGATATAAAGTATTTTTAGGTGAAGGGGATTTAAGTTTGTTTGAATGAAGGGTTTTGTTTTTTTTTAAGGGG\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-24\tXS:i:-6\tXN:i:0\tXM:i:4\tXO:i:0\tXG:i:0\tNM:i:4\tMD:Z:30G14G3A20A29\tYS:i:-15\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTAAAAAAAAATGAAATTTTTTATTTAAATAAATTTAAATTTTTTTTATTTAAAAATATTTTATATTAAATTTAATTAAAATTAATTTAATAATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTAAAAAAAAATGAAATTTTTTATTTAAATAAATTTAAATTTTTTTTATTTAAAAATATTTTATATTAAATTTAATTAAAATTAATTTAATAATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t83\tNC_035780.1_GA_converted\t8090288\t7\t26M4D74M\t=\t8090028\t-364\tAAAAAAAATAAAAATATAAAAAATTTACAAACAAACAAACAAACAAACAAACNNANNNANNNNCAAACAAACAACAAACAACAAATAATCAAAAAAACTC\tFFFFFFFF/FFFFF/B/FFFFFF/</FFFFFF<FFFFFFFFFFFFFFFF<<<##<###<####FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-26\tXS:i:-22\tXN:i:0\tXM:i:9\tXO:i:1\tXG:i:4\tNM:i:13\tMD:Z:26^ACAA26A0A1C0A0A1C0A0A0A37\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t163\tNC_035780.1_GA_converted\t8090028\t7\t100M\t=\t8090288\t364\tCCCCTTAAAAAAAAACAAAACCCTTCATTCAAACAAACTTAAATCCCCTTCACCTAAAAATACTTTATATCAAATTTAATTAAAATTAACCCAATAATTC\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-36\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:29T70\tYS:i:-26\tYT:Z:CP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_1_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    86490 (86.49%) aligned concordantly 0 times\n",
      "    5663 (5.66%) aligned concordantly exactly 1 time\n",
      "    7847 (7.85%) aligned concordantly >1 times\n",
      "13.51% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    86569 (86.57%) aligned concordantly 0 times\n",
      "    5628 (5.63%) aligned concordantly exactly 1 time\n",
      "    7803 (7.80%) aligned concordantly >1 times\n",
      "13.43% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    86672 (86.67%) aligned concordantly 0 times\n",
      "    5595 (5.59%) aligned concordantly exactly 1 time\n",
      "    7733 (7.73%) aligned concordantly >1 times\n",
      "13.33% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    86777 (86.78%) aligned concordantly 0 times\n",
      "    5615 (5.62%) aligned concordantly exactly 1 time\n",
      "    7608 (7.61%) aligned concordantly >1 times\n",
      "13.22% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_1_s1_R1.fastq.gz_C_to_T.fastq, zr2096_1_s1_R1.fastq.gz_G_to_A.fastq, zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t830788\n",
      "\n",
      "Total methylated C's in CpG context:\t72846\n",
      "Total methylated C's in CHG context:\t6793\n",
      "Total methylated C's in CHH context:\t40295\n",
      "Total methylated C's in Unknown context:\t5123\n",
      "\n",
      "Total unmethylated C's in CpG context:\t32368\n",
      "Total unmethylated C's in CHG context:\t169425\n",
      "Total unmethylated C's in CHH context:\t509061\n",
      "Total unmethylated C's in Unknown context:\t14379\n",
      "\n",
      "C methylated in CpG context:\t69.2%\n",
      "C methylated in CHG context:\t3.9%\n",
      "C methylated in CHH context:\t7.3%\n",
      "C methylated in unknown context (CN or CHN):\t26.3%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 2s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_2_s1_R1.fastq.gz to zr2096_2_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_2_s1_R1.fastq.gz to zr2096_2_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_2_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_2_s1_R2.fastq.gz to zr2096_2_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_2_s1_R2.fastq.gz to zr2096_2_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_2_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tTTTTTTATTAAAATTTATTATTTTAAAAATAAAAATTNNNNTNNNTTTTATTATTTTTTTTAATATTTTTTAATATAATTAAATAATAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTCACTTCAAAAAAATAATAAATAAAAAAAATTAATACAACAATATTTTTAATAATACAACAATATTTAAAAAATTATTTATTAATTTTTTTTATTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tCTCCTTATTAAAATCTACTATTCCAAAAATAAAAACTNNNNCNNNTTCCACCATTCCTTCTAACATCTCCCAATATAACTAAATAACAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTGTTTTGGAAAAATAATGGATAAAAAAGGTTGATGTGGTGATGTTTTTAGTGATGTGGTGGTGTTTGAGAAATTATTTATTAATTTTTTTTGTTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tCTCCTTATTAAAATCTACTATTCCAAAAATAAAAACTNNNNCNNNTTCCACCATTCCTTCTAACATCTCCCAATATAACTAAATAACAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTGTTTTGGAAAAATAATGGATAAAAAAGGTTGATGTGGTGATGTTTTTAGTGATGTGGTGGTGTTTGAGAAATTATTTATTAATTTTTTTTGTTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tTTTTTTATTAAAATTTATTATTTTAAAAATAAAAATTNNNNTNNNTTTTATTATTTTTTTTAATATTTTTTAATATAATTAAATAATAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTCACTTCAAAAAAATAATAAATAAAAAAAATTAATACAACAATATTTTTAATAATACAACAATATTTAAAAAATTATTTATTAATTTTTTTTATTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_2_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    71647 (71.65%) aligned concordantly 0 times\n",
      "    11900 (11.90%) aligned concordantly exactly 1 time\n",
      "    16453 (16.45%) aligned concordantly >1 times\n",
      "28.35% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    71946 (71.95%) aligned concordantly 0 times\n",
      "    11827 (11.83%) aligned concordantly exactly 1 time\n",
      "    16227 (16.23%) aligned concordantly >1 times\n",
      "28.05% overall alignment rate\n",
      "100000100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    71468 ( reads; of these:71.47%\n",
      ") aligned concordantly 0 times  \n",
      "    10000011794 (11.79%) aligned concordantly exactly 1 time\n",
      "    16738 (16.74%) aligned concordantly >1 times\n",
      "28.53% overall alignment rate\n",
      " (100.00%) were paired; of these:\n",
      "    71402 (71.40%) aligned concordantly 0 times\n",
      "    11953 (11.95%) aligned concordantly exactly 1 time\n",
      "    16645 (16.64%) aligned concordantly >1 times\n",
      "28.60% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_2_s1_R1.fastq.gz_C_to_T.fastq, zr2096_2_s1_R1.fastq.gz_G_to_A.fastq, zr2096_2_s1_R2.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1601294\n",
      "\n",
      "Total methylated C's in CpG context:\t153343\n",
      "Total methylated C's in CHG context:\t12481\n",
      "Total methylated C's in CHH context:\t54835\n",
      "Total methylated C's in Unknown context:\t6562\n",
      "\n",
      "Total unmethylated C's in CpG context:\t65114\n",
      "Total unmethylated C's in CHG context:\t359729\n",
      "Total unmethylated C's in CHH context:\t955792\n",
      "Total unmethylated C's in Unknown context:\t24798\n",
      "\n",
      "C methylated in CpG context:\t70.2%\n",
      "C methylated in CHG context:\t3.4%\n",
      "C methylated in CHH context:\t5.4%\n",
      "C methylated in unknown context (CN or CHN):\t20.9%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 17s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_3_s1_R1.fastq.gz to zr2096_3_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_3_s1_R1.fastq.gz to zr2096_3_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_3_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_3_s1_R2.fastq.gz to zr2096_3_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_3_s1_R2.fastq.gz to zr2096_3_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_3_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tTNTTTTAAAATAATTTAAATAATTTTATATAAATNNNNNNNNNNNNNNNTATTTAAATTAAAAATTTAAATAAATTTTTTTAATTATTTATTGTTTAATG\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAAATAAACATTAAACAATAAATAATTAAAAAAATTTATTTAAATTTTTAATTTAAATAAATTAAAAATATTAAATTTATATAAAATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tCNCTTCAAAATAATTTAAATAATCTTATATAAATNNNNNNNNNNNNNNNCACCTAAACTAAAAACTCAAATAAACTTTTCTAATCACCTATCATCCAACA\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAGATAGATGTTGGATGATAGGTGATTAGAAAAGTTTATTTGAGTTTTTAGTTTAGGTGAGTTAAAAATATTGAATTTATATAAGATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tCNCTTCAAAATAATTTAAATAATCTTATATAAATNNNNNNNNNNNNNNNCACCTAAACTAAAAACTCAAATAAACTTTTCTAATCACCTATCATCCAACA\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAGATAGATGTTGGATGATAGGTGATTAGAAAAGTTTATTTGAGTTTTTAGTTTAGGTGAGTTAAAAATATTGAATTTATATAAGATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tTNTTTTAAAATAATTTAAATAATTTTATATAAATNNNNNNNNNNNNNNNTATTTAAATTAAAAATTTAAATAAATTTTTTTAATTATTTATTGTTTAATG\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAAATAAACATTAAACAATAAATAATTAAAAAAATTTATTTAAATTTTTAATTTAAATAAATTAAAAATATTAAATTTATATAAAATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_3_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69474 (69.47%) aligned concordantly 0 times\n",
      "    12342 (12.34%) aligned concordantly exactly 1 time\n",
      "    18184 (18.18%) aligned concordantly >1 times\n",
      "30.53% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69093 (69.09%) aligned concordantly 0 times\n",
      "    12247 (12.25%) aligned concordantly exactly 1 time\n",
      "    18660 (18.66%) aligned concordantly >1 times\n",
      "30.91% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69028 (69.03%) aligned concordantly 0 times\n",
      "    12397 (12.40%) aligned concordantly exactly 1 time\n",
      "    18575 (18.57%) aligned concordantly >1 times\n",
      "30.97% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69406 (69.41%) aligned concordantly 0 times\n",
      "    12367 (12.37%) aligned concordantly exactly 1 time\n",
      "    18227 (18.23%) aligned concordantly >1 times\n",
      "30.59% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_3_s1_R1.fastq.gz_C_to_T.fastq, zr2096_3_s1_R1.fastq.gz_G_to_A.fastq, zr2096_3_s1_R2.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1911141\n",
      "\n",
      "Total methylated C's in CpG context:\t212856\n",
      "Total methylated C's in CHG context:\t13659\n",
      "Total methylated C's in CHH context:\t45545\n",
      "Total methylated C's in Unknown context:\t6337\n",
      "\n",
      "Total unmethylated C's in CpG context:\t65965\n",
      "Total unmethylated C's in CHG context:\t426703\n",
      "Total unmethylated C's in CHH context:\t1146413\n",
      "Total unmethylated C's in Unknown context:\t27502\n",
      "\n",
      "C methylated in CpG context:\t76.3%\n",
      "C methylated in CHG context:\t3.1%\n",
      "C methylated in CHH context:\t3.8%\n",
      "C methylated in unknown context (CN or CHN):\t18.7%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 39s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_4_s1_R1.fastq.gz to zr2096_4_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_4_s1_R1.fastq.gz to zr2096_4_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_4_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_4_s1_R2.fastq.gz to zr2096_4_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_4_s1_R2.fastq.gz to zr2096_4_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_4_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNGATTAGTGTATAGTGGTTGTTATGGAAATTAGNNNNNNNNNNNNNNNNNNNNNNNNNTGNANTNTTTTTATTAGAATTGTTAAGGTTTATTTGATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTCCATCATTAATTTTTAAAAAAAAAAAACCATCCTTAATATCAATTTTATAAATCAAATAAACCTTAACAATTCTAATAAAAACACTTCATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATTAATATATAATAATTATTATAAAAATTAANNNNNNNNNNNNNNNNNNNNNNNNNTANANTNTTTTTATTAAAATTATTAAAATTTATTTAATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTTTATTATTAATTTTTAAAAAAAAAAAATTATTTTTAATATTAATTTTATAAATTAAATAAATTTTAATAATTTTAATAAAAATATTTTATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATTAATATATAATAATTATTATAAAAATTAANNNNNNNNNNNNNNNNNNNNNNNNNTANANTNTTTTTATTAAAATTATTAAAATTTATTTAATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTTTATTATTAATTTTTAAAAAAAAAAAATTATTTTTAATATTAATTTTATAAATTAAATAAATTTTAATAATTTTAATAAAAATATTTTATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNGATTAGTGTATAGTGGTTGTTATGGAAATTAGNNNNNNNNNNNNNNNNNNNNNNNNNTGNANTNTTTTTATTAGAATTGTTAAGGTTTATTTGATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTCCATCATTAATTTTTAAAAAAAAAAAACCATCCTTAATATCAATTTTATAAATCAAATAAACCTTAACAATTCTAATAAAAACACTTCATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_4_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69215 (69.22%) aligned concordantly 0 times\n",
      "    12964 (12.96%) aligned concordantly exactly 1 time\n",
      "    17821 (17.82%) aligned concordantly >1 times\n",
      "30.79% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    68104 (68.10%) aligned concordantly 0 times\n",
      "    13392 (13.39%) aligned concordantly exactly 1 time\n",
      "    18504 (18.50%) aligned concordantly >1 times\n",
      "31.90% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    67910 (67.91%) aligned concordantly 0 times\n",
      "    13504 (13.50%) aligned concordantly exactly 1 time\n",
      "    18586 (18.59%) aligned concordantly >1 times\n",
      "32.09% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69042 (69.04%) aligned concordantly 0 times\n",
      "    13009 (13.01%) aligned concordantly exactly 1 time\n",
      "    17949 (17.95%) aligned concordantly >1 times\n",
      "30.96% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_4_s1_R1.fastq.gz_C_to_T.fastq, zr2096_4_s1_R1.fastq.gz_G_to_A.fastq, zr2096_4_s1_R2.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1807416\n",
      "\n",
      "Total methylated C's in CpG context:\t156286\n",
      "Total methylated C's in CHG context:\t12428\n",
      "Total methylated C's in CHH context:\t49328\n",
      "Total methylated C's in Unknown context:\t6149\n",
      "\n",
      "Total unmethylated C's in CpG context:\t82257\n",
      "Total unmethylated C's in CHG context:\t398925\n",
      "Total unmethylated C's in CHH context:\t1108192\n",
      "Total unmethylated C's in Unknown context:\t27246\n",
      "\n",
      "C methylated in CpG context:\t65.5%\n",
      "C methylated in CHG context:\t3.0%\n",
      "C methylated in CHH context:\t4.3%\n",
      "C methylated in unknown context (CN or CHN):\t18.4%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 59s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_5_s1_R1.fastq.gz to zr2096_5_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_5_s1_R1.fastq.gz to zr2096_5_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_5_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_5_s1_R2.fastq.gz to zr2096_5_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_5_s1_R2.fastq.gz to zr2096_5_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_5_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATAAAATATTATTTAAATTTTATTATTTATANNNNNNNNNNNNNNNNNNATTNTTTTTATTATATATATAAAATTATTTTAAAATTATTATATTTTTT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTCAAATATATTAAAAAAAAATTATTAATAATTTTCATTAAAAAATATAATAATTTTAAAATAATTTTATATATATAATAAAAAAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAACAAAACACCACTCAAATCTTACTATCCACANNNNNNNNNNNNNNNNNNATTNCTTCTATCATATATATAAAATCATTCCAAAATTACTATATTTTTC\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTTGAGTATATTGAAGAGGAATTATTGATAGTTTTTGTTGAAAAATATAGTAATTTTGGAATGATTTTATATATATGATAGAAGAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAACAAAACACCACTCAAATCTTACTATCCACANNNNNNNNNNNNNNNNNNATTNCTTCTATCATATATATAAAATCATTCCAAAATTACTATATTTTTC\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTTGAGTATATTGAAGAGGAATTATTGATAGTTTTTGTTGAAAAATATAGTAATTTTGGAATGATTTTATATATATGATAGAAGAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATAAAATATTATTTAAATTTTATTATTTATANNNNNNNNNNNNNNNNNNATTNTTTTTATTATATATATAAAATTATTTTAAAATTATTATATTTTTT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTCAAATATATTAAAAAAAAATTATTAATAATTTTCATTAAAAAATATAATAATTTTAAAATAATTTTATATATATAATAAAAAAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_5_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    71937 (71.94%) aligned concordantly 0 times\n",
      "    12071 (12.07%) aligned concordantly exactly 1 time\n",
      "    15992 (15.99%) aligned concordantly >1 times\n",
      "28.06% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    71853 (71.85%) aligned concordantly 0 times\n",
      "    12016 (12.02%) aligned concordantly exactly 1 time\n",
      "    16131 (16.13%) aligned concordantly >1 times\n",
      "28.15% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70840 (70.84%) aligned concordantly 0 times\n",
      "    12296 (12.30%) aligned concordantly exactly 1 time\n",
      "    16864 (16.86%) aligned concordantly >1 times\n",
      "29.16% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70617 (70.62%) aligned concordantly 0 times\n",
      "    12464 (12.46%) aligned concordantly exactly 1 time\n",
      "    16919 (16.92%) aligned concordantly >1 times\n",
      "29.38% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_5_s1_R1.fastq.gz_C_to_T.fastq, zr2096_5_s1_R1.fastq.gz_G_to_A.fastq, zr2096_5_s1_R2.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1792976\n",
      "\n",
      "Total methylated C's in CpG context:\t154384\n",
      "Total methylated C's in CHG context:\t12834\n",
      "Total methylated C's in CHH context:\t47835\n",
      "Total methylated C's in Unknown context:\t6577\n",
      "\n",
      "Total unmethylated C's in CpG context:\t75691\n",
      "Total unmethylated C's in CHG context:\t383354\n",
      "Total unmethylated C's in CHH context:\t1118878\n",
      "Total unmethylated C's in Unknown context:\t27168\n",
      "\n",
      "C methylated in CpG context:\t67.1%\n",
      "C methylated in CHG context:\t3.2%\n",
      "C methylated in CHH context:\t4.1%\n",
      "C methylated in unknown context (CN or CHN):\t19.5%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 31s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_6_s1_R1.fastq.gz to zr2096_6_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_6_s1_R1.fastq.gz to zr2096_6_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_6_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_6_s1_R2.fastq.gz to zr2096_6_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_6_s1_R2.fastq.gz to zr2096_6_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_6_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTTATTAAATTTAATTTTATTATTAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNTANANAAATATTAATTAAATATTTTTAAAATAATAAATTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAAAATTTTTTTAAATTTATTATTTTAAAAATATTTAATTAATATTTATTTATATTTTATAATTAACAATAATTATTTTTTTATTTAATAATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTCATCAAATCCAACTTTATCACCAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNCANANAAACATCAATCAAACATCTCTAAAACAATAAACTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAGAATTTTTTTAAGTTTATTGTTTTAGAGATGTTTGATTGATGTTTATTTGTATTTTGTAATTAGTGGTGGTTGTTTTTTTATTTGGTGATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTCATCAAATCCAACTTTATCACCAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNCANANAAACATCAATCAAACATCTCTAAAACAATAAACTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAGAATTTTTTTAAGTTTATTGTTTTAGAGATGTTTGATTGATGTTTATTTGTATTTTGTAATTAGTGGTGGTTGTTTTTTTATTTGGTGATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTTATTAAATTTAATTTTATTATTAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNTANANAAATATTAATTAAATATTTTTAAAATAATAAATTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAAAATTTTTTTAAATTTATTATTTTAAAAATATTTAATTAATATTTATTTATATTTTATAATTAACAATAATTATTTTTTTATTTAATAATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_6_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69708 (69.71%) aligned concordantly 0 times\n",
      "    12666 (12.67%) aligned concordantly exactly 1 time\n",
      "    17626 (17.63%) aligned concordantly >1 times\n",
      "30.29% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    67544 (67.54%) aligned concordantly 0 times\n",
      "    13694 (13.69%) aligned concordantly exactly 1 time\n",
      "    18762 (18.76%) aligned concordantly >1 times\n",
      "32.46% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69721 (69.72%) aligned concordantly 0 times\n",
      "    12698 (12.70%) aligned concordantly exactly 1 time\n",
      "    17581 (17.58%) aligned concordantly >1 times\n",
      "30.28% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    67375 (67.38%) aligned concordantly 0 times\n",
      "    13613 (13.61%) aligned concordantly exactly 1 time\n",
      "    19012 (19.01%) aligned concordantly >1 times\n",
      "32.62% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_6_s1_R1.fastq.gz_C_to_T.fastq, zr2096_6_s1_R1.fastq.gz_G_to_A.fastq, zr2096_6_s1_R2.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1839385\n",
      "\n",
      "Total methylated C's in CpG context:\t178767\n",
      "Total methylated C's in CHG context:\t13324\n",
      "Total methylated C's in CHH context:\t50010\n",
      "Total methylated C's in Unknown context:\t6495\n",
      "\n",
      "Total unmethylated C's in CpG context:\t66888\n",
      "Total unmethylated C's in CHG context:\t402617\n",
      "Total unmethylated C's in CHH context:\t1127779\n",
      "Total unmethylated C's in Unknown context:\t28732\n",
      "\n",
      "C methylated in CpG context:\t72.8%\n",
      "C methylated in CHG context:\t3.2%\n",
      "C methylated in CHH context:\t4.2%\n",
      "C methylated in unknown context (CN or CHN):\t18.4%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 42s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_7_s1_R1.fastq.gz to zr2096_7_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_7_s1_R1.fastq.gz to zr2096_7_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_7_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_7_s1_R2.fastq.gz to zr2096_7_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_7_s1_R2.fastq.gz to zr2096_7_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_7_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAAAAAATGATTATTATGATGAATTTAAATATTAATAATAAAATTTAAAAAAAATATAAATGTTAAATATTAAATTTTTTAAAAAATGATTATGG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t141\t*\t0\t0\t*\t*\t0\t0\tCCATAATCATTTTTTAAAAAATTTAATATTTAACATTTATATTTTTTTTAAATTTTATTATTAATATTTAAATTCATCATAATAATCATTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t99\tNC_035781.1_GA_converted\t24539746\t6\t95M\t=\t24539746\t-95\tAAAAAAACAACTACTACAACAAATTCAAATATTAATAACAAAATTTAAAAAAAACATAAACATTAAATATCAAATTCTTCAAAAAACAATTACAA\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-12\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:94T0\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t147\tNC_035781.1_GA_converted\t24539746\t6\t95M\t=\t24539746\t-95\tAAAAAAACAACTACTACAACAAATTCAAATATTAATAACAAAATTTAAAAAAAACATAAACATTAAATATCAAATTCTTCAAAAAACAATTACAA\tFFFFFFFFFFFFFFFFFFFFFFFBBFFFFFFBBFF/FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-6\tXS:i:-12\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:94T0\tYS:i:-6\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t83\tNC_035780.1_CT_converted\t42548681\t17\t95M\t=\t42548681\t-95\tTTGTAATTGTTTTTTGAAGAATTTGATATTTAATGTTTATGTTTTTTTTAAATTTTGTTATTAATATTTGAATTTGTTGTAGTAGTTGTTTTTTT\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-6\tXS:i:-32\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:0A94\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t163\tNC_035780.1_CT_converted\t42548681\t17\t95M\t=\t42548681\t-95\tTTGTAATTGTTTTTTGAAGAATTTGATATTTAATGTTTATGTTTTTTTTAAATTTTGTTATTAATATTTGAATTTGTTGTAGTAGTTGTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-32\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:0A94\tYS:i:-6\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAAAAAATGATTATTATGATGAATTTAAATATTAATAATAAAATTTAAAAAAAATATAAATGTTAAATATTAAATTTTTTAAAAAATGATTATGG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t141\t*\t0\t0\t*\t*\t0\t0\tCCATAATCATTTTTTAAAAAATTTAATATTTAACATTTATATTTTTTTTAAATTTTATTATTAATATTTAAATTCATCATAATAATCATTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_7_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    71530 (71.53%) aligned concordantly 0 times\n",
      "    12232 (12.23%) aligned concordantly exactly 1 time\n",
      "    16238 (16.24%) aligned concordantly >1 times\n",
      "28.47% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    72232 (72.23%) aligned concordantly 0 times\n",
      "    11838 (11.84%) aligned concordantly exactly 1 time\n",
      "    15930 (15.93%) aligned concordantly >1 times\n",
      "27.77% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    71562 (71.56%) aligned concordantly 0 times\n",
      "    12285 (12.29%) aligned concordantly exactly 1 time\n",
      "    16153 (16.15%) aligned concordantly >1 times\n",
      "28.44% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    72175 (72.17%) aligned concordantly 0 times\n",
      "    11787 (11.79%) aligned concordantly exactly 1 time\n",
      "    16038 (16.04%) aligned concordantly >1 times\n",
      "27.82% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_7_s1_R1.fastq.gz_C_to_T.fastq, zr2096_7_s1_R1.fastq.gz_G_to_A.fastq, zr2096_7_s1_R2.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1807058\n",
      "\n",
      "Total methylated C's in CpG context:\t165861\n",
      "Total methylated C's in CHG context:\t13399\n",
      "Total methylated C's in CHH context:\t47784\n",
      "Total methylated C's in Unknown context:\t6056\n",
      "\n",
      "Total unmethylated C's in CpG context:\t67849\n",
      "Total unmethylated C's in CHG context:\t393633\n",
      "Total unmethylated C's in CHH context:\t1118532\n",
      "Total unmethylated C's in Unknown context:\t26732\n",
      "\n",
      "C methylated in CpG context:\t71.0%\n",
      "C methylated in CHG context:\t3.3%\n",
      "C methylated in CHH context:\t4.1%\n",
      "C methylated in unknown context (CN or CHN):\t18.5%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 5m 25s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_8_s1_R1.fastq.gz to zr2096_8_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_8_s1_R1.fastq.gz to zr2096_8_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_8_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_8_s1_R2.fastq.gz to zr2096_8_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_8_s1_R2.fastq.gz to zr2096_8_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_8_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATTTAAAATTAAAAAATTTATTAAATTTATTATTATTTNTTATTTTTTATAAAAGAAAATAGAGAGAGAGAGAGAGAGAGAGAGAG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tCTCTCTCTCTCTCTCTCTCTCTCTCTATTTTCTTTTATAAAAAATAATAAATAATAATAAATTTAATAAATTTTTTAATTTTAAATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATCCAAAATTAAAAAACTCATTAAATCTATCATCATTCNCCATTCTTCACAAAAAAAAATAAAAAAAAAAAAAAAAAAAAAAAAAA\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTTTTTTTTTTTTTTTTTTTTTATTTTTTTTTGTGAAGAATGGTGAATGATGATAGATTTAATGAGTTTTTTAATTTTGGATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t83\tNC_007175.2_CT_converted\t14567\t0\t2M17I69M\t=\t14567\t-71\tTTTTTTTTTTTTTTTTTTTTTTTTTTATTTTTTTTTGTGAAGAATGGNGAATGATGATAGATTTAATGAGTTTTTTAATTTTGGATTA\tFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<<#FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-81\tXN:i:0\tXM:i:5\tXO:i:1\tXG:i:17\tNM:i:22\tMD:Z:1A5A0G5A15T40\tYS:i:-80\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t163\tNC_007175.2_CT_converted\t14567\t0\t2M17I69M\t=\t14567\t-71\tTTTTTTTTTTTTTTTTTTTTTTTTTTATTTTTTTTTGTGAAGAATGGTGAATGATGATAGATTTAATGAGTTTTTTAATTTTGGATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tAS:i:-80\tXN:i:0\tXM:i:4\tXO:i:1\tXG:i:17\tNM:i:21\tMD:Z:1A5A0G5A56\tYS:i:-81\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATTTAAAATTAAAAAATTTATTAAATTTATTATTATTTNTTATTTTTTATAAAAGAAAATAGAGAGAGAGAGAGAGAGAGAGAGAG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tCTCTCTCTCTCTCTCTCTCTCTCTCTATTTTCTTTTATAAAAAATAATAAATAATAATAAATTTAATAAATTTTTTAATTTTAAATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_8_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    74157 (74.16%) aligned concordantly 0 times\n",
      "    11312 (11.31%) aligned concordantly exactly 1 time\n",
      "    14531 (14.53%) aligned concordantly >1 times\n",
      "25.84% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    74306 (74.31%) aligned concordantly 0 times\n",
      "    11103 (11.10%) aligned concordantly exactly 1 time\n",
      "    14591 (14.59%) aligned concordantly >1 times\n",
      "25.69% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    74133 (74.13%) aligned concordantly 0 times\n",
      "    11331 (11.33%) aligned concordantly exactly 1 time\n",
      "    14536 (14.54%) aligned concordantly >1 times\n",
      "25.87% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    74552 (74.55%) aligned concordantly 0 times\n",
      "    11116 (11.12%) aligned concordantly exactly 1 time\n",
      "    14332 (14.33%) aligned concordantly >1 times\n",
      "25.45% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_8_s1_R1.fastq.gz_C_to_T.fastq, zr2096_8_s1_R1.fastq.gz_G_to_A.fastq, zr2096_8_s1_R2.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1598278\n",
      "\n",
      "Total methylated C's in CpG context:\t130605\n",
      "Total methylated C's in CHG context:\t11991\n",
      "Total methylated C's in CHH context:\t48523\n",
      "Total methylated C's in Unknown context:\t6038\n",
      "\n",
      "Total unmethylated C's in CpG context:\t67102\n",
      "Total unmethylated C's in CHG context:\t348741\n",
      "Total unmethylated C's in CHH context:\t991316\n",
      "Total unmethylated C's in Unknown context:\t24212\n",
      "\n",
      "C methylated in CpG context:\t66.1%\n",
      "C methylated in CHG context:\t3.3%\n",
      "C methylated in CHH context:\t4.7%\n",
      "C methylated in unknown context (CN or CHN):\t20.0%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 5m 14s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_9_s1_R1.fastq.gz to zr2096_9_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_9_s1_R1.fastq.gz to zr2096_9_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_9_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_9_s1_R2.fastq.gz to zr2096_9_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_9_s1_R2.fastq.gz to zr2096_9_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_9_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNATTAAATAAAAAATAATAAATAATAAATTANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAGG\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAAAAAATAAAAAACACATTAAAAATCTATAATTTAATTTATTATTTATTATTTTTTATTTAATTAAATCNNNNAAACATCATATAAAAAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNACCAAATAAAAAATAACAAATAATAAATCANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAAA\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAGAGGATAGGAGATATGTTAGGAATTTATAATTTGATTTATTATTTGTTATTTTTTATTTGGTTAGATTNNNNGAGTGTTGTGTAGGGAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNACCAAATAAAAAATAACAAATAATAAATCANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAAA\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAGAGGATAGGAGATATGTTAGGAATTTATAATTTGATTTATTATTTGTTATTTTTTATTTGGTTAGATTNNNNGAGTGTTGTGTAGGGAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNATTAAATAAAAAATAATAAATAATAAATTANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAGG\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAAAAAATAAAAAACACATTAAAAATCTATAATTTAATTTATTATTTATTATTTTTTATTTAATTAAATCNNNNAAACATCATATAAAAAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_9_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    73508 (73.51%) aligned concordantly 0 times\n",
      "    11236 (11.24%) aligned concordantly exactly 1 time\n",
      "    15256 (15.26%) aligned concordantly >1 times\n",
      "26.49% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    73787 (73.79%) aligned concordantly 0 times\n",
      "    11232 (11.23%) aligned concordantly exactly 1 time\n",
      "    14981 (14.98%) aligned concordantly >1 times\n",
      "26.21% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70419 (70.42%) aligned concordantly 0 times\n",
      "    12573 (12.57%) aligned concordantly exactly 1 time\n",
      "    17008 (17.01%) aligned concordantly >1 times\n",
      "29.58% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70201 (70.20%) aligned concordantly 0 times\n",
      "    12723 (12.72%) aligned concordantly exactly 1 time\n",
      "    17076 (17.08%) aligned concordantly >1 times\n",
      "29.80% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_9_s1_R1.fastq.gz_C_to_T.fastq, zr2096_9_s1_R1.fastq.gz_G_to_A.fastq, zr2096_9_s1_R2.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1849983\n",
      "\n",
      "Total methylated C's in CpG context:\t199908\n",
      "Total methylated C's in CHG context:\t14694\n",
      "Total methylated C's in CHH context:\t48531\n",
      "Total methylated C's in Unknown context:\t6764\n",
      "\n",
      "Total unmethylated C's in CpG context:\t63719\n",
      "Total unmethylated C's in CHG context:\t409648\n",
      "Total unmethylated C's in CHH context:\t1113483\n",
      "Total unmethylated C's in Unknown context:\t28006\n",
      "\n",
      "C methylated in CpG context:\t75.8%\n",
      "C methylated in CHG context:\t3.5%\n",
      "C methylated in CHH context:\t4.2%\n",
      "C methylated in unknown context (CN or CHN):\t19.5%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 5m 54s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "find /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_*R1* \\\n",
    "| xargs basename -s _s1_R1.fastq.gz | xargs -I{} /Users/Shelly/anaconda3/bin/bismark \\\n",
    "--genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ \\\n",
    "--score_min L,0,-1.2 \\\n",
    "-u 100000 \\\n",
    "--non_directional \\\n",
    "-1 /Volumes/web/seashell/bu-serine-wd/18-04-07/{}_s1_R1.fastq.gz \\\n",
    "-2 /Volumes/web/seashell/bu-serine-wd/18-04-07/{}_s1_R2.fastq.gz \\\n",
    "-o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/ \n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#run deduplication"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_10_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_1_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_2_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_3_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_4_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_5_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_6_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_7_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_8_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_9_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Processing paired-end Bismark output file(s) (SAM format):\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_10_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_1_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_2_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_3_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_4_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_5_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_6_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_7_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_8_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_9_s1_R1_bismark_bt2_pe.bam\n",
      "\n",
      "\n",
      "If there are several alignments to a single position in the genome the first alignment will be chosen. Since the input files are not in any way sorted this is a near-enough random selection of reads.\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_10_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_10_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_10_s1_R1_bismark_bt2_pe.bam:\t55304\n",
      "Total number duplicated alignments removed:\t22 (0.04%)\n",
      "Duplicated alignments were found at:\t21 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 55282 (99.96% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_1_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_1_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_1_s1_R1_bismark_bt2_pe.bam:\t25623\n",
      "Total number duplicated alignments removed:\t73 (0.28%)\n",
      "Duplicated alignments were found at:\t61 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 25550 (99.72% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_2_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_2_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_2_s1_R1_bismark_bt2_pe.bam:\t50484\n",
      "Total number duplicated alignments removed:\t59 (0.12%)\n",
      "Duplicated alignments were found at:\t42 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 50425 (99.88% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_3_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_3_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_3_s1_R1_bismark_bt2_pe.bam:\t55171\n",
      "Total number duplicated alignments removed:\t16 (0.03%)\n",
      "Duplicated alignments were found at:\t16 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 55155 (99.97% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_4_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_4_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_4_s1_R1_bismark_bt2_pe.bam:\t56412\n",
      "Total number duplicated alignments removed:\t14 (0.02%)\n",
      "Duplicated alignments were found at:\t13 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 56398 (99.98% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_5_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_5_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_5_s1_R1_bismark_bt2_pe.bam:\t53779\n",
      "Total number duplicated alignments removed:\t20 (0.04%)\n",
      "Duplicated alignments were found at:\t20 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 53759 (99.96% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_6_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_6_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_6_s1_R1_bismark_bt2_pe.bam:\t56844\n",
      "Total number duplicated alignments removed:\t12 (0.02%)\n",
      "Duplicated alignments were found at:\t12 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 56832 (99.98% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_7_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_7_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_7_s1_R1_bismark_bt2_pe.bam:\t53463\n",
      "Total number duplicated alignments removed:\t14 (0.03%)\n",
      "Duplicated alignments were found at:\t14 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 53449 (99.97% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_8_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_8_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_8_s1_R1_bismark_bt2_pe.bam:\t48884\n",
      "Total number duplicated alignments removed:\t27 (0.06%)\n",
      "Duplicated alignments were found at:\t24 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 48857 (99.94% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_9_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_9_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_9_s1_R1_bismark_bt2_pe.bam:\t52449\n",
      "Total number duplicated alignments removed:\t35 (0.07%)\n",
      "Duplicated alignments were found at:\t35 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 52414 (99.93% of total)\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "/Users/Shelly/anaconda3/bin/deduplicate_bismark \\\n",
    "--bam -p \\\n",
    "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/*.bam \\\n",
    "-o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/ \n",
    "2> /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/dedup.err"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!mkdir /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#analysis with a tighter alignment score threshold"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t35458\n",
      "Mapping efficiency:\t35.5%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t51704\n",
      "Sequence pairs did not map uniquely:\t12838\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t8804\t((converted) top strand)\n",
      "GA/CT/CT:\t8916\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t8935\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t8803\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t13744\n",
      "Mapping efficiency:\t13.7%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t81302\n",
      "Sequence pairs did not map uniquely:\t4954\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t3402\t((converted) top strand)\n",
      "GA/CT/CT:\t3480\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t3458\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t3404\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t31325\n",
      "Mapping efficiency:\t31.3%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t56828\n",
      "Sequence pairs did not map uniquely:\t11847\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t7926\t((converted) top strand)\n",
      "GA/CT/CT:\t7816\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t7775\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t7808\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t34892\n",
      "Mapping efficiency:\t34.9%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t51558\n",
      "Sequence pairs did not map uniquely:\t13550\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t8702\t((converted) top strand)\n",
      "GA/CT/CT:\t8804\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t8773\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t8613\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t35879\n",
      "Mapping efficiency:\t35.9%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t50638\n",
      "Sequence pairs did not map uniquely:\t13483\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t9000\t((converted) top strand)\n",
      "GA/CT/CT:\t8832\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t8995\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t9052\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t33466\n",
      "Mapping efficiency:\t33.5%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t54437\n",
      "Sequence pairs did not map uniquely:\t12097\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t8445\t((converted) top strand)\n",
      "GA/CT/CT:\t8308\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t8287\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t8426\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t35383\n",
      "Mapping efficiency:\t35.4%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t51529\n",
      "Sequence pairs did not map uniquely:\t13088\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t9139\t((converted) top strand)\n",
      "GA/CT/CT:\t8577\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t8499\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t9168\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t33107\n",
      "Mapping efficiency:\t33.1%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t55205\n",
      "Sequence pairs did not map uniquely:\t11688\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t8364\t((converted) top strand)\n",
      "GA/CT/CT:\t8129\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t8164\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t8450\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t30256\n",
      "Mapping efficiency:\t30.3%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t59165\n",
      "Sequence pairs did not map uniquely:\t10579\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t7666\t((converted) top strand)\n",
      "GA/CT/CT:\t7447\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t7475\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t7668\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t32194\n",
      "Mapping efficiency:\t32.2%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t55792\n",
      "Sequence pairs did not map uniquely:\t12014\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t8466\t((converted) top strand)\n",
      "GA/CT/CT:\t7691\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t7605\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t8432\t((converted) bottom strand)\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_10_s1_R1.fastq.gz to zr2096_10_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_10_s1_R1.fastq.gz to zr2096_10_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_10_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_10_s1_R2.fastq.gz to zr2096_10_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_10_s1_R2.fastq.gz to zr2096_10_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_10_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tGTTATATATAATTGTATTAATTATAAAAAATATNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATTTATTATTAAAAATATTAAATATATTATTTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNATTTTATATTTTTAATAATAAATATAATTATATATTAAAATAAAATAATATATTTAATATTTTTAATAATAAATACAATTAAATATCATAATAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tACCATATACAATCATATCAACTATAAAAAATACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATCTACTATCAAAAATACTAAACACACTACCTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNGTTTTGTATTTTTGATAGTAGATATGATTGTATATTAGGGTAAGGTAGTGTGTTTAGTATTTTTGATAGTAGATATGATTGAATATTGTGGTAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tACCATATACAATCATATCAACTATAAAAAATACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATCTACTATCAAAAATACTAAACACACTACCTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNGTTTTGTATTTTTGATAGTAGATATGATTGTATATTAGGGTAAGGTAGTGTGTTTAGTATTTTTGATAGTAGATATGATTGAATATTGTGGTAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tGTTATATATAATTGTATTAATTATAAAAAATATNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATTTATTATTAAAAATATTAAATATATTATTTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNATTTTATATTTTTAATAATAAATATAATTATATATTAAAATAAAATAATATATTTAATATTTTTAATAATAAATACAATTAAATATCATAATAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_10_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "100000100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    83458 (83.46%) aligned concordantly 0 times\n",
      "    8189 (8.19%) aligned concordantly exactly 1 time\n",
      "    8353 (8.35%) aligned concordantly >1 times\n",
      "16.54% overall alignment rate\n",
      " reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    83495 (83.50%) aligned concordantly 0 times\n",
      "    8120 (8.12%) aligned concordantly exactly 1 time\n",
      "    8385 (8.38%) aligned concordantly >1 times\n",
      "16.50% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    83320 (83.32%) aligned concordantly 0 times\n",
      "    8200 (8.20%) aligned concordantly exactly 1 time\n",
      "    8480 (8.48%) aligned concordantly >1 times\n",
      "16.68% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    83316 (83.32%) aligned concordantly 0 times\n",
      "    8207 (8.21%) aligned concordantly exactly 1 time\n",
      "    8477 (8.48%) aligned concordantly >1 times\n",
      "16.68% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_10_s1_R1.fastq.gz_C_to_T.fastq, zr2096_10_s1_R1.fastq.gz_G_to_A.fastq, zr2096_10_s1_R2.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1271427\n",
      "\n",
      "Total methylated C's in CpG context:\t132862\n",
      "Total methylated C's in CHG context:\t7807\n",
      "Total methylated C's in CHH context:\t21040\n",
      "Total methylated C's in Unknown context:\t1601\n",
      "\n",
      "Total unmethylated C's in CpG context:\t50564\n",
      "Total unmethylated C's in CHG context:\t293003\n",
      "Total unmethylated C's in CHH context:\t766151\n",
      "Total unmethylated C's in Unknown context:\t5018\n",
      "\n",
      "C methylated in CpG context:\t72.4%\n",
      "C methylated in CHG context:\t2.6%\n",
      "C methylated in CHH context:\t2.7%\n",
      "C methylated in unknown context (CN or CHN):\t24.2%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 9s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t99\tNC_035789.1_CT_converted\t4531039\t2\t37M4I59M\t=\t4531139\t200\tGAGTTTTTTTGATTATTTGTTGTTTGTTGTTTGTTTGNNNNTNNNTNNGTTTGTTTGTTTGTTTGTTTGTTTGTAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tAS:i:-22\tXS:i:-31\tXN:i:0\tXM:i:5\tXO:i:1\tXG:i:4\tNM:i:9\tMD:Z:38T0T0G1T0T52\tYS:i:-24\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t147\tNC_035789.1_CT_converted\t4531139\t2\t100M\t=\t4531039\t-200\tGAATTATTGGGTTAATTTTAATTAAATTTGATATAAAGTATTTTTAGGTGAAGGGGATTTAAGTTTGTTTGAATGAAGGGTTTTGTTTTTTTTTAAGGGG\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-24\tXS:i:-12\tXN:i:0\tXM:i:4\tXO:i:0\tXG:i:0\tNM:i:4\tMD:Z:30G14G9A14A29\tYS:i:-22\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTAAAAAAAAATGAAATTTTTTATTTAAATAAATTTAAATTTTTTTTATTTAAAAATATTTTATATTAAATTTAATTAAAATTAATTTAATAATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTAAAAAAAAATGAAATTTTTTATTTAAATAAATTTAAATTTTTTTTATTTAAAAATATTTTATATTAAATTTAATTAAAATTAATTTAATAATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t83\tNC_035780.1_GA_converted\t8090288\t6\t26M4D74M\t=\t8090028\t-364\tAAAAAAAATAAAAATATAAAAAATTTACAAACAAACAAACAAACAAACAAACNNANNNANNNNCAAACAAACAACAAACAACAAATAATCAAAAAAACTC\tFFFFFFFF/FFFFF/B/FFFFFF/</FFFFFF<FFFFFFFFFFFFFFFF<<<##<###<####FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-26\tXS:i:-9\tXN:i:0\tXM:i:9\tXO:i:1\tXG:i:4\tNM:i:13\tMD:Z:26^ACAA26A0A1C0A0A1C0A0A0A37\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t163\tNC_035780.1_GA_converted\t8090028\t6\t100M\t=\t8090288\t364\tCCCCTTAAAAAAAAACAAAACCCTTCATTCAAACAAACTTAAATCCCCTTCACCTAAAAATACTTTATATCAAATTTAATTAAAATTAACCCAATAATTC\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-18\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:29T70\tYS:i:-26\tYT:Z:CP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_1_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    93560 (93.56%) aligned concordantly 0 times\n",
      "    3220 (3.22%) aligned concordantly exactly 1 time\n",
      "    3220 (3.22%) aligned concordantly >1 times\n",
      "6.44% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    93584 (93.58%) aligned concordantly 0 times\n",
      "    3180 (3.18%) aligned concordantly exactly 1 time\n",
      "    3236 (3.24%) aligned concordantly >1 times\n",
      "6.42% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    93582 (93.58%) aligned concordantly 0 times\n",
      "    3175 (3.17%) aligned concordantly exactly 1 time\n",
      "    3243 (3.24%) aligned concordantly >1 times\n",
      "6.42% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    93611 (93.61%) aligned concordantly 0 times\n",
      "    3157 (3.16%) aligned concordantly exactly 1 time\n",
      "    3232 (3.23%) aligned concordantly >1 times\n",
      "6.39% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_1_s1_R1.fastq.gz_C_to_T.fastq, zr2096_1_s1_R1.fastq.gz_G_to_A.fastq, zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t481569\n",
      "\n",
      "Total methylated C's in CpG context:\t47945\n",
      "Total methylated C's in CHG context:\t2774\n",
      "Total methylated C's in CHH context:\t7846\n",
      "Total methylated C's in Unknown context:\t624\n",
      "\n",
      "Total unmethylated C's in CpG context:\t17126\n",
      "Total unmethylated C's in CHG context:\t107925\n",
      "Total unmethylated C's in CHH context:\t297953\n",
      "Total unmethylated C's in Unknown context:\t1993\n",
      "\n",
      "C methylated in CpG context:\t73.7%\n",
      "C methylated in CHG context:\t2.5%\n",
      "C methylated in CHH context:\t2.6%\n",
      "C methylated in unknown context (CN or CHN):\t23.8%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 3m 34s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_2_s1_R1.fastq.gz to zr2096_2_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_2_s1_R1.fastq.gz to zr2096_2_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_2_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_2_s1_R2.fastq.gz to zr2096_2_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_2_s1_R2.fastq.gz to zr2096_2_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_2_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tTTTTTTATTAAAATTTATTATTTTAAAAATAAAAATTNNNNTNNNTTTTATTATTTTTTTTAATATTTTTTAATATAATTAAATAATAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTCACTTCAAAAAAATAATAAATAAAAAAAATTAATACAACAATATTTTTAATAATACAACAATATTTAAAAAATTATTTATTAATTTTTTTTATTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tCTCCTTATTAAAATCTACTATTCCAAAAATAAAAACTNNNNCNNNTTCCACCATTCCTTCTAACATCTCCCAATATAACTAAATAACAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTGTTTTGGAAAAATAATGGATAAAAAAGGTTGATGTGGTGATGTTTTTAGTGATGTGGTGGTGTTTGAGAAATTATTTATTAATTTTTTTTGTTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tCTCCTTATTAAAATCTACTATTCCAAAAATAAAAACTNNNNCNNNTTCCACCATTCCTTCTAACATCTCCCAATATAACTAAATAACAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTGTTTTGGAAAAATAATGGATAAAAAAGGTTGATGTGGTGATGTTTTTAGTGATGTGGTGGTGTTTGAGAAATTATTTATTAATTTTTTTTGTTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tTTTTTTATTAAAATTTATTATTTTAAAAATAAAAATTNNNNTNNNTTTTATTATTTTTTTTAATATTTTTTAATATAATTAAATAATAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTCACTTCAAAAAAATAATAAATAAAAAAAATTAATACAACAATATTTTTAATAATACAACAATATTTAAAAAATTATTTATTAATTTTTTTTATTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_2_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    85277 (85.28%) aligned concordantly 0 times\n",
      "    7512 (7.51%) aligned concordantly exactly 1 time\n",
      "    7211 (7.21%) aligned concordantly >1 times\n",
      "14.72% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    85171 (85.17%) aligned concordantly 0 times\n",
      "    7449 (7.45%) aligned concordantly exactly 1 time\n",
      "    7380 (7.38%) aligned concordantly >1 times\n",
      "14.83% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84940 (84.94%) aligned concordantly 0 times\n",
      "    7464 (7.46%) aligned concordantly exactly 1 time\n",
      "    7596 (7.60%) aligned concordantly >1 times\n",
      "15.06% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    85135 (85.14%) aligned concordantly 0 times\n",
      "    7348 (7.35%) aligned concordantly exactly 1 time\n",
      "    7517 (7.52%) aligned concordantly >1 times\n",
      "14.87% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_2_s1_R1.fastq.gz_C_to_T.fastq, zr2096_2_s1_R1.fastq.gz_G_to_A.fastq, zr2096_2_s1_R2.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1039727\n",
      "\n",
      "Total methylated C's in CpG context:\t109099\n",
      "Total methylated C's in CHG context:\t6645\n",
      "Total methylated C's in CHH context:\t17729\n",
      "Total methylated C's in Unknown context:\t1331\n",
      "\n",
      "Total unmethylated C's in CpG context:\t37771\n",
      "Total unmethylated C's in CHG context:\t246689\n",
      "Total unmethylated C's in CHH context:\t621794\n",
      "Total unmethylated C's in Unknown context:\t4398\n",
      "\n",
      "C methylated in CpG context:\t74.3%\n",
      "C methylated in CHG context:\t2.6%\n",
      "C methylated in CHH context:\t2.8%\n",
      "C methylated in unknown context (CN or CHN):\t23.2%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 3m 52s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_3_s1_R1.fastq.gz to zr2096_3_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_3_s1_R1.fastq.gz to zr2096_3_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_3_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_3_s1_R2.fastq.gz to zr2096_3_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_3_s1_R2.fastq.gz to zr2096_3_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_3_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tTNTTTTAAAATAATTTAAATAATTTTATATAAATNNNNNNNNNNNNNNNTATTTAAATTAAAAATTTAAATAAATTTTTTTAATTATTTATTGTTTAATG\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAAATAAACATTAAACAATAAATAATTAAAAAAATTTATTTAAATTTTTAATTTAAATAAATTAAAAATATTAAATTTATATAAAATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tCNCTTCAAAATAATTTAAATAATCTTATATAAATNNNNNNNNNNNNNNNCACCTAAACTAAAAACTCAAATAAACTTTTCTAATCACCTATCATCCAACA\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAGATAGATGTTGGATGATAGGTGATTAGAAAAGTTTATTTGAGTTTTTAGTTTAGGTGAGTTAAAAATATTGAATTTATATAAGATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tCNCTTCAAAATAATTTAAATAATCTTATATAAATNNNNNNNNNNNNNNNCACCTAAACTAAAAACTCAAATAAACTTTTCTAATCACCTATCATCCAACA\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAGATAGATGTTGGATGATAGGTGATTAGAAAAGTTTATTTGAGTTTTTAGTTTAGGTGAGTTAAAAATATTGAATTTATATAAGATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tTNTTTTAAAATAATTTAAATAATTTTATATAAATNNNNNNNNNNNNNNNTATTTAAATTAAAAATTTAAATAAATTTTTTTAATTATTTATTGTTTAATG\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAAATAAACATTAAACAATAAATAATTAAAAAAATTTATTTAAATTTTTAATTTAAATAAATTAAAAATATTAAATTTATATAAAATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_3_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    82995 (83.00%) aligned concordantly 0 times\n",
      "    8054 (8.05%) aligned concordantly exactly 1 time\n",
      "    8951 (8.95%) aligned concordantly >1 times\n",
      "17.00% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    83095 (83.09%) aligned concordantly 0 times\n",
      "    7975 (7.97%) aligned concordantly exactly 1 time\n",
      "    8930 (8.93%) aligned concordantly >1 times\n",
      "16.91% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    83122 (83.12%) aligned concordantly 0 times\n",
      "    7830 (7.83%) aligned concordantly exactly 1 time\n",
      "    9048 (9.05%) aligned concordantly >1 times\n",
      "16.88% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    82917 (82.92%) aligned concordantly 0 times\n",
      "    8105 (8.11%) aligned concordantly exactly 1 time\n",
      "    8978 (8.98%) aligned concordantly >1 times\n",
      "17.08% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_3_s1_R1.fastq.gz_C_to_T.fastq, zr2096_3_s1_R1.fastq.gz_G_to_A.fastq, zr2096_3_s1_R2.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1252955\n",
      "\n",
      "Total methylated C's in CpG context:\t149786\n",
      "Total methylated C's in CHG context:\t7391\n",
      "Total methylated C's in CHH context:\t18812\n",
      "Total methylated C's in Unknown context:\t1441\n",
      "\n",
      "Total unmethylated C's in CpG context:\t37500\n",
      "Total unmethylated C's in CHG context:\t290378\n",
      "Total unmethylated C's in CHH context:\t749088\n",
      "Total unmethylated C's in Unknown context:\t4982\n",
      "\n",
      "C methylated in CpG context:\t80.0%\n",
      "C methylated in CHG context:\t2.5%\n",
      "C methylated in CHH context:\t2.4%\n",
      "C methylated in unknown context (CN or CHN):\t22.4%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 3m 59s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_4_s1_R1.fastq.gz to zr2096_4_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_4_s1_R1.fastq.gz to zr2096_4_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_4_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_4_s1_R2.fastq.gz to zr2096_4_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_4_s1_R2.fastq.gz to zr2096_4_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_4_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNGATTAGTGTATAGTGGTTGTTATGGAAATTAGNNNNNNNNNNNNNNNNNNNNNNNNNTGNANTNTTTTTATTAGAATTGTTAAGGTTTATTTGATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTCCATCATTAATTTTTAAAAAAAAAAAACCATCCTTAATATCAATTTTATAAATCAAATAAACCTTAACAATTCTAATAAAAACACTTCATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATTAATATATAATAATTATTATAAAAATTAANNNNNNNNNNNNNNNNNNNNNNNNNTANANTNTTTTTATTAAAATTATTAAAATTTATTTAATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTTTATTATTAATTTTTAAAAAAAAAAAATTATTTTTAATATTAATTTTATAAATTAAATAAATTTTAATAATTTTAATAAAAATATTTTATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATTAATATATAATAATTATTATAAAAATTAANNNNNNNNNNNNNNNNNNNNNNNNNTANANTNTTTTTATTAAAATTATTAAAATTTATTTAATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTTTATTATTAATTTTTAAAAAAAAAAAATTATTTTTAATATTAATTTTATAAATTAAATAAATTTTAATAATTTTAATAAAAATATTTTATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNGATTAGTGTATAGTGGTTGTTATGGAAATTAGNNNNNNNNNNNNNNNNNNNNNNNNNTGNANTNTTTTTATTAGAATTGTTAAGGTTTATTTGATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTCCATCATTAATTTTTAAAAAAAAAAAACCATCCTTAATATCAATTTTATAAATCAAATAAACCTTAACAATTCTAATAAAAACACTTCATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_4_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    83173 (83.17%) aligned concordantly 0 times\n",
      "    8539 (8.54%) aligned concordantly exactly 1 time\n",
      "    8288 (8.29%) aligned concordantly >1 times\n",
      "16.83% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    83193 (83.19%) aligned concordantly 0 times\n",
      "    8419 (8.42%) aligned concordantly exactly 1 time\n",
      "    8388 (8.39%) aligned concordantly >1 times\n",
      "16.81% overall alignment rate\n",
      "100000 reads; of these:\n",
      "100000   reads; of these:100000\n",
      " (  100000100.00 (%) were paired; of these:\n",
      "    82895 (100.00%) were paired; of these:82.89%\n",
      ") aligned concordantly 0 times    \n",
      "    828148591 ( (82.818.59%%) aligned concordantly 0 times) aligned concordantly exactly 1 time\n",
      "\n",
      "        86918514 ( (8.698.51%%) aligned concordantly exactly 1 time) aligned concordantly >1 times\n",
      "\n",
      "    17.11% overall alignment rate\n",
      "8495 (8.49%) aligned concordantly >1 times\n",
      "17.19% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_4_s1_R1.fastq.gz_C_to_T.fastq, zr2096_4_s1_R1.fastq.gz_G_to_A.fastq, zr2096_4_s1_R2.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1189881\n",
      "\n",
      "Total methylated C's in CpG context:\t112195\n",
      "Total methylated C's in CHG context:\t6744\n",
      "Total methylated C's in CHH context:\t19933\n",
      "Total methylated C's in Unknown context:\t1440\n",
      "\n",
      "Total unmethylated C's in CpG context:\t48992\n",
      "Total unmethylated C's in CHG context:\t273615\n",
      "Total unmethylated C's in CHH context:\t728402\n",
      "Total unmethylated C's in Unknown context:\t5075\n",
      "\n",
      "C methylated in CpG context:\t69.6%\n",
      "C methylated in CHG context:\t2.4%\n",
      "C methylated in CHH context:\t2.7%\n",
      "C methylated in unknown context (CN or CHN):\t22.1%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 0s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_5_s1_R1.fastq.gz to zr2096_5_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_5_s1_R1.fastq.gz to zr2096_5_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_5_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_5_s1_R2.fastq.gz to zr2096_5_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_5_s1_R2.fastq.gz to zr2096_5_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_5_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATAAAATATTATTTAAATTTTATTATTTATANNNNNNNNNNNNNNNNNNATTNTTTTTATTATATATATAAAATTATTTTAAAATTATTATATTTTTT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTCAAATATATTAAAAAAAAATTATTAATAATTTTCATTAAAAAATATAATAATTTTAAAATAATTTTATATATATAATAAAAAAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAACAAAACACCACTCAAATCTTACTATCCACANNNNNNNNNNNNNNNNNNATTNCTTCTATCATATATATAAAATCATTCCAAAATTACTATATTTTTC\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTTGAGTATATTGAAGAGGAATTATTGATAGTTTTTGTTGAAAAATATAGTAATTTTGGAATGATTTTATATATATGATAGAAGAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAACAAAACACCACTCAAATCTTACTATCCACANNNNNNNNNNNNNNNNNNATTNCTTCTATCATATATATAAAATCATTCCAAAATTACTATATTTTTC\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTTGAGTATATTGAAGAGGAATTATTGATAGTTTTTGTTGAAAAATATAGTAATTTTGGAATGATTTTATATATATGATAGAAGAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATAAAATATTATTTAAATTTTATTATTTATANNNNNNNNNNNNNNNNNNATTNTTTTTATTATATATATAAAATTATTTTAAAATTATTATATTTTTT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTCAAATATATTAAAAAAAAATTATTAATAATTTTCATTAAAAAATATAATAATTTTAAAATAATTTTATATATATAATAAAAAAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_5_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84478 (84.48%) aligned concordantly 0 times\n",
      "    7778 (7.78%) aligned concordantly exactly 1 time\n",
      "    7744 (7.74%) aligned concordantly >1 times\n",
      "15.52% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84534 (84.53%) aligned concordantly 0 times\n",
      "    7798 (7.80%) aligned concordantly exactly 1 time\n",
      "    7668 (7.67%) aligned concordantly >1 times\n",
      "15.47% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84350 (84.35%) aligned concordantly 0 times\n",
      "    7780 (7.78%) aligned concordantly exactly 1 time\n",
      "    7870 (7.87%) aligned concordantly >1 times\n",
      "15.65% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84194 (84.19%) aligned concordantly 0 times\n",
      "    7813 (7.81%) aligned concordantly exactly 1 time\n",
      "    7993 (7.99%) aligned concordantly >1 times\n",
      "15.81% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_5_s1_R1.fastq.gz_C_to_T.fastq, zr2096_5_s1_R1.fastq.gz_G_to_A.fastq, zr2096_5_s1_R2.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1159933\n",
      "\n",
      "Total methylated C's in CpG context:\t108103\n",
      "Total methylated C's in CHG context:\t6982\n",
      "Total methylated C's in CHH context:\t19266\n",
      "Total methylated C's in Unknown context:\t1384\n",
      "\n",
      "Total unmethylated C's in CpG context:\t44016\n",
      "Total unmethylated C's in CHG context:\t258610\n",
      "Total unmethylated C's in CHH context:\t722956\n",
      "Total unmethylated C's in Unknown context:\t4955\n",
      "\n",
      "C methylated in CpG context:\t71.1%\n",
      "C methylated in CHG context:\t2.6%\n",
      "C methylated in CHH context:\t2.6%\n",
      "C methylated in unknown context (CN or CHN):\t21.8%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 2s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_6_s1_R1.fastq.gz to zr2096_6_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_6_s1_R1.fastq.gz to zr2096_6_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_6_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_6_s1_R2.fastq.gz to zr2096_6_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_6_s1_R2.fastq.gz to zr2096_6_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_6_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTTATTAAATTTAATTTTATTATTAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNTANANAAATATTAATTAAATATTTTTAAAATAATAAATTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAAAATTTTTTTAAATTTATTATTTTAAAAATATTTAATTAATATTTATTTATATTTTATAATTAACAATAATTATTTTTTTATTTAATAATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTCATCAAATCCAACTTTATCACCAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNCANANAAACATCAATCAAACATCTCTAAAACAATAAACTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAGAATTTTTTTAAGTTTATTGTTTTAGAGATGTTTGATTGATGTTTATTTGTATTTTGTAATTAGTGGTGGTTGTTTTTTTATTTGGTGATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTCATCAAATCCAACTTTATCACCAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNCANANAAACATCAATCAAACATCTCTAAAACAATAAACTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAGAATTTTTTTAAGTTTATTGTTTTAGAGATGTTTGATTGATGTTTATTTGTATTTTGTAATTAGTGGTGGTTGTTTTTTTATTTGGTGATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTTATTAAATTTAATTTTATTATTAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNTANANAAATATTAATTAAATATTTTTAAAATAATAAATTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAAAATTTTTTTAAATTTATTATTTTAAAAATATTTAATTAATATTTATTTATATTTTATAATTAACAATAATTATTTTTTTATTTAATAATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_6_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84081 (84.08%) aligned concordantly 0 times\n",
      "    8033 (8.03%) aligned concordantly exactly 1 time\n",
      "    7886 (7.89%) aligned concordantly >1 times\n",
      "15.92% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    83934 (83.93%) aligned concordantly 0 times\n",
      "    8147 (8.15%) aligned concordantly exactly 1 time\n",
      "    7919 (7.92%) aligned concordantly >1 times\n",
      "16.07% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    82786 (82.79%) aligned concordantly 0 times\n",
      "    8606 (8.61%) aligned concordantly exactly 1 time\n",
      "    8608 (8.61%) aligned concordantly >1 times\n",
      "17.21% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    82685 (82.69%) aligned concordantly 0 times\n",
      "    8647 (8.65%) aligned concordantly exactly 1 time\n",
      "    8668 (8.67%) aligned concordantly >1 times\n",
      "17.32% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_6_s1_R1.fastq.gz_C_to_T.fastq, zr2096_6_s1_R1.fastq.gz_G_to_A.fastq, zr2096_6_s1_R2.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1185031\n",
      "\n",
      "Total methylated C's in CpG context:\t126300\n",
      "Total methylated C's in CHG context:\t7170\n",
      "Total methylated C's in CHH context:\t19881\n",
      "Total methylated C's in Unknown context:\t1537\n",
      "\n",
      "Total unmethylated C's in CpG context:\t36204\n",
      "Total unmethylated C's in CHG context:\t270406\n",
      "Total unmethylated C's in CHH context:\t725070\n",
      "Total unmethylated C's in Unknown context:\t5223\n",
      "\n",
      "C methylated in CpG context:\t77.7%\n",
      "C methylated in CHG context:\t2.6%\n",
      "C methylated in CHH context:\t2.7%\n",
      "C methylated in unknown context (CN or CHN):\t22.7%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 3m 58s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_7_s1_R1.fastq.gz to zr2096_7_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_7_s1_R1.fastq.gz to zr2096_7_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_7_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_7_s1_R2.fastq.gz to zr2096_7_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_7_s1_R2.fastq.gz to zr2096_7_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_7_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAAAAAATGATTATTATGATGAATTTAAATATTAATAATAAAATTTAAAAAAAATATAAATGTTAAATATTAAATTTTTTAAAAAATGATTATGG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t141\t*\t0\t0\t*\t*\t0\t0\tCCATAATCATTTTTTAAAAAATTTAATATTTAACATTTATATTTTTTTTAAATTTTATTATTAATATTTAAATTCATCATAATAATCATTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t99\tNC_035781.1_GA_converted\t24539746\t12\t95M\t=\t24539746\t-95\tAAAAAAACAACTACTACAACAAATTCAAATATTAATAACAAAATTTAAAAAAAACATAAACATTAAATATCAAATTCTTCAAAAAACAATTACAA\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-12\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:94T0\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t147\tNC_035781.1_GA_converted\t24539746\t12\t95M\t=\t24539746\t-95\tAAAAAAACAACTACTACAACAAATTCAAATATTAATAACAAAATTTAAAAAAAACATAAACATTAAATATCAAATTCTTCAAAAAACAATTACAA\tFFFFFFFFFFFFFFFFFFFFFFFBBFFFFFFBBFF/FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-6\tXS:i:-12\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:94T0\tYS:i:-6\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t83\tNC_035780.1_CT_converted\t42548681\t21\t95M\t=\t42548681\t-95\tTTGTAATTGTTTTTTGAAGAATTTGATATTTAATGTTTATGTTTTTTTTAAATTTTGTTATTAATATTTGAATTTGTTGTAGTAGTTGTTTTTTT\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-6\tXS:i:-32\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:0A94\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t163\tNC_035780.1_CT_converted\t42548681\t21\t95M\t=\t42548681\t-95\tTTGTAATTGTTTTTTGAAGAATTTGATATTTAATGTTTATGTTTTTTTTAAATTTTGTTATTAATATTTGAATTTGTTGTAGTAGTTGTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-32\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:0A94\tYS:i:-6\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAAAAAATGATTATTATGATGAATTTAAATATTAATAATAAAATTTAAAAAAAATATAAATGTTAAATATTAAATTTTTTAAAAAATGATTATGG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t141\t*\t0\t0\t*\t*\t0\t0\tCCATAATCATTTTTTAAAAAATTTAATATTTAACATTTATATTTTTTTTAAATTTTATTATTAATATTTAAATTCATCATAATAATCATTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_7_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84887 (84.89%) aligned concordantly 0 times\n",
      "    7610 (7.61%) aligned concordantly exactly 1 time\n",
      "    7503 (7.50%) aligned concordantly >1 times\n",
      "15.11% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84905 (84.91%) aligned concordantly 0 times\n",
      "    7561 (7.56%) aligned concordantly exactly 1 time\n",
      "    7534 (7.53%) aligned concordantly >1 times\n",
      "15.10% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84581 (84.58%) aligned concordantly 0 times\n",
      "    7709 (7.71%) aligned concordantly exactly 1 time\n",
      "    7710 (7.71%) aligned concordantly >1 times\n",
      "15.42% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84387 (84.39%) aligned concordantly 0 times\n",
      "    7828 (7.83%) aligned concordantly exactly 1 time\n",
      "    7785 (7.79%) aligned concordantly >1 times\n",
      "15.61% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_7_s1_R1.fastq.gz_C_to_T.fastq, zr2096_7_s1_R1.fastq.gz_G_to_A.fastq, zr2096_7_s1_R2.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1161926\n",
      "\n",
      "Total methylated C's in CpG context:\t114479\n",
      "Total methylated C's in CHG context:\t7182\n",
      "Total methylated C's in CHH context:\t20183\n",
      "Total methylated C's in Unknown context:\t1356\n",
      "\n",
      "Total unmethylated C's in CpG context:\t38201\n",
      "Total unmethylated C's in CHG context:\t262714\n",
      "Total unmethylated C's in CHH context:\t719167\n",
      "Total unmethylated C's in Unknown context:\t4916\n",
      "\n",
      "C methylated in CpG context:\t75.0%\n",
      "C methylated in CHG context:\t2.7%\n",
      "C methylated in CHH context:\t2.7%\n",
      "C methylated in unknown context (CN or CHN):\t21.6%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 5s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_8_s1_R1.fastq.gz to zr2096_8_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_8_s1_R1.fastq.gz to zr2096_8_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_8_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_8_s1_R2.fastq.gz to zr2096_8_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_8_s1_R2.fastq.gz to zr2096_8_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_8_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATTTAAAATTAAAAAATTTATTAAATTTATTATTATTTNTTATTTTTTATAAAAGAAAATAGAGAGAGAGAGAGAGAGAGAGAGAG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tCTCTCTCTCTCTCTCTCTCTCTCTCTATTTTCTTTTATAAAAAATAATAAATAATAATAAATTTAATAAATTTTTTAATTTTAAATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATCCAAAATTAAAAAACTCATTAAATCTATCATCATTCNCCATTCTTCACAAAAAAAAATAAAAAAAAAAAAAAAAAAAAAAAAAA\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTTTTTTTTTTTTTTTTTTTTTATTTTTTTTTGTGAAGAATGGTGAATGATGATAGATTTAATGAGTTTTTTAATTTTGGATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATCCAAAATTAAAAAACTCATTAAATCTATCATCATTCNCCATTCTTCACAAAAAAAAATAAAAAAAAAAAAAAAAAAAAAAAAAA\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTTTTTTTTTTTTTTTTTTTTTATTTTTTTTTGTGAAGAATGGTGAATGATGATAGATTTAATGAGTTTTTTAATTTTGGATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATTTAAAATTAAAAAATTTATTAAATTTATTATTATTTNTTATTTTTTATAAAAGAAAATAGAGAGAGAGAGAGAGAGAGAGAGAG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tCTCTCTCTCTCTCTCTCTCTCTCTCTATTTTCTTTTATAAAAAATAATAAATAATAATAAATTTAATAAATTTTTTAATTTTAAATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_8_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    85958 (85.96%) aligned concordantly 0 times\n",
      "    7118 (7.12%) aligned concordantly exactly 1 time\n",
      "    6924 (6.92%) aligned concordantly >1 times\n",
      "14.04% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    86249 (86.25%) aligned concordantly 0 times\n",
      "    7037 (7.04%) aligned concordantly exactly 1 time\n",
      "    6714 (6.71%) aligned concordantly >1 times\n",
      "13.75% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    86013 (86.01%) aligned concordantly 0 times\n",
      "    7180 (7.18%) aligned concordantly exactly 1 time\n",
      "    6807 (6.81%) aligned concordantly >1 times\n",
      "13.99% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    86322 (86.32%) aligned concordantly 0 times\n",
      "    7009 (7.01%) aligned concordantly exactly 1 time\n",
      "    6669 (6.67%) aligned concordantly >1 times\n",
      "13.68% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_8_s1_R1.fastq.gz_C_to_T.fastq, zr2096_8_s1_R1.fastq.gz_G_to_A.fastq, zr2096_8_s1_R2.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1029296\n",
      "\n",
      "Total methylated C's in CpG context:\t90766\n",
      "Total methylated C's in CHG context:\t6484\n",
      "Total methylated C's in CHH context:\t18894\n",
      "Total methylated C's in Unknown context:\t1299\n",
      "\n",
      "Total unmethylated C's in CpG context:\t39042\n",
      "Total unmethylated C's in CHG context:\t235415\n",
      "Total unmethylated C's in CHH context:\t638695\n",
      "Total unmethylated C's in Unknown context:\t4409\n",
      "\n",
      "C methylated in CpG context:\t69.9%\n",
      "C methylated in CHG context:\t2.7%\n",
      "C methylated in CHH context:\t2.9%\n",
      "C methylated in unknown context (CN or CHN):\t22.8%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 3m 52s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_9_s1_R1.fastq.gz to zr2096_9_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_9_s1_R1.fastq.gz to zr2096_9_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_9_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_9_s1_R2.fastq.gz to zr2096_9_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_9_s1_R2.fastq.gz to zr2096_9_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_9_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNATTAAATAAAAAATAATAAATAATAAATTANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAGG\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAAAAAATAAAAAACACATTAAAAATCTATAATTTAATTTATTATTTATTATTTTTTATTTAATTAAATCNNNNAAACATCATATAAAAAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNACCAAATAAAAAATAACAAATAATAAATCANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAAA\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAGAGGATAGGAGATATGTTAGGAATTTATAATTTGATTTATTATTTGTTATTTTTTATTTGGTTAGATTNNNNGAGTGTTGTGTAGGGAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNACCAAATAAAAAATAACAAATAATAAATCANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAAA\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAGAGGATAGGAGATATGTTAGGAATTTATAATTTGATTTATTATTTGTTATTTTTTATTTGGTTAGATTNNNNGAGTGTTGTGTAGGGAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNATTAAATAAAAAATAATAAATAATAAATTANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAGG\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAAAAAATAAAAAACACATTAAAAATCTATAATTTAATTTATTATTTATTATTTTTTATTTAATTAAATCNNNNAAACATCATATAAAAAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_9_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    85652 (85.65%) aligned concordantly 0 times\n",
      "    7096 (7.10%) aligned concordantly exactly 1 time\n",
      "    7252 (7.25%) aligned concordantly >1 times\n",
      "14.35% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84038 (84.04%) aligned concordantly 0 times\n",
      "    7970 (7.97%) aligned concordantly exactly 1 time\n",
      "    7992 (7.99%) aligned concordantly >1 times\n",
      "15.96% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    85781 (85.78%) aligned concordantly 0 times\n",
      "    7152 (7.15%) aligned concordantly exactly 1 time\n",
      "    7067 (7.07%) aligned concordantly >1 times\n",
      "14.22% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84011 (84.01%) aligned concordantly 0 times\n",
      "    8024 (8.02%) aligned concordantly exactly 1 time\n",
      "    7965 (7.96%) aligned concordantly >1 times\n",
      "15.99% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_9_s1_R1.fastq.gz_C_to_T.fastq, zr2096_9_s1_R1.fastq.gz_G_to_A.fastq, zr2096_9_s1_R2.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1170800\n",
      "\n",
      "Total methylated C's in CpG context:\t135804\n",
      "Total methylated C's in CHG context:\t7983\n",
      "Total methylated C's in CHH context:\t20498\n",
      "Total methylated C's in Unknown context:\t1414\n",
      "\n",
      "Total unmethylated C's in CpG context:\t34966\n",
      "Total unmethylated C's in CHG context:\t268427\n",
      "Total unmethylated C's in CHH context:\t703122\n",
      "Total unmethylated C's in Unknown context:\t4683\n",
      "\n",
      "C methylated in CpG context:\t79.5%\n",
      "C methylated in CHG context:\t2.9%\n",
      "C methylated in CHH context:\t2.8%\n",
      "C methylated in unknown context (CN or CHN):\t23.2%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 0s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "find /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_*R1* \\\n",
    "| xargs basename -s _s1_R1.fastq.gz | xargs -I{} /Users/Shelly/anaconda3/bin/bismark \\\n",
    "--genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ \\\n",
    "--score_min L,0,-0.6 \\\n",
    "-u 100000 \\\n",
    "--non_directional \\\n",
    "-1 /Volumes/web/seashell/bu-serine-wd/18-04-07/{}_s1_R1.fastq.gz \\\n",
    "-2 /Volumes/web/seashell/bu-serine-wd/18-04-07/{}_s1_R2.fastq.gz \\\n",
    "-o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/ "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#run deduplicate on stricter alignment score threshold alignments"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_10_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-0.6 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_1_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-0.6 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_2_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-0.6 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_3_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-0.6 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_4_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-0.6 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_5_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-0.6 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_6_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-0.6 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_7_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-0.6 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_8_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-0.6 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_9_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-0.6 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\"\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\n",
      "Processing paired-end Bismark output file(s) (SAM format):\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_10_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_1_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_2_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_3_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_4_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_5_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_6_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_7_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_8_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_9_s1_R1_bismark_bt2_pe.bam\n",
      "\n",
      "\n",
      "If there are several alignments to a single position in the genome the first alignment will be chosen. Since the input files are not in any way sorted this is a near-enough random selection of reads.\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_10_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_10_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_10_s1_R1_bismark_bt2_pe.bam:\t35458\n",
      "Total number duplicated alignments removed:\t15 (0.04%)\n",
      "Duplicated alignments were found at:\t14 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 35443 (99.96% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_1_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_1_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_1_s1_R1_bismark_bt2_pe.bam:\t13744\n",
      "Total number duplicated alignments removed:\t1 (0.01%)\n",
      "Duplicated alignments were found at:\t1 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 13743 (99.99% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_2_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_2_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_2_s1_R1_bismark_bt2_pe.bam:\t31325\n",
      "Total number duplicated alignments removed:\t9 (0.03%)\n",
      "Duplicated alignments were found at:\t9 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 31316 (99.97% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_3_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_3_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_3_s1_R1_bismark_bt2_pe.bam:\t34892\n",
      "Total number duplicated alignments removed:\t13 (0.04%)\n",
      "Duplicated alignments were found at:\t13 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 34879 (99.96% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_4_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_4_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_4_s1_R1_bismark_bt2_pe.bam:\t35879\n",
      "Total number duplicated alignments removed:\t9 (0.03%)\n",
      "Duplicated alignments were found at:\t8 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 35870 (99.97% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_5_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_5_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_5_s1_R1_bismark_bt2_pe.bam:\t33466\n",
      "Total number duplicated alignments removed:\t17 (0.05%)\n",
      "Duplicated alignments were found at:\t17 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 33449 (99.95% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_6_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_6_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_6_s1_R1_bismark_bt2_pe.bam:\t35383\n",
      "Total number duplicated alignments removed:\t9 (0.03%)\n",
      "Duplicated alignments were found at:\t9 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 35374 (99.97% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_7_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_7_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_7_s1_R1_bismark_bt2_pe.bam:\t33107\n",
      "Total number duplicated alignments removed:\t10 (0.03%)\n",
      "Duplicated alignments were found at:\t10 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 33097 (99.97% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_8_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_8_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_8_s1_R1_bismark_bt2_pe.bam:\t30256\n",
      "Total number duplicated alignments removed:\t10 (0.03%)\n",
      "Duplicated alignments were found at:\t10 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 30246 (99.97% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_9_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_9_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_9_s1_R1_bismark_bt2_pe.bam:\t32194\n",
      "Total number duplicated alignments removed:\t20 (0.06%)\n",
      "Duplicated alignments were found at:\t20 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 32174 (99.94% of total)\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "/Users/Shelly/anaconda3/bin/deduplicate_bismark \\\n",
    "--bam -p \\\n",
    "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/*.bam \\\n",
    "-o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/ \n",
    "2> /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/dedup.err"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#analysis with a looser alignment score threshold"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!mkdir /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t79505\n",
      "Mapping efficiency:\t79.5%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t213\n",
      "Sequence pairs did not map uniquely:\t20282\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t20127\t((converted) top strand)\n",
      "GA/CT/CT:\t19634\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t19795\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t19949\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t84845\n",
      "Mapping efficiency:\t84.8%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t573\n",
      "Sequence pairs did not map uniquely:\t14582\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t21400\t((converted) top strand)\n",
      "GA/CT/CT:\t21240\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t20948\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t21257\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t79272\n",
      "Mapping efficiency:\t79.3%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t526\n",
      "Sequence pairs did not map uniquely:\t20202\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t20068\t((converted) top strand)\n",
      "GA/CT/CT:\t19703\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t19524\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t19977\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t78192\n",
      "Mapping efficiency:\t78.2%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t310\n",
      "Sequence pairs did not map uniquely:\t21498\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t19689\t((converted) top strand)\n",
      "GA/CT/CT:\t19420\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t19283\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t19800\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t78339\n",
      "Mapping efficiency:\t78.3%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t695\n",
      "Sequence pairs did not map uniquely:\t20966\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t19925\t((converted) top strand)\n",
      "GA/CT/CT:\t19098\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t19148\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t20168\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t79734\n",
      "Mapping efficiency:\t79.7%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t374\n",
      "Sequence pairs did not map uniquely:\t19892\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t20457\t((converted) top strand)\n",
      "GA/CT/CT:\t19366\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t19341\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t20570\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t78624\n",
      "Mapping efficiency:\t78.6%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t323\n",
      "Sequence pairs did not map uniquely:\t21053\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t20580\t((converted) top strand)\n",
      "GA/CT/CT:\t18627\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t18702\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t20715\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t80059\n",
      "Mapping efficiency:\t80.1%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t386\n",
      "Sequence pairs did not map uniquely:\t19555\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t20411\t((converted) top strand)\n",
      "GA/CT/CT:\t19497\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t19740\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t20411\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t81008\n",
      "Mapping efficiency:\t81.0%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t385\n",
      "Sequence pairs did not map uniquely:\t18607\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t20390\t((converted) top strand)\n",
      "GA/CT/CT:\t20130\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t19951\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t20537\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t79357\n",
      "Mapping efficiency:\t79.4%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t540\n",
      "Sequence pairs did not map uniquely:\t20103\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t21347\t((converted) top strand)\n",
      "GA/CT/CT:\t18451\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t18365\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t21194\t((converted) bottom strand)\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_10_s1_R1.fastq.gz to zr2096_10_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_10_s1_R1.fastq.gz to zr2096_10_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_10_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_10_s1_R2.fastq.gz to zr2096_10_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_10_s1_R2.fastq.gz to zr2096_10_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_10_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tGTTATATATAATTGTATTAATTATAAAAAATATNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATTTATTATTAAAAATATTAAATATATTATTTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNATTTTATATTTTTAATAATAAATATAATTATATATTAAAATAAAATAATATATTTAATATTTTTAATAATAAATACAATTAAATATCATAATAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tACCATATACAATCATATCAACTATAAAAAATACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATCTACTATCAAAAATACTAAACACACTACCTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNGTTTTGTATTTTTGATAGTAGATATGATTGTATATTAGGGTAAGGTAGTGTGTTTAGTATTTTTGATAGTAGATATGATTGAATATTGTGGTAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tACCATATACAATCATATCAACTATAAAAAATACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATCTACTATCAAAAATACTAAACACACTACCTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNGTTTTGTATTTTTGATAGTAGATATGATTGTATATTAGGGTAAGGTAGTGTGTTTAGTATTTTTGATAGTAGATATGATTGAATATTGTGGTAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tGTTATATATAATTGTATTAATTATAAAAAATATNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATTTATTATTAAAAATATTAAATATATTATTTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNATTTTATATTTTTAATAATAAATATAATTATATATTAAAATAAAATAATATATTTAATATTTTTAATAATAAATACAATTAAATATCATAATAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_10_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    8527 (8.53%) aligned concordantly 0 times\n",
      "    3790 (3.79%) aligned concordantly exactly 1 time\n",
      "    87683 (87.68%) aligned concordantly >1 times\n",
      "91.47% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    8435 (8.44%) aligned concordantly 0 times\n",
      "    3658 (3.66%) aligned concordantly exactly 1 time\n",
      "    87907 (87.91%) aligned concordantly >1 times\n",
      "91.56% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9441 (9.44%) aligned concordantly 0 times\n",
      "    3544 (3.54%) aligned concordantly exactly 1 time\n",
      "    87015 (87.02%) aligned concordantly >1 times\n",
      "90.56% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9502 (9.50%) aligned concordantly 0 times\n",
      "    3408 (3.41%) aligned concordantly exactly 1 time\n",
      "    87090 (87.09%) aligned concordantly >1 times\n",
      "90.50% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_10_s1_R1.fastq.gz_C_to_T.fastq, zr2096_10_s1_R1.fastq.gz_G_to_A.fastq, zr2096_10_s1_R2.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t2596537\n",
      "\n",
      "Total methylated C's in CpG context:\t229383\n",
      "Total methylated C's in CHG context:\t25983\n",
      "Total methylated C's in CHH context:\t138155\n",
      "Total methylated C's in Unknown context:\t26486\n",
      "\n",
      "Total unmethylated C's in CpG context:\t118501\n",
      "Total unmethylated C's in CHG context:\t530950\n",
      "Total unmethylated C's in CHH context:\t1553565\n",
      "Total unmethylated C's in Unknown context:\t73368\n",
      "\n",
      "C methylated in CpG context:\t65.9%\n",
      "C methylated in CHG context:\t4.7%\n",
      "C methylated in CHH context:\t8.2%\n",
      "C methylated in unknown context (CN or CHN):\t26.5%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 12m 7s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t99\tNC_035786.1_CT_converted\t55112902\t1\t100M\t=\t55113162\t360\tGAGTTTTTTTGATTATTTGTTGTTTGTTGTTTGTTTGNNNNTNNNTNNGTTTGTTTGTTTGTTTGTTTGTTTGTAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tAS:i:-9\tXS:i:-22\tXN:i:0\tXM:i:9\tXO:i:0\tXG:i:0\tNM:i:9\tMD:Z:37T0T0T0G1T0T0G1T0T52\tYS:i:-30\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t147\tNC_035786.1_CT_converted\t55113162\t1\t100M\t=\t55112902\t-360\tGAATTATTGGGTTAATTTTAATTAAATTTGATATAAAGTATTTTTAGGTGAAGGGGATTTAAGTTTGTTTGAATGAAGGGTTTTGTTTTTTTTTAAGGGG\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-30\tXS:i:-6\tXN:i:0\tXM:i:5\tXO:i:0\tXG:i:0\tNM:i:5\tMD:Z:10A19G14G24A13T15\tYS:i:-9\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t99\tNC_035788.1_GA_converted\t81174826\t0\t13M2I7M3I2M2I6M2I11M2I4M3I4M1I6M1D9M8I1M1I13M\t=\t81174826\t-84\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tAS:i:-176\tXS:i:-166\tXN:i:0\tXM:i:16\tXO:i:10\tXG:i:25\tNM:i:41\tMD:Z:8A0C1C14C1A0A0C0A1A0T0A1C0A8A5^T2A10C9\tYS:i:-162\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t147\tNC_035788.1_GA_converted\t81174826\t0\t9M3I7M1I25M2I2M8I5M3I4M1D31M\t=\t81174826\t-84\tAAATTATTAAATTAATTTTAATTAAATTTAATATAAAATATTTTTAAATAAAAAAAATTTAAATTTATTTAAATAAAAAATTTCATTTTTTTTTAAAAAA\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-162\tXS:i:-162\tXN:i:0\tXM:i:13\tXO:i:6\tXG:i:18\tNM:i:31\tMD:Z:5T3C1C6C2T4C2A0C1T4C0A13^C2T3T24\tYS:i:-176\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTAAAAAAAAATGAAATTTTTTATTTAAATAAATTTAAATTTTTTTTATTTAAAAATATTTTATATTAAATTTAATTAAAATTAATTTAATAATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t83\tNC_035780.1_GA_converted\t8090288\t6\t26M4D74M\t=\t8090028\t-364\tAAAAAAAATAAAAATATAAAAAATTTACAAACAAACAAACAAACAAACAAACNNANNNANNNNCAAACAAACAACAAACAACAAATAATCAAAAAAACTC\tFFFFFFFF/FFFFF/B/FFFFFF/</FFFFFF<FFFFFFFFFFFFFFFF<<<##<###<####FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-26\tXS:i:-26\tXN:i:0\tXM:i:9\tXO:i:1\tXG:i:4\tNM:i:13\tMD:Z:26^ACAA26A0A1C0A0A1C0A0A0A37\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t163\tNC_035780.1_GA_converted\t8090028\t6\t100M\t=\t8090288\t364\tCCCCTTAAAAAAAAACAAAACCCTTCATTCAAACAAACTTAAATCCCCTTCACCTAAAAATACTTTATATCAAATTTAATTAAAATTAACCCAATAATTC\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-24\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:29T70\tYS:i:-26\tYT:Z:CP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_1_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    19427 (19.43%) aligned concordantly 0 times\n",
      "    3105 (3.10%) aligned concordantly exactly 1 time\n",
      "    77468 (77.47%) aligned concordantly >1 times\n",
      "80.57% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    19585 (19.59%) aligned concordantly 0 times\n",
      "    3108 (3.11%) aligned concordantly exactly 1 time\n",
      "    77307 (77.31%) aligned concordantly >1 times\n",
      "80.42% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    18409 (18.41%) aligned concordantly 0 times\n",
      "    3566 (3.57%) aligned concordantly exactly 1 time\n",
      "    78025 (78.03%) aligned concordantly >1 times\n",
      "81.59% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    18562 (18.56%) aligned concordantly 0 times\n",
      "    3472 (3.47%) aligned concordantly exactly 1 time\n",
      "    77966 (77.97%) aligned concordantly >1 times\n",
      "81.44% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_1_s1_R1.fastq.gz_C_to_T.fastq, zr2096_1_s1_R1.fastq.gz_G_to_A.fastq, zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t2232980\n",
      "\n",
      "Total methylated C's in CpG context:\t137788\n",
      "Total methylated C's in CHG context:\t45104\n",
      "Total methylated C's in CHH context:\t464015\n",
      "Total methylated C's in Unknown context:\t97154\n",
      "\n",
      "Total unmethylated C's in CpG context:\t77444\n",
      "Total unmethylated C's in CHG context:\t285519\n",
      "Total unmethylated C's in CHH context:\t1223110\n",
      "Total unmethylated C's in Unknown context:\t130947\n",
      "\n",
      "C methylated in CpG context:\t64.0%\n",
      "C methylated in CHG context:\t13.6%\n",
      "C methylated in CHH context:\t27.5%\n",
      "C methylated in unknown context (CN or CHN):\t42.6%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 10m 9s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_2_s1_R1.fastq.gz to zr2096_2_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_2_s1_R1.fastq.gz to zr2096_2_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_2_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_2_s1_R2.fastq.gz to zr2096_2_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_2_s1_R2.fastq.gz to zr2096_2_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_2_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t99\tNC_035788.1_CT_converted\t35644177\t2\t11M3I13M9I22M2D3M2I1M1I6M2I4M2I8M6I7M\t=\t35644107\t-147\tTTTTTTATTAAAATTTATTATTTTAAAAATAAAAATTNNNNTNNNTTTTATTATTTTTTTTAATATTTTTTAATATAATTAAATAATAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-164\tXS:i:-155\tXN:i:0\tXM:i:13\tXO:i:8\tXG:i:27\tNM:i:40\tMD:Z:9T6T4G0G2T0T0T0T1A0T0A13^GG14G10A3\tYS:i:-92\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t147\tNC_035788.1_CT_converted\t35644107\t2\t6M1I92M\t=\t35644177\t147\tAAATAATAAAAAAAATTAATAAATAATTTTTTAAATATTGTTGTATTATTAAAAATATTGTTGTATTAATTTTTTTTATTTATTATTTTTTTGAAGTGA\tFFB/FFBFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-92\tXS:i:-143\tXN:i:0\tXM:i:14\tXO:i:1\tXG:i:1\tNM:i:15\tMD:Z:1G0G1T14G4G7G0G4T13G0T38G1T1T0T0\tYS:i:-164\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tCTCCTTATTAAAATCTACTATTCCAAAAATAAAAACTNNNNCNNNTTCCACCATTCCTTCTAACATCTCCCAATATAACTAAATAACAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTGTTTTGGAAAAATAATGGATAAAAAAGGTTGATGTGGTGATGTTTTTAGTGATGTGGTGGTGTTTGAGAAATTATTTATTAATTTTTTTTGTTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t83\tNC_035780.1_CT_converted\t29673567\t2\t14M3I3M2I6M2I13M1D30M2D7M3D2M1I9M1D8M\t=\t29673488\t-178\tATTAATTTTTTTTGTTATTTAGTTATATTGGGAGATGTTAGAAGGAATGGTGGAANNNGNNNNAGTTTTTATTTTTGGAATAGTAGATTTTAATAAGGAG\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFF<<<###<####FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-176\tXS:i:-187\tXN:i:0\tXM:i:21\tXO:i:8\tXG:i:15\tNM:i:36\tMD:Z:14G9T1T2A3T2^T8A3A0T0T0T0T0T0A0A1T8^AG3A0T2^ATT3T7^T3T1T1T0\tYS:i:-41\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t163\tNC_035780.1_CT_converted\t29673488\t2\t93M2I4M\t=\t29673567\t178\tTTGTTTTGGAAAAATAATGGATAAAAAAGGTTGATGTGGTGATGTTTTTAGTGATGTGGTGGTGTTTGAGAAATTATTTATTAATTTTTTTTGTTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tAS:i:-41\tXS:i:-41\tXN:i:0\tXM:i:5\tXO:i:1\tXG:i:2\tNM:i:7\tMD:Z:0A0A0A41A48G3\tYS:i:-176\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t83\tNC_035786.1_GA_converted\t15444144\t2\t9M5I4M3I16M2I16M7I6M3I16M4I4M2D5M\t=\t15444158\t93\tATTAATTTTTTTTATTATTTAATTATATTAAAAAATATTAAAAAAAATAATAAAANNNANNNNAATTTTTATTTTTAAAATAATAAATTTTAATAAAAAA\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFF<<<###<####FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-162\tXS:i:-166\tXN:i:0\tXM:i:9\tXO:i:7\tXG:i:26\tNM:i:35\tMD:Z:1C3A0A8C1C5T12T8T12T12^TT5\tYS:i:-162\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t163\tNC_035786.1_GA_converted\t15444158\t2\t4M8I30M1D24M3I4M2I5M2I2M6I9M\t=\t15444144\t-93\tTCACTTCAAAAAAATAATAAATAAAAAAAATTAATACAACAATATTTTTAATAATACAACAATATTTAAAAAATTATTTATTAATTTTTTTTATTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tAS:i:-162\tXS:i:-163\tXN:i:0\tXM:i:11\tXO:i:6\tXG:i:22\tNM:i:33\tMD:Z:5T22A2T2^T1T9T0A1T2A17C2A3C1\tYS:i:-162\tYT:Z:CP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_2_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    13202 (13.20%) aligned concordantly 0 times\n",
      "    3521 (3.52%) aligned concordantly exactly 1 time\n",
      "    83277 (83.28%) aligned concordantly >1 times\n",
      "86.80% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    13398 (13.40%) aligned concordantly 0 times\n",
      "    3545 (3.54%) aligned concordantly exactly 1 time\n",
      "    83057 (83.06%) aligned concordantly >1 times\n",
      "86.60% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    13234 (13.23%) aligned concordantly 0 times\n",
      "    3701 (3.70%) aligned concordantly exactly 1 time\n",
      "    83065 (83.06%) aligned concordantly >1 times\n",
      "86.77% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    13114 (13.11%) aligned concordantly 0 times\n",
      "    3797 (3.80%) aligned concordantly exactly 1 time\n",
      "    83089 (83.09%) aligned concordantly >1 times\n",
      "86.89% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_2_s1_R1.fastq.gz_C_to_T.fastq, zr2096_2_s1_R1.fastq.gz_G_to_A.fastq, zr2096_2_s1_R2.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t2315114\n",
      "\n",
      "Total methylated C's in CpG context:\t191418\n",
      "Total methylated C's in CHG context:\t29517\n",
      "Total methylated C's in CHH context:\t221240\n",
      "Total methylated C's in Unknown context:\t42053\n",
      "\n",
      "Total unmethylated C's in CpG context:\t93502\n",
      "Total unmethylated C's in CHG context:\t441550\n",
      "Total unmethylated C's in CHH context:\t1337887\n",
      "Total unmethylated C's in Unknown context:\t79626\n",
      "\n",
      "C methylated in CpG context:\t67.2%\n",
      "C methylated in CHG context:\t6.3%\n",
      "C methylated in CHH context:\t14.2%\n",
      "C methylated in unknown context (CN or CHN):\t34.6%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 8m 22s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_3_s1_R1.fastq.gz to zr2096_3_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_3_s1_R1.fastq.gz to zr2096_3_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_3_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_3_s1_R2.fastq.gz to zr2096_3_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_3_s1_R2.fastq.gz to zr2096_3_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_3_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tTNTTTTAAAATAATTTAAATAATTTTATATAAATNNNNNNNNNNNNNNNTATTTAAATTAAAAATTTAAATAAATTTTTTTAATTATTTATTGTTTAATG\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAAATAAACATTAAACAATAAATAATTAAAAAAATTTATTTAAATTTTTAATTTAAATAAATTAAAAATATTAAATTTATATAAAATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tCNCTTCAAAATAATTTAAATAATCTTATATAAATNNNNNNNNNNNNNNNCACCTAAACTAAAAACTCAAATAAACTTTTCTAATCACCTATCATCCAACA\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAGATAGATGTTGGATGATAGGTGATTAGAAAAGTTTATTTGAGTTTTTAGTTTAGGTGAGTTAAAAATATTGAATTTATATAAGATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tCNCTTCAAAATAATTTAAATAATCTTATATAAATNNNNNNNNNNNNNNNCACCTAAACTAAAAACTCAAATAAACTTTTCTAATCACCTATCATCCAACA\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAGATAGATGTTGGATGATAGGTGATTAGAAAAGTTTATTTGAGTTTTTAGTTTAGGTGAGTTAAAAATATTGAATTTATATAAGATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tTNTTTTAAAATAATTTAAATAATTTTATATAAATNNNNNNNNNNNNNNNTATTTAAATTAAAAATTTAAATAAATTTTTTTAATTATTTATTGTTTAATG\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAAATAAACATTAAACAATAAATAATTAAAAAAATTTATTTAAATTTTTAATTTAAATAAATTAAAAATATTAAATTTATATAAAATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_3_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9084 (9.08%) aligned concordantly 0 times\n",
      "    3728 (3.73%) aligned concordantly exactly 1 time\n",
      "    87188 (87.19%) aligned concordantly >1 times\n",
      "90.92% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9008 (9.01%) aligned concordantly 0 times\n",
      "    3800 (3.80%) aligned concordantly exactly 1 time\n",
      "    87192 (87.19%) aligned concordantly >1 times\n",
      "90.99% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9541 (9.54%) aligned concordantly 0 times\n",
      "    3554 (3.55%) aligned concordantly exactly 1 time\n",
      "    86905 (86.91%) aligned concordantly >1 times\n",
      "90.46% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9345 (9.35%) aligned concordantly 0 times\n",
      "    3602 (3.60%) aligned concordantly exactly 1 time\n",
      "    87053 (87.05%) aligned concordantly >1 times\n",
      "90.66% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_3_s1_R1.fastq.gz_C_to_T.fastq, zr2096_3_s1_R1.fastq.gz_G_to_A.fastq, zr2096_3_s1_R2.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t2571474\n",
      "\n",
      "Total methylated C's in CpG context:\t261256\n",
      "Total methylated C's in CHG context:\t25303\n",
      "Total methylated C's in CHH context:\t130269\n",
      "Total methylated C's in Unknown context:\t24823\n",
      "\n",
      "Total unmethylated C's in CpG context:\t95714\n",
      "Total unmethylated C's in CHG context:\t531341\n",
      "Total unmethylated C's in CHH context:\t1527591\n",
      "Total unmethylated C's in Unknown context:\t73432\n",
      "\n",
      "C methylated in CpG context:\t73.2%\n",
      "C methylated in CHG context:\t4.5%\n",
      "C methylated in CHH context:\t7.9%\n",
      "C methylated in unknown context (CN or CHN):\t25.3%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 9m 32s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_4_s1_R1.fastq.gz to zr2096_4_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_4_s1_R1.fastq.gz to zr2096_4_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_4_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_4_s1_R2.fastq.gz to zr2096_4_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_4_s1_R2.fastq.gz to zr2096_4_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_4_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNGATTAGTGTATAGTGGTTGTTATGGAAATTAGNNNNNNNNNNNNNNNNNNNNNNNNNTGNANTNTTTTTATTAGAATTGTTAAGGTTTATTTGATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTCCATCATTAATTTTTAAAAAAAAAAAACCATCCTTAATATCAATTTTATAAATCAAATAAACCTTAACAATTCTAATAAAAACACTTCATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATTAATATATAATAATTATTATAAAAATTAANNNNNNNNNNNNNNNNNNNNNNNNNTANANTNTTTTTATTAAAATTATTAAAATTTATTTAATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTTTATTATTAATTTTTAAAAAAAAAAAATTATTTTTAATATTAATTTTATAAATTAAATAAATTTTAATAATTTTAATAAAAATATTTTATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATTAATATATAATAATTATTATAAAAATTAANNNNNNNNNNNNNNNNNNNNNNNNNTANANTNTTTTTATTAAAATTATTAAAATTTATTTAATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTTTATTATTAATTTTTAAAAAAAAAAAATTATTTTTAATATTAATTTTATAAATTAAATAAATTTTAATAATTTTAATAAAAATATTTTATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNGATTAGTGTATAGTGGTTGTTATGGAAATTAGNNNNNNNNNNNNNNNNNNNNNNNNNTGNANTNTTTTTATTAGAATTGTTAAGGTTTATTTGATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTCCATCATTAATTTTTAAAAAAAAAAAACCATCCTTAATATCAATTTTATAAATCAAATAAACCTTAACAATTCTAATAAAAACACTTCATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_4_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    10717 (10.72%) aligned concordantly 0 times\n",
      "    3517 (3.52%) aligned concordantly exactly 1 time\n",
      "    85766 (85.77%) aligned concordantly >1 times\n",
      "89.28% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    10802 (10.80%) aligned concordantly 0 times\n",
      "    3752 (3.75%) aligned concordantly exactly 1 time\n",
      "    85446 (85.45%) aligned concordantly >1 times\n",
      "89.20% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    10741 (10.74%) aligned concordantly 0 times\n",
      "    3801 (3.80%) aligned concordantly exactly 1 time\n",
      "    85458 (85.46%) aligned concordantly >1 times\n",
      "89.26% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    10785 (10.79%) aligned concordantly 0 times\n",
      "    3438 (3.44%) aligned concordantly exactly 1 time\n",
      "    85777 (85.78%) aligned concordantly >1 times\n",
      "89.22% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_4_s1_R1.fastq.gz_C_to_T.fastq, zr2096_4_s1_R1.fastq.gz_G_to_A.fastq, zr2096_4_s1_R2.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t2401459\n",
      "\n",
      "Total methylated C's in CpG context:\t187540\n",
      "Total methylated C's in CHG context:\t24865\n",
      "Total methylated C's in CHH context:\t139891\n",
      "Total methylated C's in Unknown context:\t26334\n",
      "\n",
      "Total unmethylated C's in CpG context:\t113392\n",
      "Total unmethylated C's in CHG context:\t483964\n",
      "Total unmethylated C's in CHH context:\t1451807\n",
      "Total unmethylated C's in Unknown context:\t72321\n",
      "\n",
      "C methylated in CpG context:\t62.3%\n",
      "C methylated in CHG context:\t4.9%\n",
      "C methylated in CHH context:\t8.8%\n",
      "C methylated in unknown context (CN or CHN):\t26.7%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 10m 9s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_5_s1_R1.fastq.gz to zr2096_5_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_5_s1_R1.fastq.gz to zr2096_5_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_5_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_5_s1_R2.fastq.gz to zr2096_5_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_5_s1_R2.fastq.gz to zr2096_5_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_5_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATAAAATATTATTTAAATTTTATTATTTATANNNNNNNNNNNNNNNNNNATTNTTTTTATTATATATATAAAATTATTTTAAAATTATTATATTTTTT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTCAAATATATTAAAAAAAAATTATTAATAATTTTCATTAAAAAATATAATAATTTTAAAATAATTTTATATATATAATAAAAAAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAACAAAACACCACTCAAATCTTACTATCCACANNNNNNNNNNNNNNNNNNATTNCTTCTATCATATATATAAAATCATTCCAAAATTACTATATTTTTC\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTTGAGTATATTGAAGAGGAATTATTGATAGTTTTTGTTGAAAAATATAGTAATTTTGGAATGATTTTATATATATGATAGAAGAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAACAAAACACCACTCAAATCTTACTATCCACANNNNNNNNNNNNNNNNNNATTNCTTCTATCATATATATAAAATCATTCCAAAATTACTATATTTTTC\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTTGAGTATATTGAAGAGGAATTATTGATAGTTTTTGTTGAAAAATATAGTAATTTTGGAATGATTTTATATATATGATAGAAGAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATAAAATATTATTTAAATTTTATTATTTATANNNNNNNNNNNNNNNNNNATTNTTTTTATTATATATATAAAATTATTTTAAAATTATTATATTTTTT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTCAAATATATTAAAAAAAAATTATTAATAATTTTCATTAAAAAATATAATAATTTTAAAATAATTTTATATATATAATAAAAAAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_5_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    8860 (8.86%) aligned concordantly 0 times\n",
      "    3285 (3.29%) aligned concordantly exactly 1 time\n",
      "    87855 (87.86%) aligned concordantly >1 times\n",
      "91.14% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9338 (9.34%) aligned concordantly 0 times\n",
      "    3142 (3.14%) aligned concordantly exactly 1 time\n",
      "    87520 (87.52%) aligned concordantly >1 times\n",
      "90.66% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    8988 (8.99%) aligned concordantly 0 times\n",
      "    3342 (3.34%) aligned concordantly exactly 1 time\n",
      "    87670 (87.67%) aligned concordantly >1 times\n",
      "91.01% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9317 (9.32%) aligned concordantly 0 times\n",
      "    3194 (3.19%) aligned concordantly exactly 1 time\n",
      "    87489 (87.49%) aligned concordantly >1 times\n",
      "90.68% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_5_s1_R1.fastq.gz_C_to_T.fastq, zr2096_5_s1_R1.fastq.gz_G_to_A.fastq, zr2096_5_s1_R2.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t2502361\n",
      "\n",
      "Total methylated C's in CpG context:\t193415\n",
      "Total methylated C's in CHG context:\t25688\n",
      "Total methylated C's in CHH context:\t154169\n",
      "Total methylated C's in Unknown context:\t29935\n",
      "\n",
      "Total unmethylated C's in CpG context:\t109516\n",
      "Total unmethylated C's in CHG context:\t484512\n",
      "Total unmethylated C's in CHH context:\t1535061\n",
      "Total unmethylated C's in Unknown context:\t78826\n",
      "\n",
      "C methylated in CpG context:\t63.8%\n",
      "C methylated in CHG context:\t5.0%\n",
      "C methylated in CHH context:\t9.1%\n",
      "C methylated in unknown context (CN or CHN):\t27.5%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 8m 38s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_6_s1_R1.fastq.gz to zr2096_6_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_6_s1_R1.fastq.gz to zr2096_6_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_6_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_6_s1_R2.fastq.gz to zr2096_6_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_6_s1_R2.fastq.gz to zr2096_6_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_6_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTTATTAAATTTAATTTTATTATTAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNTANANAAATATTAATTAAATATTTTTAAAATAATAAATTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAAAATTTTTTTAAATTTATTATTTTAAAAATATTTAATTAATATTTATTTATATTTTATAATTAACAATAATTATTTTTTTATTTAATAATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTCATCAAATCCAACTTTATCACCAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNCANANAAACATCAATCAAACATCTCTAAAACAATAAACTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAGAATTTTTTTAAGTTTATTGTTTTAGAGATGTTTGATTGATGTTTATTTGTATTTTGTAATTAGTGGTGGTTGTTTTTTTATTTGGTGATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTCATCAAATCCAACTTTATCACCAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNCANANAAACATCAATCAAACATCTCTAAAACAATAAACTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAGAATTTTTTTAAGTTTATTGTTTTAGAGATGTTTGATTGATGTTTATTTGTATTTTGTAATTAGTGGTGGTTGTTTTTTTATTTGGTGATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTTATTAAATTTAATTTTATTATTAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNTANANAAATATTAATTAAATATTTTTAAAATAATAAATTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAAAATTTTTTTAAATTTATTATTTTAAAAATATTTAATTAATATTTATTTATATTTTATAATTAACAATAATTATTTTTTTATTTAATAATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_6_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9464 (9.46%) aligned concordantly 0 times\n",
      "    3742 (3.74%) aligned concordantly exactly 1 time\n",
      "    86794 (86.79%) aligned concordantly >1 times\n",
      "90.54% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9268 (9.27%) aligned concordantly 0 times\n",
      "    3706 (3.71%) aligned concordantly exactly 1 time\n",
      "    87026 (87.03%) aligned concordantly >1 times\n",
      "90.73% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9335 (9.34%) aligned concordantly 0 times\n",
      "    3176 (3.18%) aligned concordantly exactly 1 time\n",
      "    87489 (87.49%) aligned concordantly >1 times\n",
      "90.67% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9341 (9.34%) aligned concordantly 0 times\n",
      "    3203 (3.20%) aligned concordantly exactly 1 time\n",
      "    87456 (87.46%) aligned concordantly >1 times\n",
      "90.66% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_6_s1_R1.fastq.gz_C_to_T.fastq, zr2096_6_s1_R1.fastq.gz_G_to_A.fastq, zr2096_6_s1_R2.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t2443376\n",
      "\n",
      "Total methylated C's in CpG context:\t214807\n",
      "Total methylated C's in CHG context:\t23866\n",
      "Total methylated C's in CHH context:\t130796\n",
      "Total methylated C's in Unknown context:\t23463\n",
      "\n",
      "Total unmethylated C's in CpG context:\t95641\n",
      "Total unmethylated C's in CHG context:\t494878\n",
      "Total unmethylated C's in CHH context:\t1483388\n",
      "Total unmethylated C's in Unknown context:\t71902\n",
      "\n",
      "C methylated in CpG context:\t69.2%\n",
      "C methylated in CHG context:\t4.6%\n",
      "C methylated in CHH context:\t8.1%\n",
      "C methylated in unknown context (CN or CHN):\t24.6%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 7m 54s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_7_s1_R1.fastq.gz to zr2096_7_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_7_s1_R1.fastq.gz to zr2096_7_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_7_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_7_s1_R2.fastq.gz to zr2096_7_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_7_s1_R2.fastq.gz to zr2096_7_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_7_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t99\tNC_035787.1_CT_converted\t51746759\t2\t8M5I5M5I4M3I9M4I24M1I2M2I2M6I6M4I5M\t=\t51746759\t-65\tAAAAAAATGATTATTATGATGAATTTAAATATTAATAATAAAATTTAAAAAAAATATAAATGTTAAATATTAAATTTTTTAAAAAATGATTATGG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-166\tXS:i:-185\tXN:i:0\tXM:i:6\tXO:i:8\tXG:i:30\tNM:i:36\tMD:Z:0T1G0G16T0G1T41\tYS:i:-166\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t147\tNC_035787.1_CT_converted\t51746759\t2\t8M5I5M5I4M3I9M4I24M1I2M2I2M6I6M4I5M\t=\t51746759\t-65\tAAAAAAATGATTATTATGATGAATTTAAATATTAATAATAAAATTTAAAAAAAATATAAATGTTAAATATTAAATTTTTTAAAAAATGATTATGG\tFFFFFFFFFFFFFFFFFFFFFFFBBFFFFFFBBFF/FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-166\tXN:i:0\tXM:i:6\tXO:i:8\tXG:i:30\tNM:i:36\tMD:Z:0T1G0G16T0G1T41\tYS:i:-166\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t99\tNC_035781.1_GA_converted\t24539746\t6\t95M\t=\t24539746\t-95\tAAAAAAACAACTACTACAACAAATTCAAATATTAATAACAAAATTTAAAAAAAACATAAACATTAAATATCAAATTCTTCAAAAAACAATTACAA\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-12\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:94T0\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t147\tNC_035781.1_GA_converted\t24539746\t6\t95M\t=\t24539746\t-95\tAAAAAAACAACTACTACAACAAATTCAAATATTAATAACAAAATTTAAAAAAAACATAAACATTAAATATCAAATTCTTCAAAAAACAATTACAA\tFFFFFFFFFFFFFFFFFFFFFFFBBFFFFFFBBFF/FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-6\tXS:i:-12\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:94T0\tYS:i:-6\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t83\tNC_035780.1_CT_converted\t42548681\t6\t95M\t=\t42548681\t-95\tTTGTAATTGTTTTTTGAAGAATTTGATATTTAATGTTTATGTTTTTTTTAAATTTTGTTATTAATATTTGAATTTGTTGTAGTAGTTGTTTTTTT\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-6\tXS:i:-24\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:0A94\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t163\tNC_035780.1_CT_converted\t42548681\t6\t95M\t=\t42548681\t-95\tTTGTAATTGTTTTTTGAAGAATTTGATATTTAATGTTTATGTTTTTTTTAAATTTTGTTATTAATATTTGAATTTGTTGTAGTAGTTGTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-24\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:0A94\tYS:i:-6\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t83\tNC_035782.1_GA_converted\t64740147\t2\t6M6I39M2I4M2I1M2I4M2I5M1D12M6I4M\t=\t64740147\t-76\tCCATAATCATTTTTTAAAAAATTTAATATTTAACATTTATATTTTTTTTAAATTTTATTATTAATATTTAAATTCATCATAATAATCATTTTTTT\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-164\tXS:i:-167\tXN:i:0\tXM:i:11\tXO:i:7\tXG:i:21\tNM:i:32\tMD:Z:0T0T6A26A1C3A1C15^A3A4T5A0A0\tYS:i:-164\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t163\tNC_035782.1_GA_converted\t64740147\t2\t6M6I39M2I4M2I1M2I4M2I5M1D12M6I4M\t=\t64740147\t-76\tCCATAATCATTTTTTAAAAAATTTAATATTTAACATTTATATTTTTTTTAAATTTTATTATTAATATTTAAATTCATCATAATAATCATTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-164\tXS:i:-167\tXN:i:0\tXM:i:11\tXO:i:7\tXG:i:21\tNM:i:32\tMD:Z:0T0T6A26A1C3A1C15^A3A4T5A0A0\tYS:i:-164\tYT:Z:CP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_7_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9164 (9.16%) aligned concordantly 0 times\n",
      "    3744 (3.74%) aligned concordantly exactly 1 time\n",
      "    87092 (87.09%) aligned concordantly >1 times\n",
      "90.84% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9067 (9.07%) aligned concordantly 0 times\n",
      "    3824 (3.82%) aligned concordantly exactly 1 time\n",
      "    87109 (87.11%) aligned concordantly >1 times\n",
      "90.93% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9311 (9.31%) aligned concordantly 0 times\n",
      "    3400 (3.40%) aligned concordantly exactly 1 time\n",
      "    87289 (87.29%) aligned concordantly >1 times\n",
      "90.69% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9266 (9.27%) aligned concordantly 0 times\n",
      "    3281 (3.28%) aligned concordantly exactly 1 time\n",
      "    87453 (87.45%) aligned concordantly >1 times\n",
      "90.73% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_7_s1_R1.fastq.gz_C_to_T.fastq, zr2096_7_s1_R1.fastq.gz_G_to_A.fastq, zr2096_7_s1_R2.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t2551148\n",
      "\n",
      "Total methylated C's in CpG context:\t209726\n",
      "Total methylated C's in CHG context:\t26915\n",
      "Total methylated C's in CHH context:\t146076\n",
      "Total methylated C's in Unknown context:\t27898\n",
      "\n",
      "Total unmethylated C's in CpG context:\t102122\n",
      "Total unmethylated C's in CHG context:\t507713\n",
      "Total unmethylated C's in CHH context:\t1558596\n",
      "Total unmethylated C's in Unknown context:\t80503\n",
      "\n",
      "C methylated in CpG context:\t67.3%\n",
      "C methylated in CHG context:\t5.0%\n",
      "C methylated in CHH context:\t8.6%\n",
      "C methylated in unknown context (CN or CHN):\t25.7%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 10m 24s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_8_s1_R1.fastq.gz to zr2096_8_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_8_s1_R1.fastq.gz to zr2096_8_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_8_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_8_s1_R2.fastq.gz to zr2096_8_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_8_s1_R2.fastq.gz to zr2096_8_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_8_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t99\tNC_035788.1_CT_converted\t62738085\t2\t21M1I5M2I2M2I55M\t=\t62738085\t-83\tTAATTTAAAATTAAAAAATTTATTAAATTTATTATTATTTNTTATTTTTTATAAAAGAAAATAGAGAGAGAGAGAGAGAGAGAGAGAG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tAS:i:-115\tXS:i:-121\tXN:i:0\tXM:i:15\tXO:i:3\tXG:i:5\tNM:i:20\tMD:Z:1T0T15A10A3A0A0A0A0A2A0A0A1G1A0G35\tYS:i:-114\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t147\tNC_035788.1_CT_converted\t62738085\t2\t21M1I5M2I2M2I55M\t=\t62738085\t-83\tTAATTTAAAATTAAAAAATTTATTAAATTTATTATTATTTATTATTTTTTATAAAAGAAAATAGAGAGAGAGAGAGAGAGAGAGAGAG\t/FB//BFFF<//FFFFFF/FFB/<FF<//F/<<</FF</FF<<<</<F/<FBFFFF<FFFF/FFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-114\tXS:i:-120\tXN:i:0\tXM:i:14\tXO:i:3\tXG:i:5\tNM:i:19\tMD:Z:1T0T15A10A3A0A1A0A2A0A0A1G1A0G35\tYS:i:-115\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t99\tNC_035786.1_GA_converted\t2855541\t2\t10M2I7M7I7M2I2M3I9M2I37M\t=\t2855541\t-72\tTAATCCAAAATTAAAAAACTCATTAAATCTATCATCATTCNCCATTCTTCACAAAAAAAAATAAAAAAAAAAAAAAAAAAAAAAAAAA\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tAS:i:-116\tXS:i:-115\tXN:i:0\tXM:i:8\tXO:i:5\tXG:i:16\tNM:i:24\tMD:Z:4T0A4C9A5A0A3C3T36\tYS:i:-115\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t147\tNC_035786.1_GA_converted\t2855541\t2\t10M2I7M7I7M2I2M3I9M2I37M\t=\t2855541\t-72\tTAATCCAAAATTAAAAAACTCATTAAATCTATCATCATTCACCATTCTTCACAAAAAAAAATAAAAAAAAAAAAAAAAAAAAAAAAAA\t/FB//BFFF<//FFFFFF/FFB/<FF<//F/<<</FF</FF<<<</<F/<FBFFFF<FFFF/FFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-115\tXS:i:-119\tXN:i:0\tXM:i:7\tXO:i:5\tXG:i:16\tNM:i:23\tMD:Z:4T0A4C9A6A3C3T36\tYS:i:-116\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t83\tNC_007175.2_CT_converted\t14567\t0\t2M17I69M\t=\t14567\t-71\tTTTTTTTTTTTTTTTTTTTTTTTTTTATTTTTTTTTGTGAAGAATGGNGAATGATGATAGATTTAATGAGTTTTTTAATTTTGGATTA\tFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<<#FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-81\tXS:i:-167\tXN:i:0\tXM:i:5\tXO:i:1\tXG:i:17\tNM:i:22\tMD:Z:1A5A0G5A15T40\tYS:i:-80\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t163\tNC_007175.2_CT_converted\t14567\t0\t2M17I69M\t=\t14567\t-71\tTTTTTTTTTTTTTTTTTTTTTTTTTTATTTTTTTTTGTGAAGAATGGTGAATGATGATAGATTTAATGAGTTTTTTAATTTTGGATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tAS:i:-80\tXN:i:0\tXM:i:4\tXO:i:1\tXG:i:17\tNM:i:21\tMD:Z:1A5A0G5A56\tYS:i:-81\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t83\tNC_035785.1_GA_converted\t16582012\t2\t38M3I13M1I19M4I10M\t=\t16582012\t-80\tCTCTCTCTCTCTCTCTCTCTCTCTCTATTTTCTTTTATAAAAAATAANAAATAATAATAAATTTAATAAATTTTTTAATTTTAAATTA\tFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<<#FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-118\tXS:i:-119\tXN:i:0\tXM:i:14\tXO:i:3\tXG:i:8\tNM:i:22\tMD:Z:35C0T1C1T1T1T0T0T0T1T4T0T21A0A1\tYS:i:-117\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t163\tNC_035785.1_GA_converted\t16582012\t2\t38M3I13M1I19M4I10M\t=\t16582012\t-80\tCTCTCTCTCTCTCTCTCTCTCTCTCTATTTTCTTTTATAAAAAATAATAAATAATAATAAATTTAATAAATTTTTTAATTTTAAATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tAS:i:-117\tXS:i:-118\tXN:i:0\tXM:i:13\tXO:i:3\tXG:i:8\tNM:i:21\tMD:Z:35C0T1C1T1T2T0T0T1T4T0T21A0A1\tYS:i:-118\tYT:Z:CP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_8_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    11238 (11.24%) aligned concordantly 0 times\n",
      "    3759 (3.76%) aligned concordantly exactly 1 time\n",
      "    85003 (85.00%) aligned concordantly >1 times\n",
      "88.76% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    11492 (11.49%) aligned concordantly 0 times\n",
      "    3704 (3.70%) aligned concordantly exactly 1 time\n",
      "    84804 (84.80%) aligned concordantly >1 times\n",
      "88.51% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    11544 (11.54%) aligned concordantly 0 times\n",
      "    3392 (3.39%) aligned concordantly exactly 1 time\n",
      "    85064 (85.06%) aligned concordantly >1 times\n",
      "88.46% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    11598 (11.60%) aligned concordantly 0 times\n",
      "    3343 (3.34%) aligned concordantly exactly 1 time\n",
      "    85059 (85.06%) aligned concordantly >1 times\n",
      "88.40% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_8_s1_R1.fastq.gz_C_to_T.fastq, zr2096_8_s1_R1.fastq.gz_G_to_A.fastq, zr2096_8_s1_R2.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t2427897\n",
      "\n",
      "Total methylated C's in CpG context:\t172771\n",
      "Total methylated C's in CHG context:\t29377\n",
      "Total methylated C's in CHH context:\t212768\n",
      "Total methylated C's in Unknown context:\t41165\n",
      "\n",
      "Total unmethylated C's in CpG context:\t101550\n",
      "Total unmethylated C's in CHG context:\t455554\n",
      "Total unmethylated C's in CHH context:\t1455877\n",
      "Total unmethylated C's in Unknown context:\t87081\n",
      "\n",
      "C methylated in CpG context:\t63.0%\n",
      "C methylated in CHG context:\t6.1%\n",
      "C methylated in CHH context:\t12.8%\n",
      "C methylated in unknown context (CN or CHN):\t32.1%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 8m 43s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_9_s1_R1.fastq.gz to zr2096_9_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_9_s1_R1.fastq.gz to zr2096_9_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_9_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_9_s1_R2.fastq.gz to zr2096_9_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_9_s1_R2.fastq.gz to zr2096_9_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_9_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNATTAAATAAAAAATAATAAATAATAAATTANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAGG\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAAAAAATAAAAAACACATTAAAAATCTATAATTTAATTTATTATTTATTATTTTTTATTTAATTAAATCNNNNAAACATCATATAAAAAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNACCAAATAAAAAATAACAAATAATAAATCANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAAA\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAGAGGATAGGAGATATGTTAGGAATTTATAATTTGATTTATTATTTGTTATTTTTTATTTGGTTAGATTNNNNGAGTGTTGTGTAGGGAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNACCAAATAAAAAATAACAAATAATAAATCANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAAA\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAGAGGATAGGAGATATGTTAGGAATTTATAATTTGATTTATTATTTGTTATTTTTTATTTGGTTAGATTNNNNGAGTGTTGTGTAGGGAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNATTAAATAAAAAATAATAAATAATAAATTANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAGG\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAAAAAATAAAAAACACATTAAAAATCTATAATTTAATTTATTATTTATTATTTTTTATTTAATTAAATCNNNNAAACATCATATAAAAAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_9_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9995 (9.99%) aligned concordantly 0 times\n",
      "    4021 (4.02%) aligned concordantly exactly 1 time\n",
      "    85984 (85.98%) aligned concordantly >1 times\n",
      "90.00% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9950 (9.95%) aligned concordantly 0 times\n",
      "    4042 (4.04%) aligned concordantly exactly 1 time\n",
      "    86008 (86.01%) aligned concordantly >1 times\n",
      "90.05% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    10189 (10.19%) aligned concordantly 0 times\n",
      "    3321 (3.32%) aligned concordantly exactly 1 time\n",
      "    86490 (86.49%) aligned concordantly >1 times\n",
      "89.81% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    10250 (10.25%) aligned concordantly 0 times\n",
      "    3461 (3.46%) aligned concordantly exactly 1 time\n",
      "    86289 (86.29%) aligned concordantly >1 times\n",
      "89.75% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_9_s1_R1.fastq.gz_C_to_T.fastq, zr2096_9_s1_R1.fastq.gz_G_to_A.fastq, zr2096_9_s1_R2.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t2633309\n",
      "\n",
      "Total methylated C's in CpG context:\t256604\n",
      "Total methylated C's in CHG context:\t29608\n",
      "Total methylated C's in CHH context:\t150781\n",
      "Total methylated C's in Unknown context:\t30522\n",
      "\n",
      "Total unmethylated C's in CpG context:\t97851\n",
      "Total unmethylated C's in CHG context:\t535057\n",
      "Total unmethylated C's in CHH context:\t1563408\n",
      "Total unmethylated C's in Unknown context:\t81232\n",
      "\n",
      "C methylated in CpG context:\t72.4%\n",
      "C methylated in CHG context:\t5.2%\n",
      "C methylated in CHH context:\t8.8%\n",
      "C methylated in unknown context (CN or CHN):\t27.3%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 8m 55s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "find /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_*R1* \\\n",
    "| xargs basename -s _s1_R1.fastq.gz | xargs -I{} /Users/Shelly/anaconda3/bin/bismark \\\n",
    "--genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ \\\n",
    "--score_min L,0,-2 \\\n",
    "-u 100000 \\\n",
    "--non_directional \\\n",
    "-1 /Volumes/web/seashell/bu-serine-wd/18-04-07/{}_s1_R1.fastq.gz \\\n",
    "-2 /Volumes/web/seashell/bu-serine-wd/18-04-07/{}_s1_R2.fastq.gz \\\n",
    "-o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/ "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#run deduplicate on looser alignment score threshold alignments"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "/Users/Shelly/anaconda3/bin/deduplicate_bismark \\\n",
    "--bam -p \\\n",
    "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/*.bam \\\n",
    "-o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/ \n",
    "2> /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/dedup.err"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#create directory for methylpy analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!mkdir /Volumes/web/metacarcinus/Cvirginica/methylpy"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#build methylpy minimap2 genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!mkdir /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "methylpy build-reference \\\n",
    "\t--input-files /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/Cvirginica_v300.fa \\\n",
    "\t--output-prefix /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/ \\\n",
    "\t--aligner  minimap2 \\\n",
    "\t--parallel True"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#run methylpy with minimap2 alignment"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#build methylpy bowtie2 genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "methylpy build-reference \\\n",
    "\t--input-files /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/Cvirginica_v300.fa \\\n",
    "\t--output-prefix /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/ \\\n",
    "\t--aligner  bowtie2 \\\n",
    "\t--parallel True"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#run methylpy with bowtie2 alignment"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "methylpy paired-end-pipeline \\\n",
    "\t--read1-files raw/mESC_R1.fastq.gz \\\n",
    "\t--read2-files raw/mESC_R2.fastq.gz \\\n",
    "\t--sample mESC \\\n",
    "\t--forward-ref mm10_bt2/mm10_f \\\n",
    "\t--reverse-ref mm10_bt2/mm10_r \\\n",
    "\t--ref-fasta mm10_bt2/mm10.fa \\\n",
    "\t--num-procs 2 \\\n",
    "\t--remove-clonal True \\\n",
    "\t--path-to-picard=\"picard/\" \\\n",
    "    --add-snp-info TRUE \\\n",
    "    --binom-test \\\n",
    "    --unmethylated-control"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#run DMR finder"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "methylpy DMRfind \\\n",
    "\t--allc-files allc/allc_AD_HT.tsv.gz allc/allc_AD_IT.tsv.gz \\\n",
    "\t--samples AD_HT AD_IT \\\n",
    "\t--mc-type \"CGN\" \\\n",
    "\t--chroms 1 2 3 4 5 \\\n",
    "\t--num-procs 8 \\\n",
    "\t--output-prefix DMR_HT_IT"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
