{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Bismark practice with C. virginica 2018-04-07 data\n",
    "Bismark paper:  \n",
    "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3102221/  \n",
    "Bismark user guide:  \n",
    "https://rawgit.com/FelixKrueger/Bismark/master/Docs/Bismark_User_Guide.html  \n",
    "\n",
    "First, needed to install bowtie2, samtools, and bismark locally. I already had anaconda3 on my computer, so I just needed to run the following commands:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!conda install -c bioconda bowtie2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!conda install -c bioconda samtools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!conda install -c bioconda bismark "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "FIRST:  connect to Gannett server; this will mount as /Volumes/web\n",
    "SECOND:  connect to OWL server; this will mount as /Volumes/web-1\n",
    "Then make new directories for bismark analysis in the 'metacarcinus' folder on seashell"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
    "pwd "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "copy genome to working directory folder"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "cp /Volumes/web-1/halfshell/genomic-databank/Cvirginica_v300.fa /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "run bismark genome preparation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Path to genome folder specified as: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "Aligner to be used: Bowtie 2 (default)\n",
      "Writing bisulfite genomes out into a single MFA (multi FastA) file\n",
      "\n",
      "Bismark Genome Preparation - Step I: Preparing folders\n",
      "\n",
      "Path to Bowtie 2 specified: /Users/Shelly/anaconda3/bin/\n",
      "Bisulfite Genome Indexer version v0.20.0 (last modified 26 April 2018)\n",
      "Created Bisulfite Genome folder /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/Bisulfite_Genome/\n",
      "Created Bisulfite Genome folder /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/Bisulfite_Genome/CT_conversion/\n",
      "Created Bisulfite Genome folder /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/Bisulfite_Genome/GA_conversion/\n",
      "\n",
      "Step I - Prepare genome folders - completed\n",
      "\n",
      "\n",
      "Bismark Genome Preparation - Step II: Bisulfite converting reference genome\n",
      "\n",
      "conversions performed:\n",
      "chromosome\tC->T\tG->A\n",
      "NC_035780.1\t11492316\t11528162\n",
      "NC_035781.1\t10879823\t10858285\n",
      "NC_035782.1\t13475475\t13496235\n",
      "NC_035783.1\t10484268\t10482193\n",
      "NC_035784.1\t17306783\t17253434\n",
      "NC_035785.1\t8878985\t8876470\n",
      "NC_035786.1\t9961761\t9989843\n",
      "NC_035787.1\t13145205\t13164212\n",
      "NC_035788.1\t17925605\t17932896\n",
      "NC_035789.1\t5676071\t5682065\n",
      "NC_007175.2\t2827\t3882\n",
      "\n",
      "Total number of conversions performed:\n",
      "C->T:\t119229119\n",
      "G->A:\t119267677\n",
      "\n",
      "Step II - Genome bisulfite conversions - completed\n",
      "\n",
      "\n",
      "Bismark Genome Preparation - Step III: Launching the Bowtie 2 indexer\n",
      "Please be aware that this process can - depending on genome size - take several hours!\n",
      "\n",
      "Preparing indexing of CT converted genome in /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/Bisulfite_Genome/CT_conversion/\n",
      "Parent process: Starting to index C->T converted genome with the following command:\n",
      "\n",
      "/Users/Shelly/anaconda3/bin/bowtie2-build -f genome_mfa.CT_conversion.fa BS_CT\n",
      "\n",
      "Preparing indexing of GA converted genome in /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/Bisulfite_Genome/GA_conversion/\n",
      "Child process: Starting to index G->A converted genome with the following command:\n",
      "\n",
      "/Users/Shelly/anaconda3/bin/bowtie2-build -f genome_mfa.GA_conversion.fa BS_GA\n",
      "\n",
      "Settings:\n",
      "  Output files: \"BS_CT.*.bt2\"\n",
      "  Line rate: 6 (line is 64 bytes)\n",
      "  Lines per side: 1 (side is 64 bytes)\n",
      "  Offset rate: 4 (one in 16)\n",
      "  FTable chars: 10\n",
      "  Strings: unpacked\n",
      "  Max bucket size: default\n",
      "  Max bucket size, sqrt multiplier: default\n",
      "  Max bucket size, len divisor: 4\n",
      "  Difference-cover sample period: 1024\n",
      "  Endianness: little\n",
      "  Actual local endianness: little\n",
      "  Sanity checking: disabled\n",
      "  Assertions: disabled\n",
      "  Random seed: 0\n",
      "  Sizeofs: void*:8, int:4, long:8, size_t:8\n",
      "Input files DNA, FASTA:\n",
      "  genome_mfa.CT_conversion.fa\n",
      "Building a SMALL index\n",
      "Reading reference sizes\n",
      "Settings:\n",
      "  Output files: \"BS_GA.*.bt2\"\n",
      "  Line rate: 6 (line is 64 bytes)\n",
      "  Lines per side: 1 (side is 64 bytes)\n",
      "  Offset rate: 4 (one in 16)\n",
      "  FTable chars: 10\n",
      "  Strings: unpacked\n",
      "  Max bucket size: default\n",
      "  Max bucket size, sqrt multiplier: default\n",
      "  Max bucket size, len divisor: 4\n",
      "  Difference-cover sample period: 1024\n",
      "  Endianness: little\n",
      "  Actual local endianness: little\n",
      "  Sanity checking: disabled\n",
      "  Assertions: disabled\n",
      "  Random seed: 0\n",
      "  Sizeofs: void*:8, int:4, long:8, size_t:8\n",
      "Input files DNA, FASTA:\n",
      "  genome_mfa.GA_conversion.fa\n",
      "Building a SMALL index\n",
      "Reading reference sizes\n",
      "  Time reading reference sizes: 00:01:55\n",
      "Calculating joined length\n",
      "Writing header\n",
      "Reserving space for joined string\n",
      "Joining reference sequences\n",
      "  Time reading reference sizes: 00:01:55\n",
      "Calculating joined length\n",
      "Writing header\n",
      "Reserving space for joined string\n",
      "Joining reference sequences\n",
      "  Time to join reference sequences: 00:00:12\n",
      "bmax according to bmaxDivN setting: 171168832\n",
      "Using parameters --bmax 128376624 --dcv 1024\n",
      "  Doing ahead-of-time memory usage test\n",
      "  Passed!  Constructing with these parameters: --bmax 128376624 --dcv 1024\n",
      "Constructing suffix-array element generator\n",
      "Building DifferenceCoverSample\n",
      "  Building sPrime\n",
      "  Time to join reference sequences: 00:00:13\n",
      "bmax according to bmaxDivN setting: 171168832\n",
      "Using parameters --bmax 128376624 --dcv 1024\n",
      "  Doing ahead-of-time memory usage test\n",
      "  Passed!  Constructing with these parameters: --bmax 128376624 --dcv 1024\n",
      "Constructing suffix-array element generator\n",
      "Building DifferenceCoverSample\n",
      "  Building sPrime\n",
      "  Building sPrimeOrder\n",
      "  Building sPrimeOrder\n",
      "  V-Sorting samples\n",
      "  V-Sorting samples\n",
      "  V-Sorting samples time: 00:00:47\n",
      "  Allocating rank array\n",
      "  Ranking v-sort output\n",
      "  V-Sorting samples time: 00:00:47\n",
      "  Allocating rank array\n",
      "  Ranking v-sort output\n",
      "  Ranking v-sort output time: 00:00:11\n",
      "  Invoking Larsson-Sadakane on ranks\n",
      "  Ranking v-sort output time: 00:00:11\n",
      "  Invoking Larsson-Sadakane on ranks\n",
      "  Invoking Larsson-Sadakane on ranks time: 00:00:16\n",
      "  Sanity-checking and returning\n",
      "Building samples\n",
      "Reserving space for 12 sample suffixes\n",
      "Generating random suffixes\n",
      "QSorting 12 sample offsets, eliminating duplicates\n",
      "QSorting sample offsets, eliminating duplicates time: 00:00:00\n",
      "Multikey QSorting 12 samples\n",
      "  (Using difference cover)\n",
      "  Multikey QSorting samples time: 00:00:00\n",
      "Calculating bucket sizes\n",
      "Splitting and merging\n",
      "  Splitting and merging time: 00:00:00\n",
      "Avg bucket size: 6.84675e+08 (target: 128376623)\n",
      "Converting suffix-array elements to index image\n",
      "Allocating ftab, absorbFtab\n",
      "Entering Ebwt loop\n",
      "Getting block 1 of 1\n",
      "  No samples; assembling all-inclusive block\n",
      "  Invoking Larsson-Sadakane on ranks time: 00:00:16\n",
      "  Sanity-checking and returning\n",
      "Building samples\n",
      "Reserving space for 12 sample suffixes\n",
      "Generating random suffixes\n",
      "QSorting 12 sample offsets, eliminating duplicates\n",
      "QSorting sample offsets, eliminating duplicates time: 00:00:00\n",
      "Multikey QSorting 12 samples\n",
      "  (Using difference cover)\n",
      "  Multikey QSorting samples time: 00:00:00\n",
      "Calculating bucket sizes\n",
      "Splitting and merging\n",
      "  Splitting and merging time: 00:00:00\n",
      "Avg bucket size: 6.84675e+08 (target: 128376623)\n",
      "Converting suffix-array elements to index image\n",
      "Allocating ftab, absorbFtab\n",
      "Entering Ebwt loop\n",
      "Getting block 1 of 1\n",
      "  No samples; assembling all-inclusive block\n",
      "  Sorting block of length 684675328 for bucket 1\n",
      "  (Using difference cover)\n",
      "  Sorting block of length 684675328 for bucket 1\n",
      "  (Using difference cover)\n",
      "  Sorting block time: 00:13:58\n",
      "Returning block of 684675329 for bucket 1\n",
      "  Sorting block time: 00:16:06\n",
      "Returning block of 684675329 for bucket 1\n",
      "Exited Ebwt loop\n",
      "fchr[A]: 0\n",
      "fchr[C]: 342402452\n",
      "fchr[G]: 461631571\n",
      "fchr[T]: 461631571\n",
      "fchr[$]: 684675328\n",
      "Exiting Ebwt::buildToDisk()\n",
      "Returning from initFromVector\n",
      "Wrote 232427948 bytes to primary EBWT file: BS_GA.1.bt2\n",
      "Wrote 171168840 bytes to secondary EBWT file: BS_GA.2.bt2\n",
      "Re-opening _in1 and _in2 as input streams\n",
      "Returning from Ebwt constructor\n",
      "Headers:\n",
      "    len: 684675328\n",
      "    bwtLen: 684675329\n",
      "    sz: 171168832\n",
      "    bwtSz: 171168833\n",
      "    lineRate: 6\n",
      "    offRate: 4\n",
      "    offMask: 0xfffffff0\n",
      "    ftabChars: 10\n",
      "    eftabLen: 20\n",
      "    eftabSz: 80\n",
      "    ftabLen: 1048577\n",
      "    ftabSz: 4194308\n",
      "    offsLen: 42792209\n",
      "    offsSz: 171168836\n",
      "    lineSz: 64\n",
      "    sideSz: 64\n",
      "    sideBwtSz: 48\n",
      "    sideBwtLen: 192\n",
      "    numSides: 3566018\n",
      "    numLines: 3566018\n",
      "    ebwtTotLen: 228225152\n",
      "    ebwtTotSz: 228225152\n",
      "    color: 0\n",
      "    reverse: 0\n",
      "Total time for call to driver() for forward index: 00:19:59\n",
      "Reading reference sizes\n",
      "  Time reading reference sizes: 00:00:08\n",
      "Calculating joined length\n",
      "Writing header\n",
      "Reserving space for joined string\n",
      "Joining reference sequences\n",
      "  Time to join reference sequences: 00:00:06\n",
      "  Time to reverse reference sequence: 00:00:01\n",
      "bmax according to bmaxDivN setting: 171168832\n",
      "Using parameters --bmax 128376624 --dcv 1024\n",
      "  Doing ahead-of-time memory usage test\n",
      "  Passed!  Constructing with these parameters: --bmax 128376624 --dcv 1024\n",
      "Constructing suffix-array element generator\n",
      "Building DifferenceCoverSample\n",
      "  Building sPrime\n",
      "  Building sPrimeOrder\n",
      "  V-Sorting samples\n",
      "  V-Sorting samples time: 00:00:25\n",
      "  Allocating rank array\n",
      "  Ranking v-sort output\n",
      "  Ranking v-sort output time: 00:00:07\n",
      "  Invoking Larsson-Sadakane on ranks\n",
      "  Invoking Larsson-Sadakane on ranks time: 00:00:10\n",
      "  Sanity-checking and returning\n",
      "Building samples\n",
      "Reserving space for 12 sample suffixes\n",
      "Generating random suffixes\n",
      "QSorting 12 sample offsets, eliminating duplicates\n",
      "QSorting sample offsets, eliminating duplicates time: 00:00:00\n",
      "Multikey QSorting 12 samples\n",
      "  (Using difference cover)\n",
      "  Multikey QSorting samples time: 00:00:00\n",
      "Calculating bucket sizes\n",
      "Splitting and merging\n",
      "  Splitting and merging time: 00:00:00\n",
      "Avg bucket size: 6.84675e+08 (target: 128376623)\n",
      "Converting suffix-array elements to index image\n",
      "Allocating ftab, absorbFtab\n",
      "Entering Ebwt loop\n",
      "Getting block 1 of 1\n",
      "  No samples; assembling all-inclusive block\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  Sorting block of length 684675328 for bucket 1\n",
      "  (Using difference cover)\n",
      "Exited Ebwt loop\n",
      "fchr[A]: 0\n",
      "fchr[C]: 223134775\n",
      "fchr[G]: 223134775\n",
      "fchr[T]: 342402452\n",
      "fchr[$]: 684675328\n",
      "Exiting Ebwt::buildToDisk()\n",
      "Returning from initFromVector\n",
      "Wrote 232427948 bytes to primary EBWT file: BS_CT.1.bt2\n",
      "Wrote 171168840 bytes to secondary EBWT file: BS_CT.2.bt2\n",
      "Re-opening _in1 and _in2 as input streams\n",
      "Returning from Ebwt constructor\n",
      "Headers:\n",
      "    len: 684675328\n",
      "    bwtLen: 684675329\n",
      "    sz: 171168832\n",
      "    bwtSz: 171168833\n",
      "    lineRate: 6\n",
      "    offRate: 4\n",
      "    offMask: 0xfffffff0\n",
      "    ftabChars: 10\n",
      "    eftabLen: 20\n",
      "    eftabSz: 80\n",
      "    ftabLen: 1048577\n",
      "    ftabSz: 4194308\n",
      "    offsLen: 42792209\n",
      "    offsSz: 171168836\n",
      "    lineSz: 64\n",
      "    sideSz: 64\n",
      "    sideBwtSz: 48\n",
      "    sideBwtLen: 192\n",
      "    numSides: 3566018\n",
      "    numLines: 3566018\n",
      "    ebwtTotLen: 228225152\n",
      "    ebwtTotSz: 228225152\n",
      "    color: 0\n",
      "    reverse: 0\n",
      "Total time for call to driver() for forward index: 00:22:02\n",
      "Reading reference sizes\n",
      "  Time reading reference sizes: 00:00:09\n",
      "Calculating joined length\n",
      "Writing header\n",
      "Reserving space for joined string\n",
      "Joining reference sequences\n",
      "  Time to join reference sequences: 00:00:06\n",
      "  Time to reverse reference sequence: 00:00:00\n",
      "bmax according to bmaxDivN setting: 171168832\n",
      "Using parameters --bmax 128376624 --dcv 1024\n",
      "  Doing ahead-of-time memory usage test\n",
      "  Passed!  Constructing with these parameters: --bmax 128376624 --dcv 1024\n",
      "Constructing suffix-array element generator\n",
      "Building DifferenceCoverSample\n",
      "  Building sPrime\n",
      "  Building sPrimeOrder\n",
      "  V-Sorting samples\n",
      "  V-Sorting samples time: 00:00:25\n",
      "  Allocating rank array\n",
      "  Ranking v-sort output\n",
      "  Ranking v-sort output time: 00:00:08\n",
      "  Invoking Larsson-Sadakane on ranks\n",
      "  Invoking Larsson-Sadakane on ranks time: 00:00:12\n",
      "  Sanity-checking and returning\n",
      "Building samples\n",
      "Reserving space for 12 sample suffixes\n",
      "Generating random suffixes\n",
      "QSorting 12 sample offsets, eliminating duplicates\n",
      "QSorting sample offsets, eliminating duplicates time: 00:00:00\n",
      "Multikey QSorting 12 samples\n",
      "  (Using difference cover)\n",
      "  Multikey QSorting samples time: 00:00:00\n",
      "Calculating bucket sizes\n",
      "Splitting and merging\n",
      "  Splitting and merging time: 00:00:00\n",
      "Avg bucket size: 6.84675e+08 (target: 128376623)\n",
      "Converting suffix-array elements to index image\n",
      "Allocating ftab, absorbFtab\n",
      "Entering Ebwt loop\n",
      "Getting block 1 of 1\n",
      "  No samples; assembling all-inclusive block\n",
      "  Sorting block of length 684675328 for bucket 1\n",
      "  (Using difference cover)\n",
      "  Sorting block time: 00:11:33\n",
      "Returning block of 684675329 for bucket 1\n",
      "Exited Ebwt loop\n",
      "fchr[A]: 0\n",
      "fchr[C]: 342402452\n",
      "fchr[G]: 461631571\n",
      "fchr[T]: 461631571\n",
      "fchr[$]: 684675328\n",
      "Exiting Ebwt::buildToDisk()\n",
      "Returning from initFromVector\n",
      "Wrote 232427948 bytes to primary EBWT file: BS_GA.rev.1.bt2\n",
      "Wrote 171168840 bytes to secondary EBWT file: BS_GA.rev.2.bt2\n",
      "Re-opening _in1 and _in2 as input streams\n",
      "Returning from Ebwt constructor\n",
      "Headers:\n",
      "    len: 684675328\n",
      "    bwtLen: 684675329\n",
      "    sz: 171168832\n",
      "    bwtSz: 171168833\n",
      "    lineRate: 6\n",
      "    offRate: 4\n",
      "    offMask: 0xfffffff0\n",
      "    ftabChars: 10\n",
      "    eftabLen: 20\n",
      "    eftabSz: 80\n",
      "    ftabLen: 1048577\n",
      "    ftabSz: 4194308\n",
      "    offsLen: 42792209\n",
      "    offsSz: 171168836\n",
      "    lineSz: 64\n",
      "    sideSz: 64\n",
      "    sideBwtSz: 48\n",
      "    sideBwtLen: 192\n",
      "    numSides: 3566018\n",
      "    numLines: 3566018\n",
      "    ebwtTotLen: 228225152\n",
      "    ebwtTotSz: 228225152\n",
      "    color: 0\n",
      "    reverse: 1\n",
      "Total time for backward call to driver() for mirror index: 00:15:02\n",
      "  Sorting block time: 00:14:01\n",
      "Returning block of 684675329 for bucket 1\n",
      "Exited Ebwt loop\n",
      "fchr[A]: 0\n",
      "fchr[C]: 223134775\n",
      "fchr[G]: 223134775\n",
      "fchr[T]: 342402452\n",
      "fchr[$]: 684675328\n",
      "Exiting Ebwt::buildToDisk()\n",
      "Returning from initFromVector\n",
      "Wrote 232427948 bytes to primary EBWT file: BS_CT.rev.1.bt2\n",
      "Wrote 171168840 bytes to secondary EBWT file: BS_CT.rev.2.bt2\n",
      "Re-opening _in1 and _in2 as input streams\n",
      "Returning from Ebwt constructor\n",
      "Headers:\n",
      "    len: 684675328\n",
      "    bwtLen: 684675329\n",
      "    sz: 171168832\n",
      "    bwtSz: 171168833\n",
      "    lineRate: 6\n",
      "    offRate: 4\n",
      "    offMask: 0xfffffff0\n",
      "    ftabChars: 10\n",
      "    eftabLen: 20\n",
      "    eftabSz: 80\n",
      "    ftabLen: 1048577\n",
      "    ftabSz: 4194308\n",
      "    offsLen: 42792209\n",
      "    offsSz: 171168836\n",
      "    lineSz: 64\n",
      "    sideSz: 64\n",
      "    sideBwtSz: 48\n",
      "    sideBwtLen: 192\n",
      "    numSides: 3566018\n",
      "    numLines: 3566018\n",
      "    ebwtTotLen: 228225152\n",
      "    ebwtTotSz: 228225152\n",
      "    color: 0\n",
      "    reverse: 1\n",
      "Total time for backward call to driver() for mirror index: 00:17:24\n"
     ]
    }
   ],
   "source": [
    "!bismark_genome_preparation --path_to_bowtie /Users/Shelly/anaconda3/bin/ --verbose /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Then I ran bismark subsetting only the first 10K reads from the first two .fastqs, specifying reads are non-directional, and using a minimum score threshold that Steven previously used"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "FastQ format assumed (by default)\n",
      "Processing sequences up to read no. 10000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/shellytrigg.github.io/jupyter/Cvirginica/data_analysis/Cvirg_Apr2018'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/shellytrigg.github.io/jupyter/Cvirginica/data_analysis/Cvirg_Apr2018\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 10000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R1.fastq.gz (10001 sequences in total)\n",
      "\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz: uncompress failed\n",
      "Processing reads up to sequence no. 10000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R2.fastq.gz (10001 sequences in total)\n",
      "\n",
      "Input files are zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t99\tNC_035789.1_CT_converted\t4531039\t2\t37M4I59M\t=\t4531139\t200\tGAGTTTTTTTGATTATTTGTTGTTTGTTGTTTGTTTGNNNNTNNNTNNGTTTGTTTGTTTGTTTGTTTGTTTGTAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tAS:i:-22\tXS:i:-31\tXN:i:0\tXM:i:5\tXO:i:1\tXG:i:4\tNM:i:9\tMD:Z:38T0T0G1T0T52\tYS:i:-24\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t147\tNC_035789.1_CT_converted\t4531139\t2\t100M\t=\t4531039\t-200\tGAATTATTGGGTTAATTTTAATTAAATTTGATATAAAGTATTTTTAGGTGAAGGGGATTTAAGTTTGTTTGAATGAAGGGTTTTGTTTTTTTTTAAGGGG\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-24\tXS:i:-12\tXN:i:0\tXM:i:4\tXO:i:0\tXG:i:0\tNM:i:4\tMD:Z:30G14G9A14A29\tYS:i:-22\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTAAAAAAAAATGAAATTTTTTATTTAAATAAATTTAAATTTTTTTTATTTAAAAATATTTTATATTAAATTTAATTAAAATTAATTTAATAATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTAAAAAAAAATGAAATTTTTTATTTAAATAAATTTAAATTTTTTTTATTTAAAAATATTTTATATTAAATTTAATTAAAATTAATTTAATAATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t83\tNC_035780.1_GA_converted\t8090288\t6\t26M4D74M\t=\t8090028\t-364\tAAAAAAAATAAAAATATAAAAAATTTACAAACAAACAAACAAACAAACAAACNNANNNANNNNCAAACAAACAACAAACAACAAATAATCAAAAAAACTC\tFFFFFFFF/FFFFF/B/FFFFFF/</FFFFFF<FFFFFFFFFFFFFFFF<<<##<###<####FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-26\tXS:i:-9\tXN:i:0\tXM:i:9\tXO:i:1\tXG:i:4\tNM:i:13\tMD:Z:26^ACAA26A0A1C0A0A1C0A0A0A37\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t163\tNC_035780.1_GA_converted\t8090028\t6\t100M\t=\t8090288\t364\tCCCCTTAAAAAAAAACAAAACCCTTCATTCAAACAAACTTAAATCCCCTTCACCTAAAAATACTTTATATCAAATTTAATTAAAATTAACCCAATAATTC\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-18\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:29T70\tYS:i:-26\tYT:Z:CP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_1_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "10000 reads; of these:\n",
      "  10000 (100.00%) were paired; of these:\n",
      "    9365 (93.65%) aligned concordantly 0 times\n",
      "    309 (3.09%) aligned concordantly exactly 1 time\n",
      "    326 (3.26%) aligned concordantly >1 times\n",
      "6.35% overall alignment rate\n",
      "10000 reads; of these:\n",
      "  10000 (100.00%) were paired; of these:\n",
      "    9321 (93.21%) aligned concordantly 0 times\n",
      "    353 (3.53%) aligned concordantly exactly 1 time\n",
      "    326 (3.26%) aligned concordantly >1 times\n",
      "6.79% overall alignment rate\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10000 reads; of these:\n",
      "  10000 (100.00%) were paired; of these:\n",
      "    9354 (93.54%) aligned concordantly 0 times\n",
      "    309 (3.09%) aligned concordantly exactly 1 time\n",
      "    337 (3.37%) aligned concordantly >1 times\n",
      "6.46% overall alignment rate\n",
      "10000 reads; of these:\n",
      "  10000 (100.00%) were paired; of these:\n",
      "    9335 (93.35%) aligned concordantly 0 times\n",
      "    324 (3.24%) aligned concordantly exactly 1 time\n",
      "    341 (3.41%) aligned concordantly >1 times\n",
      "6.65% overall alignment rate\n",
      "Processed 10000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_1_s1_R1.fastq.gz_C_to_T.fastq, zr2096_1_s1_R1.fastq.gz_G_to_A.fastq, zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t10000\n",
      "Number of paired-end alignments with a unique best hit:\t1386\n",
      "Mapping efficiency:\t13.9%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t8091\n",
      "Sequence pairs did not map uniquely:\t523\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t347\t((converted) top strand)\n",
      "GA/CT/CT:\t366\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t327\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t346\t((converted) bottom strand)\n",
      "\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t48011\n",
      "\n",
      "Total methylated C's in CpG context:\t4741\n",
      "Total methylated C's in CHG context:\t269\n",
      "Total methylated C's in CHH context:\t893\n",
      "Total methylated C's in Unknown context:\t66\n",
      "\n",
      "Total unmethylated C's in CpG context:\t1715\n",
      "Total unmethylated C's in CHG context:\t10962\n",
      "Total unmethylated C's in CHH context:\t29431\n",
      "Total unmethylated C's in Unknown context:\t198\n",
      "\n",
      "C methylated in CpG context:\t73.4%\n",
      "C methylated in CHG context:\t2.4%\n",
      "C methylated in CHH context:\t2.9%\n",
      "C methylated in unknown context (CN or CHN):\t25.0%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 2m 54s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz: uncompress failed\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz: uncompress failed\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz: uncompress failed\n"
     ]
    }
   ],
   "source": [
    "!bismark -u 10000 --non_directional --score_min L,0,-0.6 --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
