{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Bismark practice with C. virginica 2018-04-07 data\n",
    "Bismark paper:  \n",
    "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3102221/  \n",
    "Bismark user guide:  \n",
    "https://rawgit.com/FelixKrueger/Bismark/master/Docs/Bismark_User_Guide.html  \n",
    "\n",
    "First, needed to install bowtie2, samtools, and bismark locally. I already had anaconda3 on my computer, so I just needed to run the following commands:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!conda install -c bioconda bowtie2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!conda install -c bioconda samtools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!conda install -c bioconda bismark "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "FIRST:  connect to Gannett server; this will mount as /Volumes/web\n",
    "SECOND:  connect to OWL server; this will mount as /Volumes/web-1\n",
    "Then make new directories for bismark analysis in the 'metacarcinus' folder on seashell"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
    "pwd "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "copy genome to working directory folder"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "cp /Volumes/web-1/halfshell/genomic-databank/Cvirginica_v300.fa /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "run bismark genome preparation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Path to genome folder specified as: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "Aligner to be used: Bowtie 2 (default)\n",
      "Writing bisulfite genomes out into a single MFA (multi FastA) file\n",
      "\n",
      "Bismark Genome Preparation - Step I: Preparing folders\n",
      "\n",
      "Path to Bowtie 2 specified: /Users/Shelly/anaconda3/bin/\n",
      "Bisulfite Genome Indexer version v0.20.0 (last modified 26 April 2018)\n",
      "Created Bisulfite Genome folder /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/Bisulfite_Genome/\n",
      "Created Bisulfite Genome folder /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/Bisulfite_Genome/CT_conversion/\n",
      "Created Bisulfite Genome folder /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/Bisulfite_Genome/GA_conversion/\n",
      "\n",
      "Step I - Prepare genome folders - completed\n",
      "\n",
      "\n",
      "Bismark Genome Preparation - Step II: Bisulfite converting reference genome\n",
      "\n",
      "conversions performed:\n",
      "chromosome\tC->T\tG->A\n",
      "NC_035780.1\t11492316\t11528162\n",
      "NC_035781.1\t10879823\t10858285\n",
      "NC_035782.1\t13475475\t13496235\n",
      "NC_035783.1\t10484268\t10482193\n",
      "NC_035784.1\t17306783\t17253434\n",
      "NC_035785.1\t8878985\t8876470\n",
      "NC_035786.1\t9961761\t9989843\n",
      "NC_035787.1\t13145205\t13164212\n",
      "NC_035788.1\t17925605\t17932896\n",
      "NC_035789.1\t5676071\t5682065\n",
      "NC_007175.2\t2827\t3882\n",
      "\n",
      "Total number of conversions performed:\n",
      "C->T:\t119229119\n",
      "G->A:\t119267677\n",
      "\n",
      "Step II - Genome bisulfite conversions - completed\n",
      "\n",
      "\n",
      "Bismark Genome Preparation - Step III: Launching the Bowtie 2 indexer\n",
      "Please be aware that this process can - depending on genome size - take several hours!\n",
      "\n",
      "Preparing indexing of CT converted genome in /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/Bisulfite_Genome/CT_conversion/\n",
      "Parent process: Starting to index C->T converted genome with the following command:\n",
      "\n",
      "/Users/Shelly/anaconda3/bin/bowtie2-build -f genome_mfa.CT_conversion.fa BS_CT\n",
      "\n",
      "Preparing indexing of GA converted genome in /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/Bisulfite_Genome/GA_conversion/\n",
      "Child process: Starting to index G->A converted genome with the following command:\n",
      "\n",
      "/Users/Shelly/anaconda3/bin/bowtie2-build -f genome_mfa.GA_conversion.fa BS_GA\n",
      "\n",
      "Settings:\n",
      "  Output files: \"BS_CT.*.bt2\"\n",
      "  Line rate: 6 (line is 64 bytes)\n",
      "  Lines per side: 1 (side is 64 bytes)\n",
      "  Offset rate: 4 (one in 16)\n",
      "  FTable chars: 10\n",
      "  Strings: unpacked\n",
      "  Max bucket size: default\n",
      "  Max bucket size, sqrt multiplier: default\n",
      "  Max bucket size, len divisor: 4\n",
      "  Difference-cover sample period: 1024\n",
      "  Endianness: little\n",
      "  Actual local endianness: little\n",
      "  Sanity checking: disabled\n",
      "  Assertions: disabled\n",
      "  Random seed: 0\n",
      "  Sizeofs: void*:8, int:4, long:8, size_t:8\n",
      "Input files DNA, FASTA:\n",
      "  genome_mfa.CT_conversion.fa\n",
      "Building a SMALL index\n",
      "Reading reference sizes\n",
      "Settings:\n",
      "  Output files: \"BS_GA.*.bt2\"\n",
      "  Line rate: 6 (line is 64 bytes)\n",
      "  Lines per side: 1 (side is 64 bytes)\n",
      "  Offset rate: 4 (one in 16)\n",
      "  FTable chars: 10\n",
      "  Strings: unpacked\n",
      "  Max bucket size: default\n",
      "  Max bucket size, sqrt multiplier: default\n",
      "  Max bucket size, len divisor: 4\n",
      "  Difference-cover sample period: 1024\n",
      "  Endianness: little\n",
      "  Actual local endianness: little\n",
      "  Sanity checking: disabled\n",
      "  Assertions: disabled\n",
      "  Random seed: 0\n",
      "  Sizeofs: void*:8, int:4, long:8, size_t:8\n",
      "Input files DNA, FASTA:\n",
      "  genome_mfa.GA_conversion.fa\n",
      "Building a SMALL index\n",
      "Reading reference sizes\n",
      "  Time reading reference sizes: 00:01:55\n",
      "Calculating joined length\n",
      "Writing header\n",
      "Reserving space for joined string\n",
      "Joining reference sequences\n",
      "  Time reading reference sizes: 00:01:55\n",
      "Calculating joined length\n",
      "Writing header\n",
      "Reserving space for joined string\n",
      "Joining reference sequences\n",
      "  Time to join reference sequences: 00:00:12\n",
      "bmax according to bmaxDivN setting: 171168832\n",
      "Using parameters --bmax 128376624 --dcv 1024\n",
      "  Doing ahead-of-time memory usage test\n",
      "  Passed!  Constructing with these parameters: --bmax 128376624 --dcv 1024\n",
      "Constructing suffix-array element generator\n",
      "Building DifferenceCoverSample\n",
      "  Building sPrime\n",
      "  Time to join reference sequences: 00:00:13\n",
      "bmax according to bmaxDivN setting: 171168832\n",
      "Using parameters --bmax 128376624 --dcv 1024\n",
      "  Doing ahead-of-time memory usage test\n",
      "  Passed!  Constructing with these parameters: --bmax 128376624 --dcv 1024\n",
      "Constructing suffix-array element generator\n",
      "Building DifferenceCoverSample\n",
      "  Building sPrime\n",
      "  Building sPrimeOrder\n",
      "  Building sPrimeOrder\n",
      "  V-Sorting samples\n",
      "  V-Sorting samples\n",
      "  V-Sorting samples time: 00:00:47\n",
      "  Allocating rank array\n",
      "  Ranking v-sort output\n",
      "  V-Sorting samples time: 00:00:47\n",
      "  Allocating rank array\n",
      "  Ranking v-sort output\n",
      "  Ranking v-sort output time: 00:00:11\n",
      "  Invoking Larsson-Sadakane on ranks\n",
      "  Ranking v-sort output time: 00:00:11\n",
      "  Invoking Larsson-Sadakane on ranks\n",
      "  Invoking Larsson-Sadakane on ranks time: 00:00:16\n",
      "  Sanity-checking and returning\n",
      "Building samples\n",
      "Reserving space for 12 sample suffixes\n",
      "Generating random suffixes\n",
      "QSorting 12 sample offsets, eliminating duplicates\n",
      "QSorting sample offsets, eliminating duplicates time: 00:00:00\n",
      "Multikey QSorting 12 samples\n",
      "  (Using difference cover)\n",
      "  Multikey QSorting samples time: 00:00:00\n",
      "Calculating bucket sizes\n",
      "Splitting and merging\n",
      "  Splitting and merging time: 00:00:00\n",
      "Avg bucket size: 6.84675e+08 (target: 128376623)\n",
      "Converting suffix-array elements to index image\n",
      "Allocating ftab, absorbFtab\n",
      "Entering Ebwt loop\n",
      "Getting block 1 of 1\n",
      "  No samples; assembling all-inclusive block\n",
      "  Invoking Larsson-Sadakane on ranks time: 00:00:16\n",
      "  Sanity-checking and returning\n",
      "Building samples\n",
      "Reserving space for 12 sample suffixes\n",
      "Generating random suffixes\n",
      "QSorting 12 sample offsets, eliminating duplicates\n",
      "QSorting sample offsets, eliminating duplicates time: 00:00:00\n",
      "Multikey QSorting 12 samples\n",
      "  (Using difference cover)\n",
      "  Multikey QSorting samples time: 00:00:00\n",
      "Calculating bucket sizes\n",
      "Splitting and merging\n",
      "  Splitting and merging time: 00:00:00\n",
      "Avg bucket size: 6.84675e+08 (target: 128376623)\n",
      "Converting suffix-array elements to index image\n",
      "Allocating ftab, absorbFtab\n",
      "Entering Ebwt loop\n",
      "Getting block 1 of 1\n",
      "  No samples; assembling all-inclusive block\n",
      "  Sorting block of length 684675328 for bucket 1\n",
      "  (Using difference cover)\n",
      "  Sorting block of length 684675328 for bucket 1\n",
      "  (Using difference cover)\n",
      "  Sorting block time: 00:13:58\n",
      "Returning block of 684675329 for bucket 1\n",
      "  Sorting block time: 00:16:06\n",
      "Returning block of 684675329 for bucket 1\n",
      "Exited Ebwt loop\n",
      "fchr[A]: 0\n",
      "fchr[C]: 342402452\n",
      "fchr[G]: 461631571\n",
      "fchr[T]: 461631571\n",
      "fchr[$]: 684675328\n",
      "Exiting Ebwt::buildToDisk()\n",
      "Returning from initFromVector\n",
      "Wrote 232427948 bytes to primary EBWT file: BS_GA.1.bt2\n",
      "Wrote 171168840 bytes to secondary EBWT file: BS_GA.2.bt2\n",
      "Re-opening _in1 and _in2 as input streams\n",
      "Returning from Ebwt constructor\n",
      "Headers:\n",
      "    len: 684675328\n",
      "    bwtLen: 684675329\n",
      "    sz: 171168832\n",
      "    bwtSz: 171168833\n",
      "    lineRate: 6\n",
      "    offRate: 4\n",
      "    offMask: 0xfffffff0\n",
      "    ftabChars: 10\n",
      "    eftabLen: 20\n",
      "    eftabSz: 80\n",
      "    ftabLen: 1048577\n",
      "    ftabSz: 4194308\n",
      "    offsLen: 42792209\n",
      "    offsSz: 171168836\n",
      "    lineSz: 64\n",
      "    sideSz: 64\n",
      "    sideBwtSz: 48\n",
      "    sideBwtLen: 192\n",
      "    numSides: 3566018\n",
      "    numLines: 3566018\n",
      "    ebwtTotLen: 228225152\n",
      "    ebwtTotSz: 228225152\n",
      "    color: 0\n",
      "    reverse: 0\n",
      "Total time for call to driver() for forward index: 00:19:59\n",
      "Reading reference sizes\n",
      "  Time reading reference sizes: 00:00:08\n",
      "Calculating joined length\n",
      "Writing header\n",
      "Reserving space for joined string\n",
      "Joining reference sequences\n",
      "  Time to join reference sequences: 00:00:06\n",
      "  Time to reverse reference sequence: 00:00:01\n",
      "bmax according to bmaxDivN setting: 171168832\n",
      "Using parameters --bmax 128376624 --dcv 1024\n",
      "  Doing ahead-of-time memory usage test\n",
      "  Passed!  Constructing with these parameters: --bmax 128376624 --dcv 1024\n",
      "Constructing suffix-array element generator\n",
      "Building DifferenceCoverSample\n",
      "  Building sPrime\n",
      "  Building sPrimeOrder\n",
      "  V-Sorting samples\n",
      "  V-Sorting samples time: 00:00:25\n",
      "  Allocating rank array\n",
      "  Ranking v-sort output\n",
      "  Ranking v-sort output time: 00:00:07\n",
      "  Invoking Larsson-Sadakane on ranks\n",
      "  Invoking Larsson-Sadakane on ranks time: 00:00:10\n",
      "  Sanity-checking and returning\n",
      "Building samples\n",
      "Reserving space for 12 sample suffixes\n",
      "Generating random suffixes\n",
      "QSorting 12 sample offsets, eliminating duplicates\n",
      "QSorting sample offsets, eliminating duplicates time: 00:00:00\n",
      "Multikey QSorting 12 samples\n",
      "  (Using difference cover)\n",
      "  Multikey QSorting samples time: 00:00:00\n",
      "Calculating bucket sizes\n",
      "Splitting and merging\n",
      "  Splitting and merging time: 00:00:00\n",
      "Avg bucket size: 6.84675e+08 (target: 128376623)\n",
      "Converting suffix-array elements to index image\n",
      "Allocating ftab, absorbFtab\n",
      "Entering Ebwt loop\n",
      "Getting block 1 of 1\n",
      "  No samples; assembling all-inclusive block\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  Sorting block of length 684675328 for bucket 1\n",
      "  (Using difference cover)\n",
      "Exited Ebwt loop\n",
      "fchr[A]: 0\n",
      "fchr[C]: 223134775\n",
      "fchr[G]: 223134775\n",
      "fchr[T]: 342402452\n",
      "fchr[$]: 684675328\n",
      "Exiting Ebwt::buildToDisk()\n",
      "Returning from initFromVector\n",
      "Wrote 232427948 bytes to primary EBWT file: BS_CT.1.bt2\n",
      "Wrote 171168840 bytes to secondary EBWT file: BS_CT.2.bt2\n",
      "Re-opening _in1 and _in2 as input streams\n",
      "Returning from Ebwt constructor\n",
      "Headers:\n",
      "    len: 684675328\n",
      "    bwtLen: 684675329\n",
      "    sz: 171168832\n",
      "    bwtSz: 171168833\n",
      "    lineRate: 6\n",
      "    offRate: 4\n",
      "    offMask: 0xfffffff0\n",
      "    ftabChars: 10\n",
      "    eftabLen: 20\n",
      "    eftabSz: 80\n",
      "    ftabLen: 1048577\n",
      "    ftabSz: 4194308\n",
      "    offsLen: 42792209\n",
      "    offsSz: 171168836\n",
      "    lineSz: 64\n",
      "    sideSz: 64\n",
      "    sideBwtSz: 48\n",
      "    sideBwtLen: 192\n",
      "    numSides: 3566018\n",
      "    numLines: 3566018\n",
      "    ebwtTotLen: 228225152\n",
      "    ebwtTotSz: 228225152\n",
      "    color: 0\n",
      "    reverse: 0\n",
      "Total time for call to driver() for forward index: 00:22:02\n",
      "Reading reference sizes\n",
      "  Time reading reference sizes: 00:00:09\n",
      "Calculating joined length\n",
      "Writing header\n",
      "Reserving space for joined string\n",
      "Joining reference sequences\n",
      "  Time to join reference sequences: 00:00:06\n",
      "  Time to reverse reference sequence: 00:00:00\n",
      "bmax according to bmaxDivN setting: 171168832\n",
      "Using parameters --bmax 128376624 --dcv 1024\n",
      "  Doing ahead-of-time memory usage test\n",
      "  Passed!  Constructing with these parameters: --bmax 128376624 --dcv 1024\n",
      "Constructing suffix-array element generator\n",
      "Building DifferenceCoverSample\n",
      "  Building sPrime\n",
      "  Building sPrimeOrder\n",
      "  V-Sorting samples\n",
      "  V-Sorting samples time: 00:00:25\n",
      "  Allocating rank array\n",
      "  Ranking v-sort output\n",
      "  Ranking v-sort output time: 00:00:08\n",
      "  Invoking Larsson-Sadakane on ranks\n",
      "  Invoking Larsson-Sadakane on ranks time: 00:00:12\n",
      "  Sanity-checking and returning\n",
      "Building samples\n",
      "Reserving space for 12 sample suffixes\n",
      "Generating random suffixes\n",
      "QSorting 12 sample offsets, eliminating duplicates\n",
      "QSorting sample offsets, eliminating duplicates time: 00:00:00\n",
      "Multikey QSorting 12 samples\n",
      "  (Using difference cover)\n",
      "  Multikey QSorting samples time: 00:00:00\n",
      "Calculating bucket sizes\n",
      "Splitting and merging\n",
      "  Splitting and merging time: 00:00:00\n",
      "Avg bucket size: 6.84675e+08 (target: 128376623)\n",
      "Converting suffix-array elements to index image\n",
      "Allocating ftab, absorbFtab\n",
      "Entering Ebwt loop\n",
      "Getting block 1 of 1\n",
      "  No samples; assembling all-inclusive block\n",
      "  Sorting block of length 684675328 for bucket 1\n",
      "  (Using difference cover)\n",
      "  Sorting block time: 00:11:33\n",
      "Returning block of 684675329 for bucket 1\n",
      "Exited Ebwt loop\n",
      "fchr[A]: 0\n",
      "fchr[C]: 342402452\n",
      "fchr[G]: 461631571\n",
      "fchr[T]: 461631571\n",
      "fchr[$]: 684675328\n",
      "Exiting Ebwt::buildToDisk()\n",
      "Returning from initFromVector\n",
      "Wrote 232427948 bytes to primary EBWT file: BS_GA.rev.1.bt2\n",
      "Wrote 171168840 bytes to secondary EBWT file: BS_GA.rev.2.bt2\n",
      "Re-opening _in1 and _in2 as input streams\n",
      "Returning from Ebwt constructor\n",
      "Headers:\n",
      "    len: 684675328\n",
      "    bwtLen: 684675329\n",
      "    sz: 171168832\n",
      "    bwtSz: 171168833\n",
      "    lineRate: 6\n",
      "    offRate: 4\n",
      "    offMask: 0xfffffff0\n",
      "    ftabChars: 10\n",
      "    eftabLen: 20\n",
      "    eftabSz: 80\n",
      "    ftabLen: 1048577\n",
      "    ftabSz: 4194308\n",
      "    offsLen: 42792209\n",
      "    offsSz: 171168836\n",
      "    lineSz: 64\n",
      "    sideSz: 64\n",
      "    sideBwtSz: 48\n",
      "    sideBwtLen: 192\n",
      "    numSides: 3566018\n",
      "    numLines: 3566018\n",
      "    ebwtTotLen: 228225152\n",
      "    ebwtTotSz: 228225152\n",
      "    color: 0\n",
      "    reverse: 1\n",
      "Total time for backward call to driver() for mirror index: 00:15:02\n",
      "  Sorting block time: 00:14:01\n",
      "Returning block of 684675329 for bucket 1\n",
      "Exited Ebwt loop\n",
      "fchr[A]: 0\n",
      "fchr[C]: 223134775\n",
      "fchr[G]: 223134775\n",
      "fchr[T]: 342402452\n",
      "fchr[$]: 684675328\n",
      "Exiting Ebwt::buildToDisk()\n",
      "Returning from initFromVector\n",
      "Wrote 232427948 bytes to primary EBWT file: BS_CT.rev.1.bt2\n",
      "Wrote 171168840 bytes to secondary EBWT file: BS_CT.rev.2.bt2\n",
      "Re-opening _in1 and _in2 as input streams\n",
      "Returning from Ebwt constructor\n",
      "Headers:\n",
      "    len: 684675328\n",
      "    bwtLen: 684675329\n",
      "    sz: 171168832\n",
      "    bwtSz: 171168833\n",
      "    lineRate: 6\n",
      "    offRate: 4\n",
      "    offMask: 0xfffffff0\n",
      "    ftabChars: 10\n",
      "    eftabLen: 20\n",
      "    eftabSz: 80\n",
      "    ftabLen: 1048577\n",
      "    ftabSz: 4194308\n",
      "    offsLen: 42792209\n",
      "    offsSz: 171168836\n",
      "    lineSz: 64\n",
      "    sideSz: 64\n",
      "    sideBwtSz: 48\n",
      "    sideBwtLen: 192\n",
      "    numSides: 3566018\n",
      "    numLines: 3566018\n",
      "    ebwtTotLen: 228225152\n",
      "    ebwtTotSz: 228225152\n",
      "    color: 0\n",
      "    reverse: 1\n",
      "Total time for backward call to driver() for mirror index: 00:17:24\n"
     ]
    }
   ],
   "source": [
    "!bismark_genome_preparation --path_to_bowtie /Users/Shelly/anaconda3/bin/ \\\n",
    "--verbose /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Then I ran bismark subsetting only the first 10K reads from the first two .fastqs, specifying reads are non-directional, and using a minimum score threshold that Steven previously used"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "FastQ format assumed (by default)\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz: uncompress failed\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t99\tNC_035789.1_CT_converted\t4531039\t2\t37M4I59M\t=\t4531139\t200\tGAGTTTTTTTGATTATTTGTTGTTTGTTGTTTGTTTGNNNNTNNNTNNGTTTGTTTGTTTGTTTGTTTGTTTGTAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tAS:i:-22\tXS:i:-31\tXN:i:0\tXM:i:5\tXO:i:1\tXG:i:4\tNM:i:9\tMD:Z:38T0T0G1T0T52\tYS:i:-24\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t147\tNC_035789.1_CT_converted\t4531139\t2\t100M\t=\t4531039\t-200\tGAATTATTGGGTTAATTTTAATTAAATTTGATATAAAGTATTTTTAGGTGAAGGGGATTTAAGTTTGTTTGAATGAAGGGTTTTGTTTTTTTTTAAGGGG\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-24\tXS:i:-12\tXN:i:0\tXM:i:4\tXO:i:0\tXG:i:0\tNM:i:4\tMD:Z:30G14G9A14A29\tYS:i:-22\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTAAAAAAAAATGAAATTTTTTATTTAAATAAATTTAAATTTTTTTTATTTAAAAATATTTTATATTAAATTTAATTAAAATTAATTTAATAATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTAAAAAAAAATGAAATTTTTTATTTAAATAAATTTAAATTTTTTTTATTTAAAAATATTTTATATTAAATTTAATTAAAATTAATTTAATAATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t83\tNC_035780.1_GA_converted\t8090288\t6\t26M4D74M\t=\t8090028\t-364\tAAAAAAAATAAAAATATAAAAAATTTACAAACAAACAAACAAACAAACAAACNNANNNANNNNCAAACAAACAACAAACAACAAATAATCAAAAAAACTC\tFFFFFFFF/FFFFF/B/FFFFFF/</FFFFFF<FFFFFFFFFFFFFFFF<<<##<###<####FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-26\tXS:i:-9\tXN:i:0\tXM:i:9\tXO:i:1\tXG:i:4\tNM:i:13\tMD:Z:26^ACAA26A0A1C0A0A1C0A0A0A37\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t163\tNC_035780.1_GA_converted\t8090028\t6\t100M\t=\t8090288\t364\tCCCCTTAAAAAAAAACAAAACCCTTCATTCAAACAAACTTAAATCCCCTTCACCTAAAAATACTTTATATCAAATTTAATTAAAATTAACCCAATAATTC\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-18\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:29T70\tYS:i:-26\tYT:Z:CP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_1_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    93584 (93.58%) aligned concordantly 0 times\n",
      "    3180 (3.18%) aligned concordantly exactly 1 time\n",
      "    3236 (3.24%) aligned concordantly >1 times\n",
      "6.42% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    93560 (93.56%) aligned concordantly 0 times\n",
      "    3220 (3.22%) aligned concordantly exactly 1 time\n",
      "    3220 (3.22%) aligned concordantly >1 times\n",
      "6.44% overall alignment rate\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    93582 (93.58%) aligned concordantly 0 times\n",
      "    3175 (3.17%) aligned concordantly exactly 1 time\n",
      "    3243 (3.24%) aligned concordantly >1 times\n",
      "6.42% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    93611 (93.61%) aligned concordantly 0 times\n",
      "    3157 (3.16%) aligned concordantly exactly 1 time\n",
      "    3232 (3.23%) aligned concordantly >1 times\n",
      "6.39% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_1_s1_R1.fastq.gz_C_to_T.fastq, zr2096_1_s1_R1.fastq.gz_G_to_A.fastq, zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Number of paired-end alignments with a unique best hit:\t13744\n",
      "Mapping efficiency:\t13.7%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t81302\n",
      "Sequence pairs did not map uniquely:\t4954\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t3402\t((converted) top strand)\n",
      "GA/CT/CT:\t3480\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t3458\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t3404\t((converted) bottom strand)\n",
      "\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t481569\n",
      "\n",
      "Total methylated C's in CpG context:\t47945\n",
      "Total methylated C's in CHG context:\t2774\n",
      "Total methylated C's in CHH context:\t7846\n",
      "Total methylated C's in Unknown context:\t624\n",
      "\n",
      "Total unmethylated C's in CpG context:\t17126\n",
      "Total unmethylated C's in CHG context:\t107925\n",
      "Total unmethylated C's in CHH context:\t297953\n",
      "Total unmethylated C's in Unknown context:\t1993\n",
      "\n",
      "C methylated in CpG context:\t73.7%\n",
      "C methylated in CHG context:\t2.5%\n",
      "C methylated in CHH context:\t2.6%\n",
      "C methylated in unknown context (CN or CHN):\t23.8%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 3m 43s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz: uncompress failed\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz: uncompress failed\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz: uncompress failed\n"
     ]
    }
   ],
   "source": [
    "!bismark -u 100000 --non_directional --score_min L,0,-0.6 \\\n",
    "--genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ \\\n",
    "-1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz \\\n",
    "-2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz \\\n",
    "-o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/ "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "run Bismark Methylation extractor"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      " *** Bismark methylation extractor version v0.20.0 ***\n",
      "\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/\n",
      "Trying to determine the type of mapping from the SAM header line of file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.bam\n",
      "Treating file(s) as paired-end data (as extracted from @PG line)\n",
      "\n",
      "Setting option '--no_overlap' since this is (normally) the right thing to do for paired-end data\n",
      "\n",
      "Setting core usage to single-threaded (default). Consider using --multicore <int> to speed up the extraction process.\n",
      "\n",
      "Summarising Bismark methylation extractor parameters:\n",
      "===============================================================\n",
      "Bismark paired-end SAM format specified (default)\n",
      "Number of cores to be used: 1\n",
      "Output path specified as: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/\n",
      "\n",
      "\n",
      "Summarising bedGraph parameters:\n",
      "===============================================================\n",
      "Generating additional output in bedGraph and coverage format\n",
      "bedGraph format:\t<Chromosome> <Start Position> <End Position> <Methylation Percentage>\n",
      "coverage format:\t<Chromosome> <Start Position> <End Position> <Methylation Percentage> <count methylated> <count non-methylated>\n",
      "\n",
      "Using a cutoff of 1 read(s) to report cytosine positions\n",
      "Reporting and sorting cytosine methylation information in CpG context only (default)\n",
      "The bedGraph UNIX sort command will use the following memory setting:\t'2G'. Temporary directory used for sorting is the output directory\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "Now testing Bismark result file >/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.bam< for positional sorting (which would be bad...)\t...passed!\n",
      "Writing result file containing methylation information for C in CpG context from the original top strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Writing result file containing methylation information for C in CpG context from the complementary to original top strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Writing result file containing methylation information for C in CpG context from the complementary to original bottom strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Writing result file containing methylation information for C in CpG context from the original bottom strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "\n",
      "Writing result file containing methylation information for C in CHG context from the original top strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Writing result file containing methylation information for C in CHG context from the complementary to original top strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Writing result file containing methylation information for C in CHG context from the complementary to original bottom strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Writing result file containing methylation information for C in CHG context from the original bottom strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "\n",
      "Writing result file containing methylation information for C in CHH context from the original top strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Writing result file containing methylation information for C in CHH context from the complementary to original top strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Writing result file containing methylation information for C in CHH context from the complementary to original bottom strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Writing result file containing methylation information for C in CHH context from the original bottom strand to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "\n",
      "Now reading in Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.bam\n",
      "skipping SAM header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping SAM header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping SAM header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping SAM header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping SAM header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping SAM header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping SAM header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping SAM header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping SAM header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping SAM header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping SAM header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping SAM header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping SAM header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark -u 100000 --non_directional --score_min L,0,-0.6 --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/\"\n",
      "\n",
      "Processed 13744 lines in total\n",
      "Total number of methylation call strings processed: 27488\n",
      "\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t304266\n",
      "\n",
      "Total methylated C's in CpG context:\t29931\n",
      "Total methylated C's in CHG context:\t1873\n",
      "Total methylated C's in CHH context:\t5172\n",
      "\n",
      "Total C to T conversions in CpG context:\t10904\n",
      "Total C to T conversions in CHG context:\t67806\n",
      "Total C to T conversions in CHH context:\t188580\n",
      "\n",
      "C methylated in CpG context:\t73.3%\n",
      "C methylated in CHG context:\t2.7%\n",
      "C methylated in CHH context:\t2.7%\n",
      "\n",
      "\n",
      "Determining maximum read lengths for M-Bias plots\n",
      "Maximum read length of Read 1: 100\n",
      "Maximum read length of Read 2: 100\n",
      "\n",
      "Perl module GD::Graph::lines is not installed, skipping drawing M-bias plots (only writing out M-bias plot table)\n",
      "Determining maximum read lengths for M-Bias plots\n",
      "Maximum read length of Read 1: 100\n",
      "Maximum read length of Read 2: 100\n",
      "\n",
      "Perl module GD::Graph::lines is not installed, skipping drawing M-bias plots (only writing out M-bias plot table)\n",
      "Deleting unused files ...\n",
      "\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "gunzip: error writing to output: Broken pipe\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz contains data ->\tkept\n",
      "\n",
      "\n",
      "Using these input files: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHG_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "\n",
      "Summary of parameters for bismark2bedGraph conversion:\n",
      "======================================================\n",
      "bedGraph output:\t\tzr2096_1_s1_R1_bismark_bt2_pe.bedGraph.gz\n",
      "output directory:\t\t>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/<\n",
      "remove whitespaces:\t\tno\n",
      "CX context:\t\t\tno (CpG context only, default)\n",
      "No-header selected:\t\tno\n",
      "Sorting method:\t\t\tUnix sort-based (smaller memory footprint, but slower)\n",
      "Sort buffer size:\t\t2G\n",
      "Coverage threshold:\t\t1\n",
      "=============================================================================\n",
      "Methylation information will now be written into a bedGraph and coverage file\n",
      "=============================================================================\n",
      "\n",
      "Using the following files as Input:\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CpG_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "\n",
      "Writing bedGraph to file: zr2096_1_s1_R1_bismark_bt2_pe.bedGraph.gz\n",
      "Also writing out a coverage file including counts methylated and unmethylated residues to file: zr2096_1_s1_R1_bismark_bt2_pe.bismark.cov.gz\n",
      "\n",
      "Changed directory to /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/\n",
      "Now writing methylation information for file >>CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz<< to individual files for each chromosome\n",
      "Finished writing out individual chromosome files for CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Now writing methylation information for file >>CpG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz<< to individual files for each chromosome\n",
      "Finished writing out individual chromosome files for CpG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Now writing methylation information for file >>CpG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz<< to individual files for each chromosome\n",
      "Finished writing out individual chromosome files for CpG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "Now writing methylation information for file >>CpG_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz<< to individual files for each chromosome\n",
      "Finished writing out individual chromosome files for CpG_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz\n",
      "\n",
      "Collecting temporary chromosome file information... Processing the following input file(s):\n",
      "CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035780.1.methXtractor.temp\n",
      "CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035784.1.methXtractor.temp\n",
      "CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035782.1.methXtractor.temp\n",
      "CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035785.1.methXtractor.temp\n",
      "CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035783.1.methXtractor.temp\n",
      "CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035781.1.methXtractor.temp\n",
      "CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035787.1.methXtractor.temp\n",
      "CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035788.1.methXtractor.temp\n",
      "CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035786.1.methXtractor.temp\n",
      "CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035789.1.methXtractor.temp\n",
      "\n",
      "Sorting input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035780.1.methXtractor.temp by positions (using -S of 2G)\n",
      "Successfully deleted the temporary input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035780.1.methXtractor.temp\n",
      "\n",
      "Sorting input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035784.1.methXtractor.temp by positions (using -S of 2G)\n",
      "Successfully deleted the temporary input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035784.1.methXtractor.temp\n",
      "\n",
      "Sorting input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035782.1.methXtractor.temp by positions (using -S of 2G)\n",
      "Successfully deleted the temporary input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035782.1.methXtractor.temp\n",
      "\n",
      "Sorting input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035785.1.methXtractor.temp by positions (using -S of 2G)\n",
      "Successfully deleted the temporary input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035785.1.methXtractor.temp\n",
      "\n",
      "Sorting input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035783.1.methXtractor.temp by positions (using -S of 2G)\n",
      "Successfully deleted the temporary input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035783.1.methXtractor.temp\n",
      "\n",
      "Sorting input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035781.1.methXtractor.temp by positions (using -S of 2G)\n",
      "Successfully deleted the temporary input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035781.1.methXtractor.temp\n",
      "\n",
      "Sorting input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035787.1.methXtractor.temp by positions (using -S of 2G)\n",
      "Successfully deleted the temporary input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035787.1.methXtractor.temp\n",
      "\n",
      "Sorting input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035788.1.methXtractor.temp by positions (using -S of 2G)\n",
      "Successfully deleted the temporary input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035788.1.methXtractor.temp\n",
      "\n",
      "Sorting input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035786.1.methXtractor.temp by positions (using -S of 2G)\n",
      "Successfully deleted the temporary input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035786.1.methXtractor.temp\n",
      "\n",
      "Sorting input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035789.1.methXtractor.temp by positions (using -S of 2G)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Successfully deleted the temporary input file CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz.chrNC_035789.1.methXtractor.temp\n",
      "\n",
      "Finished BedGraph conversion ...\n",
      "\n",
      "samtools view: writing to standard output failed: Broken pipe\n",
      "samtools view: error closing standard output: -1\n",
      "gunzip: error writing to output: Broken pipe\n",
      "gunzip: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/CHH_OB_zr2096_1_s1_R1_bismark_bt2_pe.txt.gz: uncompress failed\n"
     ]
    }
   ],
   "source": [
    "!bismark_methylation_extractor --gzip -bedGraph /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.bam -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/ "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "generate Bismark Nucleotide Coverage report"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Summary of parameters for nucleotide coverage report:\n",
      "=====================================================\n",
      "Output directory:\t\t\t>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/<\n",
      "Parent directory:\t\t\t>/Users/Shelly/Documents/GitHub/C_virginica/<\n",
      "Genome directory:\t\t\t>/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/<\n",
      "Samtools installation:\t\t\t>/Users/Shelly/anaconda3/bin/samtools<\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Stored sequence information of 11 chromosomes/scaffolds in total\n",
      "\n",
      "==================================================================\n",
      "Mono- and di-nucleotide coverage will now be written into a report\n",
      "==================================================================\n",
      "\n",
      "Detected file 'genomic_nucleotide_frequencies.txt' in the genome folder /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ already. Using nucleotide frequencies contained therein ...\n",
      "============================================================================================================================================================================================\n",
      "\n",
      "\n",
      "Calculating read frequencies from file '/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.bam'\n",
      "==========================================================================================\n",
      "Determined the file to be paired-end\n",
      "\n",
      "\n",
      "Printing nucleotide stats to >> /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.nucleotide_stats.txt <<\n",
      "Final Stage: Calculating averages\n",
      "=================================\n",
      "\n",
      "(di-)nucleotide\tcount sample\tpercent sample\tcount genomic\tpercent genomic\tcoverage\n",
      "A\t397765\t27.39\t223134775\t32.59\t0.002\n",
      "C\t290760\t20.02\t119229119\t17.41\t0.002\n",
      "G\t346559\t23.86\t119267677\t17.42\t0.003\n",
      "T\t417399\t28.74\t223043757\t32.58\t0.002\n",
      "AA\t115616\t8.05\t80795037\t11.80\t0.001\n",
      "AC\t77775\t5.42\t37359527\t5.46\t0.002\n",
      "AG\t95546\t6.65\t35548672\t5.19\t0.003\n",
      "AT\t105707\t7.36\t69431301\t10.14\t0.002\n",
      "CA\t96576\t6.73\t45827414\t6.69\t0.002\n",
      "CC\t66568\t4.64\t23444678\t3.42\t0.003\n",
      "CG\t39753\t2.77\t14458703\t2.11\t0.003\n",
      "CT\t84353\t5.87\t35498217\t5.18\t0.002\n",
      "GA\t100972\t7.03\t38810489\t5.67\t0.003\n",
      "GC\t57424\t4.00\t19646670\t2.87\t0.003\n",
      "GG\t86627\t6.03\t23423030\t3.42\t0.004\n",
      "GT\t95958\t6.68\t37387368\t5.46\t0.003\n",
      "TA\t80967\t5.64\t57701608\t8.43\t0.001\n",
      "TC\t85855\t5.98\t38778132\t5.66\t0.002\n",
      "TG\t119602\t8.33\t45837169\t6.69\t0.003\n",
      "TT\t126771\t8.83\t80726644\t11.79\t0.002\n"
     ]
    }
   ],
   "source": [
    "!bam2nuc /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.bam \\\n",
    "--dir /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/ \\\n",
    "--samtools /Users/Shelly/anaconda3/bin/samtools \\\n",
    "--genome_folder /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "generate Bismark report"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Writing Bismark HTML report to >> /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_PE_report.html <<\n",
      "\n",
      "Redundant argument in sprintf at /Users/Shelly/anaconda3/bin/bismark2report line 170.\n",
      "==============================================================================================================\n",
      "Using the following alignment report:\t\t> /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_PE_report.txt <\n",
      "Processing alignment report /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_PE_report.txt ...\n",
      "Complete\n",
      "\n",
      "No deduplication report file specified, skipping this step\n",
      "Using the following splitting report:\t\t> /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe_splitting_report.txt <\n",
      "Processing splitting report /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe_splitting_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following M-bias report:\t\t> /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.M-bias.txt <\n",
      "Processing M-bias report /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.M-bias.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following nucleotide coverage report:\t> /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.nucleotide_stats.txt <\n",
      "Processing nucleotide coverage report '/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.nucleotide_stats.txt' ...\n",
      "Complete\n",
      "\n",
      "==============================================================================================================\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!bismark2report \\\n",
    "--alignment_report /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_PE_report.txt \\\n",
    "--splitting_report /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe_splitting_report.txt \\\n",
    "--mbias_report /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.M-bias.txt \\\n",
    "--nucleotide_report /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/zr2096_1_s1_R1_bismark_bt2_pe.nucleotide_stats.txt \\\n",
    "--dir /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt1/ \n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
