{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#aligning with Bismark"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!mkdir /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt7"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t55304\n",
      "Mapping efficiency:\t55.3%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t27490\n",
      "Sequence pairs did not map uniquely:\t17206\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t13863\t((converted) top strand)\n",
      "GA/CT/CT:\t13771\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t13915\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t13755\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t25623\n",
      "Mapping efficiency:\t25.6%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t66914\n",
      "Sequence pairs did not map uniquely:\t7463\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t6395\t((converted) top strand)\n",
      "GA/CT/CT:\t6412\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t6401\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t6415\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t50484\n",
      "Mapping efficiency:\t50.5%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t33358\n",
      "Sequence pairs did not map uniquely:\t16158\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t12761\t((converted) top strand)\n",
      "GA/CT/CT:\t12571\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t12410\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t12742\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t55171\n",
      "Mapping efficiency:\t55.2%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t26540\n",
      "Sequence pairs did not map uniquely:\t18289\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t13872\t((converted) top strand)\n",
      "GA/CT/CT:\t13727\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t13678\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t13894\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t56412\n",
      "Mapping efficiency:\t56.4%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t25464\n",
      "Sequence pairs did not map uniquely:\t18124\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t14301\t((converted) top strand)\n",
      "GA/CT/CT:\t13881\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t13768\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t14462\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t53779\n",
      "Mapping efficiency:\t53.8%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t29695\n",
      "Sequence pairs did not map uniquely:\t16526\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t13871\t((converted) top strand)\n",
      "GA/CT/CT:\t13053\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t12995\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t13860\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t56844\n",
      "Mapping efficiency:\t56.8%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t25108\n",
      "Sequence pairs did not map uniquely:\t18048\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t14846\t((converted) top strand)\n",
      "GA/CT/CT:\t13445\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t13483\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t15070\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t53463\n",
      "Mapping efficiency:\t53.5%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t30401\n",
      "Sequence pairs did not map uniquely:\t16136\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t13576\t((converted) top strand)\n",
      "GA/CT/CT:\t12963\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t13213\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t13711\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t48884\n",
      "Mapping efficiency:\t48.9%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t36530\n",
      "Sequence pairs did not map uniquely:\t14586\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t12344\t((converted) top strand)\n",
      "GA/CT/CT:\t12125\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t12019\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t12396\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t52449\n",
      "Mapping efficiency:\t52.4%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t30963\n",
      "Sequence pairs did not map uniquely:\t16588\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t14037\t((converted) top strand)\n",
      "GA/CT/CT:\t12205\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t12162\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t14045\t((converted) bottom strand)\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_10_s1_R1.fastq.gz to zr2096_10_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_10_s1_R1.fastq.gz to zr2096_10_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_10_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_10_s1_R2.fastq.gz to zr2096_10_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_10_s1_R2.fastq.gz to zr2096_10_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_10_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tGTTATATATAATTGTATTAATTATAAAAAATATNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATTTATTATTAAAAATATTAAATATATTATTTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNATTTTATATTTTTAATAATAAATATAATTATATATTAAAATAAAATAATATATTTAATATTTTTAATAATAAATACAATTAAATATCATAATAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tACCATATACAATCATATCAACTATAAAAAATACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATCTACTATCAAAAATACTAAACACACTACCTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNGTTTTGTATTTTTGATAGTAGATATGATTGTATATTAGGGTAAGGTAGTGTGTTTAGTATTTTTGATAGTAGATATGATTGAATATTGTGGTAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tACCATATACAATCATATCAACTATAAAAAATACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATCTACTATCAAAAATACTAAACACACTACCTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNGTTTTGTATTTTTGATAGTAGATATGATTGTATATTAGGGTAAGGTAGTGTGTTTAGTATTTTTGATAGTAGATATGATTGAATATTGTGGTAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tGTTATATATAATTGTATTAATTATAAAAAATATNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATTTATTATTAAAAATATTAAATATATTATTTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNATTTTATATTTTTAATAATAAATATAATTATATATTAAAATAAAATAATATATTTAATATTTTTAATAATAAATACAATTAAATATCATAATAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_10_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70695 (70.69%) aligned concordantly 0 times\n",
      "    12336 (12.34%) aligned concordantly exactly 1 time\n",
      "    16969 (16.97%) aligned concordantly >1 times\n",
      "29.30% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70600 (70.60%) aligned concordantly 0 times\n",
      "    12225 (12.22%) aligned concordantly exactly 1 time\n",
      "    17175 (17.18%) aligned concordantly >1 times\n",
      "29.40% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70611 (70.61%) aligned concordantly 0 times\n",
      "    12401 (12.40%) aligned concordantly exactly 1 time\n",
      "    16988 (16.99%) aligned concordantly >1 times\n",
      "29.39% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70498 (70.50%) aligned concordantly 0 times\n",
      "    12497 (12.50%) aligned concordantly exactly 1 time\n",
      "    17005 (17.00%) aligned concordantly >1 times\n",
      "29.50% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_10_s1_R1.fastq.gz_C_to_T.fastq, zr2096_10_s1_R1.fastq.gz_G_to_A.fastq, zr2096_10_s1_R2.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1911136\n",
      "\n",
      "Total methylated C's in CpG context:\t185771\n",
      "Total methylated C's in CHG context:\t13978\n",
      "Total methylated C's in CHH context:\t47702\n",
      "Total methylated C's in Unknown context:\t6403\n",
      "\n",
      "Total unmethylated C's in CpG context:\t84112\n",
      "Total unmethylated C's in CHG context:\t424427\n",
      "Total unmethylated C's in CHH context:\t1155146\n",
      "Total unmethylated C's in Unknown context:\t26070\n",
      "\n",
      "C methylated in CpG context:\t68.8%\n",
      "C methylated in CHG context:\t3.2%\n",
      "C methylated in CHH context:\t4.0%\n",
      "C methylated in unknown context (CN or CHN):\t19.7%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 46s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t99\tNC_035780.1_CT_converted\t65371909\t6\t100M\t=\t65372013\t204\tGAGTTTTTTTGATTATTTGTTGTTTGTTGTTTGTTTGNNNNTNNNTNNGTTTGTTTGTTTGTTTGTTTGTTTGTAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tAS:i:-15\tXS:i:-22\tXN:i:0\tXM:i:10\tXO:i:0\tXG:i:0\tNM:i:10\tMD:Z:37T0T0T0G1T0T0G1T0T10A41\tYS:i:-24\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t147\tNC_035780.1_CT_converted\t65372013\t6\t100M\t=\t65371909\t-204\tGAATTATTGGGTTAATTTTAATTAAATTTGATATAAAGTATTTTTAGGTGAAGGGGATTTAAGTTTGTTTGAATGAAGGGTTTTGTTTTTTTTTAAGGGG\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-24\tXS:i:-6\tXN:i:0\tXM:i:4\tXO:i:0\tXG:i:0\tNM:i:4\tMD:Z:30G14G3A20A29\tYS:i:-15\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTAAAAAAAAATGAAATTTTTTATTTAAATAAATTTAAATTTTTTTTATTTAAAAATATTTTATATTAAATTTAATTAAAATTAATTTAATAATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTAAAAAAAAATGAAATTTTTTATTTAAATAAATTTAAATTTTTTTTATTTAAAAATATTTTATATTAAATTTAATTAAAATTAATTTAATAATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t83\tNC_035780.1_GA_converted\t8090288\t7\t26M4D74M\t=\t8090028\t-364\tAAAAAAAATAAAAATATAAAAAATTTACAAACAAACAAACAAACAAACAAACNNANNNANNNNCAAACAAACAACAAACAACAAATAATCAAAAAAACTC\tFFFFFFFF/FFFFF/B/FFFFFF/</FFFFFF<FFFFFFFFFFFFFFFF<<<##<###<####FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-26\tXS:i:-22\tXN:i:0\tXM:i:9\tXO:i:1\tXG:i:4\tNM:i:13\tMD:Z:26^ACAA26A0A1C0A0A1C0A0A0A37\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t163\tNC_035780.1_GA_converted\t8090028\t7\t100M\t=\t8090288\t364\tCCCCTTAAAAAAAAACAAAACCCTTCATTCAAACAAACTTAAATCCCCTTCACCTAAAAATACTTTATATCAAATTTAATTAAAATTAACCCAATAATTC\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-36\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:29T70\tYS:i:-26\tYT:Z:CP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_1_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    86490 (86.49%) aligned concordantly 0 times\n",
      "    5663 (5.66%) aligned concordantly exactly 1 time\n",
      "    7847 (7.85%) aligned concordantly >1 times\n",
      "13.51% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    86569 (86.57%) aligned concordantly 0 times\n",
      "    5628 (5.63%) aligned concordantly exactly 1 time\n",
      "    7803 (7.80%) aligned concordantly >1 times\n",
      "13.43% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    86672 (86.67%) aligned concordantly 0 times\n",
      "    5595 (5.59%) aligned concordantly exactly 1 time\n",
      "    7733 (7.73%) aligned concordantly >1 times\n",
      "13.33% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    86777 (86.78%) aligned concordantly 0 times\n",
      "    5615 (5.62%) aligned concordantly exactly 1 time\n",
      "    7608 (7.61%) aligned concordantly >1 times\n",
      "13.22% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_1_s1_R1.fastq.gz_C_to_T.fastq, zr2096_1_s1_R1.fastq.gz_G_to_A.fastq, zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t830788\n",
      "\n",
      "Total methylated C's in CpG context:\t72846\n",
      "Total methylated C's in CHG context:\t6793\n",
      "Total methylated C's in CHH context:\t40295\n",
      "Total methylated C's in Unknown context:\t5123\n",
      "\n",
      "Total unmethylated C's in CpG context:\t32368\n",
      "Total unmethylated C's in CHG context:\t169425\n",
      "Total unmethylated C's in CHH context:\t509061\n",
      "Total unmethylated C's in Unknown context:\t14379\n",
      "\n",
      "C methylated in CpG context:\t69.2%\n",
      "C methylated in CHG context:\t3.9%\n",
      "C methylated in CHH context:\t7.3%\n",
      "C methylated in unknown context (CN or CHN):\t26.3%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 2s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_2_s1_R1.fastq.gz to zr2096_2_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_2_s1_R1.fastq.gz to zr2096_2_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_2_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_2_s1_R2.fastq.gz to zr2096_2_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_2_s1_R2.fastq.gz to zr2096_2_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_2_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tTTTTTTATTAAAATTTATTATTTTAAAAATAAAAATTNNNNTNNNTTTTATTATTTTTTTTAATATTTTTTAATATAATTAAATAATAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTCACTTCAAAAAAATAATAAATAAAAAAAATTAATACAACAATATTTTTAATAATACAACAATATTTAAAAAATTATTTATTAATTTTTTTTATTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tCTCCTTATTAAAATCTACTATTCCAAAAATAAAAACTNNNNCNNNTTCCACCATTCCTTCTAACATCTCCCAATATAACTAAATAACAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTGTTTTGGAAAAATAATGGATAAAAAAGGTTGATGTGGTGATGTTTTTAGTGATGTGGTGGTGTTTGAGAAATTATTTATTAATTTTTTTTGTTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tCTCCTTATTAAAATCTACTATTCCAAAAATAAAAACTNNNNCNNNTTCCACCATTCCTTCTAACATCTCCCAATATAACTAAATAACAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTGTTTTGGAAAAATAATGGATAAAAAAGGTTGATGTGGTGATGTTTTTAGTGATGTGGTGGTGTTTGAGAAATTATTTATTAATTTTTTTTGTTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tTTTTTTATTAAAATTTATTATTTTAAAAATAAAAATTNNNNTNNNTTTTATTATTTTTTTTAATATTTTTTAATATAATTAAATAATAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTCACTTCAAAAAAATAATAAATAAAAAAAATTAATACAACAATATTTTTAATAATACAACAATATTTAAAAAATTATTTATTAATTTTTTTTATTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_2_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    71647 (71.65%) aligned concordantly 0 times\n",
      "    11900 (11.90%) aligned concordantly exactly 1 time\n",
      "    16453 (16.45%) aligned concordantly >1 times\n",
      "28.35% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    71946 (71.95%) aligned concordantly 0 times\n",
      "    11827 (11.83%) aligned concordantly exactly 1 time\n",
      "    16227 (16.23%) aligned concordantly >1 times\n",
      "28.05% overall alignment rate\n",
      "100000100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    71468 ( reads; of these:71.47%\n",
      ") aligned concordantly 0 times  \n",
      "    10000011794 (11.79%) aligned concordantly exactly 1 time\n",
      "    16738 (16.74%) aligned concordantly >1 times\n",
      "28.53% overall alignment rate\n",
      " (100.00%) were paired; of these:\n",
      "    71402 (71.40%) aligned concordantly 0 times\n",
      "    11953 (11.95%) aligned concordantly exactly 1 time\n",
      "    16645 (16.64%) aligned concordantly >1 times\n",
      "28.60% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_2_s1_R1.fastq.gz_C_to_T.fastq, zr2096_2_s1_R1.fastq.gz_G_to_A.fastq, zr2096_2_s1_R2.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1601294\n",
      "\n",
      "Total methylated C's in CpG context:\t153343\n",
      "Total methylated C's in CHG context:\t12481\n",
      "Total methylated C's in CHH context:\t54835\n",
      "Total methylated C's in Unknown context:\t6562\n",
      "\n",
      "Total unmethylated C's in CpG context:\t65114\n",
      "Total unmethylated C's in CHG context:\t359729\n",
      "Total unmethylated C's in CHH context:\t955792\n",
      "Total unmethylated C's in Unknown context:\t24798\n",
      "\n",
      "C methylated in CpG context:\t70.2%\n",
      "C methylated in CHG context:\t3.4%\n",
      "C methylated in CHH context:\t5.4%\n",
      "C methylated in unknown context (CN or CHN):\t20.9%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 17s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_3_s1_R1.fastq.gz to zr2096_3_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_3_s1_R1.fastq.gz to zr2096_3_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_3_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_3_s1_R2.fastq.gz to zr2096_3_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_3_s1_R2.fastq.gz to zr2096_3_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_3_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tTNTTTTAAAATAATTTAAATAATTTTATATAAATNNNNNNNNNNNNNNNTATTTAAATTAAAAATTTAAATAAATTTTTTTAATTATTTATTGTTTAATG\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAAATAAACATTAAACAATAAATAATTAAAAAAATTTATTTAAATTTTTAATTTAAATAAATTAAAAATATTAAATTTATATAAAATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tCNCTTCAAAATAATTTAAATAATCTTATATAAATNNNNNNNNNNNNNNNCACCTAAACTAAAAACTCAAATAAACTTTTCTAATCACCTATCATCCAACA\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAGATAGATGTTGGATGATAGGTGATTAGAAAAGTTTATTTGAGTTTTTAGTTTAGGTGAGTTAAAAATATTGAATTTATATAAGATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tCNCTTCAAAATAATTTAAATAATCTTATATAAATNNNNNNNNNNNNNNNCACCTAAACTAAAAACTCAAATAAACTTTTCTAATCACCTATCATCCAACA\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAGATAGATGTTGGATGATAGGTGATTAGAAAAGTTTATTTGAGTTTTTAGTTTAGGTGAGTTAAAAATATTGAATTTATATAAGATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tTNTTTTAAAATAATTTAAATAATTTTATATAAATNNNNNNNNNNNNNNNTATTTAAATTAAAAATTTAAATAAATTTTTTTAATTATTTATTGTTTAATG\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAAATAAACATTAAACAATAAATAATTAAAAAAATTTATTTAAATTTTTAATTTAAATAAATTAAAAATATTAAATTTATATAAAATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_3_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69474 (69.47%) aligned concordantly 0 times\n",
      "    12342 (12.34%) aligned concordantly exactly 1 time\n",
      "    18184 (18.18%) aligned concordantly >1 times\n",
      "30.53% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69093 (69.09%) aligned concordantly 0 times\n",
      "    12247 (12.25%) aligned concordantly exactly 1 time\n",
      "    18660 (18.66%) aligned concordantly >1 times\n",
      "30.91% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69028 (69.03%) aligned concordantly 0 times\n",
      "    12397 (12.40%) aligned concordantly exactly 1 time\n",
      "    18575 (18.57%) aligned concordantly >1 times\n",
      "30.97% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69406 (69.41%) aligned concordantly 0 times\n",
      "    12367 (12.37%) aligned concordantly exactly 1 time\n",
      "    18227 (18.23%) aligned concordantly >1 times\n",
      "30.59% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_3_s1_R1.fastq.gz_C_to_T.fastq, zr2096_3_s1_R1.fastq.gz_G_to_A.fastq, zr2096_3_s1_R2.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1911141\n",
      "\n",
      "Total methylated C's in CpG context:\t212856\n",
      "Total methylated C's in CHG context:\t13659\n",
      "Total methylated C's in CHH context:\t45545\n",
      "Total methylated C's in Unknown context:\t6337\n",
      "\n",
      "Total unmethylated C's in CpG context:\t65965\n",
      "Total unmethylated C's in CHG context:\t426703\n",
      "Total unmethylated C's in CHH context:\t1146413\n",
      "Total unmethylated C's in Unknown context:\t27502\n",
      "\n",
      "C methylated in CpG context:\t76.3%\n",
      "C methylated in CHG context:\t3.1%\n",
      "C methylated in CHH context:\t3.8%\n",
      "C methylated in unknown context (CN or CHN):\t18.7%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 39s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_4_s1_R1.fastq.gz to zr2096_4_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_4_s1_R1.fastq.gz to zr2096_4_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_4_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_4_s1_R2.fastq.gz to zr2096_4_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_4_s1_R2.fastq.gz to zr2096_4_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_4_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNGATTAGTGTATAGTGGTTGTTATGGAAATTAGNNNNNNNNNNNNNNNNNNNNNNNNNTGNANTNTTTTTATTAGAATTGTTAAGGTTTATTTGATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTCCATCATTAATTTTTAAAAAAAAAAAACCATCCTTAATATCAATTTTATAAATCAAATAAACCTTAACAATTCTAATAAAAACACTTCATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATTAATATATAATAATTATTATAAAAATTAANNNNNNNNNNNNNNNNNNNNNNNNNTANANTNTTTTTATTAAAATTATTAAAATTTATTTAATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTTTATTATTAATTTTTAAAAAAAAAAAATTATTTTTAATATTAATTTTATAAATTAAATAAATTTTAATAATTTTAATAAAAATATTTTATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATTAATATATAATAATTATTATAAAAATTAANNNNNNNNNNNNNNNNNNNNNNNNNTANANTNTTTTTATTAAAATTATTAAAATTTATTTAATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTTTATTATTAATTTTTAAAAAAAAAAAATTATTTTTAATATTAATTTTATAAATTAAATAAATTTTAATAATTTTAATAAAAATATTTTATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNGATTAGTGTATAGTGGTTGTTATGGAAATTAGNNNNNNNNNNNNNNNNNNNNNNNNNTGNANTNTTTTTATTAGAATTGTTAAGGTTTATTTGATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTCCATCATTAATTTTTAAAAAAAAAAAACCATCCTTAATATCAATTTTATAAATCAAATAAACCTTAACAATTCTAATAAAAACACTTCATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_4_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69215 (69.22%) aligned concordantly 0 times\n",
      "    12964 (12.96%) aligned concordantly exactly 1 time\n",
      "    17821 (17.82%) aligned concordantly >1 times\n",
      "30.79% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    68104 (68.10%) aligned concordantly 0 times\n",
      "    13392 (13.39%) aligned concordantly exactly 1 time\n",
      "    18504 (18.50%) aligned concordantly >1 times\n",
      "31.90% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    67910 (67.91%) aligned concordantly 0 times\n",
      "    13504 (13.50%) aligned concordantly exactly 1 time\n",
      "    18586 (18.59%) aligned concordantly >1 times\n",
      "32.09% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69042 (69.04%) aligned concordantly 0 times\n",
      "    13009 (13.01%) aligned concordantly exactly 1 time\n",
      "    17949 (17.95%) aligned concordantly >1 times\n",
      "30.96% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_4_s1_R1.fastq.gz_C_to_T.fastq, zr2096_4_s1_R1.fastq.gz_G_to_A.fastq, zr2096_4_s1_R2.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1807416\n",
      "\n",
      "Total methylated C's in CpG context:\t156286\n",
      "Total methylated C's in CHG context:\t12428\n",
      "Total methylated C's in CHH context:\t49328\n",
      "Total methylated C's in Unknown context:\t6149\n",
      "\n",
      "Total unmethylated C's in CpG context:\t82257\n",
      "Total unmethylated C's in CHG context:\t398925\n",
      "Total unmethylated C's in CHH context:\t1108192\n",
      "Total unmethylated C's in Unknown context:\t27246\n",
      "\n",
      "C methylated in CpG context:\t65.5%\n",
      "C methylated in CHG context:\t3.0%\n",
      "C methylated in CHH context:\t4.3%\n",
      "C methylated in unknown context (CN or CHN):\t18.4%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 59s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_5_s1_R1.fastq.gz to zr2096_5_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_5_s1_R1.fastq.gz to zr2096_5_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_5_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_5_s1_R2.fastq.gz to zr2096_5_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_5_s1_R2.fastq.gz to zr2096_5_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_5_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATAAAATATTATTTAAATTTTATTATTTATANNNNNNNNNNNNNNNNNNATTNTTTTTATTATATATATAAAATTATTTTAAAATTATTATATTTTTT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTCAAATATATTAAAAAAAAATTATTAATAATTTTCATTAAAAAATATAATAATTTTAAAATAATTTTATATATATAATAAAAAAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAACAAAACACCACTCAAATCTTACTATCCACANNNNNNNNNNNNNNNNNNATTNCTTCTATCATATATATAAAATCATTCCAAAATTACTATATTTTTC\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTTGAGTATATTGAAGAGGAATTATTGATAGTTTTTGTTGAAAAATATAGTAATTTTGGAATGATTTTATATATATGATAGAAGAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAACAAAACACCACTCAAATCTTACTATCCACANNNNNNNNNNNNNNNNNNATTNCTTCTATCATATATATAAAATCATTCCAAAATTACTATATTTTTC\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTTGAGTATATTGAAGAGGAATTATTGATAGTTTTTGTTGAAAAATATAGTAATTTTGGAATGATTTTATATATATGATAGAAGAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATAAAATATTATTTAAATTTTATTATTTATANNNNNNNNNNNNNNNNNNATTNTTTTTATTATATATATAAAATTATTTTAAAATTATTATATTTTTT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTCAAATATATTAAAAAAAAATTATTAATAATTTTCATTAAAAAATATAATAATTTTAAAATAATTTTATATATATAATAAAAAAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_5_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    71937 (71.94%) aligned concordantly 0 times\n",
      "    12071 (12.07%) aligned concordantly exactly 1 time\n",
      "    15992 (15.99%) aligned concordantly >1 times\n",
      "28.06% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    71853 (71.85%) aligned concordantly 0 times\n",
      "    12016 (12.02%) aligned concordantly exactly 1 time\n",
      "    16131 (16.13%) aligned concordantly >1 times\n",
      "28.15% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70840 (70.84%) aligned concordantly 0 times\n",
      "    12296 (12.30%) aligned concordantly exactly 1 time\n",
      "    16864 (16.86%) aligned concordantly >1 times\n",
      "29.16% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70617 (70.62%) aligned concordantly 0 times\n",
      "    12464 (12.46%) aligned concordantly exactly 1 time\n",
      "    16919 (16.92%) aligned concordantly >1 times\n",
      "29.38% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_5_s1_R1.fastq.gz_C_to_T.fastq, zr2096_5_s1_R1.fastq.gz_G_to_A.fastq, zr2096_5_s1_R2.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1792976\n",
      "\n",
      "Total methylated C's in CpG context:\t154384\n",
      "Total methylated C's in CHG context:\t12834\n",
      "Total methylated C's in CHH context:\t47835\n",
      "Total methylated C's in Unknown context:\t6577\n",
      "\n",
      "Total unmethylated C's in CpG context:\t75691\n",
      "Total unmethylated C's in CHG context:\t383354\n",
      "Total unmethylated C's in CHH context:\t1118878\n",
      "Total unmethylated C's in Unknown context:\t27168\n",
      "\n",
      "C methylated in CpG context:\t67.1%\n",
      "C methylated in CHG context:\t3.2%\n",
      "C methylated in CHH context:\t4.1%\n",
      "C methylated in unknown context (CN or CHN):\t19.5%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 31s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_6_s1_R1.fastq.gz to zr2096_6_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_6_s1_R1.fastq.gz to zr2096_6_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_6_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_6_s1_R2.fastq.gz to zr2096_6_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_6_s1_R2.fastq.gz to zr2096_6_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_6_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTTATTAAATTTAATTTTATTATTAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNTANANAAATATTAATTAAATATTTTTAAAATAATAAATTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAAAATTTTTTTAAATTTATTATTTTAAAAATATTTAATTAATATTTATTTATATTTTATAATTAACAATAATTATTTTTTTATTTAATAATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTCATCAAATCCAACTTTATCACCAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNCANANAAACATCAATCAAACATCTCTAAAACAATAAACTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAGAATTTTTTTAAGTTTATTGTTTTAGAGATGTTTGATTGATGTTTATTTGTATTTTGTAATTAGTGGTGGTTGTTTTTTTATTTGGTGATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTCATCAAATCCAACTTTATCACCAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNCANANAAACATCAATCAAACATCTCTAAAACAATAAACTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAGAATTTTTTTAAGTTTATTGTTTTAGAGATGTTTGATTGATGTTTATTTGTATTTTGTAATTAGTGGTGGTTGTTTTTTTATTTGGTGATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTTATTAAATTTAATTTTATTATTAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNTANANAAATATTAATTAAATATTTTTAAAATAATAAATTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAAAATTTTTTTAAATTTATTATTTTAAAAATATTTAATTAATATTTATTTATATTTTATAATTAACAATAATTATTTTTTTATTTAATAATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_6_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69708 (69.71%) aligned concordantly 0 times\n",
      "    12666 (12.67%) aligned concordantly exactly 1 time\n",
      "    17626 (17.63%) aligned concordantly >1 times\n",
      "30.29% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    67544 (67.54%) aligned concordantly 0 times\n",
      "    13694 (13.69%) aligned concordantly exactly 1 time\n",
      "    18762 (18.76%) aligned concordantly >1 times\n",
      "32.46% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69721 (69.72%) aligned concordantly 0 times\n",
      "    12698 (12.70%) aligned concordantly exactly 1 time\n",
      "    17581 (17.58%) aligned concordantly >1 times\n",
      "30.28% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    67375 (67.38%) aligned concordantly 0 times\n",
      "    13613 (13.61%) aligned concordantly exactly 1 time\n",
      "    19012 (19.01%) aligned concordantly >1 times\n",
      "32.62% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_6_s1_R1.fastq.gz_C_to_T.fastq, zr2096_6_s1_R1.fastq.gz_G_to_A.fastq, zr2096_6_s1_R2.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1839385\n",
      "\n",
      "Total methylated C's in CpG context:\t178767\n",
      "Total methylated C's in CHG context:\t13324\n",
      "Total methylated C's in CHH context:\t50010\n",
      "Total methylated C's in Unknown context:\t6495\n",
      "\n",
      "Total unmethylated C's in CpG context:\t66888\n",
      "Total unmethylated C's in CHG context:\t402617\n",
      "Total unmethylated C's in CHH context:\t1127779\n",
      "Total unmethylated C's in Unknown context:\t28732\n",
      "\n",
      "C methylated in CpG context:\t72.8%\n",
      "C methylated in CHG context:\t3.2%\n",
      "C methylated in CHH context:\t4.2%\n",
      "C methylated in unknown context (CN or CHN):\t18.4%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 42s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_7_s1_R1.fastq.gz to zr2096_7_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_7_s1_R1.fastq.gz to zr2096_7_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_7_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_7_s1_R2.fastq.gz to zr2096_7_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_7_s1_R2.fastq.gz to zr2096_7_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_7_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAAAAAATGATTATTATGATGAATTTAAATATTAATAATAAAATTTAAAAAAAATATAAATGTTAAATATTAAATTTTTTAAAAAATGATTATGG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t141\t*\t0\t0\t*\t*\t0\t0\tCCATAATCATTTTTTAAAAAATTTAATATTTAACATTTATATTTTTTTTAAATTTTATTATTAATATTTAAATTCATCATAATAATCATTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t99\tNC_035781.1_GA_converted\t24539746\t6\t95M\t=\t24539746\t-95\tAAAAAAACAACTACTACAACAAATTCAAATATTAATAACAAAATTTAAAAAAAACATAAACATTAAATATCAAATTCTTCAAAAAACAATTACAA\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-12\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:94T0\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t147\tNC_035781.1_GA_converted\t24539746\t6\t95M\t=\t24539746\t-95\tAAAAAAACAACTACTACAACAAATTCAAATATTAATAACAAAATTTAAAAAAAACATAAACATTAAATATCAAATTCTTCAAAAAACAATTACAA\tFFFFFFFFFFFFFFFFFFFFFFFBBFFFFFFBBFF/FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-6\tXS:i:-12\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:94T0\tYS:i:-6\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t83\tNC_035780.1_CT_converted\t42548681\t17\t95M\t=\t42548681\t-95\tTTGTAATTGTTTTTTGAAGAATTTGATATTTAATGTTTATGTTTTTTTTAAATTTTGTTATTAATATTTGAATTTGTTGTAGTAGTTGTTTTTTT\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-6\tXS:i:-32\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:0A94\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t163\tNC_035780.1_CT_converted\t42548681\t17\t95M\t=\t42548681\t-95\tTTGTAATTGTTTTTTGAAGAATTTGATATTTAATGTTTATGTTTTTTTTAAATTTTGTTATTAATATTTGAATTTGTTGTAGTAGTTGTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-32\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:0A94\tYS:i:-6\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAAAAAATGATTATTATGATGAATTTAAATATTAATAATAAAATTTAAAAAAAATATAAATGTTAAATATTAAATTTTTTAAAAAATGATTATGG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t141\t*\t0\t0\t*\t*\t0\t0\tCCATAATCATTTTTTAAAAAATTTAATATTTAACATTTATATTTTTTTTAAATTTTATTATTAATATTTAAATTCATCATAATAATCATTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_7_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    71530 (71.53%) aligned concordantly 0 times\n",
      "    12232 (12.23%) aligned concordantly exactly 1 time\n",
      "    16238 (16.24%) aligned concordantly >1 times\n",
      "28.47% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    72232 (72.23%) aligned concordantly 0 times\n",
      "    11838 (11.84%) aligned concordantly exactly 1 time\n",
      "    15930 (15.93%) aligned concordantly >1 times\n",
      "27.77% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    71562 (71.56%) aligned concordantly 0 times\n",
      "    12285 (12.29%) aligned concordantly exactly 1 time\n",
      "    16153 (16.15%) aligned concordantly >1 times\n",
      "28.44% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    72175 (72.17%) aligned concordantly 0 times\n",
      "    11787 (11.79%) aligned concordantly exactly 1 time\n",
      "    16038 (16.04%) aligned concordantly >1 times\n",
      "27.82% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_7_s1_R1.fastq.gz_C_to_T.fastq, zr2096_7_s1_R1.fastq.gz_G_to_A.fastq, zr2096_7_s1_R2.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1807058\n",
      "\n",
      "Total methylated C's in CpG context:\t165861\n",
      "Total methylated C's in CHG context:\t13399\n",
      "Total methylated C's in CHH context:\t47784\n",
      "Total methylated C's in Unknown context:\t6056\n",
      "\n",
      "Total unmethylated C's in CpG context:\t67849\n",
      "Total unmethylated C's in CHG context:\t393633\n",
      "Total unmethylated C's in CHH context:\t1118532\n",
      "Total unmethylated C's in Unknown context:\t26732\n",
      "\n",
      "C methylated in CpG context:\t71.0%\n",
      "C methylated in CHG context:\t3.3%\n",
      "C methylated in CHH context:\t4.1%\n",
      "C methylated in unknown context (CN or CHN):\t18.5%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 5m 25s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_8_s1_R1.fastq.gz to zr2096_8_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_8_s1_R1.fastq.gz to zr2096_8_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_8_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_8_s1_R2.fastq.gz to zr2096_8_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_8_s1_R2.fastq.gz to zr2096_8_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_8_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATTTAAAATTAAAAAATTTATTAAATTTATTATTATTTNTTATTTTTTATAAAAGAAAATAGAGAGAGAGAGAGAGAGAGAGAGAG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tCTCTCTCTCTCTCTCTCTCTCTCTCTATTTTCTTTTATAAAAAATAATAAATAATAATAAATTTAATAAATTTTTTAATTTTAAATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATCCAAAATTAAAAAACTCATTAAATCTATCATCATTCNCCATTCTTCACAAAAAAAAATAAAAAAAAAAAAAAAAAAAAAAAAAA\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTTTTTTTTTTTTTTTTTTTTTATTTTTTTTTGTGAAGAATGGTGAATGATGATAGATTTAATGAGTTTTTTAATTTTGGATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t83\tNC_007175.2_CT_converted\t14567\t0\t2M17I69M\t=\t14567\t-71\tTTTTTTTTTTTTTTTTTTTTTTTTTTATTTTTTTTTGTGAAGAATGGNGAATGATGATAGATTTAATGAGTTTTTTAATTTTGGATTA\tFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<<#FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-81\tXN:i:0\tXM:i:5\tXO:i:1\tXG:i:17\tNM:i:22\tMD:Z:1A5A0G5A15T40\tYS:i:-80\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t163\tNC_007175.2_CT_converted\t14567\t0\t2M17I69M\t=\t14567\t-71\tTTTTTTTTTTTTTTTTTTTTTTTTTTATTTTTTTTTGTGAAGAATGGTGAATGATGATAGATTTAATGAGTTTTTTAATTTTGGATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tAS:i:-80\tXN:i:0\tXM:i:4\tXO:i:1\tXG:i:17\tNM:i:21\tMD:Z:1A5A0G5A56\tYS:i:-81\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATTTAAAATTAAAAAATTTATTAAATTTATTATTATTTNTTATTTTTTATAAAAGAAAATAGAGAGAGAGAGAGAGAGAGAGAGAG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tCTCTCTCTCTCTCTCTCTCTCTCTCTATTTTCTTTTATAAAAAATAATAAATAATAATAAATTTAATAAATTTTTTAATTTTAAATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_8_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    74157 (74.16%) aligned concordantly 0 times\n",
      "    11312 (11.31%) aligned concordantly exactly 1 time\n",
      "    14531 (14.53%) aligned concordantly >1 times\n",
      "25.84% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    74306 (74.31%) aligned concordantly 0 times\n",
      "    11103 (11.10%) aligned concordantly exactly 1 time\n",
      "    14591 (14.59%) aligned concordantly >1 times\n",
      "25.69% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    74133 (74.13%) aligned concordantly 0 times\n",
      "    11331 (11.33%) aligned concordantly exactly 1 time\n",
      "    14536 (14.54%) aligned concordantly >1 times\n",
      "25.87% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    74552 (74.55%) aligned concordantly 0 times\n",
      "    11116 (11.12%) aligned concordantly exactly 1 time\n",
      "    14332 (14.33%) aligned concordantly >1 times\n",
      "25.45% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_8_s1_R1.fastq.gz_C_to_T.fastq, zr2096_8_s1_R1.fastq.gz_G_to_A.fastq, zr2096_8_s1_R2.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1598278\n",
      "\n",
      "Total methylated C's in CpG context:\t130605\n",
      "Total methylated C's in CHG context:\t11991\n",
      "Total methylated C's in CHH context:\t48523\n",
      "Total methylated C's in Unknown context:\t6038\n",
      "\n",
      "Total unmethylated C's in CpG context:\t67102\n",
      "Total unmethylated C's in CHG context:\t348741\n",
      "Total unmethylated C's in CHH context:\t991316\n",
      "Total unmethylated C's in Unknown context:\t24212\n",
      "\n",
      "C methylated in CpG context:\t66.1%\n",
      "C methylated in CHG context:\t3.3%\n",
      "C methylated in CHH context:\t4.7%\n",
      "C methylated in unknown context (CN or CHN):\t20.0%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 5m 14s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_9_s1_R1.fastq.gz to zr2096_9_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_9_s1_R1.fastq.gz to zr2096_9_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_9_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_9_s1_R2.fastq.gz to zr2096_9_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_9_s1_R2.fastq.gz to zr2096_9_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_9_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNATTAAATAAAAAATAATAAATAATAAATTANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAGG\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAAAAAATAAAAAACACATTAAAAATCTATAATTTAATTTATTATTTATTATTTTTTATTTAATTAAATCNNNNAAACATCATATAAAAAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNACCAAATAAAAAATAACAAATAATAAATCANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAAA\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAGAGGATAGGAGATATGTTAGGAATTTATAATTTGATTTATTATTTGTTATTTTTTATTTGGTTAGATTNNNNGAGTGTTGTGTAGGGAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNACCAAATAAAAAATAACAAATAATAAATCANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAAA\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAGAGGATAGGAGATATGTTAGGAATTTATAATTTGATTTATTATTTGTTATTTTTTATTTGGTTAGATTNNNNGAGTGTTGTGTAGGGAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNATTAAATAAAAAATAATAAATAATAAATTANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAGG\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAAAAAATAAAAAACACATTAAAAATCTATAATTTAATTTATTATTTATTATTTTTTATTTAATTAAATCNNNNAAACATCATATAAAAAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_9_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    73508 (73.51%) aligned concordantly 0 times\n",
      "    11236 (11.24%) aligned concordantly exactly 1 time\n",
      "    15256 (15.26%) aligned concordantly >1 times\n",
      "26.49% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    73787 (73.79%) aligned concordantly 0 times\n",
      "    11232 (11.23%) aligned concordantly exactly 1 time\n",
      "    14981 (14.98%) aligned concordantly >1 times\n",
      "26.21% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70419 (70.42%) aligned concordantly 0 times\n",
      "    12573 (12.57%) aligned concordantly exactly 1 time\n",
      "    17008 (17.01%) aligned concordantly >1 times\n",
      "29.58% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70201 (70.20%) aligned concordantly 0 times\n",
      "    12723 (12.72%) aligned concordantly exactly 1 time\n",
      "    17076 (17.08%) aligned concordantly >1 times\n",
      "29.80% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_9_s1_R1.fastq.gz_C_to_T.fastq, zr2096_9_s1_R1.fastq.gz_G_to_A.fastq, zr2096_9_s1_R2.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1849983\n",
      "\n",
      "Total methylated C's in CpG context:\t199908\n",
      "Total methylated C's in CHG context:\t14694\n",
      "Total methylated C's in CHH context:\t48531\n",
      "Total methylated C's in Unknown context:\t6764\n",
      "\n",
      "Total unmethylated C's in CpG context:\t63719\n",
      "Total unmethylated C's in CHG context:\t409648\n",
      "Total unmethylated C's in CHH context:\t1113483\n",
      "Total unmethylated C's in Unknown context:\t28006\n",
      "\n",
      "C methylated in CpG context:\t75.8%\n",
      "C methylated in CHG context:\t3.5%\n",
      "C methylated in CHH context:\t4.2%\n",
      "C methylated in unknown context (CN or CHN):\t19.5%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 5m 54s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "find /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_*R1* \\\n",
    "| xargs basename -s _s1_R1.fastq.gz | xargs -I{} /gscratch/srlab/programs/Bismark-0.19.0/bismark \\\n",
    "--genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ \\\n",
    "--path_to_bowtie /gscratch/srlab/programs/bowtie2-2.3.4.1-linux-x86_64\n",
    "--score_min L,0,-1.2 \\\n",
    "-I 100 \\\n",
    "--non_directional \\\n",
    "-1 /Volumes/web/seashell/bu-serine-wd/18-04-07/{}_s1_R1.fastq.gz \\\n",
    "-2 /Volumes/web/seashell/bu-serine-wd/18-04-07/{}_s1_R2.fastq.gz \\\n",
    "-o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt7/ \\\n",
    "--samtools_path /gscratch/srlab/programs/samtools-1.9/\n",
    "\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#run deduplication"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_10_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_1_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_2_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_3_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_4_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_5_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_6_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_7_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_8_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_9_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Processing paired-end Bismark output file(s) (SAM format):\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_10_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_1_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_2_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_3_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_4_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_5_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_6_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_7_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_8_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_9_s1_R1_bismark_bt2_pe.bam\n",
      "\n",
      "\n",
      "If there are several alignments to a single position in the genome the first alignment will be chosen. Since the input files are not in any way sorted this is a near-enough random selection of reads.\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_10_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_10_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_10_s1_R1_bismark_bt2_pe.bam:\t55304\n",
      "Total number duplicated alignments removed:\t22 (0.04%)\n",
      "Duplicated alignments were found at:\t21 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 55282 (99.96% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_1_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_1_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_1_s1_R1_bismark_bt2_pe.bam:\t25623\n",
      "Total number duplicated alignments removed:\t73 (0.28%)\n",
      "Duplicated alignments were found at:\t61 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 25550 (99.72% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_2_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_2_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_2_s1_R1_bismark_bt2_pe.bam:\t50484\n",
      "Total number duplicated alignments removed:\t59 (0.12%)\n",
      "Duplicated alignments were found at:\t42 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 50425 (99.88% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_3_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_3_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_3_s1_R1_bismark_bt2_pe.bam:\t55171\n",
      "Total number duplicated alignments removed:\t16 (0.03%)\n",
      "Duplicated alignments were found at:\t16 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 55155 (99.97% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_4_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_4_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_4_s1_R1_bismark_bt2_pe.bam:\t56412\n",
      "Total number duplicated alignments removed:\t14 (0.02%)\n",
      "Duplicated alignments were found at:\t13 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 56398 (99.98% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_5_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_5_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_5_s1_R1_bismark_bt2_pe.bam:\t53779\n",
      "Total number duplicated alignments removed:\t20 (0.04%)\n",
      "Duplicated alignments were found at:\t20 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 53759 (99.96% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_6_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_6_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_6_s1_R1_bismark_bt2_pe.bam:\t56844\n",
      "Total number duplicated alignments removed:\t12 (0.02%)\n",
      "Duplicated alignments were found at:\t12 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 56832 (99.98% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_7_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_7_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_7_s1_R1_bismark_bt2_pe.bam:\t53463\n",
      "Total number duplicated alignments removed:\t14 (0.03%)\n",
      "Duplicated alignments were found at:\t14 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 53449 (99.97% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_8_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_8_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_8_s1_R1_bismark_bt2_pe.bam:\t48884\n",
      "Total number duplicated alignments removed:\t27 (0.06%)\n",
      "Duplicated alignments were found at:\t24 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 48857 (99.94% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_9_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_9_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_9_s1_R1_bismark_bt2_pe.bam:\t52449\n",
      "Total number duplicated alignments removed:\t35 (0.07%)\n",
      "Duplicated alignments were found at:\t35 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 52414 (99.93% of total)\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "/gscratch/srlab/programs/Bismark-0.19.0/deduplicate_bismark \\\n",
    "--bam -p \\\n",
    "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt7/*.bam \\\n",
    "-o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt7/ \\\n",
    "2> /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt7/dedup.err \\\n",
    "--samtools_path /gscratch/srlab/programs/samtools-1.9/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#create summary report of mapping efficiency and dedup. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt7/*PE_report.txt | \\\n",
    "grep 'Mapping\\ efficiency\\:' | \\\n",
    "cat - /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt7/*.deduplication_report.txt > /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt7/zr2096_mapping_dedup_summary.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#compiling bam files and sorting"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "find /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt7/*deduplicated.bam| \\\n",
    "xargs basename -s _s1_R1_bismark_bt2_pe.deduplicated.bam | \\\n",
    "xargs -I{} samtools \\\n",
    "sort /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt7/{}_s1_R1_bismark_bt2_pe.deduplicated.bam \\ \n",
    "-o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt7/{}_dedup.sorted.bam\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat /Volumes/web/metacarcinus/Cvirginica/20181004/*PE_report.txt | \\\n",
    "grep 'Mapping\\ efficiency\\:' | \\\n",
    "cat - /Volumes/web/metacarcinus/Cvirginica/20181004/*.deduplication_report.txt | \\\n",
    "grep 'Mapping\\ efficiency\\:\\|removed' > \\\n",
    "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_mapping_dedup_summary.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "#sort dedup files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[bam_sort_core] merging from 6 files and 1 in-memory blocks...\n",
      "[bam_sort_core] merging from 4 files and 1 in-memory blocks...\n",
      "[bam_sort_core] merging from 8 files and 1 in-memory blocks...\n",
      "[bam_sort_core] merging from 10 files and 1 in-memory blocks...\n",
      "[bam_sort_core] merging from 8 files and 1 in-memory blocks...\n",
      "[bam_sort_core] merging from 11 files and 1 in-memory blocks...\n",
      "[bam_sort_core] merging from 8 files and 1 in-memory blocks...\n",
      "[bam_sort_core] merging from 11 files and 1 in-memory blocks...\n",
      "[bam_sort_core] merging from 9 files and 1 in-memory blocks...\n",
      "[bam_sort_core] merging from 10 files and 1 in-memory blocks...\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "find /Volumes/web/metacarcinus/Cvirginica/20181004/*deduplicated.bam| \\\n",
    "xargs basename -s _s1_R1_bismark_bt2_pe.deduplicated.bam | xargs -I{} /Users/strigg/anaconda3/bin/samtools \\\n",
    "sort /Volumes/web/metacarcinus/Cvirginica/20181004/{}_s1_R1_bismark_bt2_pe.deduplicated.bam \\\n",
    "-o /Volumes/web/metacarcinus/Cvirginica/20181004/{}_dedup.sorted.bam\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "## create methylation extractor files for estimating coverage. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Now testing Bismark result file >/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_10_s1_R1_bismark_bt2_pe.deduplicated.bam< for positional sorting (which would be bad...)\t/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_10_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.1\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_10_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.2\n",
      "\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_10_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.1.mbias\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_10_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.2.mbias\n",
      "\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_OT_zr2096_10_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_CTOT_zr2096_10_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_CTOB_zr2096_10_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_OB_zr2096_10_s1_R1_bismark_bt2_pe.deduplicated.txt\n",
      "\n",
      "\n",
      "Now testing Bismark result file >/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_1_s1_R1_bismark_bt2_pe.deduplicated.bam< for positional sorting (which would be bad...)\t/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_1_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.1\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_1_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.2\n",
      "\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_1_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.1.mbias\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_1_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.2.mbias\n",
      "\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_OT_zr2096_1_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_CTOT_zr2096_1_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_CTOB_zr2096_1_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_OB_zr2096_1_s1_R1_bismark_bt2_pe.deduplicated.txt\n",
      "\n",
      "\n",
      "Now testing Bismark result file >/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_2_s1_R1_bismark_bt2_pe.deduplicated.bam< for positional sorting (which would be bad...)\t/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_2_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.1\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_2_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.2\n",
      "\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_2_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.1.mbias\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_2_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.2.mbias\n",
      "\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_OT_zr2096_2_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_CTOT_zr2096_2_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_CTOB_zr2096_2_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_OB_zr2096_2_s1_R1_bismark_bt2_pe.deduplicated.txt\n",
      "\n",
      "\n",
      "Now testing Bismark result file >/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_3_s1_R1_bismark_bt2_pe.deduplicated.bam< for positional sorting (which would be bad...)\t/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_3_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.1\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_3_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.2\n",
      "\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_3_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.1.mbias\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_3_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.2.mbias\n",
      "\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_OT_zr2096_3_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_CTOT_zr2096_3_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_CTOB_zr2096_3_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_OB_zr2096_3_s1_R1_bismark_bt2_pe.deduplicated.txt\n",
      "\n",
      "\n",
      "Now testing Bismark result file >/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_4_s1_R1_bismark_bt2_pe.deduplicated.bam< for positional sorting (which would be bad...)\t/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_4_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.1\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_4_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.2\n",
      "\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_4_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.1.mbias\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_4_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.2.mbias\n",
      "\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_OT_zr2096_4_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_CTOT_zr2096_4_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_CTOB_zr2096_4_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_OB_zr2096_4_s1_R1_bismark_bt2_pe.deduplicated.txt\n",
      "\n",
      "\n",
      "Now testing Bismark result file >/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_5_s1_R1_bismark_bt2_pe.deduplicated.bam< for positional sorting (which would be bad...)\t/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_5_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.1\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_5_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.2\n",
      "\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_5_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.1.mbias\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_5_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.2.mbias\n",
      "\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_OT_zr2096_5_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_CTOT_zr2096_5_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_CTOB_zr2096_5_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_OB_zr2096_5_s1_R1_bismark_bt2_pe.deduplicated.txt\n",
      "\n",
      "\n",
      "Now testing Bismark result file >/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_6_s1_R1_bismark_bt2_pe.deduplicated.bam< for positional sorting (which would be bad...)\t/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_6_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.1\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_6_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.2\n",
      "\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_6_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.1.mbias\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_6_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.2.mbias\n",
      "\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_OT_zr2096_6_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_CTOT_zr2096_6_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_CTOB_zr2096_6_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_OB_zr2096_6_s1_R1_bismark_bt2_pe.deduplicated.txt\n",
      "\n",
      "\n",
      "Now testing Bismark result file >/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_7_s1_R1_bismark_bt2_pe.deduplicated.bam< for positional sorting (which would be bad...)\t/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_7_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.1\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_7_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.2\n",
      "\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_7_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.1.mbias\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_7_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.2.mbias\n",
      "\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_OT_zr2096_7_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_CTOT_zr2096_7_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_CTOB_zr2096_7_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_OB_zr2096_7_s1_R1_bismark_bt2_pe.deduplicated.txt\n",
      "\n",
      "\n",
      "Now testing Bismark result file >/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_8_s1_R1_bismark_bt2_pe.deduplicated.bam< for positional sorting (which would be bad...)\t/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_8_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.1\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_8_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.2\n",
      "\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_8_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.1.mbias\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_8_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.2.mbias\n",
      "\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_OT_zr2096_8_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_CTOT_zr2096_8_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_CTOB_zr2096_8_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_OB_zr2096_8_s1_R1_bismark_bt2_pe.deduplicated.txt\n",
      "\n",
      "\n",
      "Now testing Bismark result file >/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_9_s1_R1_bismark_bt2_pe.deduplicated.bam< for positional sorting (which would be bad...)\t/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_9_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.1\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_9_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.2\n",
      "\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_9_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.1.mbias\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/zr2096_9_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt.2.mbias\n",
      "\n",
      "/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_OT_zr2096_9_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_CTOT_zr2096_9_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_CTOB_zr2096_9_s1_R1_bismark_bt2_pe.deduplicated.txt\t/Volumes/web/metacarcinus/Cvirginica/20181004/CpG_OB_zr2096_9_s1_R1_bismark_bt2_pe.deduplicated.txt\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "/Users/strigg/anaconda3/bin/bismark_methylation_extractor \\\n",
    "--paired-end --bedGraph --counts --scaffolds \\\n",
    "--multicore 2 \\\n",
    "/Volumes/web/metacarcinus/Cvirginica/20181004/*deduplicated.bam \\\n",
    "-o /Volumes/web/metacarcinus/Cvirginica/20181004/ \\\n",
    "--samtools /Users/strigg/anaconda3/bin/samtools \\\n",
    "2> /Volumes/web/metacarcinus/Cvirginica/20181004/bme.err"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "#make individual reports before making whole summary"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Found 10 alignment reports in current directory. Now trying to figure out whether there are corresponding optional reports\n",
      "\n",
      "Writing Bismark HTML report to >> zr2096_10_s1_R1_bismark_bt2_PE_report.html <<\n",
      "\n",
      "Redundant argument in sprintf at /Users/strigg/anaconda3/bin/bismark2report line 170.\n",
      "==============================================================================================================\n",
      "Using the following alignment report:\t\t> zr2096_10_s1_R1_bismark_bt2_PE_report.txt <\n",
      "Processing alignment report zr2096_10_s1_R1_bismark_bt2_PE_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following deduplication report:\t> zr2096_10_s1_R1_bismark_bt2_pe.deduplication_report.txt <\n",
      "Processing deduplication report zr2096_10_s1_R1_bismark_bt2_pe.deduplication_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following splitting report:\t\t> zr2096_10_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt <\n",
      "Processing splitting report zr2096_10_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following M-bias report:\t\t> zr2096_10_s1_R1_bismark_bt2_pe.deduplicated.M-bias.txt <\n",
      "Processing M-bias report zr2096_10_s1_R1_bismark_bt2_pe.deduplicated.M-bias.txt ...\n",
      "Complete\n",
      "\n",
      "No nucleotide coverage report file specified, skipping this step\n",
      "==============================================================================================================\n",
      "\n",
      "\n",
      "\n",
      "Writing Bismark HTML report to >> zr2096_1_s1_R1_bismark_bt2_PE_report.html <<\n",
      "\n",
      "Redundant argument in sprintf at /Users/strigg/anaconda3/bin/bismark2report line 170.\n",
      "==============================================================================================================\n",
      "Using the following alignment report:\t\t> zr2096_1_s1_R1_bismark_bt2_PE_report.txt <\n",
      "Processing alignment report zr2096_1_s1_R1_bismark_bt2_PE_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following deduplication report:\t> zr2096_1_s1_R1_bismark_bt2_pe.deduplication_report.txt <\n",
      "Processing deduplication report zr2096_1_s1_R1_bismark_bt2_pe.deduplication_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following splitting report:\t\t> zr2096_1_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt <\n",
      "Processing splitting report zr2096_1_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following M-bias report:\t\t> zr2096_1_s1_R1_bismark_bt2_pe.deduplicated.M-bias.txt <\n",
      "Processing M-bias report zr2096_1_s1_R1_bismark_bt2_pe.deduplicated.M-bias.txt ...\n",
      "Complete\n",
      "\n",
      "No nucleotide coverage report file specified, skipping this step\n",
      "==============================================================================================================\n",
      "\n",
      "\n",
      "\n",
      "Writing Bismark HTML report to >> zr2096_2_s1_R1_bismark_bt2_PE_report.html <<\n",
      "\n",
      "Redundant argument in sprintf at /Users/strigg/anaconda3/bin/bismark2report line 170.\n",
      "==============================================================================================================\n",
      "Using the following alignment report:\t\t> zr2096_2_s1_R1_bismark_bt2_PE_report.txt <\n",
      "Processing alignment report zr2096_2_s1_R1_bismark_bt2_PE_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following deduplication report:\t> zr2096_2_s1_R1_bismark_bt2_pe.deduplication_report.txt <\n",
      "Processing deduplication report zr2096_2_s1_R1_bismark_bt2_pe.deduplication_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following splitting report:\t\t> zr2096_2_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt <\n",
      "Processing splitting report zr2096_2_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following M-bias report:\t\t> zr2096_2_s1_R1_bismark_bt2_pe.deduplicated.M-bias.txt <\n",
      "Processing M-bias report zr2096_2_s1_R1_bismark_bt2_pe.deduplicated.M-bias.txt ...\n",
      "Complete\n",
      "\n",
      "No nucleotide coverage report file specified, skipping this step\n",
      "==============================================================================================================\n",
      "\n",
      "\n",
      "\n",
      "Writing Bismark HTML report to >> zr2096_3_s1_R1_bismark_bt2_PE_report.html <<\n",
      "\n",
      "Redundant argument in sprintf at /Users/strigg/anaconda3/bin/bismark2report line 170.\n",
      "==============================================================================================================\n",
      "Using the following alignment report:\t\t> zr2096_3_s1_R1_bismark_bt2_PE_report.txt <\n",
      "Processing alignment report zr2096_3_s1_R1_bismark_bt2_PE_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following deduplication report:\t> zr2096_3_s1_R1_bismark_bt2_pe.deduplication_report.txt <\n",
      "Processing deduplication report zr2096_3_s1_R1_bismark_bt2_pe.deduplication_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following splitting report:\t\t> zr2096_3_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt <\n",
      "Processing splitting report zr2096_3_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following M-bias report:\t\t> zr2096_3_s1_R1_bismark_bt2_pe.deduplicated.M-bias.txt <\n",
      "Processing M-bias report zr2096_3_s1_R1_bismark_bt2_pe.deduplicated.M-bias.txt ...\n",
      "Complete\n",
      "\n",
      "No nucleotide coverage report file specified, skipping this step\n",
      "==============================================================================================================\n",
      "\n",
      "\n",
      "\n",
      "Writing Bismark HTML report to >> zr2096_4_s1_R1_bismark_bt2_PE_report.html <<\n",
      "\n",
      "Redundant argument in sprintf at /Users/strigg/anaconda3/bin/bismark2report line 170.\n",
      "==============================================================================================================\n",
      "Using the following alignment report:\t\t> zr2096_4_s1_R1_bismark_bt2_PE_report.txt <\n",
      "Processing alignment report zr2096_4_s1_R1_bismark_bt2_PE_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following deduplication report:\t> zr2096_4_s1_R1_bismark_bt2_pe.deduplication_report.txt <\n",
      "Processing deduplication report zr2096_4_s1_R1_bismark_bt2_pe.deduplication_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following splitting report:\t\t> zr2096_4_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt <\n",
      "Processing splitting report zr2096_4_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following M-bias report:\t\t> zr2096_4_s1_R1_bismark_bt2_pe.deduplicated.M-bias.txt <\n",
      "Processing M-bias report zr2096_4_s1_R1_bismark_bt2_pe.deduplicated.M-bias.txt ...\n",
      "Complete\n",
      "\n",
      "No nucleotide coverage report file specified, skipping this step\n",
      "==============================================================================================================\n",
      "\n",
      "\n",
      "\n",
      "Writing Bismark HTML report to >> zr2096_5_s1_R1_bismark_bt2_PE_report.html <<\n",
      "\n",
      "Redundant argument in sprintf at /Users/strigg/anaconda3/bin/bismark2report line 170.\n",
      "==============================================================================================================\n",
      "Using the following alignment report:\t\t> zr2096_5_s1_R1_bismark_bt2_PE_report.txt <\n",
      "Processing alignment report zr2096_5_s1_R1_bismark_bt2_PE_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following deduplication report:\t> zr2096_5_s1_R1_bismark_bt2_pe.deduplication_report.txt <\n",
      "Processing deduplication report zr2096_5_s1_R1_bismark_bt2_pe.deduplication_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following splitting report:\t\t> zr2096_5_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt <\n",
      "Processing splitting report zr2096_5_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following M-bias report:\t\t> zr2096_5_s1_R1_bismark_bt2_pe.deduplicated.M-bias.txt <\n",
      "Processing M-bias report zr2096_5_s1_R1_bismark_bt2_pe.deduplicated.M-bias.txt ...\n",
      "Complete\n",
      "\n",
      "No nucleotide coverage report file specified, skipping this step\n",
      "==============================================================================================================\n",
      "\n",
      "\n",
      "\n",
      "Writing Bismark HTML report to >> zr2096_6_s1_R1_bismark_bt2_PE_report.html <<\n",
      "\n",
      "Redundant argument in sprintf at /Users/strigg/anaconda3/bin/bismark2report line 170.\n",
      "==============================================================================================================\n",
      "Using the following alignment report:\t\t> zr2096_6_s1_R1_bismark_bt2_PE_report.txt <\n",
      "Processing alignment report zr2096_6_s1_R1_bismark_bt2_PE_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following deduplication report:\t> zr2096_6_s1_R1_bismark_bt2_pe.deduplication_report.txt <\n",
      "Processing deduplication report zr2096_6_s1_R1_bismark_bt2_pe.deduplication_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following splitting report:\t\t> zr2096_6_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt <\n",
      "Processing splitting report zr2096_6_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following M-bias report:\t\t> zr2096_6_s1_R1_bismark_bt2_pe.deduplicated.M-bias.txt <\n",
      "Processing M-bias report zr2096_6_s1_R1_bismark_bt2_pe.deduplicated.M-bias.txt ...\n",
      "Complete\n",
      "\n",
      "No nucleotide coverage report file specified, skipping this step\n",
      "==============================================================================================================\n",
      "\n",
      "\n",
      "\n",
      "Writing Bismark HTML report to >> zr2096_7_s1_R1_bismark_bt2_PE_report.html <<\n",
      "\n",
      "Redundant argument in sprintf at /Users/strigg/anaconda3/bin/bismark2report line 170.\n",
      "==============================================================================================================\n",
      "Using the following alignment report:\t\t> zr2096_7_s1_R1_bismark_bt2_PE_report.txt <\n",
      "Processing alignment report zr2096_7_s1_R1_bismark_bt2_PE_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following deduplication report:\t> zr2096_7_s1_R1_bismark_bt2_pe.deduplication_report.txt <\n",
      "Processing deduplication report zr2096_7_s1_R1_bismark_bt2_pe.deduplication_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following splitting report:\t\t> zr2096_7_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt <\n",
      "Processing splitting report zr2096_7_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following M-bias report:\t\t> zr2096_7_s1_R1_bismark_bt2_pe.deduplicated.M-bias.txt <\n",
      "Processing M-bias report zr2096_7_s1_R1_bismark_bt2_pe.deduplicated.M-bias.txt ...\n",
      "Complete\n",
      "\n",
      "No nucleotide coverage report file specified, skipping this step\n",
      "==============================================================================================================\n",
      "\n",
      "\n",
      "\n",
      "Writing Bismark HTML report to >> zr2096_8_s1_R1_bismark_bt2_PE_report.html <<\n",
      "\n",
      "Redundant argument in sprintf at /Users/strigg/anaconda3/bin/bismark2report line 170.\n",
      "==============================================================================================================\n",
      "Using the following alignment report:\t\t> zr2096_8_s1_R1_bismark_bt2_PE_report.txt <\n",
      "Processing alignment report zr2096_8_s1_R1_bismark_bt2_PE_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following deduplication report:\t> zr2096_8_s1_R1_bismark_bt2_pe.deduplication_report.txt <\n",
      "Processing deduplication report zr2096_8_s1_R1_bismark_bt2_pe.deduplication_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following splitting report:\t\t> zr2096_8_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt <\n",
      "Processing splitting report zr2096_8_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following M-bias report:\t\t> zr2096_8_s1_R1_bismark_bt2_pe.deduplicated.M-bias.txt <\n",
      "Processing M-bias report zr2096_8_s1_R1_bismark_bt2_pe.deduplicated.M-bias.txt ...\n",
      "Complete\n",
      "\n",
      "No nucleotide coverage report file specified, skipping this step\n",
      "==============================================================================================================\n",
      "\n",
      "\n",
      "\n",
      "Writing Bismark HTML report to >> zr2096_9_s1_R1_bismark_bt2_PE_report.html <<\n",
      "\n",
      "Redundant argument in sprintf at /Users/strigg/anaconda3/bin/bismark2report line 170.\n",
      "==============================================================================================================\n",
      "Using the following alignment report:\t\t> zr2096_9_s1_R1_bismark_bt2_PE_report.txt <\n",
      "Processing alignment report zr2096_9_s1_R1_bismark_bt2_PE_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following deduplication report:\t> zr2096_9_s1_R1_bismark_bt2_pe.deduplication_report.txt <\n",
      "Processing deduplication report zr2096_9_s1_R1_bismark_bt2_pe.deduplication_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following splitting report:\t\t> zr2096_9_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt <\n",
      "Processing splitting report zr2096_9_s1_R1_bismark_bt2_pe.deduplicated_splitting_report.txt ...\n",
      "Complete\n",
      "\n",
      "Using the following M-bias report:\t\t> zr2096_9_s1_R1_bismark_bt2_pe.deduplicated.M-bias.txt <\n",
      "Processing M-bias report zr2096_9_s1_R1_bismark_bt2_pe.deduplicated.M-bias.txt ...\n",
      "Complete\n",
      "\n",
      "No nucleotide coverage report file specified, skipping this step\n",
      "==============================================================================================================\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd /Volumes/web/metacarcinus/Cvirginica/20181004/\n",
    "/Users/strigg/anaconda3/bin/bismark2report"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "#make whole summary report"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "No Bismark/Bowtie2 single-end BAM files detected\n",
      "Found Bismark/Bowtie2 paired-end files\n",
      "No Bismark/Bowtie single-end BAM files detected\n",
      "No Bismark/Bowtie paired-end BAM files detected\n",
      "\n",
      "Generating Bismark summary report from 10 Bismark BAM file(s)...\n",
      ">> Reading from Bismark report: zr2096_10_s1_R1_bismark_bt2_PE_report.txt\n",
      ">> Reading from Bismark report: zr2096_1_s1_R1_bismark_bt2_PE_report.txt\n",
      ">> Reading from Bismark report: zr2096_2_s1_R1_bismark_bt2_PE_report.txt\n",
      ">> Reading from Bismark report: zr2096_3_s1_R1_bismark_bt2_PE_report.txt\n",
      ">> Reading from Bismark report: zr2096_4_s1_R1_bismark_bt2_PE_report.txt\n",
      ">> Reading from Bismark report: zr2096_5_s1_R1_bismark_bt2_PE_report.txt\n",
      ">> Reading from Bismark report: zr2096_6_s1_R1_bismark_bt2_PE_report.txt\n",
      ">> Reading from Bismark report: zr2096_7_s1_R1_bismark_bt2_PE_report.txt\n",
      ">> Reading from Bismark report: zr2096_8_s1_R1_bismark_bt2_PE_report.txt\n",
      ">> Reading from Bismark report: zr2096_9_s1_R1_bismark_bt2_PE_report.txt\n",
      "\n",
      "Wrote Bismark project summary to >> bismark_summary_report.html <<\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd /Volumes/web/metacarcinus/Cvirginica/20181004/\n",
    "/Users/strigg/anaconda3/bin/bismark2summary"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
