{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#Subsampling from each fastq file and aligning with Bismark"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t55304\n",
      "Mapping efficiency:\t55.3%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t27490\n",
      "Sequence pairs did not map uniquely:\t17206\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t13863\t((converted) top strand)\n",
      "GA/CT/CT:\t13771\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t13915\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t13755\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t25623\n",
      "Mapping efficiency:\t25.6%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t66914\n",
      "Sequence pairs did not map uniquely:\t7463\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t6395\t((converted) top strand)\n",
      "GA/CT/CT:\t6412\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t6401\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t6415\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t50484\n",
      "Mapping efficiency:\t50.5%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t33358\n",
      "Sequence pairs did not map uniquely:\t16158\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t12761\t((converted) top strand)\n",
      "GA/CT/CT:\t12571\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t12410\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t12742\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t55171\n",
      "Mapping efficiency:\t55.2%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t26540\n",
      "Sequence pairs did not map uniquely:\t18289\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t13872\t((converted) top strand)\n",
      "GA/CT/CT:\t13727\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t13678\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t13894\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t56412\n",
      "Mapping efficiency:\t56.4%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t25464\n",
      "Sequence pairs did not map uniquely:\t18124\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t14301\t((converted) top strand)\n",
      "GA/CT/CT:\t13881\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t13768\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t14462\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t53779\n",
      "Mapping efficiency:\t53.8%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t29695\n",
      "Sequence pairs did not map uniquely:\t16526\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t13871\t((converted) top strand)\n",
      "GA/CT/CT:\t13053\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t12995\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t13860\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t56844\n",
      "Mapping efficiency:\t56.8%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t25108\n",
      "Sequence pairs did not map uniquely:\t18048\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t14846\t((converted) top strand)\n",
      "GA/CT/CT:\t13445\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t13483\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t15070\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t53463\n",
      "Mapping efficiency:\t53.5%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t30401\n",
      "Sequence pairs did not map uniquely:\t16136\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t13576\t((converted) top strand)\n",
      "GA/CT/CT:\t12963\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t13213\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t13711\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t48884\n",
      "Mapping efficiency:\t48.9%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t36530\n",
      "Sequence pairs did not map uniquely:\t14586\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t12344\t((converted) top strand)\n",
      "GA/CT/CT:\t12125\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t12019\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t12396\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t52449\n",
      "Mapping efficiency:\t52.4%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t30963\n",
      "Sequence pairs did not map uniquely:\t16588\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t14037\t((converted) top strand)\n",
      "GA/CT/CT:\t12205\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t12162\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t14045\t((converted) bottom strand)\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_10_s1_R1.fastq.gz to zr2096_10_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_10_s1_R1.fastq.gz to zr2096_10_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_10_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_10_s1_R2.fastq.gz to zr2096_10_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_10_s1_R2.fastq.gz to zr2096_10_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_10_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tGTTATATATAATTGTATTAATTATAAAAAATATNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATTTATTATTAAAAATATTAAATATATTATTTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNATTTTATATTTTTAATAATAAATATAATTATATATTAAAATAAAATAATATATTTAATATTTTTAATAATAAATACAATTAAATATCATAATAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tACCATATACAATCATATCAACTATAAAAAATACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATCTACTATCAAAAATACTAAACACACTACCTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNGTTTTGTATTTTTGATAGTAGATATGATTGTATATTAGGGTAAGGTAGTGTGTTTAGTATTTTTGATAGTAGATATGATTGAATATTGTGGTAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tACCATATACAATCATATCAACTATAAAAAATACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATCTACTATCAAAAATACTAAACACACTACCTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNGTTTTGTATTTTTGATAGTAGATATGATTGTATATTAGGGTAAGGTAGTGTGTTTAGTATTTTTGATAGTAGATATGATTGAATATTGTGGTAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tGTTATATATAATTGTATTAATTATAAAAAATATNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATTTATTATTAAAAATATTAAATATATTATTTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNATTTTATATTTTTAATAATAAATATAATTATATATTAAAATAAAATAATATATTTAATATTTTTAATAATAAATACAATTAAATATCATAATAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_10_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70695 (70.69%) aligned concordantly 0 times\n",
      "    12336 (12.34%) aligned concordantly exactly 1 time\n",
      "    16969 (16.97%) aligned concordantly >1 times\n",
      "29.30% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70600 (70.60%) aligned concordantly 0 times\n",
      "    12225 (12.22%) aligned concordantly exactly 1 time\n",
      "    17175 (17.18%) aligned concordantly >1 times\n",
      "29.40% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70611 (70.61%) aligned concordantly 0 times\n",
      "    12401 (12.40%) aligned concordantly exactly 1 time\n",
      "    16988 (16.99%) aligned concordantly >1 times\n",
      "29.39% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70498 (70.50%) aligned concordantly 0 times\n",
      "    12497 (12.50%) aligned concordantly exactly 1 time\n",
      "    17005 (17.00%) aligned concordantly >1 times\n",
      "29.50% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_10_s1_R1.fastq.gz_C_to_T.fastq, zr2096_10_s1_R1.fastq.gz_G_to_A.fastq, zr2096_10_s1_R2.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1911136\n",
      "\n",
      "Total methylated C's in CpG context:\t185771\n",
      "Total methylated C's in CHG context:\t13978\n",
      "Total methylated C's in CHH context:\t47702\n",
      "Total methylated C's in Unknown context:\t6403\n",
      "\n",
      "Total unmethylated C's in CpG context:\t84112\n",
      "Total unmethylated C's in CHG context:\t424427\n",
      "Total unmethylated C's in CHH context:\t1155146\n",
      "Total unmethylated C's in Unknown context:\t26070\n",
      "\n",
      "C methylated in CpG context:\t68.8%\n",
      "C methylated in CHG context:\t3.2%\n",
      "C methylated in CHH context:\t4.0%\n",
      "C methylated in unknown context (CN or CHN):\t19.7%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 46s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t99\tNC_035780.1_CT_converted\t65371909\t6\t100M\t=\t65372013\t204\tGAGTTTTTTTGATTATTTGTTGTTTGTTGTTTGTTTGNNNNTNNNTNNGTTTGTTTGTTTGTTTGTTTGTTTGTAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tAS:i:-15\tXS:i:-22\tXN:i:0\tXM:i:10\tXO:i:0\tXG:i:0\tNM:i:10\tMD:Z:37T0T0T0G1T0T0G1T0T10A41\tYS:i:-24\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t147\tNC_035780.1_CT_converted\t65372013\t6\t100M\t=\t65371909\t-204\tGAATTATTGGGTTAATTTTAATTAAATTTGATATAAAGTATTTTTAGGTGAAGGGGATTTAAGTTTGTTTGAATGAAGGGTTTTGTTTTTTTTTAAGGGG\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-24\tXS:i:-6\tXN:i:0\tXM:i:4\tXO:i:0\tXG:i:0\tNM:i:4\tMD:Z:30G14G3A20A29\tYS:i:-15\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTAAAAAAAAATGAAATTTTTTATTTAAATAAATTTAAATTTTTTTTATTTAAAAATATTTTATATTAAATTTAATTAAAATTAATTTAATAATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTAAAAAAAAATGAAATTTTTTATTTAAATAAATTTAAATTTTTTTTATTTAAAAATATTTTATATTAAATTTAATTAAAATTAATTTAATAATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t83\tNC_035780.1_GA_converted\t8090288\t7\t26M4D74M\t=\t8090028\t-364\tAAAAAAAATAAAAATATAAAAAATTTACAAACAAACAAACAAACAAACAAACNNANNNANNNNCAAACAAACAACAAACAACAAATAATCAAAAAAACTC\tFFFFFFFF/FFFFF/B/FFFFFF/</FFFFFF<FFFFFFFFFFFFFFFF<<<##<###<####FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-26\tXS:i:-22\tXN:i:0\tXM:i:9\tXO:i:1\tXG:i:4\tNM:i:13\tMD:Z:26^ACAA26A0A1C0A0A1C0A0A0A37\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t163\tNC_035780.1_GA_converted\t8090028\t7\t100M\t=\t8090288\t364\tCCCCTTAAAAAAAAACAAAACCCTTCATTCAAACAAACTTAAATCCCCTTCACCTAAAAATACTTTATATCAAATTTAATTAAAATTAACCCAATAATTC\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-36\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:29T70\tYS:i:-26\tYT:Z:CP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_1_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    86490 (86.49%) aligned concordantly 0 times\n",
      "    5663 (5.66%) aligned concordantly exactly 1 time\n",
      "    7847 (7.85%) aligned concordantly >1 times\n",
      "13.51% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    86569 (86.57%) aligned concordantly 0 times\n",
      "    5628 (5.63%) aligned concordantly exactly 1 time\n",
      "    7803 (7.80%) aligned concordantly >1 times\n",
      "13.43% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    86672 (86.67%) aligned concordantly 0 times\n",
      "    5595 (5.59%) aligned concordantly exactly 1 time\n",
      "    7733 (7.73%) aligned concordantly >1 times\n",
      "13.33% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    86777 (86.78%) aligned concordantly 0 times\n",
      "    5615 (5.62%) aligned concordantly exactly 1 time\n",
      "    7608 (7.61%) aligned concordantly >1 times\n",
      "13.22% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_1_s1_R1.fastq.gz_C_to_T.fastq, zr2096_1_s1_R1.fastq.gz_G_to_A.fastq, zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t830788\n",
      "\n",
      "Total methylated C's in CpG context:\t72846\n",
      "Total methylated C's in CHG context:\t6793\n",
      "Total methylated C's in CHH context:\t40295\n",
      "Total methylated C's in Unknown context:\t5123\n",
      "\n",
      "Total unmethylated C's in CpG context:\t32368\n",
      "Total unmethylated C's in CHG context:\t169425\n",
      "Total unmethylated C's in CHH context:\t509061\n",
      "Total unmethylated C's in Unknown context:\t14379\n",
      "\n",
      "C methylated in CpG context:\t69.2%\n",
      "C methylated in CHG context:\t3.9%\n",
      "C methylated in CHH context:\t7.3%\n",
      "C methylated in unknown context (CN or CHN):\t26.3%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 2s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_2_s1_R1.fastq.gz to zr2096_2_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_2_s1_R1.fastq.gz to zr2096_2_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_2_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_2_s1_R2.fastq.gz to zr2096_2_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_2_s1_R2.fastq.gz to zr2096_2_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_2_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tTTTTTTATTAAAATTTATTATTTTAAAAATAAAAATTNNNNTNNNTTTTATTATTTTTTTTAATATTTTTTAATATAATTAAATAATAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTCACTTCAAAAAAATAATAAATAAAAAAAATTAATACAACAATATTTTTAATAATACAACAATATTTAAAAAATTATTTATTAATTTTTTTTATTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tCTCCTTATTAAAATCTACTATTCCAAAAATAAAAACTNNNNCNNNTTCCACCATTCCTTCTAACATCTCCCAATATAACTAAATAACAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTGTTTTGGAAAAATAATGGATAAAAAAGGTTGATGTGGTGATGTTTTTAGTGATGTGGTGGTGTTTGAGAAATTATTTATTAATTTTTTTTGTTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tCTCCTTATTAAAATCTACTATTCCAAAAATAAAAACTNNNNCNNNTTCCACCATTCCTTCTAACATCTCCCAATATAACTAAATAACAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTGTTTTGGAAAAATAATGGATAAAAAAGGTTGATGTGGTGATGTTTTTAGTGATGTGGTGGTGTTTGAGAAATTATTTATTAATTTTTTTTGTTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tTTTTTTATTAAAATTTATTATTTTAAAAATAAAAATTNNNNTNNNTTTTATTATTTTTTTTAATATTTTTTAATATAATTAAATAATAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTCACTTCAAAAAAATAATAAATAAAAAAAATTAATACAACAATATTTTTAATAATACAACAATATTTAAAAAATTATTTATTAATTTTTTTTATTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_2_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    71647 (71.65%) aligned concordantly 0 times\n",
      "    11900 (11.90%) aligned concordantly exactly 1 time\n",
      "    16453 (16.45%) aligned concordantly >1 times\n",
      "28.35% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    71946 (71.95%) aligned concordantly 0 times\n",
      "    11827 (11.83%) aligned concordantly exactly 1 time\n",
      "    16227 (16.23%) aligned concordantly >1 times\n",
      "28.05% overall alignment rate\n",
      "100000100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    71468 ( reads; of these:71.47%\n",
      ") aligned concordantly 0 times  \n",
      "    10000011794 (11.79%) aligned concordantly exactly 1 time\n",
      "    16738 (16.74%) aligned concordantly >1 times\n",
      "28.53% overall alignment rate\n",
      " (100.00%) were paired; of these:\n",
      "    71402 (71.40%) aligned concordantly 0 times\n",
      "    11953 (11.95%) aligned concordantly exactly 1 time\n",
      "    16645 (16.64%) aligned concordantly >1 times\n",
      "28.60% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_2_s1_R1.fastq.gz_C_to_T.fastq, zr2096_2_s1_R1.fastq.gz_G_to_A.fastq, zr2096_2_s1_R2.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1601294\n",
      "\n",
      "Total methylated C's in CpG context:\t153343\n",
      "Total methylated C's in CHG context:\t12481\n",
      "Total methylated C's in CHH context:\t54835\n",
      "Total methylated C's in Unknown context:\t6562\n",
      "\n",
      "Total unmethylated C's in CpG context:\t65114\n",
      "Total unmethylated C's in CHG context:\t359729\n",
      "Total unmethylated C's in CHH context:\t955792\n",
      "Total unmethylated C's in Unknown context:\t24798\n",
      "\n",
      "C methylated in CpG context:\t70.2%\n",
      "C methylated in CHG context:\t3.4%\n",
      "C methylated in CHH context:\t5.4%\n",
      "C methylated in unknown context (CN or CHN):\t20.9%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 17s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_3_s1_R1.fastq.gz to zr2096_3_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_3_s1_R1.fastq.gz to zr2096_3_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_3_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_3_s1_R2.fastq.gz to zr2096_3_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_3_s1_R2.fastq.gz to zr2096_3_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_3_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tTNTTTTAAAATAATTTAAATAATTTTATATAAATNNNNNNNNNNNNNNNTATTTAAATTAAAAATTTAAATAAATTTTTTTAATTATTTATTGTTTAATG\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAAATAAACATTAAACAATAAATAATTAAAAAAATTTATTTAAATTTTTAATTTAAATAAATTAAAAATATTAAATTTATATAAAATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tCNCTTCAAAATAATTTAAATAATCTTATATAAATNNNNNNNNNNNNNNNCACCTAAACTAAAAACTCAAATAAACTTTTCTAATCACCTATCATCCAACA\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAGATAGATGTTGGATGATAGGTGATTAGAAAAGTTTATTTGAGTTTTTAGTTTAGGTGAGTTAAAAATATTGAATTTATATAAGATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tCNCTTCAAAATAATTTAAATAATCTTATATAAATNNNNNNNNNNNNNNNCACCTAAACTAAAAACTCAAATAAACTTTTCTAATCACCTATCATCCAACA\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAGATAGATGTTGGATGATAGGTGATTAGAAAAGTTTATTTGAGTTTTTAGTTTAGGTGAGTTAAAAATATTGAATTTATATAAGATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tTNTTTTAAAATAATTTAAATAATTTTATATAAATNNNNNNNNNNNNNNNTATTTAAATTAAAAATTTAAATAAATTTTTTTAATTATTTATTGTTTAATG\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAAATAAACATTAAACAATAAATAATTAAAAAAATTTATTTAAATTTTTAATTTAAATAAATTAAAAATATTAAATTTATATAAAATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_3_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69474 (69.47%) aligned concordantly 0 times\n",
      "    12342 (12.34%) aligned concordantly exactly 1 time\n",
      "    18184 (18.18%) aligned concordantly >1 times\n",
      "30.53% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69093 (69.09%) aligned concordantly 0 times\n",
      "    12247 (12.25%) aligned concordantly exactly 1 time\n",
      "    18660 (18.66%) aligned concordantly >1 times\n",
      "30.91% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69028 (69.03%) aligned concordantly 0 times\n",
      "    12397 (12.40%) aligned concordantly exactly 1 time\n",
      "    18575 (18.57%) aligned concordantly >1 times\n",
      "30.97% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69406 (69.41%) aligned concordantly 0 times\n",
      "    12367 (12.37%) aligned concordantly exactly 1 time\n",
      "    18227 (18.23%) aligned concordantly >1 times\n",
      "30.59% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_3_s1_R1.fastq.gz_C_to_T.fastq, zr2096_3_s1_R1.fastq.gz_G_to_A.fastq, zr2096_3_s1_R2.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1911141\n",
      "\n",
      "Total methylated C's in CpG context:\t212856\n",
      "Total methylated C's in CHG context:\t13659\n",
      "Total methylated C's in CHH context:\t45545\n",
      "Total methylated C's in Unknown context:\t6337\n",
      "\n",
      "Total unmethylated C's in CpG context:\t65965\n",
      "Total unmethylated C's in CHG context:\t426703\n",
      "Total unmethylated C's in CHH context:\t1146413\n",
      "Total unmethylated C's in Unknown context:\t27502\n",
      "\n",
      "C methylated in CpG context:\t76.3%\n",
      "C methylated in CHG context:\t3.1%\n",
      "C methylated in CHH context:\t3.8%\n",
      "C methylated in unknown context (CN or CHN):\t18.7%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 39s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_4_s1_R1.fastq.gz to zr2096_4_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_4_s1_R1.fastq.gz to zr2096_4_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_4_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_4_s1_R2.fastq.gz to zr2096_4_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_4_s1_R2.fastq.gz to zr2096_4_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_4_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNGATTAGTGTATAGTGGTTGTTATGGAAATTAGNNNNNNNNNNNNNNNNNNNNNNNNNTGNANTNTTTTTATTAGAATTGTTAAGGTTTATTTGATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTCCATCATTAATTTTTAAAAAAAAAAAACCATCCTTAATATCAATTTTATAAATCAAATAAACCTTAACAATTCTAATAAAAACACTTCATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATTAATATATAATAATTATTATAAAAATTAANNNNNNNNNNNNNNNNNNNNNNNNNTANANTNTTTTTATTAAAATTATTAAAATTTATTTAATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTTTATTATTAATTTTTAAAAAAAAAAAATTATTTTTAATATTAATTTTATAAATTAAATAAATTTTAATAATTTTAATAAAAATATTTTATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATTAATATATAATAATTATTATAAAAATTAANNNNNNNNNNNNNNNNNNNNNNNNNTANANTNTTTTTATTAAAATTATTAAAATTTATTTAATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTTTATTATTAATTTTTAAAAAAAAAAAATTATTTTTAATATTAATTTTATAAATTAAATAAATTTTAATAATTTTAATAAAAATATTTTATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNGATTAGTGTATAGTGGTTGTTATGGAAATTAGNNNNNNNNNNNNNNNNNNNNNNNNNTGNANTNTTTTTATTAGAATTGTTAAGGTTTATTTGATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTCCATCATTAATTTTTAAAAAAAAAAAACCATCCTTAATATCAATTTTATAAATCAAATAAACCTTAACAATTCTAATAAAAACACTTCATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_4_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69215 (69.22%) aligned concordantly 0 times\n",
      "    12964 (12.96%) aligned concordantly exactly 1 time\n",
      "    17821 (17.82%) aligned concordantly >1 times\n",
      "30.79% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    68104 (68.10%) aligned concordantly 0 times\n",
      "    13392 (13.39%) aligned concordantly exactly 1 time\n",
      "    18504 (18.50%) aligned concordantly >1 times\n",
      "31.90% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    67910 (67.91%) aligned concordantly 0 times\n",
      "    13504 (13.50%) aligned concordantly exactly 1 time\n",
      "    18586 (18.59%) aligned concordantly >1 times\n",
      "32.09% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69042 (69.04%) aligned concordantly 0 times\n",
      "    13009 (13.01%) aligned concordantly exactly 1 time\n",
      "    17949 (17.95%) aligned concordantly >1 times\n",
      "30.96% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_4_s1_R1.fastq.gz_C_to_T.fastq, zr2096_4_s1_R1.fastq.gz_G_to_A.fastq, zr2096_4_s1_R2.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1807416\n",
      "\n",
      "Total methylated C's in CpG context:\t156286\n",
      "Total methylated C's in CHG context:\t12428\n",
      "Total methylated C's in CHH context:\t49328\n",
      "Total methylated C's in Unknown context:\t6149\n",
      "\n",
      "Total unmethylated C's in CpG context:\t82257\n",
      "Total unmethylated C's in CHG context:\t398925\n",
      "Total unmethylated C's in CHH context:\t1108192\n",
      "Total unmethylated C's in Unknown context:\t27246\n",
      "\n",
      "C methylated in CpG context:\t65.5%\n",
      "C methylated in CHG context:\t3.0%\n",
      "C methylated in CHH context:\t4.3%\n",
      "C methylated in unknown context (CN or CHN):\t18.4%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 59s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_5_s1_R1.fastq.gz to zr2096_5_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_5_s1_R1.fastq.gz to zr2096_5_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_5_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_5_s1_R2.fastq.gz to zr2096_5_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_5_s1_R2.fastq.gz to zr2096_5_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_5_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATAAAATATTATTTAAATTTTATTATTTATANNNNNNNNNNNNNNNNNNATTNTTTTTATTATATATATAAAATTATTTTAAAATTATTATATTTTTT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTCAAATATATTAAAAAAAAATTATTAATAATTTTCATTAAAAAATATAATAATTTTAAAATAATTTTATATATATAATAAAAAAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAACAAAACACCACTCAAATCTTACTATCCACANNNNNNNNNNNNNNNNNNATTNCTTCTATCATATATATAAAATCATTCCAAAATTACTATATTTTTC\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTTGAGTATATTGAAGAGGAATTATTGATAGTTTTTGTTGAAAAATATAGTAATTTTGGAATGATTTTATATATATGATAGAAGAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAACAAAACACCACTCAAATCTTACTATCCACANNNNNNNNNNNNNNNNNNATTNCTTCTATCATATATATAAAATCATTCCAAAATTACTATATTTTTC\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTTGAGTATATTGAAGAGGAATTATTGATAGTTTTTGTTGAAAAATATAGTAATTTTGGAATGATTTTATATATATGATAGAAGAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATAAAATATTATTTAAATTTTATTATTTATANNNNNNNNNNNNNNNNNNATTNTTTTTATTATATATATAAAATTATTTTAAAATTATTATATTTTTT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTCAAATATATTAAAAAAAAATTATTAATAATTTTCATTAAAAAATATAATAATTTTAAAATAATTTTATATATATAATAAAAAAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_5_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    71937 (71.94%) aligned concordantly 0 times\n",
      "    12071 (12.07%) aligned concordantly exactly 1 time\n",
      "    15992 (15.99%) aligned concordantly >1 times\n",
      "28.06% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    71853 (71.85%) aligned concordantly 0 times\n",
      "    12016 (12.02%) aligned concordantly exactly 1 time\n",
      "    16131 (16.13%) aligned concordantly >1 times\n",
      "28.15% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70840 (70.84%) aligned concordantly 0 times\n",
      "    12296 (12.30%) aligned concordantly exactly 1 time\n",
      "    16864 (16.86%) aligned concordantly >1 times\n",
      "29.16% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70617 (70.62%) aligned concordantly 0 times\n",
      "    12464 (12.46%) aligned concordantly exactly 1 time\n",
      "    16919 (16.92%) aligned concordantly >1 times\n",
      "29.38% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_5_s1_R1.fastq.gz_C_to_T.fastq, zr2096_5_s1_R1.fastq.gz_G_to_A.fastq, zr2096_5_s1_R2.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1792976\n",
      "\n",
      "Total methylated C's in CpG context:\t154384\n",
      "Total methylated C's in CHG context:\t12834\n",
      "Total methylated C's in CHH context:\t47835\n",
      "Total methylated C's in Unknown context:\t6577\n",
      "\n",
      "Total unmethylated C's in CpG context:\t75691\n",
      "Total unmethylated C's in CHG context:\t383354\n",
      "Total unmethylated C's in CHH context:\t1118878\n",
      "Total unmethylated C's in Unknown context:\t27168\n",
      "\n",
      "C methylated in CpG context:\t67.1%\n",
      "C methylated in CHG context:\t3.2%\n",
      "C methylated in CHH context:\t4.1%\n",
      "C methylated in unknown context (CN or CHN):\t19.5%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 31s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_6_s1_R1.fastq.gz to zr2096_6_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_6_s1_R1.fastq.gz to zr2096_6_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_6_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_6_s1_R2.fastq.gz to zr2096_6_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_6_s1_R2.fastq.gz to zr2096_6_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_6_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTTATTAAATTTAATTTTATTATTAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNTANANAAATATTAATTAAATATTTTTAAAATAATAAATTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAAAATTTTTTTAAATTTATTATTTTAAAAATATTTAATTAATATTTATTTATATTTTATAATTAACAATAATTATTTTTTTATTTAATAATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTCATCAAATCCAACTTTATCACCAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNCANANAAACATCAATCAAACATCTCTAAAACAATAAACTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAGAATTTTTTTAAGTTTATTGTTTTAGAGATGTTTGATTGATGTTTATTTGTATTTTGTAATTAGTGGTGGTTGTTTTTTTATTTGGTGATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTCATCAAATCCAACTTTATCACCAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNCANANAAACATCAATCAAACATCTCTAAAACAATAAACTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAGAATTTTTTTAAGTTTATTGTTTTAGAGATGTTTGATTGATGTTTATTTGTATTTTGTAATTAGTGGTGGTTGTTTTTTTATTTGGTGATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTTATTAAATTTAATTTTATTATTAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNTANANAAATATTAATTAAATATTTTTAAAATAATAAATTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAAAATTTTTTTAAATTTATTATTTTAAAAATATTTAATTAATATTTATTTATATTTTATAATTAACAATAATTATTTTTTTATTTAATAATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_6_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69708 (69.71%) aligned concordantly 0 times\n",
      "    12666 (12.67%) aligned concordantly exactly 1 time\n",
      "    17626 (17.63%) aligned concordantly >1 times\n",
      "30.29% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    67544 (67.54%) aligned concordantly 0 times\n",
      "    13694 (13.69%) aligned concordantly exactly 1 time\n",
      "    18762 (18.76%) aligned concordantly >1 times\n",
      "32.46% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    69721 (69.72%) aligned concordantly 0 times\n",
      "    12698 (12.70%) aligned concordantly exactly 1 time\n",
      "    17581 (17.58%) aligned concordantly >1 times\n",
      "30.28% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    67375 (67.38%) aligned concordantly 0 times\n",
      "    13613 (13.61%) aligned concordantly exactly 1 time\n",
      "    19012 (19.01%) aligned concordantly >1 times\n",
      "32.62% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_6_s1_R1.fastq.gz_C_to_T.fastq, zr2096_6_s1_R1.fastq.gz_G_to_A.fastq, zr2096_6_s1_R2.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1839385\n",
      "\n",
      "Total methylated C's in CpG context:\t178767\n",
      "Total methylated C's in CHG context:\t13324\n",
      "Total methylated C's in CHH context:\t50010\n",
      "Total methylated C's in Unknown context:\t6495\n",
      "\n",
      "Total unmethylated C's in CpG context:\t66888\n",
      "Total unmethylated C's in CHG context:\t402617\n",
      "Total unmethylated C's in CHH context:\t1127779\n",
      "Total unmethylated C's in Unknown context:\t28732\n",
      "\n",
      "C methylated in CpG context:\t72.8%\n",
      "C methylated in CHG context:\t3.2%\n",
      "C methylated in CHH context:\t4.2%\n",
      "C methylated in unknown context (CN or CHN):\t18.4%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 42s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_7_s1_R1.fastq.gz to zr2096_7_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_7_s1_R1.fastq.gz to zr2096_7_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_7_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_7_s1_R2.fastq.gz to zr2096_7_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_7_s1_R2.fastq.gz to zr2096_7_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_7_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAAAAAATGATTATTATGATGAATTTAAATATTAATAATAAAATTTAAAAAAAATATAAATGTTAAATATTAAATTTTTTAAAAAATGATTATGG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t141\t*\t0\t0\t*\t*\t0\t0\tCCATAATCATTTTTTAAAAAATTTAATATTTAACATTTATATTTTTTTTAAATTTTATTATTAATATTTAAATTCATCATAATAATCATTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t99\tNC_035781.1_GA_converted\t24539746\t6\t95M\t=\t24539746\t-95\tAAAAAAACAACTACTACAACAAATTCAAATATTAATAACAAAATTTAAAAAAAACATAAACATTAAATATCAAATTCTTCAAAAAACAATTACAA\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-12\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:94T0\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t147\tNC_035781.1_GA_converted\t24539746\t6\t95M\t=\t24539746\t-95\tAAAAAAACAACTACTACAACAAATTCAAATATTAATAACAAAATTTAAAAAAAACATAAACATTAAATATCAAATTCTTCAAAAAACAATTACAA\tFFFFFFFFFFFFFFFFFFFFFFFBBFFFFFFBBFF/FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-6\tXS:i:-12\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:94T0\tYS:i:-6\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t83\tNC_035780.1_CT_converted\t42548681\t17\t95M\t=\t42548681\t-95\tTTGTAATTGTTTTTTGAAGAATTTGATATTTAATGTTTATGTTTTTTTTAAATTTTGTTATTAATATTTGAATTTGTTGTAGTAGTTGTTTTTTT\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-6\tXS:i:-32\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:0A94\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t163\tNC_035780.1_CT_converted\t42548681\t17\t95M\t=\t42548681\t-95\tTTGTAATTGTTTTTTGAAGAATTTGATATTTAATGTTTATGTTTTTTTTAAATTTTGTTATTAATATTTGAATTTGTTGTAGTAGTTGTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-32\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:0A94\tYS:i:-6\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAAAAAATGATTATTATGATGAATTTAAATATTAATAATAAAATTTAAAAAAAATATAAATGTTAAATATTAAATTTTTTAAAAAATGATTATGG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t141\t*\t0\t0\t*\t*\t0\t0\tCCATAATCATTTTTTAAAAAATTTAATATTTAACATTTATATTTTTTTTAAATTTTATTATTAATATTTAAATTCATCATAATAATCATTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_7_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    71530 (71.53%) aligned concordantly 0 times\n",
      "    12232 (12.23%) aligned concordantly exactly 1 time\n",
      "    16238 (16.24%) aligned concordantly >1 times\n",
      "28.47% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    72232 (72.23%) aligned concordantly 0 times\n",
      "    11838 (11.84%) aligned concordantly exactly 1 time\n",
      "    15930 (15.93%) aligned concordantly >1 times\n",
      "27.77% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    71562 (71.56%) aligned concordantly 0 times\n",
      "    12285 (12.29%) aligned concordantly exactly 1 time\n",
      "    16153 (16.15%) aligned concordantly >1 times\n",
      "28.44% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    72175 (72.17%) aligned concordantly 0 times\n",
      "    11787 (11.79%) aligned concordantly exactly 1 time\n",
      "    16038 (16.04%) aligned concordantly >1 times\n",
      "27.82% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_7_s1_R1.fastq.gz_C_to_T.fastq, zr2096_7_s1_R1.fastq.gz_G_to_A.fastq, zr2096_7_s1_R2.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1807058\n",
      "\n",
      "Total methylated C's in CpG context:\t165861\n",
      "Total methylated C's in CHG context:\t13399\n",
      "Total methylated C's in CHH context:\t47784\n",
      "Total methylated C's in Unknown context:\t6056\n",
      "\n",
      "Total unmethylated C's in CpG context:\t67849\n",
      "Total unmethylated C's in CHG context:\t393633\n",
      "Total unmethylated C's in CHH context:\t1118532\n",
      "Total unmethylated C's in Unknown context:\t26732\n",
      "\n",
      "C methylated in CpG context:\t71.0%\n",
      "C methylated in CHG context:\t3.3%\n",
      "C methylated in CHH context:\t4.1%\n",
      "C methylated in unknown context (CN or CHN):\t18.5%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 5m 25s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_8_s1_R1.fastq.gz to zr2096_8_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_8_s1_R1.fastq.gz to zr2096_8_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_8_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_8_s1_R2.fastq.gz to zr2096_8_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_8_s1_R2.fastq.gz to zr2096_8_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_8_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATTTAAAATTAAAAAATTTATTAAATTTATTATTATTTNTTATTTTTTATAAAAGAAAATAGAGAGAGAGAGAGAGAGAGAGAGAG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tCTCTCTCTCTCTCTCTCTCTCTCTCTATTTTCTTTTATAAAAAATAATAAATAATAATAAATTTAATAAATTTTTTAATTTTAAATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATCCAAAATTAAAAAACTCATTAAATCTATCATCATTCNCCATTCTTCACAAAAAAAAATAAAAAAAAAAAAAAAAAAAAAAAAAA\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTTTTTTTTTTTTTTTTTTTTTATTTTTTTTTGTGAAGAATGGTGAATGATGATAGATTTAATGAGTTTTTTAATTTTGGATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t83\tNC_007175.2_CT_converted\t14567\t0\t2M17I69M\t=\t14567\t-71\tTTTTTTTTTTTTTTTTTTTTTTTTTTATTTTTTTTTGTGAAGAATGGNGAATGATGATAGATTTAATGAGTTTTTTAATTTTGGATTA\tFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<<#FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-81\tXN:i:0\tXM:i:5\tXO:i:1\tXG:i:17\tNM:i:22\tMD:Z:1A5A0G5A15T40\tYS:i:-80\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t163\tNC_007175.2_CT_converted\t14567\t0\t2M17I69M\t=\t14567\t-71\tTTTTTTTTTTTTTTTTTTTTTTTTTTATTTTTTTTTGTGAAGAATGGTGAATGATGATAGATTTAATGAGTTTTTTAATTTTGGATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tAS:i:-80\tXN:i:0\tXM:i:4\tXO:i:1\tXG:i:17\tNM:i:21\tMD:Z:1A5A0G5A56\tYS:i:-81\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATTTAAAATTAAAAAATTTATTAAATTTATTATTATTTNTTATTTTTTATAAAAGAAAATAGAGAGAGAGAGAGAGAGAGAGAGAG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tCTCTCTCTCTCTCTCTCTCTCTCTCTATTTTCTTTTATAAAAAATAATAAATAATAATAAATTTAATAAATTTTTTAATTTTAAATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_8_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    74157 (74.16%) aligned concordantly 0 times\n",
      "    11312 (11.31%) aligned concordantly exactly 1 time\n",
      "    14531 (14.53%) aligned concordantly >1 times\n",
      "25.84% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    74306 (74.31%) aligned concordantly 0 times\n",
      "    11103 (11.10%) aligned concordantly exactly 1 time\n",
      "    14591 (14.59%) aligned concordantly >1 times\n",
      "25.69% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    74133 (74.13%) aligned concordantly 0 times\n",
      "    11331 (11.33%) aligned concordantly exactly 1 time\n",
      "    14536 (14.54%) aligned concordantly >1 times\n",
      "25.87% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    74552 (74.55%) aligned concordantly 0 times\n",
      "    11116 (11.12%) aligned concordantly exactly 1 time\n",
      "    14332 (14.33%) aligned concordantly >1 times\n",
      "25.45% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_8_s1_R1.fastq.gz_C_to_T.fastq, zr2096_8_s1_R1.fastq.gz_G_to_A.fastq, zr2096_8_s1_R2.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1598278\n",
      "\n",
      "Total methylated C's in CpG context:\t130605\n",
      "Total methylated C's in CHG context:\t11991\n",
      "Total methylated C's in CHH context:\t48523\n",
      "Total methylated C's in Unknown context:\t6038\n",
      "\n",
      "Total unmethylated C's in CpG context:\t67102\n",
      "Total unmethylated C's in CHG context:\t348741\n",
      "Total unmethylated C's in CHH context:\t991316\n",
      "Total unmethylated C's in Unknown context:\t24212\n",
      "\n",
      "C methylated in CpG context:\t66.1%\n",
      "C methylated in CHG context:\t3.3%\n",
      "C methylated in CHH context:\t4.7%\n",
      "C methylated in unknown context (CN or CHN):\t20.0%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 5m 14s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_9_s1_R1.fastq.gz to zr2096_9_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_9_s1_R1.fastq.gz to zr2096_9_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_9_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_9_s1_R2.fastq.gz to zr2096_9_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_9_s1_R2.fastq.gz to zr2096_9_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_9_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNATTAAATAAAAAATAATAAATAATAAATTANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAGG\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAAAAAATAAAAAACACATTAAAAATCTATAATTTAATTTATTATTTATTATTTTTTATTTAATTAAATCNNNNAAACATCATATAAAAAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNACCAAATAAAAAATAACAAATAATAAATCANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAAA\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAGAGGATAGGAGATATGTTAGGAATTTATAATTTGATTTATTATTTGTTATTTTTTATTTGGTTAGATTNNNNGAGTGTTGTGTAGGGAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNACCAAATAAAAAATAACAAATAATAAATCANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAAA\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAGAGGATAGGAGATATGTTAGGAATTTATAATTTGATTTATTATTTGTTATTTTTTATTTGGTTAGATTNNNNGAGTGTTGTGTAGGGAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNATTAAATAAAAAATAATAAATAATAAATTANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAGG\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAAAAAATAAAAAACACATTAAAAATCTATAATTTAATTTATTATTTATTATTTTTTATTTAATTAAATCNNNNAAACATCATATAAAAAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_9_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    73508 (73.51%) aligned concordantly 0 times\n",
      "    11236 (11.24%) aligned concordantly exactly 1 time\n",
      "    15256 (15.26%) aligned concordantly >1 times\n",
      "26.49% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    73787 (73.79%) aligned concordantly 0 times\n",
      "    11232 (11.23%) aligned concordantly exactly 1 time\n",
      "    14981 (14.98%) aligned concordantly >1 times\n",
      "26.21% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70419 (70.42%) aligned concordantly 0 times\n",
      "    12573 (12.57%) aligned concordantly exactly 1 time\n",
      "    17008 (17.01%) aligned concordantly >1 times\n",
      "29.58% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    70201 (70.20%) aligned concordantly 0 times\n",
      "    12723 (12.72%) aligned concordantly exactly 1 time\n",
      "    17076 (17.08%) aligned concordantly >1 times\n",
      "29.80% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_9_s1_R1.fastq.gz_C_to_T.fastq, zr2096_9_s1_R1.fastq.gz_G_to_A.fastq, zr2096_9_s1_R2.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1849983\n",
      "\n",
      "Total methylated C's in CpG context:\t199908\n",
      "Total methylated C's in CHG context:\t14694\n",
      "Total methylated C's in CHH context:\t48531\n",
      "Total methylated C's in Unknown context:\t6764\n",
      "\n",
      "Total unmethylated C's in CpG context:\t63719\n",
      "Total unmethylated C's in CHG context:\t409648\n",
      "Total unmethylated C's in CHH context:\t1113483\n",
      "Total unmethylated C's in Unknown context:\t28006\n",
      "\n",
      "C methylated in CpG context:\t75.8%\n",
      "C methylated in CHG context:\t3.5%\n",
      "C methylated in CHH context:\t4.2%\n",
      "C methylated in unknown context (CN or CHN):\t19.5%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 5m 54s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "find /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_*R1* \\\n",
    "| xargs basename -s _s1_R1.fastq.gz | xargs -I{} /Users/Shelly/anaconda3/bin/bismark \\\n",
    "--genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ \\\n",
    "--score_min L,0,-1.2 \\\n",
    "-u 100000 \\\n",
    "--non_directional \\\n",
    "-1 /Volumes/web/seashell/bu-serine-wd/18-04-07/{}_s1_R1.fastq.gz \\\n",
    "-2 /Volumes/web/seashell/bu-serine-wd/18-04-07/{}_s1_R2.fastq.gz \\\n",
    "-o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/ \n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#run deduplication"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_10_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_1_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_2_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_3_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_4_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_5_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_6_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_7_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_8_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_9_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\"\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/\n",
      "Processing paired-end Bismark output file(s) (SAM format):\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_10_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_1_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_2_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_3_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_4_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_5_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_6_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_7_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_8_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_9_s1_R1_bismark_bt2_pe.bam\n",
      "\n",
      "\n",
      "If there are several alignments to a single position in the genome the first alignment will be chosen. Since the input files are not in any way sorted this is a near-enough random selection of reads.\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_10_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_10_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_10_s1_R1_bismark_bt2_pe.bam:\t55304\n",
      "Total number duplicated alignments removed:\t22 (0.04%)\n",
      "Duplicated alignments were found at:\t21 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 55282 (99.96% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_1_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_1_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_1_s1_R1_bismark_bt2_pe.bam:\t25623\n",
      "Total number duplicated alignments removed:\t73 (0.28%)\n",
      "Duplicated alignments were found at:\t61 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 25550 (99.72% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_2_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_2_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_2_s1_R1_bismark_bt2_pe.bam:\t50484\n",
      "Total number duplicated alignments removed:\t59 (0.12%)\n",
      "Duplicated alignments were found at:\t42 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 50425 (99.88% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_3_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_3_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_3_s1_R1_bismark_bt2_pe.bam:\t55171\n",
      "Total number duplicated alignments removed:\t16 (0.03%)\n",
      "Duplicated alignments were found at:\t16 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 55155 (99.97% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_4_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_4_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_4_s1_R1_bismark_bt2_pe.bam:\t56412\n",
      "Total number duplicated alignments removed:\t14 (0.02%)\n",
      "Duplicated alignments were found at:\t13 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 56398 (99.98% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_5_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_5_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_5_s1_R1_bismark_bt2_pe.bam:\t53779\n",
      "Total number duplicated alignments removed:\t20 (0.04%)\n",
      "Duplicated alignments were found at:\t20 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 53759 (99.96% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_6_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_6_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_6_s1_R1_bismark_bt2_pe.bam:\t56844\n",
      "Total number duplicated alignments removed:\t12 (0.02%)\n",
      "Duplicated alignments were found at:\t12 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 56832 (99.98% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_7_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_7_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_7_s1_R1_bismark_bt2_pe.bam:\t53463\n",
      "Total number duplicated alignments removed:\t14 (0.03%)\n",
      "Duplicated alignments were found at:\t14 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 53449 (99.97% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_8_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_8_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_8_s1_R1_bismark_bt2_pe.bam:\t48884\n",
      "Total number duplicated alignments removed:\t27 (0.06%)\n",
      "Duplicated alignments were found at:\t24 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 48857 (99.94% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_9_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_9_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_9_s1_R1_bismark_bt2_pe.bam:\t52449\n",
      "Total number duplicated alignments removed:\t35 (0.07%)\n",
      "Duplicated alignments were found at:\t35 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 52414 (99.93% of total)\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "/Users/Shelly/anaconda3/bin/deduplicate_bismark \\\n",
    "--bam -p \\\n",
    "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/*.bam \\\n",
    "-o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/ \n",
    "2> /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/dedup.err"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#create summary report of mapping efficiency and dedup. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/*PE_report.txt | grep 'Mapping\\ efficiency\\:' | cat - /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/*.deduplication_report.txt > /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_mapping_dedup_summary.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!mkdir /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#analysis with a tighter alignment score threshold"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t35458\n",
      "Mapping efficiency:\t35.5%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t51704\n",
      "Sequence pairs did not map uniquely:\t12838\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t8804\t((converted) top strand)\n",
      "GA/CT/CT:\t8916\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t8935\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t8803\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t13744\n",
      "Mapping efficiency:\t13.7%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t81302\n",
      "Sequence pairs did not map uniquely:\t4954\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t3402\t((converted) top strand)\n",
      "GA/CT/CT:\t3480\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t3458\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t3404\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t31325\n",
      "Mapping efficiency:\t31.3%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t56828\n",
      "Sequence pairs did not map uniquely:\t11847\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t7926\t((converted) top strand)\n",
      "GA/CT/CT:\t7816\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t7775\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t7808\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t34892\n",
      "Mapping efficiency:\t34.9%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t51558\n",
      "Sequence pairs did not map uniquely:\t13550\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t8702\t((converted) top strand)\n",
      "GA/CT/CT:\t8804\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t8773\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t8613\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t35879\n",
      "Mapping efficiency:\t35.9%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t50638\n",
      "Sequence pairs did not map uniquely:\t13483\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t9000\t((converted) top strand)\n",
      "GA/CT/CT:\t8832\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t8995\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t9052\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t33466\n",
      "Mapping efficiency:\t33.5%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t54437\n",
      "Sequence pairs did not map uniquely:\t12097\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t8445\t((converted) top strand)\n",
      "GA/CT/CT:\t8308\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t8287\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t8426\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t35383\n",
      "Mapping efficiency:\t35.4%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t51529\n",
      "Sequence pairs did not map uniquely:\t13088\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t9139\t((converted) top strand)\n",
      "GA/CT/CT:\t8577\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t8499\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t9168\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t33107\n",
      "Mapping efficiency:\t33.1%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t55205\n",
      "Sequence pairs did not map uniquely:\t11688\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t8364\t((converted) top strand)\n",
      "GA/CT/CT:\t8129\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t8164\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t8450\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t30256\n",
      "Mapping efficiency:\t30.3%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t59165\n",
      "Sequence pairs did not map uniquely:\t10579\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t7666\t((converted) top strand)\n",
      "GA/CT/CT:\t7447\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t7475\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t7668\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t32194\n",
      "Mapping efficiency:\t32.2%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t55792\n",
      "Sequence pairs did not map uniquely:\t12014\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t8466\t((converted) top strand)\n",
      "GA/CT/CT:\t7691\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t7605\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t8432\t((converted) bottom strand)\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_10_s1_R1.fastq.gz to zr2096_10_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_10_s1_R1.fastq.gz to zr2096_10_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_10_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_10_s1_R2.fastq.gz to zr2096_10_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_10_s1_R2.fastq.gz to zr2096_10_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_10_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tGTTATATATAATTGTATTAATTATAAAAAATATNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATTTATTATTAAAAATATTAAATATATTATTTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNATTTTATATTTTTAATAATAAATATAATTATATATTAAAATAAAATAATATATTTAATATTTTTAATAATAAATACAATTAAATATCATAATAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tACCATATACAATCATATCAACTATAAAAAATACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATCTACTATCAAAAATACTAAACACACTACCTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNGTTTTGTATTTTTGATAGTAGATATGATTGTATATTAGGGTAAGGTAGTGTGTTTAGTATTTTTGATAGTAGATATGATTGAATATTGTGGTAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tACCATATACAATCATATCAACTATAAAAAATACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATCTACTATCAAAAATACTAAACACACTACCTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNGTTTTGTATTTTTGATAGTAGATATGATTGTATATTAGGGTAAGGTAGTGTGTTTAGTATTTTTGATAGTAGATATGATTGAATATTGTGGTAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tGTTATATATAATTGTATTAATTATAAAAAATATNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATTTATTATTAAAAATATTAAATATATTATTTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNATTTTATATTTTTAATAATAAATATAATTATATATTAAAATAAAATAATATATTTAATATTTTTAATAATAAATACAATTAAATATCATAATAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_10_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "100000100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    83458 (83.46%) aligned concordantly 0 times\n",
      "    8189 (8.19%) aligned concordantly exactly 1 time\n",
      "    8353 (8.35%) aligned concordantly >1 times\n",
      "16.54% overall alignment rate\n",
      " reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    83495 (83.50%) aligned concordantly 0 times\n",
      "    8120 (8.12%) aligned concordantly exactly 1 time\n",
      "    8385 (8.38%) aligned concordantly >1 times\n",
      "16.50% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    83320 (83.32%) aligned concordantly 0 times\n",
      "    8200 (8.20%) aligned concordantly exactly 1 time\n",
      "    8480 (8.48%) aligned concordantly >1 times\n",
      "16.68% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    83316 (83.32%) aligned concordantly 0 times\n",
      "    8207 (8.21%) aligned concordantly exactly 1 time\n",
      "    8477 (8.48%) aligned concordantly >1 times\n",
      "16.68% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_10_s1_R1.fastq.gz_C_to_T.fastq, zr2096_10_s1_R1.fastq.gz_G_to_A.fastq, zr2096_10_s1_R2.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1271427\n",
      "\n",
      "Total methylated C's in CpG context:\t132862\n",
      "Total methylated C's in CHG context:\t7807\n",
      "Total methylated C's in CHH context:\t21040\n",
      "Total methylated C's in Unknown context:\t1601\n",
      "\n",
      "Total unmethylated C's in CpG context:\t50564\n",
      "Total unmethylated C's in CHG context:\t293003\n",
      "Total unmethylated C's in CHH context:\t766151\n",
      "Total unmethylated C's in Unknown context:\t5018\n",
      "\n",
      "C methylated in CpG context:\t72.4%\n",
      "C methylated in CHG context:\t2.6%\n",
      "C methylated in CHH context:\t2.7%\n",
      "C methylated in unknown context (CN or CHN):\t24.2%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 9s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t99\tNC_035789.1_CT_converted\t4531039\t2\t37M4I59M\t=\t4531139\t200\tGAGTTTTTTTGATTATTTGTTGTTTGTTGTTTGTTTGNNNNTNNNTNNGTTTGTTTGTTTGTTTGTTTGTTTGTAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tAS:i:-22\tXS:i:-31\tXN:i:0\tXM:i:5\tXO:i:1\tXG:i:4\tNM:i:9\tMD:Z:38T0T0G1T0T52\tYS:i:-24\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t147\tNC_035789.1_CT_converted\t4531139\t2\t100M\t=\t4531039\t-200\tGAATTATTGGGTTAATTTTAATTAAATTTGATATAAAGTATTTTTAGGTGAAGGGGATTTAAGTTTGTTTGAATGAAGGGTTTTGTTTTTTTTTAAGGGG\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-24\tXS:i:-12\tXN:i:0\tXM:i:4\tXO:i:0\tXG:i:0\tNM:i:4\tMD:Z:30G14G9A14A29\tYS:i:-22\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTAAAAAAAAATGAAATTTTTTATTTAAATAAATTTAAATTTTTTTTATTTAAAAATATTTTATATTAAATTTAATTAAAATTAATTTAATAATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTAAAAAAAAATGAAATTTTTTATTTAAATAAATTTAAATTTTTTTTATTTAAAAATATTTTATATTAAATTTAATTAAAATTAATTTAATAATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t83\tNC_035780.1_GA_converted\t8090288\t6\t26M4D74M\t=\t8090028\t-364\tAAAAAAAATAAAAATATAAAAAATTTACAAACAAACAAACAAACAAACAAACNNANNNANNNNCAAACAAACAACAAACAACAAATAATCAAAAAAACTC\tFFFFFFFF/FFFFF/B/FFFFFF/</FFFFFF<FFFFFFFFFFFFFFFF<<<##<###<####FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-26\tXS:i:-9\tXN:i:0\tXM:i:9\tXO:i:1\tXG:i:4\tNM:i:13\tMD:Z:26^ACAA26A0A1C0A0A1C0A0A0A37\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t163\tNC_035780.1_GA_converted\t8090028\t6\t100M\t=\t8090288\t364\tCCCCTTAAAAAAAAACAAAACCCTTCATTCAAACAAACTTAAATCCCCTTCACCTAAAAATACTTTATATCAAATTTAATTAAAATTAACCCAATAATTC\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-18\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:29T70\tYS:i:-26\tYT:Z:CP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_1_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    93560 (93.56%) aligned concordantly 0 times\n",
      "    3220 (3.22%) aligned concordantly exactly 1 time\n",
      "    3220 (3.22%) aligned concordantly >1 times\n",
      "6.44% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    93584 (93.58%) aligned concordantly 0 times\n",
      "    3180 (3.18%) aligned concordantly exactly 1 time\n",
      "    3236 (3.24%) aligned concordantly >1 times\n",
      "6.42% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    93582 (93.58%) aligned concordantly 0 times\n",
      "    3175 (3.17%) aligned concordantly exactly 1 time\n",
      "    3243 (3.24%) aligned concordantly >1 times\n",
      "6.42% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    93611 (93.61%) aligned concordantly 0 times\n",
      "    3157 (3.16%) aligned concordantly exactly 1 time\n",
      "    3232 (3.23%) aligned concordantly >1 times\n",
      "6.39% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_1_s1_R1.fastq.gz_C_to_T.fastq, zr2096_1_s1_R1.fastq.gz_G_to_A.fastq, zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t481569\n",
      "\n",
      "Total methylated C's in CpG context:\t47945\n",
      "Total methylated C's in CHG context:\t2774\n",
      "Total methylated C's in CHH context:\t7846\n",
      "Total methylated C's in Unknown context:\t624\n",
      "\n",
      "Total unmethylated C's in CpG context:\t17126\n",
      "Total unmethylated C's in CHG context:\t107925\n",
      "Total unmethylated C's in CHH context:\t297953\n",
      "Total unmethylated C's in Unknown context:\t1993\n",
      "\n",
      "C methylated in CpG context:\t73.7%\n",
      "C methylated in CHG context:\t2.5%\n",
      "C methylated in CHH context:\t2.6%\n",
      "C methylated in unknown context (CN or CHN):\t23.8%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 3m 34s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_2_s1_R1.fastq.gz to zr2096_2_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_2_s1_R1.fastq.gz to zr2096_2_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_2_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_2_s1_R2.fastq.gz to zr2096_2_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_2_s1_R2.fastq.gz to zr2096_2_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_2_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tTTTTTTATTAAAATTTATTATTTTAAAAATAAAAATTNNNNTNNNTTTTATTATTTTTTTTAATATTTTTTAATATAATTAAATAATAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTCACTTCAAAAAAATAATAAATAAAAAAAATTAATACAACAATATTTTTAATAATACAACAATATTTAAAAAATTATTTATTAATTTTTTTTATTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tCTCCTTATTAAAATCTACTATTCCAAAAATAAAAACTNNNNCNNNTTCCACCATTCCTTCTAACATCTCCCAATATAACTAAATAACAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTGTTTTGGAAAAATAATGGATAAAAAAGGTTGATGTGGTGATGTTTTTAGTGATGTGGTGGTGTTTGAGAAATTATTTATTAATTTTTTTTGTTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tCTCCTTATTAAAATCTACTATTCCAAAAATAAAAACTNNNNCNNNTTCCACCATTCCTTCTAACATCTCCCAATATAACTAAATAACAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTGTTTTGGAAAAATAATGGATAAAAAAGGTTGATGTGGTGATGTTTTTAGTGATGTGGTGGTGTTTGAGAAATTATTTATTAATTTTTTTTGTTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tTTTTTTATTAAAATTTATTATTTTAAAAATAAAAATTNNNNTNNNTTTTATTATTTTTTTTAATATTTTTTAATATAATTAAATAATAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTCACTTCAAAAAAATAATAAATAAAAAAAATTAATACAACAATATTTTTAATAATACAACAATATTTAAAAAATTATTTATTAATTTTTTTTATTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_2_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    85277 (85.28%) aligned concordantly 0 times\n",
      "    7512 (7.51%) aligned concordantly exactly 1 time\n",
      "    7211 (7.21%) aligned concordantly >1 times\n",
      "14.72% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    85171 (85.17%) aligned concordantly 0 times\n",
      "    7449 (7.45%) aligned concordantly exactly 1 time\n",
      "    7380 (7.38%) aligned concordantly >1 times\n",
      "14.83% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84940 (84.94%) aligned concordantly 0 times\n",
      "    7464 (7.46%) aligned concordantly exactly 1 time\n",
      "    7596 (7.60%) aligned concordantly >1 times\n",
      "15.06% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    85135 (85.14%) aligned concordantly 0 times\n",
      "    7348 (7.35%) aligned concordantly exactly 1 time\n",
      "    7517 (7.52%) aligned concordantly >1 times\n",
      "14.87% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_2_s1_R1.fastq.gz_C_to_T.fastq, zr2096_2_s1_R1.fastq.gz_G_to_A.fastq, zr2096_2_s1_R2.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1039727\n",
      "\n",
      "Total methylated C's in CpG context:\t109099\n",
      "Total methylated C's in CHG context:\t6645\n",
      "Total methylated C's in CHH context:\t17729\n",
      "Total methylated C's in Unknown context:\t1331\n",
      "\n",
      "Total unmethylated C's in CpG context:\t37771\n",
      "Total unmethylated C's in CHG context:\t246689\n",
      "Total unmethylated C's in CHH context:\t621794\n",
      "Total unmethylated C's in Unknown context:\t4398\n",
      "\n",
      "C methylated in CpG context:\t74.3%\n",
      "C methylated in CHG context:\t2.6%\n",
      "C methylated in CHH context:\t2.8%\n",
      "C methylated in unknown context (CN or CHN):\t23.2%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 3m 52s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_3_s1_R1.fastq.gz to zr2096_3_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_3_s1_R1.fastq.gz to zr2096_3_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_3_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_3_s1_R2.fastq.gz to zr2096_3_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_3_s1_R2.fastq.gz to zr2096_3_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_3_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tTNTTTTAAAATAATTTAAATAATTTTATATAAATNNNNNNNNNNNNNNNTATTTAAATTAAAAATTTAAATAAATTTTTTTAATTATTTATTGTTTAATG\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAAATAAACATTAAACAATAAATAATTAAAAAAATTTATTTAAATTTTTAATTTAAATAAATTAAAAATATTAAATTTATATAAAATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tCNCTTCAAAATAATTTAAATAATCTTATATAAATNNNNNNNNNNNNNNNCACCTAAACTAAAAACTCAAATAAACTTTTCTAATCACCTATCATCCAACA\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAGATAGATGTTGGATGATAGGTGATTAGAAAAGTTTATTTGAGTTTTTAGTTTAGGTGAGTTAAAAATATTGAATTTATATAAGATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tCNCTTCAAAATAATTTAAATAATCTTATATAAATNNNNNNNNNNNNNNNCACCTAAACTAAAAACTCAAATAAACTTTTCTAATCACCTATCATCCAACA\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAGATAGATGTTGGATGATAGGTGATTAGAAAAGTTTATTTGAGTTTTTAGTTTAGGTGAGTTAAAAATATTGAATTTATATAAGATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tTNTTTTAAAATAATTTAAATAATTTTATATAAATNNNNNNNNNNNNNNNTATTTAAATTAAAAATTTAAATAAATTTTTTTAATTATTTATTGTTTAATG\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAAATAAACATTAAACAATAAATAATTAAAAAAATTTATTTAAATTTTTAATTTAAATAAATTAAAAATATTAAATTTATATAAAATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_3_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    82995 (83.00%) aligned concordantly 0 times\n",
      "    8054 (8.05%) aligned concordantly exactly 1 time\n",
      "    8951 (8.95%) aligned concordantly >1 times\n",
      "17.00% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    83095 (83.09%) aligned concordantly 0 times\n",
      "    7975 (7.97%) aligned concordantly exactly 1 time\n",
      "    8930 (8.93%) aligned concordantly >1 times\n",
      "16.91% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    83122 (83.12%) aligned concordantly 0 times\n",
      "    7830 (7.83%) aligned concordantly exactly 1 time\n",
      "    9048 (9.05%) aligned concordantly >1 times\n",
      "16.88% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    82917 (82.92%) aligned concordantly 0 times\n",
      "    8105 (8.11%) aligned concordantly exactly 1 time\n",
      "    8978 (8.98%) aligned concordantly >1 times\n",
      "17.08% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_3_s1_R1.fastq.gz_C_to_T.fastq, zr2096_3_s1_R1.fastq.gz_G_to_A.fastq, zr2096_3_s1_R2.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1252955\n",
      "\n",
      "Total methylated C's in CpG context:\t149786\n",
      "Total methylated C's in CHG context:\t7391\n",
      "Total methylated C's in CHH context:\t18812\n",
      "Total methylated C's in Unknown context:\t1441\n",
      "\n",
      "Total unmethylated C's in CpG context:\t37500\n",
      "Total unmethylated C's in CHG context:\t290378\n",
      "Total unmethylated C's in CHH context:\t749088\n",
      "Total unmethylated C's in Unknown context:\t4982\n",
      "\n",
      "C methylated in CpG context:\t80.0%\n",
      "C methylated in CHG context:\t2.5%\n",
      "C methylated in CHH context:\t2.4%\n",
      "C methylated in unknown context (CN or CHN):\t22.4%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 3m 59s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_4_s1_R1.fastq.gz to zr2096_4_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_4_s1_R1.fastq.gz to zr2096_4_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_4_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_4_s1_R2.fastq.gz to zr2096_4_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_4_s1_R2.fastq.gz to zr2096_4_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_4_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNGATTAGTGTATAGTGGTTGTTATGGAAATTAGNNNNNNNNNNNNNNNNNNNNNNNNNTGNANTNTTTTTATTAGAATTGTTAAGGTTTATTTGATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTCCATCATTAATTTTTAAAAAAAAAAAACCATCCTTAATATCAATTTTATAAATCAAATAAACCTTAACAATTCTAATAAAAACACTTCATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATTAATATATAATAATTATTATAAAAATTAANNNNNNNNNNNNNNNNNNNNNNNNNTANANTNTTTTTATTAAAATTATTAAAATTTATTTAATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTTTATTATTAATTTTTAAAAAAAAAAAATTATTTTTAATATTAATTTTATAAATTAAATAAATTTTAATAATTTTAATAAAAATATTTTATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATTAATATATAATAATTATTATAAAAATTAANNNNNNNNNNNNNNNNNNNNNNNNNTANANTNTTTTTATTAAAATTATTAAAATTTATTTAATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTTTATTATTAATTTTTAAAAAAAAAAAATTATTTTTAATATTAATTTTATAAATTAAATAAATTTTAATAATTTTAATAAAAATATTTTATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNGATTAGTGTATAGTGGTTGTTATGGAAATTAGNNNNNNNNNNNNNNNNNNNNNNNNNTGNANTNTTTTTATTAGAATTGTTAAGGTTTATTTGATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTCCATCATTAATTTTTAAAAAAAAAAAACCATCCTTAATATCAATTTTATAAATCAAATAAACCTTAACAATTCTAATAAAAACACTTCATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_4_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    83173 (83.17%) aligned concordantly 0 times\n",
      "    8539 (8.54%) aligned concordantly exactly 1 time\n",
      "    8288 (8.29%) aligned concordantly >1 times\n",
      "16.83% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    83193 (83.19%) aligned concordantly 0 times\n",
      "    8419 (8.42%) aligned concordantly exactly 1 time\n",
      "    8388 (8.39%) aligned concordantly >1 times\n",
      "16.81% overall alignment rate\n",
      "100000 reads; of these:\n",
      "100000   reads; of these:100000\n",
      " (  100000100.00 (%) were paired; of these:\n",
      "    82895 (100.00%) were paired; of these:82.89%\n",
      ") aligned concordantly 0 times    \n",
      "    828148591 ( (82.818.59%%) aligned concordantly 0 times) aligned concordantly exactly 1 time\n",
      "\n",
      "        86918514 ( (8.698.51%%) aligned concordantly exactly 1 time) aligned concordantly >1 times\n",
      "\n",
      "    17.11% overall alignment rate\n",
      "8495 (8.49%) aligned concordantly >1 times\n",
      "17.19% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_4_s1_R1.fastq.gz_C_to_T.fastq, zr2096_4_s1_R1.fastq.gz_G_to_A.fastq, zr2096_4_s1_R2.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1189881\n",
      "\n",
      "Total methylated C's in CpG context:\t112195\n",
      "Total methylated C's in CHG context:\t6744\n",
      "Total methylated C's in CHH context:\t19933\n",
      "Total methylated C's in Unknown context:\t1440\n",
      "\n",
      "Total unmethylated C's in CpG context:\t48992\n",
      "Total unmethylated C's in CHG context:\t273615\n",
      "Total unmethylated C's in CHH context:\t728402\n",
      "Total unmethylated C's in Unknown context:\t5075\n",
      "\n",
      "C methylated in CpG context:\t69.6%\n",
      "C methylated in CHG context:\t2.4%\n",
      "C methylated in CHH context:\t2.7%\n",
      "C methylated in unknown context (CN or CHN):\t22.1%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 0s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_5_s1_R1.fastq.gz to zr2096_5_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_5_s1_R1.fastq.gz to zr2096_5_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_5_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_5_s1_R2.fastq.gz to zr2096_5_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_5_s1_R2.fastq.gz to zr2096_5_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_5_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATAAAATATTATTTAAATTTTATTATTTATANNNNNNNNNNNNNNNNNNATTNTTTTTATTATATATATAAAATTATTTTAAAATTATTATATTTTTT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTCAAATATATTAAAAAAAAATTATTAATAATTTTCATTAAAAAATATAATAATTTTAAAATAATTTTATATATATAATAAAAAAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAACAAAACACCACTCAAATCTTACTATCCACANNNNNNNNNNNNNNNNNNATTNCTTCTATCATATATATAAAATCATTCCAAAATTACTATATTTTTC\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTTGAGTATATTGAAGAGGAATTATTGATAGTTTTTGTTGAAAAATATAGTAATTTTGGAATGATTTTATATATATGATAGAAGAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAACAAAACACCACTCAAATCTTACTATCCACANNNNNNNNNNNNNNNNNNATTNCTTCTATCATATATATAAAATCATTCCAAAATTACTATATTTTTC\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTTGAGTATATTGAAGAGGAATTATTGATAGTTTTTGTTGAAAAATATAGTAATTTTGGAATGATTTTATATATATGATAGAAGAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATAAAATATTATTTAAATTTTATTATTTATANNNNNNNNNNNNNNNNNNATTNTTTTTATTATATATATAAAATTATTTTAAAATTATTATATTTTTT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTCAAATATATTAAAAAAAAATTATTAATAATTTTCATTAAAAAATATAATAATTTTAAAATAATTTTATATATATAATAAAAAAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_5_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84478 (84.48%) aligned concordantly 0 times\n",
      "    7778 (7.78%) aligned concordantly exactly 1 time\n",
      "    7744 (7.74%) aligned concordantly >1 times\n",
      "15.52% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84534 (84.53%) aligned concordantly 0 times\n",
      "    7798 (7.80%) aligned concordantly exactly 1 time\n",
      "    7668 (7.67%) aligned concordantly >1 times\n",
      "15.47% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84350 (84.35%) aligned concordantly 0 times\n",
      "    7780 (7.78%) aligned concordantly exactly 1 time\n",
      "    7870 (7.87%) aligned concordantly >1 times\n",
      "15.65% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84194 (84.19%) aligned concordantly 0 times\n",
      "    7813 (7.81%) aligned concordantly exactly 1 time\n",
      "    7993 (7.99%) aligned concordantly >1 times\n",
      "15.81% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_5_s1_R1.fastq.gz_C_to_T.fastq, zr2096_5_s1_R1.fastq.gz_G_to_A.fastq, zr2096_5_s1_R2.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1159933\n",
      "\n",
      "Total methylated C's in CpG context:\t108103\n",
      "Total methylated C's in CHG context:\t6982\n",
      "Total methylated C's in CHH context:\t19266\n",
      "Total methylated C's in Unknown context:\t1384\n",
      "\n",
      "Total unmethylated C's in CpG context:\t44016\n",
      "Total unmethylated C's in CHG context:\t258610\n",
      "Total unmethylated C's in CHH context:\t722956\n",
      "Total unmethylated C's in Unknown context:\t4955\n",
      "\n",
      "C methylated in CpG context:\t71.1%\n",
      "C methylated in CHG context:\t2.6%\n",
      "C methylated in CHH context:\t2.6%\n",
      "C methylated in unknown context (CN or CHN):\t21.8%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 2s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_6_s1_R1.fastq.gz to zr2096_6_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_6_s1_R1.fastq.gz to zr2096_6_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_6_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_6_s1_R2.fastq.gz to zr2096_6_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_6_s1_R2.fastq.gz to zr2096_6_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_6_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTTATTAAATTTAATTTTATTATTAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNTANANAAATATTAATTAAATATTTTTAAAATAATAAATTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAAAATTTTTTTAAATTTATTATTTTAAAAATATTTAATTAATATTTATTTATATTTTATAATTAACAATAATTATTTTTTTATTTAATAATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTCATCAAATCCAACTTTATCACCAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNCANANAAACATCAATCAAACATCTCTAAAACAATAAACTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAGAATTTTTTTAAGTTTATTGTTTTAGAGATGTTTGATTGATGTTTATTTGTATTTTGTAATTAGTGGTGGTTGTTTTTTTATTTGGTGATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTCATCAAATCCAACTTTATCACCAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNCANANAAACATCAATCAAACATCTCTAAAACAATAAACTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAGAATTTTTTTAAGTTTATTGTTTTAGAGATGTTTGATTGATGTTTATTTGTATTTTGTAATTAGTGGTGGTTGTTTTTTTATTTGGTGATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTTATTAAATTTAATTTTATTATTAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNTANANAAATATTAATTAAATATTTTTAAAATAATAAATTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAAAATTTTTTTAAATTTATTATTTTAAAAATATTTAATTAATATTTATTTATATTTTATAATTAACAATAATTATTTTTTTATTTAATAATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_6_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84081 (84.08%) aligned concordantly 0 times\n",
      "    8033 (8.03%) aligned concordantly exactly 1 time\n",
      "    7886 (7.89%) aligned concordantly >1 times\n",
      "15.92% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    83934 (83.93%) aligned concordantly 0 times\n",
      "    8147 (8.15%) aligned concordantly exactly 1 time\n",
      "    7919 (7.92%) aligned concordantly >1 times\n",
      "16.07% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    82786 (82.79%) aligned concordantly 0 times\n",
      "    8606 (8.61%) aligned concordantly exactly 1 time\n",
      "    8608 (8.61%) aligned concordantly >1 times\n",
      "17.21% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    82685 (82.69%) aligned concordantly 0 times\n",
      "    8647 (8.65%) aligned concordantly exactly 1 time\n",
      "    8668 (8.67%) aligned concordantly >1 times\n",
      "17.32% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_6_s1_R1.fastq.gz_C_to_T.fastq, zr2096_6_s1_R1.fastq.gz_G_to_A.fastq, zr2096_6_s1_R2.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1185031\n",
      "\n",
      "Total methylated C's in CpG context:\t126300\n",
      "Total methylated C's in CHG context:\t7170\n",
      "Total methylated C's in CHH context:\t19881\n",
      "Total methylated C's in Unknown context:\t1537\n",
      "\n",
      "Total unmethylated C's in CpG context:\t36204\n",
      "Total unmethylated C's in CHG context:\t270406\n",
      "Total unmethylated C's in CHH context:\t725070\n",
      "Total unmethylated C's in Unknown context:\t5223\n",
      "\n",
      "C methylated in CpG context:\t77.7%\n",
      "C methylated in CHG context:\t2.6%\n",
      "C methylated in CHH context:\t2.7%\n",
      "C methylated in unknown context (CN or CHN):\t22.7%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 3m 58s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_7_s1_R1.fastq.gz to zr2096_7_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_7_s1_R1.fastq.gz to zr2096_7_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_7_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_7_s1_R2.fastq.gz to zr2096_7_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_7_s1_R2.fastq.gz to zr2096_7_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_7_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAAAAAATGATTATTATGATGAATTTAAATATTAATAATAAAATTTAAAAAAAATATAAATGTTAAATATTAAATTTTTTAAAAAATGATTATGG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t141\t*\t0\t0\t*\t*\t0\t0\tCCATAATCATTTTTTAAAAAATTTAATATTTAACATTTATATTTTTTTTAAATTTTATTATTAATATTTAAATTCATCATAATAATCATTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t99\tNC_035781.1_GA_converted\t24539746\t12\t95M\t=\t24539746\t-95\tAAAAAAACAACTACTACAACAAATTCAAATATTAATAACAAAATTTAAAAAAAACATAAACATTAAATATCAAATTCTTCAAAAAACAATTACAA\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-12\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:94T0\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t147\tNC_035781.1_GA_converted\t24539746\t12\t95M\t=\t24539746\t-95\tAAAAAAACAACTACTACAACAAATTCAAATATTAATAACAAAATTTAAAAAAAACATAAACATTAAATATCAAATTCTTCAAAAAACAATTACAA\tFFFFFFFFFFFFFFFFFFFFFFFBBFFFFFFBBFF/FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-6\tXS:i:-12\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:94T0\tYS:i:-6\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t83\tNC_035780.1_CT_converted\t42548681\t21\t95M\t=\t42548681\t-95\tTTGTAATTGTTTTTTGAAGAATTTGATATTTAATGTTTATGTTTTTTTTAAATTTTGTTATTAATATTTGAATTTGTTGTAGTAGTTGTTTTTTT\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-6\tXS:i:-32\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:0A94\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t163\tNC_035780.1_CT_converted\t42548681\t21\t95M\t=\t42548681\t-95\tTTGTAATTGTTTTTTGAAGAATTTGATATTTAATGTTTATGTTTTTTTTAAATTTTGTTATTAATATTTGAATTTGTTGTAGTAGTTGTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-32\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:0A94\tYS:i:-6\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAAAAAATGATTATTATGATGAATTTAAATATTAATAATAAAATTTAAAAAAAATATAAATGTTAAATATTAAATTTTTTAAAAAATGATTATGG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t141\t*\t0\t0\t*\t*\t0\t0\tCCATAATCATTTTTTAAAAAATTTAATATTTAACATTTATATTTTTTTTAAATTTTATTATTAATATTTAAATTCATCATAATAATCATTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_7_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84887 (84.89%) aligned concordantly 0 times\n",
      "    7610 (7.61%) aligned concordantly exactly 1 time\n",
      "    7503 (7.50%) aligned concordantly >1 times\n",
      "15.11% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84905 (84.91%) aligned concordantly 0 times\n",
      "    7561 (7.56%) aligned concordantly exactly 1 time\n",
      "    7534 (7.53%) aligned concordantly >1 times\n",
      "15.10% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84581 (84.58%) aligned concordantly 0 times\n",
      "    7709 (7.71%) aligned concordantly exactly 1 time\n",
      "    7710 (7.71%) aligned concordantly >1 times\n",
      "15.42% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84387 (84.39%) aligned concordantly 0 times\n",
      "    7828 (7.83%) aligned concordantly exactly 1 time\n",
      "    7785 (7.79%) aligned concordantly >1 times\n",
      "15.61% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_7_s1_R1.fastq.gz_C_to_T.fastq, zr2096_7_s1_R1.fastq.gz_G_to_A.fastq, zr2096_7_s1_R2.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1161926\n",
      "\n",
      "Total methylated C's in CpG context:\t114479\n",
      "Total methylated C's in CHG context:\t7182\n",
      "Total methylated C's in CHH context:\t20183\n",
      "Total methylated C's in Unknown context:\t1356\n",
      "\n",
      "Total unmethylated C's in CpG context:\t38201\n",
      "Total unmethylated C's in CHG context:\t262714\n",
      "Total unmethylated C's in CHH context:\t719167\n",
      "Total unmethylated C's in Unknown context:\t4916\n",
      "\n",
      "C methylated in CpG context:\t75.0%\n",
      "C methylated in CHG context:\t2.7%\n",
      "C methylated in CHH context:\t2.7%\n",
      "C methylated in unknown context (CN or CHN):\t21.6%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 5s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_8_s1_R1.fastq.gz to zr2096_8_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_8_s1_R1.fastq.gz to zr2096_8_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_8_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_8_s1_R2.fastq.gz to zr2096_8_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_8_s1_R2.fastq.gz to zr2096_8_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_8_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATTTAAAATTAAAAAATTTATTAAATTTATTATTATTTNTTATTTTTTATAAAAGAAAATAGAGAGAGAGAGAGAGAGAGAGAGAG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tCTCTCTCTCTCTCTCTCTCTCTCTCTATTTTCTTTTATAAAAAATAATAAATAATAATAAATTTAATAAATTTTTTAATTTTAAATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATCCAAAATTAAAAAACTCATTAAATCTATCATCATTCNCCATTCTTCACAAAAAAAAATAAAAAAAAAAAAAAAAAAAAAAAAAA\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTTTTTTTTTTTTTTTTTTTTTATTTTTTTTTGTGAAGAATGGTGAATGATGATAGATTTAATGAGTTTTTTAATTTTGGATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATCCAAAATTAAAAAACTCATTAAATCTATCATCATTCNCCATTCTTCACAAAAAAAAATAAAAAAAAAAAAAAAAAAAAAAAAAA\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTTTTTTTTTTTTTTTTTTTTTATTTTTTTTTGTGAAGAATGGTGAATGATGATAGATTTAATGAGTTTTTTAATTTTGGATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATTTAAAATTAAAAAATTTATTAAATTTATTATTATTTNTTATTTTTTATAAAAGAAAATAGAGAGAGAGAGAGAGAGAGAGAGAG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tCTCTCTCTCTCTCTCTCTCTCTCTCTATTTTCTTTTATAAAAAATAATAAATAATAATAAATTTAATAAATTTTTTAATTTTAAATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_8_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    85958 (85.96%) aligned concordantly 0 times\n",
      "    7118 (7.12%) aligned concordantly exactly 1 time\n",
      "    6924 (6.92%) aligned concordantly >1 times\n",
      "14.04% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    86249 (86.25%) aligned concordantly 0 times\n",
      "    7037 (7.04%) aligned concordantly exactly 1 time\n",
      "    6714 (6.71%) aligned concordantly >1 times\n",
      "13.75% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    86013 (86.01%) aligned concordantly 0 times\n",
      "    7180 (7.18%) aligned concordantly exactly 1 time\n",
      "    6807 (6.81%) aligned concordantly >1 times\n",
      "13.99% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    86322 (86.32%) aligned concordantly 0 times\n",
      "    7009 (7.01%) aligned concordantly exactly 1 time\n",
      "    6669 (6.67%) aligned concordantly >1 times\n",
      "13.68% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_8_s1_R1.fastq.gz_C_to_T.fastq, zr2096_8_s1_R1.fastq.gz_G_to_A.fastq, zr2096_8_s1_R2.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1029296\n",
      "\n",
      "Total methylated C's in CpG context:\t90766\n",
      "Total methylated C's in CHG context:\t6484\n",
      "Total methylated C's in CHH context:\t18894\n",
      "Total methylated C's in Unknown context:\t1299\n",
      "\n",
      "Total unmethylated C's in CpG context:\t39042\n",
      "Total unmethylated C's in CHG context:\t235415\n",
      "Total unmethylated C's in CHH context:\t638695\n",
      "Total unmethylated C's in Unknown context:\t4409\n",
      "\n",
      "C methylated in CpG context:\t69.9%\n",
      "C methylated in CHG context:\t2.7%\n",
      "C methylated in CHH context:\t2.9%\n",
      "C methylated in unknown context (CN or CHN):\t22.8%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 3m 52s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_9_s1_R1.fastq.gz to zr2096_9_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_9_s1_R1.fastq.gz to zr2096_9_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_9_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_9_s1_R2.fastq.gz to zr2096_9_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_9_s1_R2.fastq.gz to zr2096_9_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_9_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNATTAAATAAAAAATAATAAATAATAAATTANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAGG\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAAAAAATAAAAAACACATTAAAAATCTATAATTTAATTTATTATTTATTATTTTTTATTTAATTAAATCNNNNAAACATCATATAAAAAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNACCAAATAAAAAATAACAAATAATAAATCANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAAA\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAGAGGATAGGAGATATGTTAGGAATTTATAATTTGATTTATTATTTGTTATTTTTTATTTGGTTAGATTNNNNGAGTGTTGTGTAGGGAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNACCAAATAAAAAATAACAAATAATAAATCANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAAA\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAGAGGATAGGAGATATGTTAGGAATTTATAATTTGATTTATTATTTGTTATTTTTTATTTGGTTAGATTNNNNGAGTGTTGTGTAGGGAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-0.6 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNATTAAATAAAAAATAATAAATAATAAATTANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAGG\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAAAAAATAAAAAACACATTAAAAATCTATAATTTAATTTATTATTTATTATTTTTTATTTAATTAAATCNNNNAAACATCATATAAAAAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_9_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    85652 (85.65%) aligned concordantly 0 times\n",
      "    7096 (7.10%) aligned concordantly exactly 1 time\n",
      "    7252 (7.25%) aligned concordantly >1 times\n",
      "14.35% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84038 (84.04%) aligned concordantly 0 times\n",
      "    7970 (7.97%) aligned concordantly exactly 1 time\n",
      "    7992 (7.99%) aligned concordantly >1 times\n",
      "15.96% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    85781 (85.78%) aligned concordantly 0 times\n",
      "    7152 (7.15%) aligned concordantly exactly 1 time\n",
      "    7067 (7.07%) aligned concordantly >1 times\n",
      "14.22% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    84011 (84.01%) aligned concordantly 0 times\n",
      "    8024 (8.02%) aligned concordantly exactly 1 time\n",
      "    7965 (7.96%) aligned concordantly >1 times\n",
      "15.99% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_9_s1_R1.fastq.gz_C_to_T.fastq, zr2096_9_s1_R1.fastq.gz_G_to_A.fastq, zr2096_9_s1_R2.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1170800\n",
      "\n",
      "Total methylated C's in CpG context:\t135804\n",
      "Total methylated C's in CHG context:\t7983\n",
      "Total methylated C's in CHH context:\t20498\n",
      "Total methylated C's in Unknown context:\t1414\n",
      "\n",
      "Total unmethylated C's in CpG context:\t34966\n",
      "Total unmethylated C's in CHG context:\t268427\n",
      "Total unmethylated C's in CHH context:\t703122\n",
      "Total unmethylated C's in Unknown context:\t4683\n",
      "\n",
      "C methylated in CpG context:\t79.5%\n",
      "C methylated in CHG context:\t2.9%\n",
      "C methylated in CHH context:\t2.8%\n",
      "C methylated in unknown context (CN or CHN):\t23.2%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 0s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "find /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_*R1* \\\n",
    "| xargs basename -s _s1_R1.fastq.gz | xargs -I{} /Users/Shelly/anaconda3/bin/bismark \\\n",
    "--genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ \\\n",
    "--score_min L,0,-0.6 \\\n",
    "-u 100000 \\\n",
    "--non_directional \\\n",
    "-1 /Volumes/web/seashell/bu-serine-wd/18-04-07/{}_s1_R1.fastq.gz \\\n",
    "-2 /Volumes/web/seashell/bu-serine-wd/18-04-07/{}_s1_R2.fastq.gz \\\n",
    "-o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/ "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#run deduplicate on stricter alignment score threshold alignments"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_10_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-0.6 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_1_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-0.6 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_2_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-0.6 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_3_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-0.6 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_4_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-0.6 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_5_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-0.6 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_6_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-0.6 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_7_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-0.6 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_8_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-0.6 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_9_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-0.6 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\"\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/\n",
      "Processing paired-end Bismark output file(s) (SAM format):\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_10_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_1_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_2_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_3_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_4_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_5_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_6_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_7_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_8_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_9_s1_R1_bismark_bt2_pe.bam\n",
      "\n",
      "\n",
      "If there are several alignments to a single position in the genome the first alignment will be chosen. Since the input files are not in any way sorted this is a near-enough random selection of reads.\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_10_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_10_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_10_s1_R1_bismark_bt2_pe.bam:\t35458\n",
      "Total number duplicated alignments removed:\t15 (0.04%)\n",
      "Duplicated alignments were found at:\t14 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 35443 (99.96% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_1_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_1_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_1_s1_R1_bismark_bt2_pe.bam:\t13744\n",
      "Total number duplicated alignments removed:\t1 (0.01%)\n",
      "Duplicated alignments were found at:\t1 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 13743 (99.99% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_2_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_2_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_2_s1_R1_bismark_bt2_pe.bam:\t31325\n",
      "Total number duplicated alignments removed:\t9 (0.03%)\n",
      "Duplicated alignments were found at:\t9 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 31316 (99.97% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_3_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_3_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_3_s1_R1_bismark_bt2_pe.bam:\t34892\n",
      "Total number duplicated alignments removed:\t13 (0.04%)\n",
      "Duplicated alignments were found at:\t13 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 34879 (99.96% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_4_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_4_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_4_s1_R1_bismark_bt2_pe.bam:\t35879\n",
      "Total number duplicated alignments removed:\t9 (0.03%)\n",
      "Duplicated alignments were found at:\t8 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 35870 (99.97% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_5_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_5_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_5_s1_R1_bismark_bt2_pe.bam:\t33466\n",
      "Total number duplicated alignments removed:\t17 (0.05%)\n",
      "Duplicated alignments were found at:\t17 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 33449 (99.95% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_6_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_6_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_6_s1_R1_bismark_bt2_pe.bam:\t35383\n",
      "Total number duplicated alignments removed:\t9 (0.03%)\n",
      "Duplicated alignments were found at:\t9 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 35374 (99.97% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_7_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_7_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_7_s1_R1_bismark_bt2_pe.bam:\t33107\n",
      "Total number duplicated alignments removed:\t10 (0.03%)\n",
      "Duplicated alignments were found at:\t10 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 33097 (99.97% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_8_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_8_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_8_s1_R1_bismark_bt2_pe.bam:\t30256\n",
      "Total number duplicated alignments removed:\t10 (0.03%)\n",
      "Duplicated alignments were found at:\t10 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 30246 (99.97% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_9_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_9_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_9_s1_R1_bismark_bt2_pe.bam:\t32194\n",
      "Total number duplicated alignments removed:\t20 (0.06%)\n",
      "Duplicated alignments were found at:\t20 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 32174 (99.94% of total)\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "/Users/Shelly/anaconda3/bin/deduplicate_bismark \\\n",
    "--bam -p \\\n",
    "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/*.bam \\\n",
    "-o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/ \n",
    "2> /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/dedup.err"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#create summary report of mapping efficiency and dedup. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/*PE_report.txt | grep 'Mapping\\ efficiency\\:' | cat - /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/*.deduplication_report.txt > /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_mapping_dedup_summary.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#analysis with a looser alignment score threshold"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!mkdir /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t79505\n",
      "Mapping efficiency:\t79.5%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t213\n",
      "Sequence pairs did not map uniquely:\t20282\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t20127\t((converted) top strand)\n",
      "GA/CT/CT:\t19634\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t19795\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t19949\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t84845\n",
      "Mapping efficiency:\t84.8%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t573\n",
      "Sequence pairs did not map uniquely:\t14582\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t21400\t((converted) top strand)\n",
      "GA/CT/CT:\t21240\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t20948\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t21257\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t79272\n",
      "Mapping efficiency:\t79.3%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t526\n",
      "Sequence pairs did not map uniquely:\t20202\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t20068\t((converted) top strand)\n",
      "GA/CT/CT:\t19703\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t19524\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t19977\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t78192\n",
      "Mapping efficiency:\t78.2%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t310\n",
      "Sequence pairs did not map uniquely:\t21498\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t19689\t((converted) top strand)\n",
      "GA/CT/CT:\t19420\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t19283\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t19800\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t78339\n",
      "Mapping efficiency:\t78.3%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t695\n",
      "Sequence pairs did not map uniquely:\t20966\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t19925\t((converted) top strand)\n",
      "GA/CT/CT:\t19098\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t19148\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t20168\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t79734\n",
      "Mapping efficiency:\t79.7%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t374\n",
      "Sequence pairs did not map uniquely:\t19892\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t20457\t((converted) top strand)\n",
      "GA/CT/CT:\t19366\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t19341\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t20570\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t78624\n",
      "Mapping efficiency:\t78.6%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t323\n",
      "Sequence pairs did not map uniquely:\t21053\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t20580\t((converted) top strand)\n",
      "GA/CT/CT:\t18627\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t18702\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t20715\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t80059\n",
      "Mapping efficiency:\t80.1%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t386\n",
      "Sequence pairs did not map uniquely:\t19555\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t20411\t((converted) top strand)\n",
      "GA/CT/CT:\t19497\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t19740\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t20411\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t81008\n",
      "Mapping efficiency:\t81.0%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t385\n",
      "Sequence pairs did not map uniquely:\t18607\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t20390\t((converted) top strand)\n",
      "GA/CT/CT:\t20130\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t19951\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t20537\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t79357\n",
      "Mapping efficiency:\t79.4%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t540\n",
      "Sequence pairs did not map uniquely:\t20103\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t21347\t((converted) top strand)\n",
      "GA/CT/CT:\t18451\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t18365\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t21194\t((converted) bottom strand)\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_10_s1_R1.fastq.gz to zr2096_10_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_10_s1_R1.fastq.gz to zr2096_10_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_10_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_10_s1_R2.fastq.gz to zr2096_10_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_10_s1_R2.fastq.gz to zr2096_10_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_10_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tGTTATATATAATTGTATTAATTATAAAAAATATNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATTTATTATTAAAAATATTAAATATATTATTTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNATTTTATATTTTTAATAATAAATATAATTATATATTAAAATAAAATAATATATTTAATATTTTTAATAATAAATACAATTAAATATCATAATAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tACCATATACAATCATATCAACTATAAAAAATACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATCTACTATCAAAAATACTAAACACACTACCTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNGTTTTGTATTTTTGATAGTAGATATGATTGTATATTAGGGTAAGGTAGTGTGTTTAGTATTTTTGATAGTAGATATGATTGAATATTGTGGTAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tACCATATACAATCATATCAACTATAAAAAATACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATCTACTATCAAAAATACTAAACACACTACCTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNGTTTTGTATTTTTGATAGTAGATATGATTGTATATTAGGGTAAGGTAGTGTGTTTAGTATTTTTGATAGTAGATATGATTGAATATTGTGGTAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tGTTATATATAATTGTATTAATTATAAAAAATATNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATTTATTATTAAAAATATTAAATATATTATTTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNATTTTATATTTTTAATAATAAATATAATTATATATTAAAATAAAATAATATATTTAATATTTTTAATAATAAATACAATTAAATATCATAATAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_10_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    8527 (8.53%) aligned concordantly 0 times\n",
      "    3790 (3.79%) aligned concordantly exactly 1 time\n",
      "    87683 (87.68%) aligned concordantly >1 times\n",
      "91.47% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    8435 (8.44%) aligned concordantly 0 times\n",
      "    3658 (3.66%) aligned concordantly exactly 1 time\n",
      "    87907 (87.91%) aligned concordantly >1 times\n",
      "91.56% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9441 (9.44%) aligned concordantly 0 times\n",
      "    3544 (3.54%) aligned concordantly exactly 1 time\n",
      "    87015 (87.02%) aligned concordantly >1 times\n",
      "90.56% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9502 (9.50%) aligned concordantly 0 times\n",
      "    3408 (3.41%) aligned concordantly exactly 1 time\n",
      "    87090 (87.09%) aligned concordantly >1 times\n",
      "90.50% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_10_s1_R1.fastq.gz_C_to_T.fastq, zr2096_10_s1_R1.fastq.gz_G_to_A.fastq, zr2096_10_s1_R2.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t2596537\n",
      "\n",
      "Total methylated C's in CpG context:\t229383\n",
      "Total methylated C's in CHG context:\t25983\n",
      "Total methylated C's in CHH context:\t138155\n",
      "Total methylated C's in Unknown context:\t26486\n",
      "\n",
      "Total unmethylated C's in CpG context:\t118501\n",
      "Total unmethylated C's in CHG context:\t530950\n",
      "Total unmethylated C's in CHH context:\t1553565\n",
      "Total unmethylated C's in Unknown context:\t73368\n",
      "\n",
      "C methylated in CpG context:\t65.9%\n",
      "C methylated in CHG context:\t4.7%\n",
      "C methylated in CHH context:\t8.2%\n",
      "C methylated in unknown context (CN or CHN):\t26.5%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 12m 7s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t99\tNC_035786.1_CT_converted\t55112902\t1\t100M\t=\t55113162\t360\tGAGTTTTTTTGATTATTTGTTGTTTGTTGTTTGTTTGNNNNTNNNTNNGTTTGTTTGTTTGTTTGTTTGTTTGTAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tAS:i:-9\tXS:i:-22\tXN:i:0\tXM:i:9\tXO:i:0\tXG:i:0\tNM:i:9\tMD:Z:37T0T0T0G1T0T0G1T0T52\tYS:i:-30\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t147\tNC_035786.1_CT_converted\t55113162\t1\t100M\t=\t55112902\t-360\tGAATTATTGGGTTAATTTTAATTAAATTTGATATAAAGTATTTTTAGGTGAAGGGGATTTAAGTTTGTTTGAATGAAGGGTTTTGTTTTTTTTTAAGGGG\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-30\tXS:i:-6\tXN:i:0\tXM:i:5\tXO:i:0\tXG:i:0\tNM:i:5\tMD:Z:10A19G14G24A13T15\tYS:i:-9\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t99\tNC_035788.1_GA_converted\t81174826\t0\t13M2I7M3I2M2I6M2I11M2I4M3I4M1I6M1D9M8I1M1I13M\t=\t81174826\t-84\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tAS:i:-176\tXS:i:-166\tXN:i:0\tXM:i:16\tXO:i:10\tXG:i:25\tNM:i:41\tMD:Z:8A0C1C14C1A0A0C0A1A0T0A1C0A8A5^T2A10C9\tYS:i:-162\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t147\tNC_035788.1_GA_converted\t81174826\t0\t9M3I7M1I25M2I2M8I5M3I4M1D31M\t=\t81174826\t-84\tAAATTATTAAATTAATTTTAATTAAATTTAATATAAAATATTTTTAAATAAAAAAAATTTAAATTTATTTAAATAAAAAATTTCATTTTTTTTTAAAAAA\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-162\tXS:i:-162\tXN:i:0\tXM:i:13\tXO:i:6\tXG:i:18\tNM:i:31\tMD:Z:5T3C1C6C2T4C2A0C1T4C0A13^C2T3T24\tYS:i:-176\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTAAAAAAAAATGAAATTTTTTATTTAAATAAATTTAAATTTTTTTTATTTAAAAATATTTTATATTAAATTTAATTAAAATTAATTTAATAATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t83\tNC_035780.1_GA_converted\t8090288\t6\t26M4D74M\t=\t8090028\t-364\tAAAAAAAATAAAAATATAAAAAATTTACAAACAAACAAACAAACAAACAAACNNANNNANNNNCAAACAAACAACAAACAACAAATAATCAAAAAAACTC\tFFFFFFFF/FFFFF/B/FFFFFF/</FFFFFF<FFFFFFFFFFFFFFFF<<<##<###<####FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-26\tXS:i:-26\tXN:i:0\tXM:i:9\tXO:i:1\tXG:i:4\tNM:i:13\tMD:Z:26^ACAA26A0A1C0A0A1C0A0A0A37\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t163\tNC_035780.1_GA_converted\t8090028\t6\t100M\t=\t8090288\t364\tCCCCTTAAAAAAAAACAAAACCCTTCATTCAAACAAACTTAAATCCCCTTCACCTAAAAATACTTTATATCAAATTTAATTAAAATTAACCCAATAATTC\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-24\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:29T70\tYS:i:-26\tYT:Z:CP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_1_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    19427 (19.43%) aligned concordantly 0 times\n",
      "    3105 (3.10%) aligned concordantly exactly 1 time\n",
      "    77468 (77.47%) aligned concordantly >1 times\n",
      "80.57% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    19585 (19.59%) aligned concordantly 0 times\n",
      "    3108 (3.11%) aligned concordantly exactly 1 time\n",
      "    77307 (77.31%) aligned concordantly >1 times\n",
      "80.42% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    18409 (18.41%) aligned concordantly 0 times\n",
      "    3566 (3.57%) aligned concordantly exactly 1 time\n",
      "    78025 (78.03%) aligned concordantly >1 times\n",
      "81.59% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    18562 (18.56%) aligned concordantly 0 times\n",
      "    3472 (3.47%) aligned concordantly exactly 1 time\n",
      "    77966 (77.97%) aligned concordantly >1 times\n",
      "81.44% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_1_s1_R1.fastq.gz_C_to_T.fastq, zr2096_1_s1_R1.fastq.gz_G_to_A.fastq, zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t2232980\n",
      "\n",
      "Total methylated C's in CpG context:\t137788\n",
      "Total methylated C's in CHG context:\t45104\n",
      "Total methylated C's in CHH context:\t464015\n",
      "Total methylated C's in Unknown context:\t97154\n",
      "\n",
      "Total unmethylated C's in CpG context:\t77444\n",
      "Total unmethylated C's in CHG context:\t285519\n",
      "Total unmethylated C's in CHH context:\t1223110\n",
      "Total unmethylated C's in Unknown context:\t130947\n",
      "\n",
      "C methylated in CpG context:\t64.0%\n",
      "C methylated in CHG context:\t13.6%\n",
      "C methylated in CHH context:\t27.5%\n",
      "C methylated in unknown context (CN or CHN):\t42.6%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 10m 9s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_2_s1_R1.fastq.gz to zr2096_2_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_2_s1_R1.fastq.gz to zr2096_2_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_2_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_2_s1_R2.fastq.gz to zr2096_2_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_2_s1_R2.fastq.gz to zr2096_2_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_2_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t99\tNC_035788.1_CT_converted\t35644177\t2\t11M3I13M9I22M2D3M2I1M1I6M2I4M2I8M6I7M\t=\t35644107\t-147\tTTTTTTATTAAAATTTATTATTTTAAAAATAAAAATTNNNNTNNNTTTTATTATTTTTTTTAATATTTTTTAATATAATTAAATAATAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-164\tXS:i:-155\tXN:i:0\tXM:i:13\tXO:i:8\tXG:i:27\tNM:i:40\tMD:Z:9T6T4G0G2T0T0T0T1A0T0A13^GG14G10A3\tYS:i:-92\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t147\tNC_035788.1_CT_converted\t35644107\t2\t6M1I92M\t=\t35644177\t147\tAAATAATAAAAAAAATTAATAAATAATTTTTTAAATATTGTTGTATTATTAAAAATATTGTTGTATTAATTTTTTTTATTTATTATTTTTTTGAAGTGA\tFFB/FFBFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-92\tXS:i:-143\tXN:i:0\tXM:i:14\tXO:i:1\tXG:i:1\tNM:i:15\tMD:Z:1G0G1T14G4G7G0G4T13G0T38G1T1T0T0\tYS:i:-164\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tCTCCTTATTAAAATCTACTATTCCAAAAATAAAAACTNNNNCNNNTTCCACCATTCCTTCTAACATCTCCCAATATAACTAAATAACAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTGTTTTGGAAAAATAATGGATAAAAAAGGTTGATGTGGTGATGTTTTTAGTGATGTGGTGGTGTTTGAGAAATTATTTATTAATTTTTTTTGTTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t83\tNC_035780.1_CT_converted\t29673567\t2\t14M3I3M2I6M2I13M1D30M2D7M3D2M1I9M1D8M\t=\t29673488\t-178\tATTAATTTTTTTTGTTATTTAGTTATATTGGGAGATGTTAGAAGGAATGGTGGAANNNGNNNNAGTTTTTATTTTTGGAATAGTAGATTTTAATAAGGAG\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFF<<<###<####FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-176\tXS:i:-187\tXN:i:0\tXM:i:21\tXO:i:8\tXG:i:15\tNM:i:36\tMD:Z:14G9T1T2A3T2^T8A3A0T0T0T0T0T0A0A1T8^AG3A0T2^ATT3T7^T3T1T1T0\tYS:i:-41\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t163\tNC_035780.1_CT_converted\t29673488\t2\t93M2I4M\t=\t29673567\t178\tTTGTTTTGGAAAAATAATGGATAAAAAAGGTTGATGTGGTGATGTTTTTAGTGATGTGGTGGTGTTTGAGAAATTATTTATTAATTTTTTTTGTTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tAS:i:-41\tXS:i:-41\tXN:i:0\tXM:i:5\tXO:i:1\tXG:i:2\tNM:i:7\tMD:Z:0A0A0A41A48G3\tYS:i:-176\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t83\tNC_035786.1_GA_converted\t15444144\t2\t9M5I4M3I16M2I16M7I6M3I16M4I4M2D5M\t=\t15444158\t93\tATTAATTTTTTTTATTATTTAATTATATTAAAAAATATTAAAAAAAATAATAAAANNNANNNNAATTTTTATTTTTAAAATAATAAATTTTAATAAAAAA\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFF<<<###<####FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-162\tXS:i:-166\tXN:i:0\tXM:i:9\tXO:i:7\tXG:i:26\tNM:i:35\tMD:Z:1C3A0A8C1C5T12T8T12T12^TT5\tYS:i:-162\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t163\tNC_035786.1_GA_converted\t15444158\t2\t4M8I30M1D24M3I4M2I5M2I2M6I9M\t=\t15444144\t-93\tTCACTTCAAAAAAATAATAAATAAAAAAAATTAATACAACAATATTTTTAATAATACAACAATATTTAAAAAATTATTTATTAATTTTTTTTATTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tAS:i:-162\tXS:i:-163\tXN:i:0\tXM:i:11\tXO:i:6\tXG:i:22\tNM:i:33\tMD:Z:5T22A2T2^T1T9T0A1T2A17C2A3C1\tYS:i:-162\tYT:Z:CP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_2_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    13202 (13.20%) aligned concordantly 0 times\n",
      "    3521 (3.52%) aligned concordantly exactly 1 time\n",
      "    83277 (83.28%) aligned concordantly >1 times\n",
      "86.80% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    13398 (13.40%) aligned concordantly 0 times\n",
      "    3545 (3.54%) aligned concordantly exactly 1 time\n",
      "    83057 (83.06%) aligned concordantly >1 times\n",
      "86.60% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    13234 (13.23%) aligned concordantly 0 times\n",
      "    3701 (3.70%) aligned concordantly exactly 1 time\n",
      "    83065 (83.06%) aligned concordantly >1 times\n",
      "86.77% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    13114 (13.11%) aligned concordantly 0 times\n",
      "    3797 (3.80%) aligned concordantly exactly 1 time\n",
      "    83089 (83.09%) aligned concordantly >1 times\n",
      "86.89% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_2_s1_R1.fastq.gz_C_to_T.fastq, zr2096_2_s1_R1.fastq.gz_G_to_A.fastq, zr2096_2_s1_R2.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t2315114\n",
      "\n",
      "Total methylated C's in CpG context:\t191418\n",
      "Total methylated C's in CHG context:\t29517\n",
      "Total methylated C's in CHH context:\t221240\n",
      "Total methylated C's in Unknown context:\t42053\n",
      "\n",
      "Total unmethylated C's in CpG context:\t93502\n",
      "Total unmethylated C's in CHG context:\t441550\n",
      "Total unmethylated C's in CHH context:\t1337887\n",
      "Total unmethylated C's in Unknown context:\t79626\n",
      "\n",
      "C methylated in CpG context:\t67.2%\n",
      "C methylated in CHG context:\t6.3%\n",
      "C methylated in CHH context:\t14.2%\n",
      "C methylated in unknown context (CN or CHN):\t34.6%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 8m 22s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_3_s1_R1.fastq.gz to zr2096_3_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_3_s1_R1.fastq.gz to zr2096_3_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_3_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_3_s1_R2.fastq.gz to zr2096_3_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_3_s1_R2.fastq.gz to zr2096_3_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_3_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tTNTTTTAAAATAATTTAAATAATTTTATATAAATNNNNNNNNNNNNNNNTATTTAAATTAAAAATTTAAATAAATTTTTTTAATTATTTATTGTTTAATG\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAAATAAACATTAAACAATAAATAATTAAAAAAATTTATTTAAATTTTTAATTTAAATAAATTAAAAATATTAAATTTATATAAAATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tCNCTTCAAAATAATTTAAATAATCTTATATAAATNNNNNNNNNNNNNNNCACCTAAACTAAAAACTCAAATAAACTTTTCTAATCACCTATCATCCAACA\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAGATAGATGTTGGATGATAGGTGATTAGAAAAGTTTATTTGAGTTTTTAGTTTAGGTGAGTTAAAAATATTGAATTTATATAAGATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tCNCTTCAAAATAATTTAAATAATCTTATATAAATNNNNNNNNNNNNNNNCACCTAAACTAAAAACTCAAATAAACTTTTCTAATCACCTATCATCCAACA\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAGATAGATGTTGGATGATAGGTGATTAGAAAAGTTTATTTGAGTTTTTAGTTTAGGTGAGTTAAAAATATTGAATTTATATAAGATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tTNTTTTAAAATAATTTAAATAATTTTATATAAATNNNNNNNNNNNNNNNTATTTAAATTAAAAATTTAAATAAATTTTTTTAATTATTTATTGTTTAATG\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAAATAAACATTAAACAATAAATAATTAAAAAAATTTATTTAAATTTTTAATTTAAATAAATTAAAAATATTAAATTTATATAAAATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_3_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9084 (9.08%) aligned concordantly 0 times\n",
      "    3728 (3.73%) aligned concordantly exactly 1 time\n",
      "    87188 (87.19%) aligned concordantly >1 times\n",
      "90.92% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9008 (9.01%) aligned concordantly 0 times\n",
      "    3800 (3.80%) aligned concordantly exactly 1 time\n",
      "    87192 (87.19%) aligned concordantly >1 times\n",
      "90.99% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9541 (9.54%) aligned concordantly 0 times\n",
      "    3554 (3.55%) aligned concordantly exactly 1 time\n",
      "    86905 (86.91%) aligned concordantly >1 times\n",
      "90.46% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9345 (9.35%) aligned concordantly 0 times\n",
      "    3602 (3.60%) aligned concordantly exactly 1 time\n",
      "    87053 (87.05%) aligned concordantly >1 times\n",
      "90.66% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_3_s1_R1.fastq.gz_C_to_T.fastq, zr2096_3_s1_R1.fastq.gz_G_to_A.fastq, zr2096_3_s1_R2.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t2571474\n",
      "\n",
      "Total methylated C's in CpG context:\t261256\n",
      "Total methylated C's in CHG context:\t25303\n",
      "Total methylated C's in CHH context:\t130269\n",
      "Total methylated C's in Unknown context:\t24823\n",
      "\n",
      "Total unmethylated C's in CpG context:\t95714\n",
      "Total unmethylated C's in CHG context:\t531341\n",
      "Total unmethylated C's in CHH context:\t1527591\n",
      "Total unmethylated C's in Unknown context:\t73432\n",
      "\n",
      "C methylated in CpG context:\t73.2%\n",
      "C methylated in CHG context:\t4.5%\n",
      "C methylated in CHH context:\t7.9%\n",
      "C methylated in unknown context (CN or CHN):\t25.3%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 9m 32s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_4_s1_R1.fastq.gz to zr2096_4_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_4_s1_R1.fastq.gz to zr2096_4_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_4_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_4_s1_R2.fastq.gz to zr2096_4_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_4_s1_R2.fastq.gz to zr2096_4_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_4_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNGATTAGTGTATAGTGGTTGTTATGGAAATTAGNNNNNNNNNNNNNNNNNNNNNNNNNTGNANTNTTTTTATTAGAATTGTTAAGGTTTATTTGATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTCCATCATTAATTTTTAAAAAAAAAAAACCATCCTTAATATCAATTTTATAAATCAAATAAACCTTAACAATTCTAATAAAAACACTTCATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATTAATATATAATAATTATTATAAAAATTAANNNNNNNNNNNNNNNNNNNNNNNNNTANANTNTTTTTATTAAAATTATTAAAATTTATTTAATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTTTATTATTAATTTTTAAAAAAAAAAAATTATTTTTAATATTAATTTTATAAATTAAATAAATTTTAATAATTTTAATAAAAATATTTTATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATTAATATATAATAATTATTATAAAAATTAANNNNNNNNNNNNNNNNNNNNNNNNNTANANTNTTTTTATTAAAATTATTAAAATTTATTTAATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTTTATTATTAATTTTTAAAAAAAAAAAATTATTTTTAATATTAATTTTATAAATTAAATAAATTTTAATAATTTTAATAAAAATATTTTATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNGATTAGTGTATAGTGGTTGTTATGGAAATTAGNNNNNNNNNNNNNNNNNNNNNNNNNTGNANTNTTTTTATTAGAATTGTTAAGGTTTATTTGATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTCCATCATTAATTTTTAAAAAAAAAAAACCATCCTTAATATCAATTTTATAAATCAAATAAACCTTAACAATTCTAATAAAAACACTTCATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_4_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    10717 (10.72%) aligned concordantly 0 times\n",
      "    3517 (3.52%) aligned concordantly exactly 1 time\n",
      "    85766 (85.77%) aligned concordantly >1 times\n",
      "89.28% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    10802 (10.80%) aligned concordantly 0 times\n",
      "    3752 (3.75%) aligned concordantly exactly 1 time\n",
      "    85446 (85.45%) aligned concordantly >1 times\n",
      "89.20% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    10741 (10.74%) aligned concordantly 0 times\n",
      "    3801 (3.80%) aligned concordantly exactly 1 time\n",
      "    85458 (85.46%) aligned concordantly >1 times\n",
      "89.26% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    10785 (10.79%) aligned concordantly 0 times\n",
      "    3438 (3.44%) aligned concordantly exactly 1 time\n",
      "    85777 (85.78%) aligned concordantly >1 times\n",
      "89.22% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_4_s1_R1.fastq.gz_C_to_T.fastq, zr2096_4_s1_R1.fastq.gz_G_to_A.fastq, zr2096_4_s1_R2.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t2401459\n",
      "\n",
      "Total methylated C's in CpG context:\t187540\n",
      "Total methylated C's in CHG context:\t24865\n",
      "Total methylated C's in CHH context:\t139891\n",
      "Total methylated C's in Unknown context:\t26334\n",
      "\n",
      "Total unmethylated C's in CpG context:\t113392\n",
      "Total unmethylated C's in CHG context:\t483964\n",
      "Total unmethylated C's in CHH context:\t1451807\n",
      "Total unmethylated C's in Unknown context:\t72321\n",
      "\n",
      "C methylated in CpG context:\t62.3%\n",
      "C methylated in CHG context:\t4.9%\n",
      "C methylated in CHH context:\t8.8%\n",
      "C methylated in unknown context (CN or CHN):\t26.7%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 10m 9s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_5_s1_R1.fastq.gz to zr2096_5_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_5_s1_R1.fastq.gz to zr2096_5_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_5_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_5_s1_R2.fastq.gz to zr2096_5_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_5_s1_R2.fastq.gz to zr2096_5_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_5_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATAAAATATTATTTAAATTTTATTATTTATANNNNNNNNNNNNNNNNNNATTNTTTTTATTATATATATAAAATTATTTTAAAATTATTATATTTTTT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTCAAATATATTAAAAAAAAATTATTAATAATTTTCATTAAAAAATATAATAATTTTAAAATAATTTTATATATATAATAAAAAAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAACAAAACACCACTCAAATCTTACTATCCACANNNNNNNNNNNNNNNNNNATTNCTTCTATCATATATATAAAATCATTCCAAAATTACTATATTTTTC\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTTGAGTATATTGAAGAGGAATTATTGATAGTTTTTGTTGAAAAATATAGTAATTTTGGAATGATTTTATATATATGATAGAAGAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAACAAAACACCACTCAAATCTTACTATCCACANNNNNNNNNNNNNNNNNNATTNCTTCTATCATATATATAAAATCATTCCAAAATTACTATATTTTTC\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTTGAGTATATTGAAGAGGAATTATTGATAGTTTTTGTTGAAAAATATAGTAATTTTGGAATGATTTTATATATATGATAGAAGAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATAAAATATTATTTAAATTTTATTATTTATANNNNNNNNNNNNNNNNNNATTNTTTTTATTATATATATAAAATTATTTTAAAATTATTATATTTTTT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTCAAATATATTAAAAAAAAATTATTAATAATTTTCATTAAAAAATATAATAATTTTAAAATAATTTTATATATATAATAAAAAAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_5_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    8860 (8.86%) aligned concordantly 0 times\n",
      "    3285 (3.29%) aligned concordantly exactly 1 time\n",
      "    87855 (87.86%) aligned concordantly >1 times\n",
      "91.14% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9338 (9.34%) aligned concordantly 0 times\n",
      "    3142 (3.14%) aligned concordantly exactly 1 time\n",
      "    87520 (87.52%) aligned concordantly >1 times\n",
      "90.66% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    8988 (8.99%) aligned concordantly 0 times\n",
      "    3342 (3.34%) aligned concordantly exactly 1 time\n",
      "    87670 (87.67%) aligned concordantly >1 times\n",
      "91.01% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9317 (9.32%) aligned concordantly 0 times\n",
      "    3194 (3.19%) aligned concordantly exactly 1 time\n",
      "    87489 (87.49%) aligned concordantly >1 times\n",
      "90.68% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_5_s1_R1.fastq.gz_C_to_T.fastq, zr2096_5_s1_R1.fastq.gz_G_to_A.fastq, zr2096_5_s1_R2.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t2502361\n",
      "\n",
      "Total methylated C's in CpG context:\t193415\n",
      "Total methylated C's in CHG context:\t25688\n",
      "Total methylated C's in CHH context:\t154169\n",
      "Total methylated C's in Unknown context:\t29935\n",
      "\n",
      "Total unmethylated C's in CpG context:\t109516\n",
      "Total unmethylated C's in CHG context:\t484512\n",
      "Total unmethylated C's in CHH context:\t1535061\n",
      "Total unmethylated C's in Unknown context:\t78826\n",
      "\n",
      "C methylated in CpG context:\t63.8%\n",
      "C methylated in CHG context:\t5.0%\n",
      "C methylated in CHH context:\t9.1%\n",
      "C methylated in unknown context (CN or CHN):\t27.5%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 8m 38s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_6_s1_R1.fastq.gz to zr2096_6_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_6_s1_R1.fastq.gz to zr2096_6_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_6_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_6_s1_R2.fastq.gz to zr2096_6_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_6_s1_R2.fastq.gz to zr2096_6_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_6_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTTATTAAATTTAATTTTATTATTAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNTANANAAATATTAATTAAATATTTTTAAAATAATAAATTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAAAATTTTTTTAAATTTATTATTTTAAAAATATTTAATTAATATTTATTTATATTTTATAATTAACAATAATTATTTTTTTATTTAATAATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTCATCAAATCCAACTTTATCACCAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNCANANAAACATCAATCAAACATCTCTAAAACAATAAACTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAGAATTTTTTTAAGTTTATTGTTTTAGAGATGTTTGATTGATGTTTATTTGTATTTTGTAATTAGTGGTGGTTGTTTTTTTATTTGGTGATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTCATCAAATCCAACTTTATCACCAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNCANANAAACATCAATCAAACATCTCTAAAACAATAAACTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAGAATTTTTTTAAGTTTATTGTTTTAGAGATGTTTGATTGATGTTTATTTGTATTTTGTAATTAGTGGTGGTTGTTTTTTTATTTGGTGATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTTATTAAATTTAATTTTATTATTAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNTANANAAATATTAATTAAATATTTTTAAAATAATAAATTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAAAATTTTTTTAAATTTATTATTTTAAAAATATTTAATTAATATTTATTTATATTTTATAATTAACAATAATTATTTTTTTATTTAATAATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_6_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9464 (9.46%) aligned concordantly 0 times\n",
      "    3742 (3.74%) aligned concordantly exactly 1 time\n",
      "    86794 (86.79%) aligned concordantly >1 times\n",
      "90.54% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9268 (9.27%) aligned concordantly 0 times\n",
      "    3706 (3.71%) aligned concordantly exactly 1 time\n",
      "    87026 (87.03%) aligned concordantly >1 times\n",
      "90.73% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9335 (9.34%) aligned concordantly 0 times\n",
      "    3176 (3.18%) aligned concordantly exactly 1 time\n",
      "    87489 (87.49%) aligned concordantly >1 times\n",
      "90.67% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9341 (9.34%) aligned concordantly 0 times\n",
      "    3203 (3.20%) aligned concordantly exactly 1 time\n",
      "    87456 (87.46%) aligned concordantly >1 times\n",
      "90.66% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_6_s1_R1.fastq.gz_C_to_T.fastq, zr2096_6_s1_R1.fastq.gz_G_to_A.fastq, zr2096_6_s1_R2.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t2443376\n",
      "\n",
      "Total methylated C's in CpG context:\t214807\n",
      "Total methylated C's in CHG context:\t23866\n",
      "Total methylated C's in CHH context:\t130796\n",
      "Total methylated C's in Unknown context:\t23463\n",
      "\n",
      "Total unmethylated C's in CpG context:\t95641\n",
      "Total unmethylated C's in CHG context:\t494878\n",
      "Total unmethylated C's in CHH context:\t1483388\n",
      "Total unmethylated C's in Unknown context:\t71902\n",
      "\n",
      "C methylated in CpG context:\t69.2%\n",
      "C methylated in CHG context:\t4.6%\n",
      "C methylated in CHH context:\t8.1%\n",
      "C methylated in unknown context (CN or CHN):\t24.6%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 7m 54s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_7_s1_R1.fastq.gz to zr2096_7_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_7_s1_R1.fastq.gz to zr2096_7_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_7_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_7_s1_R2.fastq.gz to zr2096_7_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_7_s1_R2.fastq.gz to zr2096_7_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_7_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t99\tNC_035787.1_CT_converted\t51746759\t2\t8M5I5M5I4M3I9M4I24M1I2M2I2M6I6M4I5M\t=\t51746759\t-65\tAAAAAAATGATTATTATGATGAATTTAAATATTAATAATAAAATTTAAAAAAAATATAAATGTTAAATATTAAATTTTTTAAAAAATGATTATGG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-166\tXS:i:-185\tXN:i:0\tXM:i:6\tXO:i:8\tXG:i:30\tNM:i:36\tMD:Z:0T1G0G16T0G1T41\tYS:i:-166\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t147\tNC_035787.1_CT_converted\t51746759\t2\t8M5I5M5I4M3I9M4I24M1I2M2I2M6I6M4I5M\t=\t51746759\t-65\tAAAAAAATGATTATTATGATGAATTTAAATATTAATAATAAAATTTAAAAAAAATATAAATGTTAAATATTAAATTTTTTAAAAAATGATTATGG\tFFFFFFFFFFFFFFFFFFFFFFFBBFFFFFFBBFF/FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-166\tXN:i:0\tXM:i:6\tXO:i:8\tXG:i:30\tNM:i:36\tMD:Z:0T1G0G16T0G1T41\tYS:i:-166\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t99\tNC_035781.1_GA_converted\t24539746\t6\t95M\t=\t24539746\t-95\tAAAAAAACAACTACTACAACAAATTCAAATATTAATAACAAAATTTAAAAAAAACATAAACATTAAATATCAAATTCTTCAAAAAACAATTACAA\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-12\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:94T0\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t147\tNC_035781.1_GA_converted\t24539746\t6\t95M\t=\t24539746\t-95\tAAAAAAACAACTACTACAACAAATTCAAATATTAATAACAAAATTTAAAAAAAACATAAACATTAAATATCAAATTCTTCAAAAAACAATTACAA\tFFFFFFFFFFFFFFFFFFFFFFFBBFFFFFFBBFF/FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-6\tXS:i:-12\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:94T0\tYS:i:-6\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t83\tNC_035780.1_CT_converted\t42548681\t6\t95M\t=\t42548681\t-95\tTTGTAATTGTTTTTTGAAGAATTTGATATTTAATGTTTATGTTTTTTTTAAATTTTGTTATTAATATTTGAATTTGTTGTAGTAGTTGTTTTTTT\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-6\tXS:i:-24\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:0A94\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t163\tNC_035780.1_CT_converted\t42548681\t6\t95M\t=\t42548681\t-95\tTTGTAATTGTTTTTTGAAGAATTTGATATTTAATGTTTATGTTTTTTTTAAATTTTGTTATTAATATTTGAATTTGTTGTAGTAGTTGTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-24\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:0A94\tYS:i:-6\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t83\tNC_035782.1_GA_converted\t64740147\t2\t6M6I39M2I4M2I1M2I4M2I5M1D12M6I4M\t=\t64740147\t-76\tCCATAATCATTTTTTAAAAAATTTAATATTTAACATTTATATTTTTTTTAAATTTTATTATTAATATTTAAATTCATCATAATAATCATTTTTTT\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-164\tXS:i:-167\tXN:i:0\tXM:i:11\tXO:i:7\tXG:i:21\tNM:i:32\tMD:Z:0T0T6A26A1C3A1C15^A3A4T5A0A0\tYS:i:-164\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t163\tNC_035782.1_GA_converted\t64740147\t2\t6M6I39M2I4M2I1M2I4M2I5M1D12M6I4M\t=\t64740147\t-76\tCCATAATCATTTTTTAAAAAATTTAATATTTAACATTTATATTTTTTTTAAATTTTATTATTAATATTTAAATTCATCATAATAATCATTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-164\tXS:i:-167\tXN:i:0\tXM:i:11\tXO:i:7\tXG:i:21\tNM:i:32\tMD:Z:0T0T6A26A1C3A1C15^A3A4T5A0A0\tYS:i:-164\tYT:Z:CP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_7_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9164 (9.16%) aligned concordantly 0 times\n",
      "    3744 (3.74%) aligned concordantly exactly 1 time\n",
      "    87092 (87.09%) aligned concordantly >1 times\n",
      "90.84% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9067 (9.07%) aligned concordantly 0 times\n",
      "    3824 (3.82%) aligned concordantly exactly 1 time\n",
      "    87109 (87.11%) aligned concordantly >1 times\n",
      "90.93% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9311 (9.31%) aligned concordantly 0 times\n",
      "    3400 (3.40%) aligned concordantly exactly 1 time\n",
      "    87289 (87.29%) aligned concordantly >1 times\n",
      "90.69% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9266 (9.27%) aligned concordantly 0 times\n",
      "    3281 (3.28%) aligned concordantly exactly 1 time\n",
      "    87453 (87.45%) aligned concordantly >1 times\n",
      "90.73% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_7_s1_R1.fastq.gz_C_to_T.fastq, zr2096_7_s1_R1.fastq.gz_G_to_A.fastq, zr2096_7_s1_R2.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t2551148\n",
      "\n",
      "Total methylated C's in CpG context:\t209726\n",
      "Total methylated C's in CHG context:\t26915\n",
      "Total methylated C's in CHH context:\t146076\n",
      "Total methylated C's in Unknown context:\t27898\n",
      "\n",
      "Total unmethylated C's in CpG context:\t102122\n",
      "Total unmethylated C's in CHG context:\t507713\n",
      "Total unmethylated C's in CHH context:\t1558596\n",
      "Total unmethylated C's in Unknown context:\t80503\n",
      "\n",
      "C methylated in CpG context:\t67.3%\n",
      "C methylated in CHG context:\t5.0%\n",
      "C methylated in CHH context:\t8.6%\n",
      "C methylated in unknown context (CN or CHN):\t25.7%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 10m 24s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_8_s1_R1.fastq.gz to zr2096_8_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_8_s1_R1.fastq.gz to zr2096_8_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_8_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_8_s1_R2.fastq.gz to zr2096_8_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_8_s1_R2.fastq.gz to zr2096_8_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_8_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t99\tNC_035788.1_CT_converted\t62738085\t2\t21M1I5M2I2M2I55M\t=\t62738085\t-83\tTAATTTAAAATTAAAAAATTTATTAAATTTATTATTATTTNTTATTTTTTATAAAAGAAAATAGAGAGAGAGAGAGAGAGAGAGAGAG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tAS:i:-115\tXS:i:-121\tXN:i:0\tXM:i:15\tXO:i:3\tXG:i:5\tNM:i:20\tMD:Z:1T0T15A10A3A0A0A0A0A2A0A0A1G1A0G35\tYS:i:-114\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t147\tNC_035788.1_CT_converted\t62738085\t2\t21M1I5M2I2M2I55M\t=\t62738085\t-83\tTAATTTAAAATTAAAAAATTTATTAAATTTATTATTATTTATTATTTTTTATAAAAGAAAATAGAGAGAGAGAGAGAGAGAGAGAGAG\t/FB//BFFF<//FFFFFF/FFB/<FF<//F/<<</FF</FF<<<</<F/<FBFFFF<FFFF/FFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-114\tXS:i:-120\tXN:i:0\tXM:i:14\tXO:i:3\tXG:i:5\tNM:i:19\tMD:Z:1T0T15A10A3A0A1A0A2A0A0A1G1A0G35\tYS:i:-115\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t99\tNC_035786.1_GA_converted\t2855541\t2\t10M2I7M7I7M2I2M3I9M2I37M\t=\t2855541\t-72\tTAATCCAAAATTAAAAAACTCATTAAATCTATCATCATTCNCCATTCTTCACAAAAAAAAATAAAAAAAAAAAAAAAAAAAAAAAAAA\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tAS:i:-116\tXS:i:-115\tXN:i:0\tXM:i:8\tXO:i:5\tXG:i:16\tNM:i:24\tMD:Z:4T0A4C9A5A0A3C3T36\tYS:i:-115\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t147\tNC_035786.1_GA_converted\t2855541\t2\t10M2I7M7I7M2I2M3I9M2I37M\t=\t2855541\t-72\tTAATCCAAAATTAAAAAACTCATTAAATCTATCATCATTCACCATTCTTCACAAAAAAAAATAAAAAAAAAAAAAAAAAAAAAAAAAA\t/FB//BFFF<//FFFFFF/FFB/<FF<//F/<<</FF</FF<<<</<F/<FBFFFF<FFFF/FFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-115\tXS:i:-119\tXN:i:0\tXM:i:7\tXO:i:5\tXG:i:16\tNM:i:23\tMD:Z:4T0A4C9A6A3C3T36\tYS:i:-116\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t83\tNC_007175.2_CT_converted\t14567\t0\t2M17I69M\t=\t14567\t-71\tTTTTTTTTTTTTTTTTTTTTTTTTTTATTTTTTTTTGTGAAGAATGGNGAATGATGATAGATTTAATGAGTTTTTTAATTTTGGATTA\tFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<<#FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-81\tXS:i:-167\tXN:i:0\tXM:i:5\tXO:i:1\tXG:i:17\tNM:i:22\tMD:Z:1A5A0G5A15T40\tYS:i:-80\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t163\tNC_007175.2_CT_converted\t14567\t0\t2M17I69M\t=\t14567\t-71\tTTTTTTTTTTTTTTTTTTTTTTTTTTATTTTTTTTTGTGAAGAATGGTGAATGATGATAGATTTAATGAGTTTTTTAATTTTGGATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tAS:i:-80\tXN:i:0\tXM:i:4\tXO:i:1\tXG:i:17\tNM:i:21\tMD:Z:1A5A0G5A56\tYS:i:-81\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t83\tNC_035785.1_GA_converted\t16582012\t2\t38M3I13M1I19M4I10M\t=\t16582012\t-80\tCTCTCTCTCTCTCTCTCTCTCTCTCTATTTTCTTTTATAAAAAATAANAAATAATAATAAATTTAATAAATTTTTTAATTTTAAATTA\tFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<<#FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-118\tXS:i:-119\tXN:i:0\tXM:i:14\tXO:i:3\tXG:i:8\tNM:i:22\tMD:Z:35C0T1C1T1T1T0T0T0T1T4T0T21A0A1\tYS:i:-117\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t163\tNC_035785.1_GA_converted\t16582012\t2\t38M3I13M1I19M4I10M\t=\t16582012\t-80\tCTCTCTCTCTCTCTCTCTCTCTCTCTATTTTCTTTTATAAAAAATAATAAATAATAATAAATTTAATAAATTTTTTAATTTTAAATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tAS:i:-117\tXS:i:-118\tXN:i:0\tXM:i:13\tXO:i:3\tXG:i:8\tNM:i:21\tMD:Z:35C0T1C1T1T2T0T0T1T4T0T21A0A1\tYS:i:-118\tYT:Z:CP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_8_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    11238 (11.24%) aligned concordantly 0 times\n",
      "    3759 (3.76%) aligned concordantly exactly 1 time\n",
      "    85003 (85.00%) aligned concordantly >1 times\n",
      "88.76% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    11492 (11.49%) aligned concordantly 0 times\n",
      "    3704 (3.70%) aligned concordantly exactly 1 time\n",
      "    84804 (84.80%) aligned concordantly >1 times\n",
      "88.51% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    11544 (11.54%) aligned concordantly 0 times\n",
      "    3392 (3.39%) aligned concordantly exactly 1 time\n",
      "    85064 (85.06%) aligned concordantly >1 times\n",
      "88.46% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    11598 (11.60%) aligned concordantly 0 times\n",
      "    3343 (3.34%) aligned concordantly exactly 1 time\n",
      "    85059 (85.06%) aligned concordantly >1 times\n",
      "88.40% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_8_s1_R1.fastq.gz_C_to_T.fastq, zr2096_8_s1_R1.fastq.gz_G_to_A.fastq, zr2096_8_s1_R2.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t2427897\n",
      "\n",
      "Total methylated C's in CpG context:\t172771\n",
      "Total methylated C's in CHG context:\t29377\n",
      "Total methylated C's in CHH context:\t212768\n",
      "Total methylated C's in Unknown context:\t41165\n",
      "\n",
      "Total unmethylated C's in CpG context:\t101550\n",
      "Total unmethylated C's in CHG context:\t455554\n",
      "Total unmethylated C's in CHH context:\t1455877\n",
      "Total unmethylated C's in Unknown context:\t87081\n",
      "\n",
      "C methylated in CpG context:\t63.0%\n",
      "C methylated in CHG context:\t6.1%\n",
      "C methylated in CHH context:\t12.8%\n",
      "C methylated in unknown context (CN or CHN):\t32.1%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 8m 43s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_9_s1_R1.fastq.gz to zr2096_9_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_9_s1_R1.fastq.gz to zr2096_9_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_9_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_9_s1_R2.fastq.gz to zr2096_9_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_9_s1_R2.fastq.gz to zr2096_9_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_9_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNATTAAATAAAAAATAATAAATAATAAATTANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAGG\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAAAAAATAAAAAACACATTAAAAATCTATAATTTAATTTATTATTTATTATTTTTTATTTAATTAAATCNNNNAAACATCATATAAAAAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNACCAAATAAAAAATAACAAATAATAAATCANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAAA\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAGAGGATAGGAGATATGTTAGGAATTTATAATTTGATTTATTATTTGTTATTTTTTATTTGGTTAGATTNNNNGAGTGTTGTGTAGGGAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNACCAAATAAAAAATAACAAATAATAAATCANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAAA\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAGAGGATAGGAGATATGTTAGGAATTTATAATTTGATTTATTATTTGTTATTTTTTATTTGGTTAGATTNNNNGAGTGTTGTGTAGGGAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-2 --ignore-quals --no-mixed --no-discordant --dovetail --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNATTAAATAAAAAATAATAAATAATAAATTANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAGG\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAAAAAATAAAAAACACATTAAAAATCTATAATTTAATTTATTATTTATTATTTTTTATTTAATTAAATCNNNNAAACATCATATAAAAAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_9_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9995 (9.99%) aligned concordantly 0 times\n",
      "    4021 (4.02%) aligned concordantly exactly 1 time\n",
      "    85984 (85.98%) aligned concordantly >1 times\n",
      "90.00% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    9950 (9.95%) aligned concordantly 0 times\n",
      "    4042 (4.04%) aligned concordantly exactly 1 time\n",
      "    86008 (86.01%) aligned concordantly >1 times\n",
      "90.05% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    10189 (10.19%) aligned concordantly 0 times\n",
      "    3321 (3.32%) aligned concordantly exactly 1 time\n",
      "    86490 (86.49%) aligned concordantly >1 times\n",
      "89.81% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    10250 (10.25%) aligned concordantly 0 times\n",
      "    3461 (3.46%) aligned concordantly exactly 1 time\n",
      "    86289 (86.29%) aligned concordantly >1 times\n",
      "89.75% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_9_s1_R1.fastq.gz_C_to_T.fastq, zr2096_9_s1_R1.fastq.gz_G_to_A.fastq, zr2096_9_s1_R2.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t2633309\n",
      "\n",
      "Total methylated C's in CpG context:\t256604\n",
      "Total methylated C's in CHG context:\t29608\n",
      "Total methylated C's in CHH context:\t150781\n",
      "Total methylated C's in Unknown context:\t30522\n",
      "\n",
      "Total unmethylated C's in CpG context:\t97851\n",
      "Total unmethylated C's in CHG context:\t535057\n",
      "Total unmethylated C's in CHH context:\t1563408\n",
      "Total unmethylated C's in Unknown context:\t81232\n",
      "\n",
      "C methylated in CpG context:\t72.4%\n",
      "C methylated in CHG context:\t5.2%\n",
      "C methylated in CHH context:\t8.8%\n",
      "C methylated in unknown context (CN or CHN):\t27.3%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 8m 55s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "find /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_*R1* \\\n",
    "| xargs basename -s _s1_R1.fastq.gz | xargs -I{} /Users/Shelly/anaconda3/bin/bismark \\\n",
    "--genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ \\\n",
    "--score_min L,0,-2 \\\n",
    "-u 100000 \\\n",
    "--non_directional \\\n",
    "-1 /Volumes/web/seashell/bu-serine-wd/18-04-07/{}_s1_R1.fastq.gz \\\n",
    "-2 /Volumes/web/seashell/bu-serine-wd/18-04-07/{}_s1_R2.fastq.gz \\\n",
    "-o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/ "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#run deduplicate on looser alignment score threshold alignments"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_10_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_1_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_2_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_3_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_4_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_5_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_6_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_7_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_8_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_9_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-2 -u 100000 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\"\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/\n",
      "Processing paired-end Bismark output file(s) (SAM format):\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_10_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_1_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_2_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_3_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_4_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_5_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_6_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_7_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_8_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_9_s1_R1_bismark_bt2_pe.bam\n",
      "\n",
      "\n",
      "If there are several alignments to a single position in the genome the first alignment will be chosen. Since the input files are not in any way sorted this is a near-enough random selection of reads.\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_10_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_10_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_10_s1_R1_bismark_bt2_pe.bam:\t79505\n",
      "Total number duplicated alignments removed:\t32 (0.04%)\n",
      "Duplicated alignments were found at:\t28 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 79473 (99.96% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_1_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_1_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_1_s1_R1_bismark_bt2_pe.bam:\t84845\n",
      "Total number duplicated alignments removed:\t645 (0.76%)\n",
      "Duplicated alignments were found at:\t537 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 84200 (99.24% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_2_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_2_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_2_s1_R1_bismark_bt2_pe.bam:\t79272\n",
      "Total number duplicated alignments removed:\t207 (0.26%)\n",
      "Duplicated alignments were found at:\t166 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 79065 (99.74% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_3_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_3_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_3_s1_R1_bismark_bt2_pe.bam:\t78192\n",
      "Total number duplicated alignments removed:\t19 (0.02%)\n",
      "Duplicated alignments were found at:\t19 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 78173 (99.98% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_4_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_4_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_4_s1_R1_bismark_bt2_pe.bam:\t78339\n",
      "Total number duplicated alignments removed:\t20 (0.03%)\n",
      "Duplicated alignments were found at:\t19 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 78319 (99.97% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_5_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_5_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_5_s1_R1_bismark_bt2_pe.bam:\t79734\n",
      "Total number duplicated alignments removed:\t30 (0.04%)\n",
      "Duplicated alignments were found at:\t30 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 79704 (99.96% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_6_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_6_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_6_s1_R1_bismark_bt2_pe.bam:\t78624\n",
      "Total number duplicated alignments removed:\t14 (0.02%)\n",
      "Duplicated alignments were found at:\t14 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 78610 (99.98% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_7_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_7_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_7_s1_R1_bismark_bt2_pe.bam:\t80059\n",
      "Total number duplicated alignments removed:\t17 (0.02%)\n",
      "Duplicated alignments were found at:\t17 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 80042 (99.98% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_8_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_8_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_8_s1_R1_bismark_bt2_pe.bam:\t81008\n",
      "Total number duplicated alignments removed:\t120 (0.15%)\n",
      "Duplicated alignments were found at:\t103 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 80888 (99.85% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_9_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_9_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_9_s1_R1_bismark_bt2_pe.bam:\t79357\n",
      "Total number duplicated alignments removed:\t169 (0.21%)\n",
      "Duplicated alignments were found at:\t97 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 79188 (99.79% of total)\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "/Users/Shelly/anaconda3/bin/deduplicate_bismark \\\n",
    "--bam -p \\\n",
    "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/*.bam \\\n",
    "-o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/ \n",
    "2> /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/dedup.err"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#create summary report of mapping efficiency and dedup. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/*PE_report.txt | \\\n",
    "grep 'Mapping\\ efficiency\\:' | \\\n",
    "cat - /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/*.deduplication_report.txt > /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_mapping_dedup_summary.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#Bismark analysis with fragment length threshold (60bp) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!mkdir /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t52694\n",
      "Mapping efficiency:\t52.7%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t31468\n",
      "Sequence pairs did not map uniquely:\t15838\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t13187\t((converted) top strand)\n",
      "GA/CT/CT:\t13076\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t13300\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t13131\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t22900\n",
      "Mapping efficiency:\t22.9%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t70581\n",
      "Sequence pairs did not map uniquely:\t6519\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t5684\t((converted) top strand)\n",
      "GA/CT/CT:\t5733\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t5718\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t5765\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t45073\n",
      "Mapping efficiency:\t45.1%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t41197\n",
      "Sequence pairs did not map uniquely:\t13730\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t11397\t((converted) top strand)\n",
      "GA/CT/CT:\t11214\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t11082\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t11380\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t51869\n",
      "Mapping efficiency:\t51.9%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t31536\n",
      "Sequence pairs did not map uniquely:\t16595\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t13049\t((converted) top strand)\n",
      "GA/CT/CT:\t12928\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t12824\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t13068\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t51868\n",
      "Mapping efficiency:\t51.9%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t32383\n",
      "Sequence pairs did not map uniquely:\t15749\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t13196\t((converted) top strand)\n",
      "GA/CT/CT:\t12740\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t12633\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t13299\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t50679\n",
      "Mapping efficiency:\t50.7%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t34356\n",
      "Sequence pairs did not map uniquely:\t14965\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t13032\t((converted) top strand)\n",
      "GA/CT/CT:\t12327\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t12248\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t13072\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t52453\n",
      "Mapping efficiency:\t52.5%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t31628\n",
      "Sequence pairs did not map uniquely:\t15919\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t13781\t((converted) top strand)\n",
      "GA/CT/CT:\t12327\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t12471\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t13874\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t50979\n",
      "Mapping efficiency:\t51.0%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t34216\n",
      "Sequence pairs did not map uniquely:\t14805\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t12999\t((converted) top strand)\n",
      "GA/CT/CT:\t12330\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t12617\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t13033\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t45836\n",
      "Mapping efficiency:\t45.8%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t40924\n",
      "Sequence pairs did not map uniquely:\t13240\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t11596\t((converted) top strand)\n",
      "GA/CT/CT:\t11367\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t11289\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t11584\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t49112\n",
      "Mapping efficiency:\t49.1%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t35939\n",
      "Sequence pairs did not map uniquely:\t14949\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t13133\t((converted) top strand)\n",
      "GA/CT/CT:\t11436\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t11407\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t13136\t((converted) bottom strand)\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_10_s1_R1.fastq.gz to zr2096_10_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_10_s1_R1.fastq.gz to zr2096_10_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_10_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_10_s1_R2.fastq.gz to zr2096_10_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_10_s1_R2.fastq.gz to zr2096_10_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_10_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tGTTATATATAATTGTATTAATTATAAAAAATATNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATTTATTATTAAAAATATTAAATATATTATTTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNATTTTATATTTTTAATAATAAATATAATTATATATTAAAATAAAATAATATATTTAATATTTTTAATAATAAATACAATTAAATATCATAATAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tACCATATACAATCATATCAACTATAAAAAATACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATCTACTATCAAAAATACTAAACACACTACCTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNGTTTTGTATTTTTGATAGTAGATATGATTGTATATTAGGGTAAGGTAGTGTGTTTAGTATTTTTGATAGTAGATATGATTGAATATTGTGGTAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tACCATATACAATCATATCAACTATAAAAAATACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATCTACTATCAAAAATACTAAACACACTACCTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNGTTTTGTATTTTTGATAGTAGATATGATTGTATATTAGGGTAAGGTAGTGTGTTTAGTATTTTTGATAGTAGATATGATTGAATATTGTGGTAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tGTTATATATAATTGTATTAATTATAAAAAATATNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATTTATTATTAAAAATATTAAATATATTATTTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNATTTTATATTTTTAATAATAAATATAATTATATATTAAAATAAAATAATATATTTAATATTTTTAATAATAAATACAATTAAATATCATAATAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_10_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    73640 (73.64%) aligned concordantly 0 times\n",
      "    11329 (11.33%) aligned concordantly exactly 1 time\n",
      "    15031 (15.03%) aligned concordantly >1 times\n",
      "26.36% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    73718 (73.72%) aligned concordantly 0 times\n",
      "    11261 (11.26%) aligned concordantly exactly 1 time\n",
      "    15021 (15.02%) aligned concordantly >1 times\n",
      "26.28% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    73649 (73.65%) aligned concordantly 0 times\n",
      "    11198 (11.20%) aligned concordantly exactly 1 time\n",
      "    15153 (15.15%) aligned concordantly >1 times\n",
      "26.35% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    73542 (73.54%) aligned concordantly 0 times\n",
      "    11416 (11.42%) aligned concordantly exactly 1 time\n",
      "    15042 (15.04%) aligned concordantly >1 times\n",
      "26.46% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_10_s1_R1.fastq.gz_C_to_T.fastq, zr2096_10_s1_R1.fastq.gz_G_to_A.fastq, zr2096_10_s1_R2.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1858912\n",
      "\n",
      "Total methylated C's in CpG context:\t180434\n",
      "Total methylated C's in CHG context:\t13482\n",
      "Total methylated C's in CHH context:\t45621\n",
      "Total methylated C's in Unknown context:\t6075\n",
      "\n",
      "Total unmethylated C's in CpG context:\t81969\n",
      "Total unmethylated C's in CHG context:\t411935\n",
      "Total unmethylated C's in CHH context:\t1125471\n",
      "Total unmethylated C's in Unknown context:\t25306\n",
      "\n",
      "C methylated in CpG context:\t68.8%\n",
      "C methylated in CHG context:\t3.2%\n",
      "C methylated in CHH context:\t3.9%\n",
      "C methylated in unknown context (CN or CHN):\t19.4%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 37s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t99\tNC_035780.1_CT_converted\t65371909\t6\t100M\t=\t65372013\t204\tGAGTTTTTTTGATTATTTGTTGTTTGTTGTTTGTTTGNNNNTNNNTNNGTTTGTTTGTTTGTTTGTTTGTTTGTAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tAS:i:-15\tXS:i:-22\tXN:i:0\tXM:i:10\tXO:i:0\tXG:i:0\tNM:i:10\tMD:Z:37T0T0T0G1T0T0G1T0T10A41\tYS:i:-24\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t147\tNC_035780.1_CT_converted\t65372013\t6\t100M\t=\t65371909\t-204\tGAATTATTGGGTTAATTTTAATTAAATTTGATATAAAGTATTTTTAGGTGAAGGGGATTTAAGTTTGTTTGAATGAAGGGTTTTGTTTTTTTTTAAGGGG\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-24\tXS:i:-6\tXN:i:0\tXM:i:4\tXO:i:0\tXG:i:0\tNM:i:4\tMD:Z:30G14G3A20A29\tYS:i:-15\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTAAAAAAAAATGAAATTTTTTATTTAAATAAATTTAAATTTTTTTTATTTAAAAATATTTTATATTAAATTTAATTAAAATTAATTTAATAATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTAAAAAAAAATGAAATTTTTTATTTAAATAAATTTAAATTTTTTTTATTTAAAAATATTTTATATTAAATTTAATTAAAATTAATTTAATAATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t83\tNC_035780.1_GA_converted\t8090288\t7\t26M4D74M\t=\t8090028\t-364\tAAAAAAAATAAAAATATAAAAAATTTACAAACAAACAAACAAACAAACAAACNNANNNANNNNCAAACAAACAACAAACAACAAATAATCAAAAAAACTC\tFFFFFFFF/FFFFF/B/FFFFFF/</FFFFFF<FFFFFFFFFFFFFFFF<<<##<###<####FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-26\tXS:i:-22\tXN:i:0\tXM:i:9\tXO:i:1\tXG:i:4\tNM:i:13\tMD:Z:26^ACAA26A0A1C0A0A1C0A0A0A37\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t163\tNC_035780.1_GA_converted\t8090028\t7\t100M\t=\t8090288\t364\tCCCCTTAAAAAAAAACAAAACCCTTCATTCAAACAAACTTAAATCCCCTTCACCTAAAAATACTTTATATCAAATTTAATTAAAATTAACCCAATAATTC\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-36\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:29T70\tYS:i:-26\tYT:Z:CP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_1_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    88629 (88.63%) aligned concordantly 0 times\n",
      "    4908 (4.91%) aligned concordantly exactly 1 time\n",
      "    6463 (6.46%) aligned concordantly >1 times\n",
      "11.37% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    88691 (88.69%) aligned concordantly 0 times\n",
      "    4888 (4.89%) aligned concordantly exactly 1 time\n",
      "    6421 (6.42%) aligned concordantly >1 times\n",
      "11.31% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    88873 (88.87%) aligned concordantly 0 times\n",
      "    4833 (4.83%) aligned concordantly exactly 1 time\n",
      "    6294 (6.29%) aligned concordantly >1 times\n",
      "11.13% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    88699 (88.70%) aligned concordantly 0 times\n",
      "    4878 (4.88%) aligned concordantly exactly 1 time\n",
      "    6423 (6.42%) aligned concordantly >1 times\n",
      "11.30% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_1_s1_R1.fastq.gz_C_to_T.fastq, zr2096_1_s1_R1.fastq.gz_G_to_A.fastq, zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t786616\n",
      "\n",
      "Total methylated C's in CpG context:\t70027\n",
      "Total methylated C's in CHG context:\t5876\n",
      "Total methylated C's in CHH context:\t29196\n",
      "Total methylated C's in Unknown context:\t3658\n",
      "\n",
      "Total unmethylated C's in CpG context:\t31078\n",
      "Total unmethylated C's in CHG context:\t163846\n",
      "Total unmethylated C's in CHH context:\t486593\n",
      "Total unmethylated C's in Unknown context:\t12665\n",
      "\n",
      "C methylated in CpG context:\t69.3%\n",
      "C methylated in CHG context:\t3.5%\n",
      "C methylated in CHH context:\t5.7%\n",
      "C methylated in unknown context (CN or CHN):\t22.4%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 1m 41s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_2_s1_R1.fastq.gz to zr2096_2_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_2_s1_R1.fastq.gz to zr2096_2_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_2_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_2_s1_R2.fastq.gz to zr2096_2_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_2_s1_R2.fastq.gz to zr2096_2_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_2_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tTTTTTTATTAAAATTTATTATTTTAAAAATAAAAATTNNNNTNNNTTTTATTATTTTTTTTAATATTTTTTAATATAATTAAATAATAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTCACTTCAAAAAAATAATAAATAAAAAAAATTAATACAACAATATTTTTAATAATACAACAATATTTAAAAAATTATTTATTAATTTTTTTTATTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tCTCCTTATTAAAATCTACTATTCCAAAAATAAAAACTNNNNCNNNTTCCACCATTCCTTCTAACATCTCCCAATATAACTAAATAACAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTGTTTTGGAAAAATAATGGATAAAAAAGGTTGATGTGGTGATGTTTTTAGTGATGTGGTGGTGTTTGAGAAATTATTTATTAATTTTTTTTGTTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tCTCCTTATTAAAATCTACTATTCCAAAAATAAAAACTNNNNCNNNTTCCACCATTCCTTCTAACATCTCCCAATATAACTAAATAACAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTGTTTTGGAAAAATAATGGATAAAAAAGGTTGATGTGGTGATGTTTTTAGTGATGTGGTGGTGTTTGAGAAATTATTTATTAATTTTTTTTGTTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tTTTTTTATTAAAATTTATTATTTTAAAAATAAAAATTNNNNTNNNTTTTATTATTTTTTTTAATATTTTTTAATATAATTAAATAATAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTCACTTCAAAAAAATAATAAATAAAAAAAATTAATACAACAATATTTTTAATAATACAACAATATTTAAAAAATTATTTATTAATTTTTTTTATTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_2_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    77244 (77.24%) aligned concordantly 0 times\n",
      "    9948 (9.95%) aligned concordantly exactly 1 time\n",
      "    12808 (12.81%) aligned concordantly >1 times\n",
      "22.76% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    77510 (77.51%) aligned concordantly 0 times\n",
      "    9883 (9.88%) aligned concordantly exactly 1 time\n",
      "    12607 (12.61%) aligned concordantly >1 times\n",
      "22.49% overall alignment rate\n",
      "100000100000 reads; of these: reads; of these:\n",
      "  \n",
      "100000   (100000 (100.00%) were paired; of these:100.00\n",
      "%    ) were paired; of these:\n",
      "77023     (7698177.02 (%) aligned concordantly 0 times76.98\n",
      "%    ) aligned concordantly 0 times9880\n",
      " (    9.8810090% () aligned concordantly exactly 1 time\n",
      "10.09    %13097 (13.10) aligned concordantly exactly 1 time%) aligned concordantly >1 times\n",
      "\n",
      "    22.9812929% overall alignment rate\n",
      " (12.93%) aligned concordantly >1 times\n",
      "23.02% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_2_s1_R1.fastq.gz_C_to_T.fastq, zr2096_2_s1_R1.fastq.gz_G_to_A.fastq, zr2096_2_s1_R2.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1504178\n",
      "\n",
      "Total methylated C's in CpG context:\t144161\n",
      "Total methylated C's in CHG context:\t11195\n",
      "Total methylated C's in CHH context:\t46340\n",
      "Total methylated C's in Unknown context:\t5291\n",
      "\n",
      "Total unmethylated C's in CpG context:\t61859\n",
      "Total unmethylated C's in CHG context:\t338198\n",
      "Total unmethylated C's in CHH context:\t902425\n",
      "Total unmethylated C's in Unknown context:\t22883\n",
      "\n",
      "C methylated in CpG context:\t70.0%\n",
      "C methylated in CHG context:\t3.2%\n",
      "C methylated in CHH context:\t4.9%\n",
      "C methylated in unknown context (CN or CHN):\t18.8%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 2m 2s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_3_s1_R1.fastq.gz to zr2096_3_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_3_s1_R1.fastq.gz to zr2096_3_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_3_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_3_s1_R2.fastq.gz to zr2096_3_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_3_s1_R2.fastq.gz to zr2096_3_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_3_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tTNTTTTAAAATAATTTAAATAATTTTATATAAATNNNNNNNNNNNNNNNTATTTAAATTAAAAATTTAAATAAATTTTTTTAATTATTTATTGTTTAATG\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAAATAAACATTAAACAATAAATAATTAAAAAAATTTATTTAAATTTTTAATTTAAATAAATTAAAAATATTAAATTTATATAAAATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tCNCTTCAAAATAATTTAAATAATCTTATATAAATNNNNNNNNNNNNNNNCACCTAAACTAAAAACTCAAATAAACTTTTCTAATCACCTATCATCCAACA\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAGATAGATGTTGGATGATAGGTGATTAGAAAAGTTTATTTGAGTTTTTAGTTTAGGTGAGTTAAAAATATTGAATTTATATAAGATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tCNCTTCAAAATAATTTAAATAATCTTATATAAATNNNNNNNNNNNNNNNCACCTAAACTAAAAACTCAAATAAACTTTTCTAATCACCTATCATCCAACA\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAGATAGATGTTGGATGATAGGTGATTAGAAAAGTTTATTTGAGTTTTTAGTTTAGGTGAGTTAAAAATATTGAATTTATATAAGATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tTNTTTTAAAATAATTTAAATAATTTTATATAAATNNNNNNNNNNNNNNNTATTTAAATTAAAAATTTAAATAAATTTTTTTAATTATTTATTGTTTAATG\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAAATAAACATTAAACAATAAATAATTAAAAAAATTTATTTAAATTTTTAATTTAAATAAATTAAAAATATTAAATTTATATAAAATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_3_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    72857 (72.86%) aligned concordantly 0 times\n",
      "    11109 (11.11%) aligned concordantly exactly 1 time\n",
      "    16034 (16.03%) aligned concordantly >1 times\n",
      "27.14% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    73267 (73.27%) aligned concordantly 0 times\n",
      "    11104 (11.10%) aligned concordantly exactly 1 time\n",
      "    15629 (15.63%) aligned concordantly >1 times\n",
      "26.73% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    73256 (73.26%) aligned concordantly 0 times\n",
      "    11099 (11.10%) aligned concordantly exactly 1 time\n",
      "    15645 (15.64%) aligned concordantly >1 times\n",
      "26.74% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    72899 (72.90%) aligned concordantly 0 times\n",
      "    11000 (11.00%) aligned concordantly exactly 1 time\n",
      "    16101 (16.10%) aligned concordantly >1 times\n",
      "27.10% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_3_s1_R1.fastq.gz_C_to_T.fastq, zr2096_3_s1_R1.fastq.gz_G_to_A.fastq, zr2096_3_s1_R2.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1846054\n",
      "\n",
      "Total methylated C's in CpG context:\t205232\n",
      "Total methylated C's in CHG context:\t13018\n",
      "Total methylated C's in CHH context:\t43410\n",
      "Total methylated C's in Unknown context:\t5954\n",
      "\n",
      "Total unmethylated C's in CpG context:\t64206\n",
      "Total unmethylated C's in CHG context:\t410867\n",
      "Total unmethylated C's in CHH context:\t1109321\n",
      "Total unmethylated C's in Unknown context:\t26408\n",
      "\n",
      "C methylated in CpG context:\t76.2%\n",
      "C methylated in CHG context:\t3.1%\n",
      "C methylated in CHH context:\t3.8%\n",
      "C methylated in unknown context (CN or CHN):\t18.4%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 2m 21s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_4_s1_R1.fastq.gz to zr2096_4_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_4_s1_R1.fastq.gz to zr2096_4_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_4_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_4_s1_R2.fastq.gz to zr2096_4_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_4_s1_R2.fastq.gz to zr2096_4_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_4_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNGATTAGTGTATAGTGGTTGTTATGGAAATTAGNNNNNNNNNNNNNNNNNNNNNNNNNTGNANTNTTTTTATTAGAATTGTTAAGGTTTATTTGATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTCCATCATTAATTTTTAAAAAAAAAAAACCATCCTTAATATCAATTTTATAAATCAAATAAACCTTAACAATTCTAATAAAAACACTTCATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATTAATATATAATAATTATTATAAAAATTAANNNNNNNNNNNNNNNNNNNNNNNNNTANANTNTTTTTATTAAAATTATTAAAATTTATTTAATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTTTATTATTAATTTTTAAAAAAAAAAAATTATTTTTAATATTAATTTTATAAATTAAATAAATTTTAATAATTTTAATAAAAATATTTTATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATTAATATATAATAATTATTATAAAAATTAANNNNNNNNNNNNNNNNNNNNNNNNNTANANTNTTTTTATTAAAATTATTAAAATTTATTTAATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTTTATTATTAATTTTTAAAAAAAAAAAATTATTTTTAATATTAATTTTATAAATTAAATAAATTTTAATAATTTTAATAAAAATATTTTATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNGATTAGTGTATAGTGGTTGTTATGGAAATTAGNNNNNNNNNNNNNNNNNNNNNNNNNTGNANTNTTTTTATTAGAATTGTTAAGGTTTATTTGATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTCCATCATTAATTTTTAAAAAAAAAAAACCATCCTTAATATCAATTTTATAAATCAAATAAACCTTAACAATTCTAATAAAAACACTTCATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_4_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    74565 (74.56%) aligned concordantly 0 times\n",
      "    11163 (11.16%) aligned concordantly exactly 1 time\n",
      "    14272 (14.27%) aligned concordantly >1 times\n",
      "25.43% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    74362 (74.36%) aligned concordantly 0 times\n",
      "    11177 (11.18%) aligned concordantly exactly 1 time\n",
      "    14461 (14.46%) aligned concordantly >1 times\n",
      "25.64% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    73448 (73.45%) aligned concordantly 0 times\n",
      "    11608 (11.61%) aligned concordantly exactly 1 time\n",
      "    14944 (14.94%) aligned concordantly >1 times\n",
      "26.55% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    73348 (73.35%) aligned concordantly 0 times\n",
      "    11651 (11.65%) aligned concordantly exactly 1 time\n",
      "    15001 (15.00%) aligned concordantly >1 times\n",
      "26.65% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_4_s1_R1.fastq.gz_C_to_T.fastq, zr2096_4_s1_R1.fastq.gz_G_to_A.fastq, zr2096_4_s1_R2.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1722505\n",
      "\n",
      "Total methylated C's in CpG context:\t147996\n",
      "Total methylated C's in CHG context:\t11743\n",
      "Total methylated C's in CHH context:\t46769\n",
      "Total methylated C's in Unknown context:\t5744\n",
      "\n",
      "Total unmethylated C's in CpG context:\t78905\n",
      "Total unmethylated C's in CHG context:\t378220\n",
      "Total unmethylated C's in CHH context:\t1058872\n",
      "Total unmethylated C's in Unknown context:\t25949\n",
      "\n",
      "C methylated in CpG context:\t65.2%\n",
      "C methylated in CHG context:\t3.0%\n",
      "C methylated in CHH context:\t4.2%\n",
      "C methylated in unknown context (CN or CHN):\t18.1%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 2m 18s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_5_s1_R1.fastq.gz to zr2096_5_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_5_s1_R1.fastq.gz to zr2096_5_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_5_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_5_s1_R2.fastq.gz to zr2096_5_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_5_s1_R2.fastq.gz to zr2096_5_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_5_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATAAAATATTATTTAAATTTTATTATTTATANNNNNNNNNNNNNNNNNNATTNTTTTTATTATATATATAAAATTATTTTAAAATTATTATATTTTTT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTCAAATATATTAAAAAAAAATTATTAATAATTTTCATTAAAAAATATAATAATTTTAAAATAATTTTATATATATAATAAAAAAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAACAAAACACCACTCAAATCTTACTATCCACANNNNNNNNNNNNNNNNNNATTNCTTCTATCATATATATAAAATCATTCCAAAATTACTATATTTTTC\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTTGAGTATATTGAAGAGGAATTATTGATAGTTTTTGTTGAAAAATATAGTAATTTTGGAATGATTTTATATATATGATAGAAGAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAACAAAACACCACTCAAATCTTACTATCCACANNNNNNNNNNNNNNNNNNATTNCTTCTATCATATATATAAAATCATTCCAAAATTACTATATTTTTC\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTTGAGTATATTGAAGAGGAATTATTGATAGTTTTTGTTGAAAAATATAGTAATTTTGGAATGATTTTATATATATGATAGAAGAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATAAAATATTATTTAAATTTTATTATTTATANNNNNNNNNNNNNNNNNNATTNTTTTTATTATATATATAAAATTATTTTAAAATTATTATATTTTTT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTCAAATATATTAAAAAAAAATTATTAATAATTTTCATTAAAAAATATAATAATTTTAAAATAATTTTATATATATAATAAAAAAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_5_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    75423 (75.42%) aligned concordantly 0 times\n",
      "    10829 (10.83%) aligned concordantly exactly 1 time\n",
      "    13748 (13.75%) aligned concordantly >1 times\n",
      "24.58% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    75332 (75.33%) aligned concordantly 0 times\n",
      "    10788 (10.79%) aligned concordantly exactly 1 time\n",
      "    13880 (13.88%) aligned concordantly >1 times\n",
      "24.67% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    74296 (74.30%) aligned concordantly 0 times\n",
      "    11111 (11.11%) aligned concordantly exactly 1 time\n",
      "    14593 (14.59%) aligned concordantly >1 times\n",
      "25.70% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    74142 (74.14%) aligned concordantly 0 times\n",
      "    11221 (11.22%) aligned concordantly exactly 1 time\n",
      "    14637 (14.64%) aligned concordantly >1 times\n",
      "25.86% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_5_s1_R1.fastq.gz_C_to_T.fastq, zr2096_5_s1_R1.fastq.gz_G_to_A.fastq, zr2096_5_s1_R2.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1732925\n",
      "\n",
      "Total methylated C's in CpG context:\t148635\n",
      "Total methylated C's in CHG context:\t12330\n",
      "Total methylated C's in CHH context:\t45576\n",
      "Total methylated C's in Unknown context:\t6184\n",
      "\n",
      "Total unmethylated C's in CpG context:\t73546\n",
      "Total unmethylated C's in CHG context:\t369593\n",
      "Total unmethylated C's in CHH context:\t1083245\n",
      "Total unmethylated C's in Unknown context:\t26179\n",
      "\n",
      "C methylated in CpG context:\t66.9%\n",
      "C methylated in CHG context:\t3.2%\n",
      "C methylated in CHH context:\t4.0%\n",
      "C methylated in unknown context (CN or CHN):\t19.1%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 58m 45s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_6_s1_R1.fastq.gz to zr2096_6_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_6_s1_R1.fastq.gz to zr2096_6_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_6_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_6_s1_R2.fastq.gz to zr2096_6_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_6_s1_R2.fastq.gz to zr2096_6_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_6_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTTATTAAATTTAATTTTATTATTAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNTANANAAATATTAATTAAATATTTTTAAAATAATAAATTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAAAATTTTTTTAAATTTATTATTTTAAAAATATTTAATTAATATTTATTTATATTTTATAATTAACAATAATTATTTTTTTATTTAATAATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTCATCAAATCCAACTTTATCACCAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNCANANAAACATCAATCAAACATCTCTAAAACAATAAACTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAGAATTTTTTTAAGTTTATTGTTTTAGAGATGTTTGATTGATGTTTATTTGTATTTTGTAATTAGTGGTGGTTGTTTTTTTATTTGGTGATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTCATCAAATCCAACTTTATCACCAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNCANANAAACATCAATCAAACATCTCTAAAACAATAAACTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAGAATTTTTTTAAGTTTATTGTTTTAGAGATGTTTGATTGATGTTTATTTGTATTTTGTAATTAGTGGTGGTTGTTTTTTTATTTGGTGATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTTATTAAATTTAATTTTATTATTAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNTANANAAATATTAATTAAATATTTTTAAAATAATAAATTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAAAATTTTTTTAAATTTATTATTTTAAAAATATTTAATTAATATTTATTTATATTTTATAATTAACAATAATTATTTTTTTATTTAATAATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_6_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    74703 (74.70%) aligned concordantly 0 times\n",
      "    10970 (10.97%) aligned concordantly exactly 1 time\n",
      "    14327 (14.33%) aligned concordantly >1 times\n",
      "25.30% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    74678 (74.68%) aligned concordantly 0 times\n",
      "    11024 (11.02%) aligned concordantly exactly 1 time\n",
      "    14298 (14.30%) aligned concordantly >1 times\n",
      "25.32% overall alignment rate\n",
      "100000100000 reads; of these:\n",
      "   reads; of these:100000 (\n",
      "100.00%) were paired; of these:\n",
      "      72507100000 ( (72.51%) aligned concordantly 0 times\n",
      "    100.0012031% () were paired; of these:12.03\n",
      "%    ) aligned concordantly exactly 1 time\n",
      "72316     (1546272.32 (%) aligned concordantly 0 times\n",
      "    15.4611944% () aligned concordantly >1 times11.94\n",
      "%) aligned concordantly exactly 1 time27.49\n",
      "    %15740 overall alignment rate (\n",
      "15.74%) aligned concordantly >1 times\n",
      "27.68% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_6_s1_R1.fastq.gz_C_to_T.fastq, zr2096_6_s1_R1.fastq.gz_G_to_A.fastq, zr2096_6_s1_R2.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1757283\n",
      "\n",
      "Total methylated C's in CpG context:\t170109\n",
      "Total methylated C's in CHG context:\t12567\n",
      "Total methylated C's in CHH context:\t47214\n",
      "Total methylated C's in Unknown context:\t5978\n",
      "\n",
      "Total unmethylated C's in CpG context:\t64088\n",
      "Total unmethylated C's in CHG context:\t383166\n",
      "Total unmethylated C's in CHH context:\t1080139\n",
      "Total unmethylated C's in Unknown context:\t27238\n",
      "\n",
      "C methylated in CpG context:\t72.6%\n",
      "C methylated in CHG context:\t3.2%\n",
      "C methylated in CHH context:\t4.2%\n",
      "C methylated in unknown context (CN or CHN):\t18.0%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 2m 27s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_7_s1_R1.fastq.gz to zr2096_7_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_7_s1_R1.fastq.gz to zr2096_7_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_7_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_7_s1_R2.fastq.gz to zr2096_7_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_7_s1_R2.fastq.gz to zr2096_7_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_7_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAAAAAATGATTATTATGATGAATTTAAATATTAATAATAAAATTTAAAAAAAATATAAATGTTAAATATTAAATTTTTTAAAAAATGATTATGG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t141\t*\t0\t0\t*\t*\t0\t0\tCCATAATCATTTTTTAAAAAATTTAATATTTAACATTTATATTTTTTTTAAATTTTATTATTAATATTTAAATTCATCATAATAATCATTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t99\tNC_035781.1_GA_converted\t24539746\t6\t95M\t=\t24539746\t-95\tAAAAAAACAACTACTACAACAAATTCAAATATTAATAACAAAATTTAAAAAAAACATAAACATTAAATATCAAATTCTTCAAAAAACAATTACAA\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-12\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:94T0\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t147\tNC_035781.1_GA_converted\t24539746\t6\t95M\t=\t24539746\t-95\tAAAAAAACAACTACTACAACAAATTCAAATATTAATAACAAAATTTAAAAAAAACATAAACATTAAATATCAAATTCTTCAAAAAACAATTACAA\tFFFFFFFFFFFFFFFFFFFFFFFBBFFFFFFBBFF/FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-6\tXS:i:-12\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:94T0\tYS:i:-6\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t83\tNC_035780.1_CT_converted\t42548681\t17\t95M\t=\t42548681\t-95\tTTGTAATTGTTTTTTGAAGAATTTGATATTTAATGTTTATGTTTTTTTTAAATTTTGTTATTAATATTTGAATTTGTTGTAGTAGTTGTTTTTTT\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-6\tXS:i:-32\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:0A94\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t163\tNC_035780.1_CT_converted\t42548681\t17\t95M\t=\t42548681\t-95\tTTGTAATTGTTTTTTGAAGAATTTGATATTTAATGTTTATGTTTTTTTTAAATTTTGTTATTAATATTTGAATTTGTTGTAGTAGTTGTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-32\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:0A94\tYS:i:-6\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAAAAAATGATTATTATGATGAATTTAAATATTAATAATAAAATTTAAAAAAAATATAAATGTTAAATATTAAATTTTTTAAAAAATGATTATGG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t141\t*\t0\t0\t*\t*\t0\t0\tCCATAATCATTTTTTAAAAAATTTAATATTTAACATTTATATTTTTTTTAAATTTTATTATTAATATTTAAATTCATCATAATAATCATTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_7_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    74392 (74.39%) aligned concordantly 0 times\n",
      "    11232 (11.23%) aligned concordantly exactly 1 time\n",
      "    14376 (14.38%) aligned concordantly >1 times\n",
      "25.61% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    75126 (75.13%) aligned concordantly 0 times\n",
      "    10861 (10.86%) aligned concordantly exactly 1 time\n",
      "    14013 (14.01%) aligned concordantly >1 times\n",
      "24.87% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    74489 (74.49%) aligned concordantly 0 times\n",
      "    11197 (11.20%) aligned concordantly exactly 1 time\n",
      "    14314 (14.31%) aligned concordantly >1 times\n",
      "25.51% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    75079 (75.08%) aligned concordantly 0 times\n",
      "    10839 (10.84%) aligned concordantly exactly 1 time\n",
      "    14082 (14.08%) aligned concordantly >1 times\n",
      "24.92% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_7_s1_R1.fastq.gz_C_to_T.fastq, zr2096_7_s1_R1.fastq.gz_G_to_A.fastq, zr2096_7_s1_R2.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1759007\n",
      "\n",
      "Total methylated C's in CpG context:\t161109\n",
      "Total methylated C's in CHG context:\t12969\n",
      "Total methylated C's in CHH context:\t46088\n",
      "Total methylated C's in Unknown context:\t5728\n",
      "\n",
      "Total unmethylated C's in CpG context:\t66119\n",
      "Total unmethylated C's in CHG context:\t382089\n",
      "Total unmethylated C's in CHH context:\t1090633\n",
      "Total unmethylated C's in Unknown context:\t25887\n",
      "\n",
      "C methylated in CpG context:\t70.9%\n",
      "C methylated in CHG context:\t3.3%\n",
      "C methylated in CHH context:\t4.1%\n",
      "C methylated in unknown context (CN or CHN):\t18.1%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 2m 49s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_8_s1_R1.fastq.gz to zr2096_8_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_8_s1_R1.fastq.gz to zr2096_8_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_8_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_8_s1_R2.fastq.gz to zr2096_8_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_8_s1_R2.fastq.gz to zr2096_8_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_8_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATTTAAAATTAAAAAATTTATTAAATTTATTATTATTTNTTATTTTTTATAAAAGAAAATAGAGAGAGAGAGAGAGAGAGAGAGAG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tCTCTCTCTCTCTCTCTCTCTCTCTCTATTTTCTTTTATAAAAAATAATAAATAATAATAAATTTAATAAATTTTTTAATTTTAAATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATCCAAAATTAAAAAACTCATTAAATCTATCATCATTCNCCATTCTTCACAAAAAAAAATAAAAAAAAAAAAAAAAAAAAAAAAAA\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTTTTTTTTTTTTTTTTTTTTTATTTTTTTTTGTGAAGAATGGTGAATGATGATAGATTTAATGAGTTTTTTAATTTTGGATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t83\tNC_007175.2_CT_converted\t14567\t0\t2M17I69M\t=\t14567\t-71\tTTTTTTTTTTTTTTTTTTTTTTTTTTATTTTTTTTTGTGAAGAATGGNGAATGATGATAGATTTAATGAGTTTTTTAATTTTGGATTA\tFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<<#FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-81\tXN:i:0\tXM:i:5\tXO:i:1\tXG:i:17\tNM:i:22\tMD:Z:1A5A0G5A15T40\tYS:i:-80\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t163\tNC_007175.2_CT_converted\t14567\t0\t2M17I69M\t=\t14567\t-71\tTTTTTTTTTTTTTTTTTTTTTTTTTTATTTTTTTTTGTGAAGAATGGTGAATGATGATAGATTTAATGAGTTTTTTAATTTTGGATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tAS:i:-80\tXN:i:0\tXM:i:4\tXO:i:1\tXG:i:17\tNM:i:21\tMD:Z:1A5A0G5A56\tYS:i:-81\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATTTAAAATTAAAAAATTTATTAAATTTATTATTATTTNTTATTTTTTATAAAAGAAAATAGAGAGAGAGAGAGAGAGAGAGAGAG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tCTCTCTCTCTCTCTCTCTCTCTCTCTATTTTCTTTTATAAAAAATAATAAATAATAATAAATTTAATAAATTTTTTAATTTTAAATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_8_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    77556 (77.56%) aligned concordantly 0 times\n",
      "    9974 (9.97%) aligned concordantly exactly 1 time\n",
      "    12470 (12.47%) aligned concordantly >1 times\n",
      "22.44% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    77349 (77.35%) aligned concordantly 0 times\n",
      "    10145 (10.14%) aligned concordantly exactly 1 time\n",
      "    12506 (12.51%) aligned concordantly >1 times\n",
      "22.65% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    77761 (77.76%) aligned concordantly 0 times\n",
      "    9983 (9.98%) aligned concordantly exactly 1 time\n",
      "    12256 (12.26%) aligned concordantly >1 times\n",
      "22.24% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    77359 (77.36%) aligned concordantly 0 times\n",
      "    10158 (10.16%) aligned concordantly exactly 1 time\n",
      "    12483 (12.48%) aligned concordantly >1 times\n",
      "22.64% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_8_s1_R1.fastq.gz_C_to_T.fastq, zr2096_8_s1_R1.fastq.gz_G_to_A.fastq, zr2096_8_s1_R2.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1542276\n",
      "\n",
      "Total methylated C's in CpG context:\t125986\n",
      "Total methylated C's in CHG context:\t11350\n",
      "Total methylated C's in CHH context:\t43949\n",
      "Total methylated C's in Unknown context:\t5383\n",
      "\n",
      "Total unmethylated C's in CpG context:\t65071\n",
      "Total unmethylated C's in CHG context:\t336699\n",
      "Total unmethylated C's in CHH context:\t959221\n",
      "Total unmethylated C's in Unknown context:\t23052\n",
      "\n",
      "C methylated in CpG context:\t65.9%\n",
      "C methylated in CHG context:\t3.3%\n",
      "C methylated in CHH context:\t4.4%\n",
      "C methylated in unknown context (CN or CHN):\t18.9%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 3m 17s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_9_s1_R1.fastq.gz to zr2096_9_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_9_s1_R1.fastq.gz to zr2096_9_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_9_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_9_s1_R2.fastq.gz to zr2096_9_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_9_s1_R2.fastq.gz to zr2096_9_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_9_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNATTAAATAAAAAATAATAAATAATAAATTANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAGG\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAAAAAATAAAAAACACATTAAAAATCTATAATTTAATTTATTATTTATTATTTTTTATTTAATTAAATCNNNNAAACATCATATAAAAAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNACCAAATAAAAAATAACAAATAATAAATCANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAAA\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAGAGGATAGGAGATATGTTAGGAATTTATAATTTGATTTATTATTTGTTATTTTTTATTTGGTTAGATTNNNNGAGTGTTGTGTAGGGAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNACCAAATAAAAAATAACAAATAATAAATCANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAAA\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAGAGGATAGGAGATATGTTAGGAATTTATAATTTGATTTATTATTTGTTATTTTTTATTTGGTTAGATTNNNNGAGTGTTGTGTAGGGAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 60 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNATTAAATAAAAAATAATAAATAATAAATTANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAGG\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAAAAAATAAAAAACACATTAAAAATCTATAATTTAATTTATTATTTATTATTTTTTATTTAATTAAATCNNNNAAACATCATATAAAAAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_9_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    77171 (77.17%) aligned concordantly 0 times\n",
      "    9961 (9.96%) aligned concordantly exactly 1 time\n",
      "    12868 (12.87%) aligned concordantly >1 times\n",
      "22.83% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    77405 (77.41%) aligned concordantly 0 times\n",
      "    9952 (9.95%) aligned concordantly exactly 1 time\n",
      "    12643 (12.64%) aligned concordantly >1 times\n",
      "22.59% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    73894 (73.89%) aligned concordantly 0 times\n",
      "    11380 (11.38%) aligned concordantly exactly 1 time\n",
      "    14726 (14.73%) aligned concordantly >1 times\n",
      "26.11% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    74109 (74.11%) aligned concordantly 0 times\n",
      "    11230 (11.23%) aligned concordantly exactly 1 time\n",
      "    14661 (14.66%) aligned concordantly >1 times\n",
      "25.89% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_9_s1_R1.fastq.gz_C_to_T.fastq, zr2096_9_s1_R1.fastq.gz_G_to_A.fastq, zr2096_9_s1_R2.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t1782066\n",
      "\n",
      "Total methylated C's in CpG context:\t191619\n",
      "Total methylated C's in CHG context:\t14039\n",
      "Total methylated C's in CHH context:\t46105\n",
      "Total methylated C's in Unknown context:\t6367\n",
      "\n",
      "Total unmethylated C's in CpG context:\t61672\n",
      "Total unmethylated C's in CHG context:\t393266\n",
      "Total unmethylated C's in CHH context:\t1075365\n",
      "Total unmethylated C's in Unknown context:\t26921\n",
      "\n",
      "C methylated in CpG context:\t75.7%\n",
      "C methylated in CHG context:\t3.4%\n",
      "C methylated in CHH context:\t4.1%\n",
      "C methylated in unknown context (CN or CHN):\t19.1%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 2m 55s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "find /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_*R1* \\\n",
    "| xargs basename -s _s1_R1.fastq.gz | xargs -I{} /Users/Shelly/anaconda3/bin/bismark \\\n",
    "--genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ \\\n",
    "--score_min L,0,-1.2 \\\n",
    "-u 100000 \\\n",
    "-I 60 \\\n",
    "--non_directional \\\n",
    "-1 /Volumes/web/seashell/bu-serine-wd/18-04-07/{}_s1_R1.fastq.gz \\\n",
    "-2 /Volumes/web/seashell/bu-serine-wd/18-04-07/{}_s1_R2.fastq.gz \\\n",
    "-o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/ "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#run deduplicate on fragment length threshold (60bp) alignments"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_10_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 60 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_1_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 60 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_2_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 60 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_3_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 60 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_4_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 60 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_5_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 60 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_6_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 60 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_7_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 60 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_8_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 60 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_9_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 60 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/\"\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/\n",
      "Processing paired-end Bismark output file(s) (SAM format):\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_10_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_1_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_2_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_3_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_4_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_5_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_6_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_7_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_8_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_9_s1_R1_bismark_bt2_pe.bam\n",
      "\n",
      "\n",
      "If there are several alignments to a single position in the genome the first alignment will be chosen. Since the input files are not in any way sorted this is a near-enough random selection of reads.\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_10_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_10_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_10_s1_R1_bismark_bt2_pe.bam:\t52694\n",
      "Total number duplicated alignments removed:\t20 (0.04%)\n",
      "Duplicated alignments were found at:\t19 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 52674 (99.96% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_1_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_1_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_1_s1_R1_bismark_bt2_pe.bam:\t22900\n",
      "Total number duplicated alignments removed:\t29 (0.13%)\n",
      "Duplicated alignments were found at:\t23 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 22871 (99.87% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_2_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_2_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_2_s1_R1_bismark_bt2_pe.bam:\t45073\n",
      "Total number duplicated alignments removed:\t29 (0.06%)\n",
      "Duplicated alignments were found at:\t20 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 45044 (99.94% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_3_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_3_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_3_s1_R1_bismark_bt2_pe.bam:\t51869\n",
      "Total number duplicated alignments removed:\t17 (0.03%)\n",
      "Duplicated alignments were found at:\t17 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 51852 (99.97% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_4_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_4_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_4_s1_R1_bismark_bt2_pe.bam:\t51868\n",
      "Total number duplicated alignments removed:\t13 (0.03%)\n",
      "Duplicated alignments were found at:\t12 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 51855 (99.97% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_5_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_5_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_5_s1_R1_bismark_bt2_pe.bam:\t50679\n",
      "Total number duplicated alignments removed:\t19 (0.04%)\n",
      "Duplicated alignments were found at:\t19 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 50660 (99.96% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_6_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_6_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_6_s1_R1_bismark_bt2_pe.bam:\t52453\n",
      "Total number duplicated alignments removed:\t12 (0.02%)\n",
      "Duplicated alignments were found at:\t12 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 52441 (99.98% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_7_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_7_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_7_s1_R1_bismark_bt2_pe.bam:\t50979\n",
      "Total number duplicated alignments removed:\t13 (0.03%)\n",
      "Duplicated alignments were found at:\t13 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 50966 (99.97% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_8_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_8_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_8_s1_R1_bismark_bt2_pe.bam:\t45836\n",
      "Total number duplicated alignments removed:\t13 (0.03%)\n",
      "Duplicated alignments were found at:\t12 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 45823 (99.97% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_9_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_9_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_9_s1_R1_bismark_bt2_pe.bam:\t49112\n",
      "Total number duplicated alignments removed:\t36 (0.07%)\n",
      "Duplicated alignments were found at:\t36 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 49076 (99.93% of total)\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "/Users/Shelly/anaconda3/bin/deduplicate_bismark \\\n",
    "--bam -p \\\n",
    "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/*.bam \\\n",
    "-o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/ \n",
    "2> /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/dedup.err"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#create summary report of mapping efficiency and dedup. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/*PE_report.txt | \\\n",
    "grep 'Mapping\\ efficiency\\:' | \\\n",
    "cat - /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/*.deduplication_report.txt > /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_mapping_dedup_summary.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#make a new directory for fragment length threshold (150bp)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "mkdir /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#Bismark analysis with fragment length threshold (150bp) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t16486\n",
      "Mapping efficiency:\t16.5%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t78915\n",
      "Sequence pairs did not map uniquely:\t4599\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t4115\t((converted) top strand)\n",
      "GA/CT/CT:\t4057\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t4238\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t4076\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t7693\n",
      "Mapping efficiency:\t7.7%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t90152\n",
      "Sequence pairs did not map uniquely:\t2155\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t1851\t((converted) top strand)\n",
      "GA/CT/CT:\t1914\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t1930\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t1998\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t10983\n",
      "Mapping efficiency:\t11.0%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t85462\n",
      "Sequence pairs did not map uniquely:\t3555\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t2817\t((converted) top strand)\n",
      "GA/CT/CT:\t2703\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t2748\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t2715\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t16093\n",
      "Mapping efficiency:\t16.1%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t79129\n",
      "Sequence pairs did not map uniquely:\t4778\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t4075\t((converted) top strand)\n",
      "GA/CT/CT:\t3984\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t3987\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t4047\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t14079\n",
      "Mapping efficiency:\t14.1%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t81720\n",
      "Sequence pairs did not map uniquely:\t4201\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t3462\t((converted) top strand)\n",
      "GA/CT/CT:\t3472\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t3558\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t3587\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t14504\n",
      "Mapping efficiency:\t14.5%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t81332\n",
      "Sequence pairs did not map uniquely:\t4164\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t3647\t((converted) top strand)\n",
      "GA/CT/CT:\t3586\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t3574\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t3697\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t14166\n",
      "Mapping efficiency:\t14.2%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t81561\n",
      "Sequence pairs did not map uniquely:\t4273\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t3709\t((converted) top strand)\n",
      "GA/CT/CT:\t3478\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t3378\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t3601\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t16821\n",
      "Mapping efficiency:\t16.8%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t78580\n",
      "Sequence pairs did not map uniquely:\t4599\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t4235\t((converted) top strand)\n",
      "GA/CT/CT:\t4195\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t4252\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t4139\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t14354\n",
      "Mapping efficiency:\t14.4%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t81679\n",
      "Sequence pairs did not map uniquely:\t3967\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t3601\t((converted) top strand)\n",
      "GA/CT/CT:\t3584\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t3585\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t3584\t((converted) bottom strand)\n",
      "\n",
      "FastQ format assumed (by default)\n",
      "chr NC_035780.1 (65668440 bp)\n",
      "chr NC_035781.1 (61752955 bp)\n",
      "chr NC_035782.1 (77061148 bp)\n",
      "chr NC_035783.1 (59691872 bp)\n",
      "chr NC_035784.1 (98698416 bp)\n",
      "chr NC_035785.1 (51258098 bp)\n",
      "chr NC_035786.1 (57830854 bp)\n",
      "chr NC_035787.1 (75944018 bp)\n",
      "chr NC_035788.1 (104168038 bp)\n",
      "chr NC_035789.1 (32650045 bp)\n",
      "chr NC_007175.2 (17244 bp)\n",
      "\n",
      "Number of paired-end alignments with a unique best hit:\t14107\n",
      "Mapping efficiency:\t14.1%\n",
      "\n",
      "Sequence pairs with no alignments under any condition:\t81677\n",
      "Sequence pairs did not map uniquely:\t4216\n",
      "Sequence pairs which were discarded because genomic sequence could not be extracted:\t0\n",
      "\n",
      "Number of sequence pairs with unique best (first) alignment came from the bowtie output:\n",
      "CT/GA/CT:\t3686\t((converted) top strand)\n",
      "GA/CT/CT:\t3327\t(complementary to (converted) top strand)\n",
      "GA/CT/GA:\t3351\t(complementary to (converted) bottom strand)\n",
      "CT/GA/GA:\t3743\t((converted) bottom strand)\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_10_s1_R1.fastq.gz to zr2096_10_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_10_s1_R1.fastq.gz to zr2096_10_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_10_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_10_s1_R2.fastq.gz to zr2096_10_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_10_s1_R2.fastq.gz to zr2096_10_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_10_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tGTTATATATAATTGTATTAATTATAAAAAATATNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATTTATTATTAAAAATATTAAATATATTATTTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNATTTTATATTTTTAATAATAAATATAATTATATATTAAAATAAAATAATATATTTAATATTTTTAATAATAAATACAATTAAATATCATAATAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tACCATATACAATCATATCAACTATAAAAAATACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATCTACTATCAAAAATACTAAACACACTACCTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNGTTTTGTATTTTTGATAGTAGATATGATTGTATATTAGGGTAAGGTAGTGTGTTTAGTATTTTTGATAGTAGATATGATTGAATATTGTGGTAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_G_to_A.fastq and zr2096_10_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tACCATATACAATCATATCAACTATAAAAAATACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATCTACTATCAAAAATACTAAACACACTACCTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNGTTTTGTATTTTTGATAGTAGATATGATTGTATATTAGGGTAAGGTAGTGTGTTTAGTATTTTTGATAGTAGATATGATTGAATATTGTGGTAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_10_s1_R1.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_1:N:0:GATCAG/1\t77\t*\t0\t0\t*\t*\t0\t0\tGTTATATATAATTGTATTAATTATAAAAAATATNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNATTTATTATTAAAAATATTAAATATATTATTTTA\tBBBBBFFFFFFFFFFFFFFBFFFFFFFFFFFFF#################################<</<FFFFFFFFFFFFFFFFFFFFFFFFFF<FFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1232:2155_2:N:0:GATCAG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNATTTTATATTTTTAATAATAAATATAATTATATATTAAAATAAAATAATATATTTAATATTTTTAATAATAAATACAATTAAATATCATAATAA\t#####BB<FFFFFFFFFFFFFBFFFFFFFFFFFFFFFFBFFBFFFFFFFFFBFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_10_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    91934 (91.93%) aligned concordantly 0 times\n",
      "    2526 (2.53%) aligned concordantly exactly 1 time\n",
      "    5540 (5.54%) aligned concordantly >1 times\n",
      "8.07% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    91798 (91.80%) aligned concordantly 0 times\n",
      "    2508 (2.51%) aligned concordantly exactly 1 time\n",
      "    5694 (5.69%) aligned concordantly >1 times\n",
      "8.20% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    91896 (91.90%) aligned concordantly 0 times\n",
      "    2504 (2.50%) aligned concordantly exactly 1 time\n",
      "    5600 (5.60%) aligned concordantly >1 times\n",
      "8.10% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    91725 (91.72%) aligned concordantly 0 times\n",
      "    2534 (2.53%) aligned concordantly exactly 1 time\n",
      "    5741 (5.74%) aligned concordantly >1 times\n",
      "8.28% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_10_s1_R1.fastq.gz_C_to_T.fastq, zr2096_10_s1_R1.fastq.gz_G_to_A.fastq, zr2096_10_s1_R2.fastq.gz_C_to_T.fastq and zr2096_10_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t589731\n",
      "\n",
      "Total methylated C's in CpG context:\t49950\n",
      "Total methylated C's in CHG context:\t4083\n",
      "Total methylated C's in CHH context:\t17719\n",
      "Total methylated C's in Unknown context:\t1985\n",
      "\n",
      "Total unmethylated C's in CpG context:\t27314\n",
      "Total unmethylated C's in CHG context:\t121352\n",
      "Total unmethylated C's in CHH context:\t369313\n",
      "Total unmethylated C's in Unknown context:\t7737\n",
      "\n",
      "C methylated in CpG context:\t64.6%\n",
      "C methylated in CHG context:\t3.3%\n",
      "C methylated in CHH context:\t4.6%\n",
      "C methylated in unknown context (CN or CHN):\t20.4%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 5m 2s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R1.fastq.gz to zr2096_1_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_1_s1_R2.fastq.gz to zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_1_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t99\tNC_035780.1_CT_converted\t65371909\t6\t100M\t=\t65372013\t204\tGAGTTTTTTTGATTATTTGTTGTTTGTTGTTTGTTTGNNNNTNNNTNNGTTTGTTTGTTTGTTTGTTTGTTTGTAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tAS:i:-15\tXS:i:-22\tXN:i:0\tXM:i:10\tXO:i:0\tXG:i:0\tNM:i:10\tMD:Z:37T0T0T0G1T0T0G1T0T10A41\tYS:i:-24\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t147\tNC_035780.1_CT_converted\t65372013\t6\t100M\t=\t65371909\t-204\tGAATTATTGGGTTAATTTTAATTAAATTTGATATAAAGTATTTTTAGGTGAAGGGGATTTAAGTTTGTTTGAATGAAGGGTTTTGTTTTTTTTTAAGGGG\tFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-24\tXS:i:-6\tXN:i:0\tXM:i:4\tXO:i:0\tXG:i:0\tNM:i:4\tMD:Z:30G14G3A20A29\tYS:i:-15\tYT:Z:CP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTAAAAAAAAATGAAATTTTTTATTTAAATAAATTTAAATTTTTTTTATTTAAAAATATTTTATATTAAATTTAATTAAAATTAATTTAATAATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_G_to_A.fastq and zr2096_1_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAATTTTTTTAATTATTTATCATTCATCATTTATTTANNNNTNNNTNNATTTATTTATTTATTTATTCATTTATAAATTTTTTATATTTTTATTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<##<<<FFFFFFFFFFFFFFFF<FFFFFF/</FFFFFF/B/FFFFF/FFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTAAAAAAAAATGAAATTTTTTATTTAAATAAATTTAAATTTTTTTTATTTAAAAATATTTTATATTAAATTTAATTAAAATTAATTTAATAATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_1_s1_R1.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_1:N:0:CGATGT/1\t83\tNC_035780.1_GA_converted\t8090288\t7\t26M4D74M\t=\t8090028\t-364\tAAAAAAAATAAAAATATAAAAAATTTACAAACAAACAAACAAACAAACAAACNNANNNANNNNCAAACAAACAACAAACAACAAATAATCAAAAAAACTC\tFFFFFFFF/FFFFF/B/FFFFFF/</FFFFFF<FFFFFFFFFFFFFFFF<<<##<###<####FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBBBBB\tAS:i:-26\tXS:i:-22\tXN:i:0\tXM:i:9\tXO:i:1\tXG:i:4\tNM:i:13\tMD:Z:26^ACAA26A0A1C0A0A1C0A0A0A37\tYS:i:-6\tYT:Z:CP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1249:2156_2:N:0:CGATGT/2\t163\tNC_035780.1_GA_converted\t8090028\t7\t100M\t=\t8090288\t364\tCCCCTTAAAAAAAAACAAAACCCTTCATTCAAACAAACTTAAATCCCCTTCACCTAAAAATACTTTATATCAAATTTAATTAAAATTAACCCAATAATTC\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tAS:i:-6\tXS:i:-36\tXN:i:0\tXM:i:1\tXO:i:0\tXG:i:0\tNM:i:1\tMD:Z:29T70\tYS:i:-26\tYT:Z:CP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_1_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    96177 (96.18%) aligned concordantly 0 times\n",
      "    1175 (1.18%) aligned concordantly exactly 1 time\n",
      "    2648 (2.65%) aligned concordantly >1 times\n",
      "3.82% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    96162 (96.16%) aligned concordantly 0 times\n",
      "    1147 (1.15%) aligned concordantly exactly 1 time\n",
      "    2691 (2.69%) aligned concordantly >1 times\n",
      "3.84% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    96242 (96.24%) aligned concordantly 0 times\n",
      "    1159 (1.16%) aligned concordantly exactly 1 time\n",
      "    2599 (2.60%) aligned concordantly >1 times\n",
      "3.76% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    96112 (96.11%) aligned concordantly 0 times\n",
      "    1237 (1.24%) aligned concordantly exactly 1 time\n",
      "    2651 (2.65%) aligned concordantly >1 times\n",
      "3.89% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_1_s1_R1.fastq.gz_C_to_T.fastq, zr2096_1_s1_R1.fastq.gz_G_to_A.fastq, zr2096_1_s1_R2.fastq.gz_C_to_T.fastq and zr2096_1_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t271321\n",
      "\n",
      "Total methylated C's in CpG context:\t21511\n",
      "Total methylated C's in CHG context:\t2077\n",
      "Total methylated C's in CHH context:\t10907\n",
      "Total methylated C's in Unknown context:\t1131\n",
      "\n",
      "Total unmethylated C's in CpG context:\t10746\n",
      "Total unmethylated C's in CHG context:\t52709\n",
      "Total unmethylated C's in CHH context:\t173371\n",
      "Total unmethylated C's in Unknown context:\t4292\n",
      "\n",
      "C methylated in CpG context:\t66.7%\n",
      "C methylated in CHG context:\t3.8%\n",
      "C methylated in CHH context:\t5.9%\n",
      "C methylated in unknown context (CN or CHN):\t20.9%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 3m 57s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_2_s1_R1.fastq.gz to zr2096_2_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_2_s1_R1.fastq.gz to zr2096_2_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_2_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_2_s1_R2.fastq.gz to zr2096_2_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_2_s1_R2.fastq.gz to zr2096_2_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_2_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tTTTTTTATTAAAATTTATTATTTTAAAAATAAAAATTNNNNTNNNTTTTATTATTTTTTTTAATATTTTTTAATATAATTAAATAATAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTCACTTCAAAAAAATAATAAATAAAAAAAATTAATACAACAATATTTTTAATAATACAACAATATTTAAAAAATTATTTATTAATTTTTTTTATTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tCTCCTTATTAAAATCTACTATTCCAAAAATAAAAACTNNNNCNNNTTCCACCATTCCTTCTAACATCTCCCAATATAACTAAATAACAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTGTTTTGGAAAAATAATGGATAAAAAAGGTTGATGTGGTGATGTTTTTAGTGATGTGGTGGTGTTTGAGAAATTATTTATTAATTTTTTTTGTTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_G_to_A.fastq and zr2096_2_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tCTCCTTATTAAAATCTACTATTCCAAAAATAAAAACTNNNNCNNNTTCCACCATTCCTTCTAACATCTCCCAATATAACTAAATAACAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTGTTTTGGAAAAATAATGGATAAAAAAGGTTGATGTGGTGATGTTTTTAGTGATGTGGTGGTGTTTGAGAAATTATTTATTAATTTTTTTTGTTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_2_s1_R1.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_1:N:0:TGACCA/1\t77\t*\t0\t0\t*\t*\t0\t0\tTTTTTTATTAAAATTTATTATTTTAAAAATAAAAATTNNNNTNNNTTTTATTATTTTTTTTAATATTTTTTAATATAATTAAATAATAAAAAAAATTAAT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF####<###<<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1148:2157_2:N:0:TGACCA/2\t141\t*\t0\t0\t*\t*\t0\t0\tTCACTTCAAAAAAATAATAAATAAAAAAAATTAATACAACAATATTTTTAATAATACAACAATATTTAAAAAATTATTTATTAATTTTTTTTATTATTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFFFFBFFFFFFFFBFF/BFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_2_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    94284 (94.28%) aligned concordantly 0 times\n",
      "    1408 (1.41%) aligned concordantly exactly 1 time\n",
      "    4308 (4.31%) aligned concordantly >1 times\n",
      "5.72% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    94169 (94.17%) aligned concordantly 0 times\n",
      "    1473 (1.47%) aligned concordantly exactly 1 time\n",
      "    4358 (4.36%) aligned concordantly >1 times\n",
      "5.83% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    94068 (94.07%) aligned concordantly 0 times\n",
      "    1488 (1.49%) aligned concordantly exactly 1 time\n",
      "    4444 (4.44%) aligned concordantly >1 times\n",
      "5.93% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    94251 (94.25%) aligned concordantly 0 times\n",
      "    1453 (1.45%) aligned concordantly exactly 1 time\n",
      "    4296 (4.30%) aligned concordantly >1 times\n",
      "5.75% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_2_s1_R1.fastq.gz_C_to_T.fastq, zr2096_2_s1_R1.fastq.gz_G_to_A.fastq, zr2096_2_s1_R2.fastq.gz_C_to_T.fastq and zr2096_2_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t367378\n",
      "\n",
      "Total methylated C's in CpG context:\t28334\n",
      "Total methylated C's in CHG context:\t2922\n",
      "Total methylated C's in CHH context:\t18231\n",
      "Total methylated C's in Unknown context:\t1512\n",
      "\n",
      "Total unmethylated C's in CpG context:\t16610\n",
      "Total unmethylated C's in CHG context:\t74763\n",
      "Total unmethylated C's in CHH context:\t226518\n",
      "Total unmethylated C's in Unknown context:\t6037\n",
      "\n",
      "C methylated in CpG context:\t63.0%\n",
      "C methylated in CHG context:\t3.8%\n",
      "C methylated in CHH context:\t7.4%\n",
      "C methylated in unknown context (CN or CHN):\t20.0%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 20s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_3_s1_R1.fastq.gz to zr2096_3_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_3_s1_R1.fastq.gz to zr2096_3_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_3_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_3_s1_R2.fastq.gz to zr2096_3_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_3_s1_R2.fastq.gz to zr2096_3_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_3_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tTNTTTTAAAATAATTTAAATAATTTTATATAAATNNNNNNNNNNNNNNNTATTTAAATTAAAAATTTAAATAAATTTTTTTAATTATTTATTGTTTAATG\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAAATAAACATTAAACAATAAATAATTAAAAAAATTTATTTAAATTTTTAATTTAAATAAATTAAAAATATTAAATTTATATAAAATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tCNCTTCAAAATAATTTAAATAATCTTATATAAATNNNNNNNNNNNNNNNCACCTAAACTAAAAACTCAAATAAACTTTTCTAATCACCTATCATCCAACA\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAGATAGATGTTGGATGATAGGTGATTAGAAAAGTTTATTTGAGTTTTTAGTTTAGGTGAGTTAAAAATATTGAATTTATATAAGATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_G_to_A.fastq and zr2096_3_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tCNCTTCAAAATAATTTAAATAATCTTATATAAATNNNNNNNNNNNNNNNCACCTAAACTAAAAACTCAAATAAACTTTTCTAATCACCTATCATCCAACA\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAGATAGATGTTGGATGATAGGTGATTAGAAAAGTTTATTTGAGTTTTTAGTTTAGGTGAGTTAAAAATATTGAATTTATATAAGATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_3_s1_R1.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_1:N:0:ACAGTG/1\t77\t*\t0\t0\t*\t*\t0\t0\tTNTTTTAAAATAATTTAAATAATTTTATATAAATNNNNNNNNNNNNNNNTATTTAAATTAAAAATTTAAATAAATTTTTTTAATTATTTATTGTTTAATG\tB#<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFF###############<<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1059:2153_2:N:0:ACAGTG/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNAAATAAACATTAAACAATAAATAATTAAAAAAATTTATTTAAATTTTTAATTTAAATAAATTAAAAATATTAAATTTATATAAAATTATTTAAAT\t####<BBFFFFFFFFFFFFF<FFFFFF<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFB\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_3_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    91863 (91.86%) aligned concordantly 0 times\n",
      "    2399 (2.40%) aligned concordantly exactly 1 time\n",
      "    5738 (5.74%) aligned concordantly >1 times\n",
      "8.14% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    91783 (91.78%) aligned concordantly 0 times\n",
      "    2455 (2.46%) aligned concordantly exactly 1 time\n",
      "    5762 (5.76%) aligned concordantly >1 times\n",
      "8.22% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    91808 (91.81%) aligned concordantly 0 times\n",
      "    2410 (2.41%) aligned concordantly exactly 1 time\n",
      "    5782 (5.78%) aligned concordantly >1 times\n",
      "8.19% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    91799 (91.80%) aligned concordantly 0 times\n",
      "    2405 (2.40%) aligned concordantly exactly 1 time\n",
      "    5796 (5.80%) aligned concordantly >1 times\n",
      "8.20% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_3_s1_R1.fastq.gz_C_to_T.fastq, zr2096_3_s1_R1.fastq.gz_G_to_A.fastq, zr2096_3_s1_R2.fastq.gz_C_to_T.fastq and zr2096_3_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t576214\n",
      "\n",
      "Total methylated C's in CpG context:\t56019\n",
      "Total methylated C's in CHG context:\t4247\n",
      "Total methylated C's in CHH context:\t18724\n",
      "Total methylated C's in Unknown context:\t2114\n",
      "\n",
      "Total unmethylated C's in CpG context:\t22188\n",
      "Total unmethylated C's in CHG context:\t118727\n",
      "Total unmethylated C's in CHH context:\t356309\n",
      "Total unmethylated C's in Unknown context:\t8473\n",
      "\n",
      "C methylated in CpG context:\t71.6%\n",
      "C methylated in CHG context:\t3.5%\n",
      "C methylated in CHH context:\t5.0%\n",
      "C methylated in unknown context (CN or CHN):\t20.0%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 31s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_4_s1_R1.fastq.gz to zr2096_4_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_4_s1_R1.fastq.gz to zr2096_4_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_4_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_4_s1_R2.fastq.gz to zr2096_4_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_4_s1_R2.fastq.gz to zr2096_4_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_4_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNGATTAGTGTATAGTGGTTGTTATGGAAATTAGNNNNNNNNNNNNNNNNNNNNNNNNNTGNANTNTTTTTATTAGAATTGTTAAGGTTTATTTGATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTCCATCATTAATTTTTAAAAAAAAAAAACCATCCTTAATATCAATTTTATAAATCAAATAAACCTTAACAATTCTAATAAAAACACTTCATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATTAATATATAATAATTATTATAAAAATTAANNNNNNNNNNNNNNNNNNNNNNNNNTANANTNTTTTTATTAAAATTATTAAAATTTATTTAATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTTTATTATTAATTTTTAAAAAAAAAAAATTATTTTTAATATTAATTTTATAAATTAAATAAATTTTAATAATTTTAATAAAAATATTTTATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_G_to_A.fastq and zr2096_4_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATTAATATATAATAATTATTATAAAAATTAANNNNNNNNNNNNNNNNNNNNNNNNNTANANTNTTTTTATTAAAATTATTAAAATTTATTTAATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTTTATTATTAATTTTTAAAAAAAAAAAATTATTTTTAATATTAATTTTATAAATTAAATAAATTTTAATAATTTTAATAAAAATATTTTATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_4_s1_R1.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_1:N:0:GCCAAT/1\t77\t*\t0\t0\t*\t*\t0\t0\tNGATTAGTGTATAGTGGTTGTTATGGAAATTAGNNNNNNNNNNNNNNNNNNNNNNNNNTGNANTNTTTTTATTAGAATTGTTAAGGTTTATTTGATTTAT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF#########################<<#<#/#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1343:2133_2:N:0:GCCAAT/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTCCATCATTAATTTTTAAAAAAAAAAAACCATCCTTAATATCAATTTTATAAATCAAATAAACCTTAACAATTCTAATAAAAACACTTCATATAT\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFFF/BBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_4_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    92886 (92.89%) aligned concordantly 0 times\n",
      "    1942 (1.94%) aligned concordantly exactly 1 time\n",
      "    5172 (5.17%) aligned concordantly >1 times\n",
      "7.11% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    92793 (92.79%) aligned concordantly 0 times\n",
      "    1988 (1.99%) aligned concordantly exactly 1 time\n",
      "    5219 (5.22%) aligned concordantly >1 times\n",
      "7.21% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    92883 (92.88%) aligned concordantly 0 times\n",
      "    1896 (1.90%) aligned concordantly exactly 1 time\n",
      "    5221 (5.22%) aligned concordantly >1 times\n",
      "7.12% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    92757 (92.76%) aligned concordantly 0 times\n",
      "    2045 (2.04%) aligned concordantly exactly 1 time\n",
      "    5198 (5.20%) aligned concordantly >1 times\n",
      "7.24% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_4_s1_R1.fastq.gz_C_to_T.fastq, zr2096_4_s1_R1.fastq.gz_G_to_A.fastq, zr2096_4_s1_R2.fastq.gz_C_to_T.fastq and zr2096_4_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t473549\n",
      "\n",
      "Total methylated C's in CpG context:\t33523\n",
      "Total methylated C's in CHG context:\t3283\n",
      "Total methylated C's in CHH context:\t20431\n",
      "Total methylated C's in Unknown context:\t1818\n",
      "\n",
      "Total unmethylated C's in CpG context:\t22212\n",
      "Total unmethylated C's in CHG context:\t93475\n",
      "Total unmethylated C's in CHH context:\t300625\n",
      "Total unmethylated C's in Unknown context:\t7507\n",
      "\n",
      "C methylated in CpG context:\t60.1%\n",
      "C methylated in CHG context:\t3.4%\n",
      "C methylated in CHH context:\t6.4%\n",
      "C methylated in unknown context (CN or CHN):\t19.5%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 16s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_5_s1_R1.fastq.gz to zr2096_5_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_5_s1_R1.fastq.gz to zr2096_5_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_5_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_5_s1_R2.fastq.gz to zr2096_5_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_5_s1_R2.fastq.gz to zr2096_5_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_5_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATAAAATATTATTTAAATTTTATTATTTATANNNNNNNNNNNNNNNNNNATTNTTTTTATTATATATATAAAATTATTTTAAAATTATTATATTTTTT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTCAAATATATTAAAAAAAAATTATTAATAATTTTCATTAAAAAATATAATAATTTTAAAATAATTTTATATATATAATAAAAAAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAACAAAACACCACTCAAATCTTACTATCCACANNNNNNNNNNNNNNNNNNATTNCTTCTATCATATATATAAAATCATTCCAAAATTACTATATTTTTC\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTTGAGTATATTGAAGAGGAATTATTGATAGTTTTTGTTGAAAAATATAGTAATTTTGGAATGATTTTATATATATGATAGAAGAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_G_to_A.fastq and zr2096_5_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAACAAAACACCACTCAAATCTTACTATCCACANNNNNNNNNNNNNNNNNNATTNCTTCTATCATATATATAAAATCATTCCAAAATTACTATATTTTTC\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTTGAGTATATTGAAGAGGAATTATTGATAGTTTTTGTTGAAAAATATAGTAATTTTGGAATGATTTTATATATATGATAGAAGAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_5_s1_R1.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_1:N:0:CAGATC/1\t77\t*\t0\t0\t*\t*\t0\t0\tNAATAAAATATTATTTAAATTTTATTATTTATANNNNNNNNNNNNNNNNNNATTNTTTTTATTATATATATAAAATTATTTTAAAATTATTATATTTTTT\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF##################<<<#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1218:2142_2:N:0:CAGATC/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNTCAAATATATTAAAAAAAAATTATTAATAATTTTCATTAAAAAATATAATAATTTTAAAATAATTTTATATATATAATAAAAAAAATAATAATAAT\t###<<BFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFB\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_5_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    92759 (92.76%) aligned concordantly 0 times\n",
      "    2102 (2.10%) aligned concordantly exactly 1 time\n",
      "    5139 (5.14%) aligned concordantly >1 times\n",
      "7.24% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    92716 (92.72%) aligned concordantly 0 times\n",
      "    2057 (2.06%) aligned concordantly exactly 1 time\n",
      "    5227 (5.23%) aligned concordantly >1 times\n",
      "7.28% overall alignment rate\n",
      "100000100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    92603 (92.60%) aligned concordantly 0 times\n",
      "    2140 (2.14%) aligned concordantly exactly 1 time\n",
      "    5257 reads; of these: (5.26%) aligned concordantly >1 times\n",
      "\n",
      "  100000 (7.40100.00%% overall alignment rate\n",
      ") were paired; of these:\n",
      "    92582 (92.58%) aligned concordantly 0 times\n",
      "    2093 (2.09%) aligned concordantly exactly 1 time\n",
      "    5325 (5.33%) aligned concordantly >1 times\n",
      "7.42% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_5_s1_R1.fastq.gz_C_to_T.fastq, zr2096_5_s1_R1.fastq.gz_G_to_A.fastq, zr2096_5_s1_R2.fastq.gz_C_to_T.fastq and zr2096_5_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t502895\n",
      "\n",
      "Total methylated C's in CpG context:\t35809\n",
      "Total methylated C's in CHG context:\t3499\n",
      "Total methylated C's in CHH context:\t17515\n",
      "Total methylated C's in Unknown context:\t1864\n",
      "\n",
      "Total unmethylated C's in CpG context:\t22276\n",
      "Total unmethylated C's in CHG context:\t97253\n",
      "Total unmethylated C's in CHH context:\t326543\n",
      "Total unmethylated C's in Unknown context:\t7594\n",
      "\n",
      "C methylated in CpG context:\t61.6%\n",
      "C methylated in CHG context:\t3.5%\n",
      "C methylated in CHH context:\t5.1%\n",
      "C methylated in unknown context (CN or CHN):\t19.7%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 37s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_6_s1_R1.fastq.gz to zr2096_6_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_6_s1_R1.fastq.gz to zr2096_6_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_6_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_6_s1_R2.fastq.gz to zr2096_6_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_6_s1_R2.fastq.gz to zr2096_6_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_6_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTTATTAAATTTAATTTTATTATTAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNTANANAAATATTAATTAAATATTTTTAAAATAATAAATTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAAAATTTTTTTAAATTTATTATTTTAAAAATATTTAATTAATATTTATTTATATTTTATAATTAACAATAATTATTTTTTTATTTAATAATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTCATCAAATCCAACTTTATCACCAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNCANANAAACATCAATCAAACATCTCTAAAACAATAAACTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAGAATTTTTTTAAGTTTATTGTTTTAGAGATGTTTGATTGATGTTTATTTGTATTTTGTAATTAGTGGTGGTTGTTTTTTTATTTGGTGATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_G_to_A.fastq and zr2096_6_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTCATCAAATCCAACTTTATCACCAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNCANANAAACATCAATCAAACATCTCTAAAACAATAAACTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAGAATTTTTTTAAGTTTATTGTTTTAGAGATGTTTGATTGATGTTTATTTGTATTTTGTAATTAGTGGTGGTTGTTTTTTTATTTGGTGATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_6_s1_R1.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_1:N:0:CTTGTA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNTTATTAAATTTAATTTTATTATTAAATAAAAANNNNNNNNNNNNNNNNNNNNNNANNTANANAAATATTAATTAAATATTTTTAAAATAATAAATTTAA\t#<<BBFFFFFFFFFFFFFFFFFFFFFFFFFFFF######################<##<<#<#<<FFFFFFFFBFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1093:2134_2:N:0:CTTGTA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNTAAAAATTTTTTTAAATTTATTATTTTAAAAATATTTAATTAATATTTATTTATATTTTATAATTAACAATAATTATTTTTTTATTTAATAATAA\t#####BBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFBFFFFFFFFFFFFFFBFFFFBFFFFB/\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_6_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    92820 (92.82%) aligned concordantly 0 times\n",
      "    2036 (2.04%) aligned concordantly exactly 1 time\n",
      "    5144 (5.14%) aligned concordantly >1 times\n",
      "7.18% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    92979 (92.98%) aligned concordantly 0 times\n",
      "    1916 (1.92%) aligned concordantly exactly 1 time\n",
      "    5105 (5.11%) aligned concordantly >1 times\n",
      "7.02% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    92455 (92.45%) aligned concordantly 0 times\n",
      "    2103 (2.10%) aligned concordantly exactly 1 time\n",
      "    5442 (5.44%) aligned concordantly >1 times\n",
      "7.54% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    92512 (92.51%) aligned concordantly 0 times\n",
      "    2113 (2.11%) aligned concordantly exactly 1 time\n",
      "    5375 (5.38%) aligned concordantly >1 times\n",
      "7.49% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_6_s1_R1.fastq.gz_C_to_T.fastq, zr2096_6_s1_R1.fastq.gz_G_to_A.fastq, zr2096_6_s1_R2.fastq.gz_C_to_T.fastq and zr2096_6_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t480251\n",
      "\n",
      "Total methylated C's in CpG context:\t38844\n",
      "Total methylated C's in CHG context:\t3593\n",
      "Total methylated C's in CHH context:\t18891\n",
      "Total methylated C's in Unknown context:\t1850\n",
      "\n",
      "Total unmethylated C's in CpG context:\t18198\n",
      "Total unmethylated C's in CHG context:\t95056\n",
      "Total unmethylated C's in CHH context:\t305669\n",
      "Total unmethylated C's in Unknown context:\t7667\n",
      "\n",
      "C methylated in CpG context:\t68.1%\n",
      "C methylated in CHG context:\t3.6%\n",
      "C methylated in CHH context:\t5.8%\n",
      "C methylated in unknown context (CN or CHN):\t19.4%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 21s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_7_s1_R1.fastq.gz to zr2096_7_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_7_s1_R1.fastq.gz to zr2096_7_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_7_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_7_s1_R2.fastq.gz to zr2096_7_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_7_s1_R2.fastq.gz to zr2096_7_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_7_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAAAAAATGATTATTATGATGAATTTAAATATTAATAATAAAATTTAAAAAAAATATAAATGTTAAATATTAAATTTTTTAAAAAATGATTATGG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t141\t*\t0\t0\t*\t*\t0\t0\tCCATAATCATTTTTTAAAAAATTTAATATTTAACATTTATATTTTTTTTAAATTTTATTATTAATATTTAAATTCATCATAATAATCATTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAAAAAACAACTACTACAACAAATTCAAATATTAATAACAAAATTTAAAAAAAACATAAACATTAAATATCAAATTCTTCAAAAAACAATTACAA\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTGTAATTGTTTTTTGAAGAATTTGATATTTAATGTTTATGTTTTTTTTAAATTTTGTTATTAATATTTGAATTTGTTGTAGTAGTTGTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_G_to_A.fastq and zr2096_7_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAAAAAACAACTACTACAACAAATTCAAATATTAATAACAAAATTTAAAAAAAACATAAACATTAAATATCAAATTCTTCAAAAAACAATTACAA\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTGTAATTGTTTTTTGAAGAATTTGATATTTAATGTTTATGTTTTTTTTAAATTTTGTTATTAATATTTGAATTTGTTGTAGTAGTTGTTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_7_s1_R1.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_1:N:0:ATCACG/1\t77\t*\t0\t0\t*\t*\t0\t0\tAAAAAAATGATTATTATGATGAATTTAAATATTAATAATAAAATTTAAAAAAAATATAAATGTTAAATATTAAATTTTTTAAAAAATGATTATGG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1132:2189_2:N:0:ATCACG/2\t141\t*\t0\t0\t*\t*\t0\t0\tCCATAATCATTTTTTAAAAAATTTAATATTTAACATTTATATTTTTTTTAAATTTTATTATTAATATTTAAATTCATCATAATAATCATTTTTTT\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF/FFBBFFFFFFBBFFFFFFFFFFFFFFFFFFFFFFF\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_7_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    91587 (91.59%) aligned concordantly 0 times\n",
      "    2630 (2.63%) aligned concordantly exactly 1 time\n",
      "    5783 (5.78%) aligned concordantly >1 times\n",
      "8.41% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    91812 (91.81%) aligned concordantly 0 times\n",
      "    2575 (2.58%) aligned concordantly exactly 1 time\n",
      "    5613 (5.61%) aligned concordantly >1 times\n",
      "8.19% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    91716 (91.72%) aligned concordantly 0 times\n",
      "    2704 (2.70%) aligned concordantly exactly 1 time\n",
      "    5580 (5.58%) aligned concordantly >1 times\n",
      "8.28% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    91655 (91.66%) aligned concordantly 0 times\n",
      "    2630 (2.63%) aligned concordantly exactly 1 time\n",
      "    5715 (5.71%) aligned concordantly >1 times\n",
      "8.35% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_7_s1_R1.fastq.gz_C_to_T.fastq, zr2096_7_s1_R1.fastq.gz_G_to_A.fastq, zr2096_7_s1_R2.fastq.gz_C_to_T.fastq and zr2096_7_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t589017\n",
      "\n",
      "Total methylated C's in CpG context:\t46592\n",
      "Total methylated C's in CHG context:\t4453\n",
      "Total methylated C's in CHH context:\t20285\n",
      "Total methylated C's in Unknown context:\t2133\n",
      "\n",
      "Total unmethylated C's in CpG context:\t23209\n",
      "Total unmethylated C's in CHG context:\t118241\n",
      "Total unmethylated C's in CHH context:\t376237\n",
      "Total unmethylated C's in Unknown context:\t8629\n",
      "\n",
      "C methylated in CpG context:\t66.7%\n",
      "C methylated in CHG context:\t3.6%\n",
      "C methylated in CHH context:\t5.1%\n",
      "C methylated in unknown context (CN or CHN):\t19.8%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 26s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_8_s1_R1.fastq.gz to zr2096_8_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_8_s1_R1.fastq.gz to zr2096_8_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_8_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_8_s1_R2.fastq.gz to zr2096_8_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_8_s1_R2.fastq.gz to zr2096_8_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_8_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATTTAAAATTAAAAAATTTATTAAATTTATTATTATTTNTTATTTTTTATAAAAGAAAATAGAGAGAGAGAGAGAGAGAGAGAGAG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tCTCTCTCTCTCTCTCTCTCTCTCTCTATTTTCTTTTATAAAAAATAATAAATAATAATAAATTTAATAAATTTTTTAATTTTAAATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATCCAAAATTAAAAAACTCATTAAATCTATCATCATTCNCCATTCTTCACAAAAAAAAATAAAAAAAAAAAAAAAAAAAAAAAAAA\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTTTTTTTTTTTTTTTTTTTTTATTTTTTTTTGTGAAGAATGGTGAATGATGATAGATTTAATGAGTTTTTTAATTTTGGATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_G_to_A.fastq and zr2096_8_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATCCAAAATTAAAAAACTCATTAAATCTATCATCATTCNCCATTCTTCACAAAAAAAAATAAAAAAAAAAAAAAAAAAAAAAAAAA\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tTTTTTTTTTTTTTTTTTTTTTTTTTTATTTTTTTTTGTGAAGAATGGTGAATGATGATAGATTTAATGAGTTTTTTAATTTTGGATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_8_s1_R1.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_1:N:0:TTAGGC/1\t77\t*\t0\t0\t*\t*\t0\t0\tTAATTTAAAATTAAAAAATTTATTAAATTTATTATTATTTNTTATTTTTTATAAAAGAAAATAGAGAGAGAGAGAGAGAGAGAGAGAG\tBBBBBFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF#<<FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF<FFFFFFFFF\tYT:Z:UP\n",
      "HWI-C00124:321:CC781ANXX:1:1101:1281:2170_2:N:0:TTAGGC/2\t141\t*\t0\t0\t*\t*\t0\t0\tCTCTCTCTCTCTCTCTCTCTCTCTCTATTTTCTTTTATAAAAAATAATAAATAATAATAAATTTAATAAATTTTTTAATTTTAAATTA\tBBBBBFFFFFFFFFFFFFFFFFFFFF/FFFF<FFFFBF</F</<<<<FF/<FF/<<</F//<FF</BFF/FFFFFF//<FFFB//BF/\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_8_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    92934 (92.93%) aligned concordantly 0 times\n",
      "    2153 (2.15%) aligned concordantly exactly 1 time\n",
      "    4913 (4.91%) aligned concordantly >1 times\n",
      "7.07% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    92847 (92.85%) aligned concordantly 0 times\n",
      "    2258 (2.26%) aligned concordantly exactly 1 time\n",
      "    4895 (4.89%) aligned concordantly >1 times\n",
      "7.15% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    92895 (92.89%) aligned concordantly 0 times\n",
      "    2191 (2.19%) aligned concordantly exactly 1 time\n",
      "    4914 (4.91%) aligned concordantly >1 times\n",
      "7.11% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    92932 (92.93%) aligned concordantly 0 times\n",
      "    2216 (2.22%) aligned concordantly exactly 1 time\n",
      "    4852 (4.85%) aligned concordantly >1 times\n",
      "7.07% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_8_s1_R1.fastq.gz_C_to_T.fastq, zr2096_8_s1_R1.fastq.gz_G_to_A.fastq, zr2096_8_s1_R2.fastq.gz_C_to_T.fastq and zr2096_8_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t492720\n",
      "\n",
      "Total methylated C's in CpG context:\t34212\n",
      "Total methylated C's in CHG context:\t3523\n",
      "Total methylated C's in CHH context:\t19647\n",
      "Total methylated C's in Unknown context:\t1841\n",
      "\n",
      "Total unmethylated C's in CpG context:\t21124\n",
      "Total unmethylated C's in CHG context:\t99079\n",
      "Total unmethylated C's in CHH context:\t315135\n",
      "Total unmethylated C's in Unknown context:\t7158\n",
      "\n",
      "C methylated in CpG context:\t61.8%\n",
      "C methylated in CHG context:\t3.4%\n",
      "C methylated in CHH context:\t5.9%\n",
      "C methylated in unknown context (CN or CHN):\t20.5%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 6s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n",
      "Path to Bowtie 2 specified as: bowtie2\n",
      "Bowtie seems to be working fine (tested command 'bowtie2 --version' [2.3.4])\n",
      "Output format is BAM (default)\n",
      "Alignments will be written out in BAM format. Samtools found here: '/Users/Shelly/anaconda3/bin/samtools'\n",
      "Reference genome folder provided is /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\t(absolute path is '/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/)'\n",
      "Processing sequences up to read no. 100000 from the input file\n",
      "\n",
      "Input files to be analysed (in current folder '/Users/Shelly/Documents/GitHub/C_virginica'):\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz\n",
      "/Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Library was specified to be not strand-specific (non-directional), therefore alignments to all four possible bisulfite strands (OT, CTOT, OB and CTOB) will be reported\n",
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\n",
      "Setting parallelization to single-threaded (default)\n",
      "\n",
      "Current working directory is: /Users/Shelly/Documents/GitHub/C_virginica\n",
      "\n",
      "Now reading in and storing sequence information of the genome specified in: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/\n",
      "\n",
      "Single-core mode: setting pid to 1\n",
      "\n",
      "Paired-end alignments will be performed\n",
      "=======================================\n",
      "\n",
      "The provided filenames for paired-end alignments are /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Input files are in FastQ format\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_9_s1_R1.fastq.gz to zr2096_9_s1_R1.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_9_s1_R1.fastq.gz to zr2096_9_s1_R1.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_9_s1_R1.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Processing reads up to sequence no. 100000 from /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "Writing a C -> T converted version of the input file zr2096_9_s1_R2.fastq.gz to zr2096_9_s1_R2.fastq.gz_C_to_T.fastq\n",
      "Writing a G -> A converted version of the input file zr2096_9_s1_R2.fastq.gz to zr2096_9_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Created C -> T as well as G -> A converted versions of the FastQ file zr2096_9_s1_R2.fastq.gz (100001 sequences in total)\n",
      "\n",
      "Input files are zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq (FastQ)\n",
      "Now running 4 individual instances of Bowtie 2 against the bisulfite genome of /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ with the specified options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500\n",
      "\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2CTgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNATTAAATAAAAAATAATAAATAATAAATTANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAGG\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAAAAAATAAAAAACACATTAAAAATCTATAATTTAATTTATTATTTATTATTTTTTATTTAATTAAATCNNNNAAACATCATATAAAAAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2GAgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --norc))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNACCAAATAAAAAATAACAAATAATAAATCANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAAA\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAGAGGATAGGAGATATGTTAGGAATTTATAATTTGATTTATTATTTGTTATTTTTTATTTGGTTAGATTNNNNGAGTGTTGTGTAGGGAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for GAread1CTread2CTgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_G_to_A.fastq and zr2096_9_s1_R2.fastq.gz_C_to_T.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNACCAAATAAAAAATAACAAATAATAAATCANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAAA\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAGAGGATAGGAGATATGTTAGGAATTTATAATTTGATTTATTATTTGTTATTTTTTATTTGGTTAGATTNNNNGAGTGTTGTGTAGGGAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "Now starting a Bowtie 2 paired-end alignment for CTread1GAread2GAgenome (reading in sequences from zr2096_9_s1_R1.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq, with the options: -q --score-min L,0,-1.2 --ignore-quals --no-mixed --no-discordant --dovetail --minins 150 --maxins 500 --nofw))\n",
      "Found first alignment:\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_1:N:0:ACTTGA/1\t77\t*\t0\t0\t*\t*\t0\t0\tNATTAAATAAAAAATAATAAATAATAAATTANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTATAGG\t#/<<</<<B<<B<FF<BBFFBFFFF<B/FFF####################################<<<<FBF\tYT:Z:UP\tYF:Z:NS\n",
      "HWI-C00124:321:CC781ANXX:2:1101:1135:2128_2:N:0:ACTTGA/2\t141\t*\t0\t0\t*\t*\t0\t0\tNNNNNAANTAAAAAAATAAAAAACACATTAAAAATCTATAATTTAATTTATTATTTATTATTTTTTATTTAATTAAATCNNNNAAACATCATATAAAAAA\t#####<<#</BFFFB/FBFBF<FFFFF/FBFF//<FFF/</<FFF//</<<B/FB//FB/<FFFFF/B/F/FFFFB///####<</FFF<FBFF/F</7B\tYT:Z:UP\n",
      "\n",
      ">>> Writing bisulfite mapping results to zr2096_9_s1_R1_bismark_bt2_pe.bam <<<\n",
      "\n",
      "\n",
      "Reading in the sequence files /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz and /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    93300 (93.30%) aligned concordantly 0 times\n",
      "    2067 (2.07%) aligned concordantly exactly 1 time\n",
      "    4633 (4.63%) aligned concordantly >1 times\n",
      "6.70% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    93391 (93.39%) aligned concordantly 0 times\n",
      "    2010 (2.01%) aligned concordantly exactly 1 time\n",
      "    4599 (4.60%) aligned concordantly >1 times\n",
      "6.61% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    92479 (92.48%) aligned concordantly 0 times\n",
      "    2233 (2.23%) aligned concordantly exactly 1 time\n",
      "    5288 (5.29%) aligned concordantly >1 times\n",
      "7.52% overall alignment rate\n",
      "100000 reads; of these:\n",
      "  100000 (100.00%) were paired; of these:\n",
      "    92498 (92.50%) aligned concordantly 0 times\n",
      "    2267 (2.27%) aligned concordantly exactly 1 time\n",
      "    5235 (5.24%) aligned concordantly >1 times\n",
      "7.50% overall alignment rate\n",
      "Processed 100000 sequences in total\n",
      "\n",
      "\n",
      "Successfully deleted the temporary files zr2096_9_s1_R1.fastq.gz_C_to_T.fastq, zr2096_9_s1_R1.fastq.gz_G_to_A.fastq, zr2096_9_s1_R2.fastq.gz_C_to_T.fastq and zr2096_9_s1_R2.fastq.gz_G_to_A.fastq\n",
      "\n",
      "Final Alignment report\n",
      "======================\n",
      "Sequence pairs analysed in total:\t100000\n",
      "Final Cytosine Methylation Report\n",
      "=================================\n",
      "Total number of C's analysed:\t508437\n",
      "\n",
      "Total methylated C's in CpG context:\t46216\n",
      "Total methylated C's in CHG context:\t3861\n",
      "Total methylated C's in CHH context:\t17749\n",
      "Total methylated C's in Unknown context:\t2103\n",
      "\n",
      "Total unmethylated C's in CpG context:\t19413\n",
      "Total unmethylated C's in CHG context:\t102854\n",
      "Total unmethylated C's in CHH context:\t318344\n",
      "Total unmethylated C's in Unknown context:\t7946\n",
      "\n",
      "C methylated in CpG context:\t70.4%\n",
      "C methylated in CHG context:\t3.6%\n",
      "C methylated in CHH context:\t5.3%\n",
      "C methylated in unknown context (CN or CHN):\t20.9%\n",
      "\n",
      "\n",
      "Bismark completed in 0d 0h 4m 27s\n",
      "\n",
      "====================\n",
      "Bismark run complete\n",
      "====================\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "find /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_*R1* \\\n",
    "| xargs basename -s _s1_R1.fastq.gz | xargs -I{} /Users/Shelly/anaconda3/bin/bismark \\\n",
    "--genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ \\\n",
    "--score_min L,0,-1.2 \\\n",
    "-u 100000 \\\n",
    "-I 150 \\\n",
    "--non_directional \\\n",
    "-1 /Volumes/web/seashell/bu-serine-wd/18-04-07/{}_s1_R1.fastq.gz \\\n",
    "-2 /Volumes/web/seashell/bu-serine-wd/18-04-07/{}_s1_R2.fastq.gz \\\n",
    "-o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/ "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#run deduplicate on fragment length threshold (150bp) alignments"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_10_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 150 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_10_s1_R1_bismark_bt2_pe.deduplicated.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 150 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_10_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_1_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 150 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_1_s1_R1_bismark_bt2_pe.deduplicated.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 150 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_1_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_2_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 150 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_2_s1_R1_bismark_bt2_pe.deduplicated.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 150 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_2_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_3_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 150 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_3_s1_R1_bismark_bt2_pe.deduplicated.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 150 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_3_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_4_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 150 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_4_s1_R1_bismark_bt2_pe.deduplicated.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 150 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_4_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_5_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 150 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_5_s1_R1_bismark_bt2_pe.deduplicated.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 150 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_5_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_6_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 150 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_6_s1_R1_bismark_bt2_pe.deduplicated.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 150 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_6_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_7_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 150 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_7_s1_R1_bismark_bt2_pe.deduplicated.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 150 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_7_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_8_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 150 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_8_s1_R1_bismark_bt2_pe.deduplicated.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 150 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_8_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_9_s1_R1_bismark_bt2_pe.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 150 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\"\n",
      "\n",
      "Now testing Bismark result file /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_9_s1_R1_bismark_bt2_pe.deduplicated.bam for positional sorting (which would be bad...)\tskipping header line:\t@HD\tVN:1.0\tSO:unsorted\n",
      "skipping header line:\t@SQ\tSN:NC_035780.1\tLN:65668440\n",
      "skipping header line:\t@SQ\tSN:NC_035781.1\tLN:61752955\n",
      "skipping header line:\t@SQ\tSN:NC_035782.1\tLN:77061148\n",
      "skipping header line:\t@SQ\tSN:NC_035783.1\tLN:59691872\n",
      "skipping header line:\t@SQ\tSN:NC_035784.1\tLN:98698416\n",
      "skipping header line:\t@SQ\tSN:NC_035785.1\tLN:51258098\n",
      "skipping header line:\t@SQ\tSN:NC_035786.1\tLN:57830854\n",
      "skipping header line:\t@SQ\tSN:NC_035787.1\tLN:75944018\n",
      "skipping header line:\t@SQ\tSN:NC_035788.1\tLN:104168038\n",
      "skipping header line:\t@SQ\tSN:NC_035789.1\tLN:32650045\n",
      "skipping header line:\t@SQ\tSN:NC_007175.2\tLN:17244\n",
      "skipping header line:\t@PG\tID:Bismark\tVN:v0.20.0\tCL:\"bismark --genome /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/ --score_min L,0,-1.2 -u 100000 -I 150 --non_directional -1 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R1.fastq.gz -2 /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_9_s1_R2.fastq.gz -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\"\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Output will be written into the directory: /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/\n",
      "Processing paired-end Bismark output file(s) (SAM format):\n",
      "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_10_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_10_s1_R1_bismark_bt2_pe.deduplicated.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_1_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_1_s1_R1_bismark_bt2_pe.deduplicated.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_2_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_2_s1_R1_bismark_bt2_pe.deduplicated.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_3_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_3_s1_R1_bismark_bt2_pe.deduplicated.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_4_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_4_s1_R1_bismark_bt2_pe.deduplicated.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_5_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_5_s1_R1_bismark_bt2_pe.deduplicated.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_6_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_6_s1_R1_bismark_bt2_pe.deduplicated.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_7_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_7_s1_R1_bismark_bt2_pe.deduplicated.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_8_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_8_s1_R1_bismark_bt2_pe.deduplicated.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_9_s1_R1_bismark_bt2_pe.bam\t/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_9_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "If there are several alignments to a single position in the genome the first alignment will be chosen. Since the input files are not in any way sorted this is a near-enough random selection of reads.\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_10_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_10_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_10_s1_R1_bismark_bt2_pe.bam:\t16486\n",
      "Total number duplicated alignments removed:\t3 (0.02%)\n",
      "Duplicated alignments were found at:\t3 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 16483 (99.98% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_10_s1_R1_bismark_bt2_pe.deduplicated.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_10_s1_R1_bismark_bt2_pe.deduplicated.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_10_s1_R1_bismark_bt2_pe.deduplicated.bam:\t16483\n",
      "Total number duplicated alignments removed:\t0 (0.00%)\n",
      "Duplicated alignments were found at:\t0 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 16483 (100.00% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_1_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_1_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_1_s1_R1_bismark_bt2_pe.bam:\t7693\n",
      "Total number duplicated alignments removed:\t10 (0.13%)\n",
      "Duplicated alignments were found at:\t6 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 7683 (99.87% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_1_s1_R1_bismark_bt2_pe.deduplicated.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_1_s1_R1_bismark_bt2_pe.deduplicated.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_1_s1_R1_bismark_bt2_pe.deduplicated.bam:\t7683\n",
      "Total number duplicated alignments removed:\t0 (0.00%)\n",
      "Duplicated alignments were found at:\t0 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 7683 (100.00% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_2_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_2_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_2_s1_R1_bismark_bt2_pe.bam:\t10983\n",
      "Total number duplicated alignments removed:\t9 (0.08%)\n",
      "Duplicated alignments were found at:\t8 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 10974 (99.92% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_2_s1_R1_bismark_bt2_pe.deduplicated.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_2_s1_R1_bismark_bt2_pe.deduplicated.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_2_s1_R1_bismark_bt2_pe.deduplicated.bam:\t10974\n",
      "Total number duplicated alignments removed:\t0 (0.00%)\n",
      "Duplicated alignments were found at:\t0 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 10974 (100.00% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_3_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_3_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_3_s1_R1_bismark_bt2_pe.bam:\t16093\n",
      "Total number duplicated alignments removed:\t3 (0.02%)\n",
      "Duplicated alignments were found at:\t3 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 16090 (99.98% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_3_s1_R1_bismark_bt2_pe.deduplicated.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_3_s1_R1_bismark_bt2_pe.deduplicated.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_3_s1_R1_bismark_bt2_pe.deduplicated.bam:\t16090\n",
      "Total number duplicated alignments removed:\t0 (0.00%)\n",
      "Duplicated alignments were found at:\t0 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 16090 (100.00% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_4_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_4_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_4_s1_R1_bismark_bt2_pe.bam:\t14079\n",
      "Total number duplicated alignments removed:\t3 (0.02%)\n",
      "Duplicated alignments were found at:\t3 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 14076 (99.98% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_4_s1_R1_bismark_bt2_pe.deduplicated.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_4_s1_R1_bismark_bt2_pe.deduplicated.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_4_s1_R1_bismark_bt2_pe.deduplicated.bam:\t14076\n",
      "Total number duplicated alignments removed:\t0 (0.00%)\n",
      "Duplicated alignments were found at:\t0 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 14076 (100.00% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_5_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_5_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_5_s1_R1_bismark_bt2_pe.bam:\t14504\n",
      "Total number duplicated alignments removed:\t4 (0.03%)\n",
      "Duplicated alignments were found at:\t4 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 14500 (99.97% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_5_s1_R1_bismark_bt2_pe.deduplicated.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_5_s1_R1_bismark_bt2_pe.deduplicated.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_5_s1_R1_bismark_bt2_pe.deduplicated.bam:\t14500\n",
      "Total number duplicated alignments removed:\t0 (0.00%)\n",
      "Duplicated alignments were found at:\t0 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 14500 (100.00% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_6_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_6_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_6_s1_R1_bismark_bt2_pe.bam:\t14166\n",
      "Total number duplicated alignments removed:\t2 (0.01%)\n",
      "Duplicated alignments were found at:\t2 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 14164 (99.99% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_6_s1_R1_bismark_bt2_pe.deduplicated.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_6_s1_R1_bismark_bt2_pe.deduplicated.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_6_s1_R1_bismark_bt2_pe.deduplicated.bam:\t14164\n",
      "Total number duplicated alignments removed:\t0 (0.00%)\n",
      "Duplicated alignments were found at:\t0 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 14164 (100.00% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_7_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_7_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_7_s1_R1_bismark_bt2_pe.bam:\t16821\n",
      "Total number duplicated alignments removed:\t8 (0.05%)\n",
      "Duplicated alignments were found at:\t8 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 16813 (99.95% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_7_s1_R1_bismark_bt2_pe.deduplicated.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_7_s1_R1_bismark_bt2_pe.deduplicated.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_7_s1_R1_bismark_bt2_pe.deduplicated.bam:\t16813\n",
      "Total number duplicated alignments removed:\t0 (0.00%)\n",
      "Duplicated alignments were found at:\t0 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 16813 (100.00% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_8_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_8_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_8_s1_R1_bismark_bt2_pe.bam:\t14354\n",
      "Total number duplicated alignments removed:\t2 (0.01%)\n",
      "Duplicated alignments were found at:\t2 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 14352 (99.99% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_8_s1_R1_bismark_bt2_pe.deduplicated.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_8_s1_R1_bismark_bt2_pe.deduplicated.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_8_s1_R1_bismark_bt2_pe.deduplicated.bam:\t14352\n",
      "Total number duplicated alignments removed:\t0 (0.00%)\n",
      "Duplicated alignments were found at:\t0 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 14352 (100.00% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_9_s1_R1_bismark_bt2_pe.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_9_s1_R1_bismark_bt2_pe.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_9_s1_R1_bismark_bt2_pe.bam:\t14107\n",
      "Total number duplicated alignments removed:\t9 (0.06%)\n",
      "Duplicated alignments were found at:\t9 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 14098 (99.94% of total)\n",
      "\n",
      "Checking file >>/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_9_s1_R1_bismark_bt2_pe.deduplicated.bam<< for signs of file truncation...\n",
      "\n",
      "\n",
      "...passed!\n",
      "Output file is: zr2096_9_s1_R1_bismark_bt2_pe.deduplicated.deduplicated.bam\n",
      "\n",
      "\n",
      "Total number of alignments analysed in /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_9_s1_R1_bismark_bt2_pe.deduplicated.bam:\t14098\n",
      "Total number duplicated alignments removed:\t0 (0.00%)\n",
      "Duplicated alignments were found at:\t0 different position(s)\n",
      "\n",
      "Total count of deduplicated leftover sequences: 14098 (100.00% of total)\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "/Users/Shelly/anaconda3/bin/deduplicate_bismark \\\n",
    "--bam -p \\\n",
    "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/*.bam \\\n",
    "-o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/ \n",
    "2> /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/dedup.err"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#create summary report of mapping efficiency and dedup. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/*PE_report.txt | \\\n",
    "grep 'Mapping\\ efficiency\\:' | \\\n",
    "cat - /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/*.deduplication_report.txt > /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_mapping_dedup_summary.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#combine summary mapping efficiency and dedup reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!paste -d' ' \\\n",
    "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/zr2096_mapping_dedup_summary.txt \\\n",
    "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/zr2096_mapping_dedup_summary.txt \\\n",
    "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/zr2096_mapping_dedup_summary.txt \\\n",
    "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/zr2096_mapping_dedup_summary.txt \\\n",
    "/Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/zr2096_mapping_dedup_summary.txt \\\n",
    "| grep 'Mapping\\ efficiency\\:\\|removed' \\\n",
    "> /Volumes/web/metacarcinus/Cvirginica/bismarkASthreshold_comparison.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#clean up summary report file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#this first sed code replaces the string 'Mapping efficiency:' with nothing\n",
    "#this 2nd sed code replaces the string 'Total number duplicated alignments removed:' with nothing\n",
    "#this 3rd sed code replaces all spaces with nothing; awk code fixes spacing between columns\n",
    "#next I opened the file in vim and added a header ('AS_06' = L,0,-0.6;'AS_12' = L,0,-1.2; 'AS_2' = L,0,-2; 'AS_12I60' = L,0,-12 & -I 60 ) and row names ('Lib[1-10]')\n",
    "!sed 's/Mapping\\ efficiency\\://g' /Volumes/web/metacarcinus/Cvirginica/bismarkASthreshold_comparison.txt | \\\n",
    "sed 's/Total\\ number\\ duplicated\\ alignments\\ removed\\://g' | \\\n",
    "sed 's/ //g' | awk '{print $1,$2,$3,$4,$5}' > /Volumes/web/metacarcinus/Cvirginica/bismarkASthreshold_comparison_clean.txt\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#compiling bam files and sorting"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "find /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/*deduplicated.bam| \\\n",
    "xargs basename -s _s1_R1_bismark_bt2_pe.deduplicated.bam | xargs -I{} /Users/Shelly/anaconda3/bin/samtools \\\n",
    "sort /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/{}_s1_R1_bismark_bt2_pe.deduplicated.bam -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt2/{}_dedup.sorted.bam\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "find /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/*deduplicated.bam| \\\n",
    "xargs basename -s _s1_R1_bismark_bt2_pe.deduplicated.bam | xargs -I{} /Users/Shelly/anaconda3/bin/samtools \\\n",
    "sort /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/{}_s1_R1_bismark_bt2_pe.deduplicated.bam -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt3/{}_dedup.sorted.bam\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "find /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/*deduplicated.bam| \\\n",
    "xargs basename -s _s1_R1_bismark_bt2_pe.deduplicated.bam | xargs -I{} /Users/Shelly/anaconda3/bin/samtools \\\n",
    "sort /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/{}_s1_R1_bismark_bt2_pe.deduplicated.bam -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt4/{}_dedup.sorted.bam\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "find /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/*deduplicated.bam| \\\n",
    "xargs basename -s _s1_R1_bismark_bt2_pe.deduplicated.bam | xargs -I{} /Users/Shelly/anaconda3/bin/samtools \\\n",
    "sort /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/{}_s1_R1_bismark_bt2_pe.deduplicated.bam -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt5/{}_dedup.sorted.bam\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "find /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/*deduplicated.bam| \\\n",
    "xargs basename -s _s1_R1_bismark_bt2_pe.deduplicated.bam | xargs -I{} /Users/Shelly/anaconda3/bin/samtools \\\n",
    "sort /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/{}_s1_R1_bismark_bt2_pe.deduplicated.bam -o /Volumes/web/metacarcinus/Cvirginica/Bismark_attempt6/{}_dedup.sorted.bam\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
