{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#################METHYLPY#######################"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#create directory for methylpy analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!mkdir /Volumes/web/metacarcinus/Cvirginica/methylpy"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#build methylpy minimap2 genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!mkdir /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[M::mm_idx_gen::38.404*0.95] collected minimizers\n",
      "[M::mm_idx_gen::47.421*1.13] sorted minimizers\n",
      "[M::main::2013.673*0.04] loaded/built the index for 11 target sequence(s)\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 11\n",
      "[M::mm_idx_stat::2013.735*0.04] distinct minimizers: 5077906 (20.29% are singletons); average occurrences: 25.657; average spacing: 5.256\n",
      "[M::main] Version: 2.12-r827\n",
      "[M::main] CMD: /Users/Shelly/anaconda3/bin//minimap2 -t 2 -d /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/methylpy_mm_f.mmi /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/methylpy_mm_f.fasta\n",
      "[M::main] Real time: 2025.256 sec; CPU: 71.179 sec\n",
      "[M::mm_idx_gen::100.105*0.39] collected minimizers\n",
      "[M::mm_idx_gen::108.670*0.51] sorted minimizers\n",
      "[M::main::2443.507*0.03] loaded/built the index for 11 target sequence(s)\n",
      "[M::mm_idx_stat] kmer size: 15; skip: 10; is_hpc: 0; #seq: 11\n",
      "[M::mm_idx_stat::2443.591*0.03] distinct minimizers: 5077799 (20.31% are singletons); average occurrences: 25.660; average spacing: 5.255\n",
      "[M::main] Version: 2.12-r827\n",
      "[M::main] CMD: /Users/Shelly/anaconda3/bin//minimap2 -t 2 -d /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/methylpy_mm_r.mmi /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/methylpy_mm_r.fasta\n",
      "[M::main] Real time: 2455.184 sec; CPU: 80.244 sec\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "methylpy build-reference \\\n",
    "--input-files /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/Cvirginica_v300.fa \\\n",
    "--output-prefix /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/methylpy_mm \\\n",
    "--num-procs 2 \\\n",
    "--path-to-aligner /Users/Shelly/anaconda3/bin/ \\\n",
    "--aligner minimap2 \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#run methylpy with minimap2 alignment"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "find /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_*R1* \\\n",
    "| xargs basename -s _s1_R1.fastq.gz | xargs -I{} /Users/Shelly/anaconda3/bin/methylpy paired-end-pipeline \\\n",
    "--read1-files /Volumes/web/seashell/bu-serine-wd/18-04-07/{}_s1_R1.fastq.gz \\\n",
    "--read2-files /Volumes/web/seashell/bu-serine-wd/18-04-07/{}_s1_R2.fastq.gz \\\n",
    "--sample  {}_Cvirgv300_pe_mm\\\n",
    "--path-to-output /Volumes/web/metacarcinus/Cvirginica/methylpy/ \\\n",
    "--forward-ref /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/methylpy_mm_f \\\n",
    "--reverse-ref /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/methylpy_mm_r \\\n",
    "--ref-fasta /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/Cvirginica_v300.fa \\\n",
    "--num-procs 2 \\\n",
    "--path-to-aligner /Users/Shelly/anaconda3/bin/minimap2 \\\n",
    "--aligner minimap2 \\\n",
    "--aligner_options \\\n",
    "--remove-clonal True \\\n",
    "--path-to-picard=\"picard/\" \\\n",
    "--add-snp-info True \\\n",
    "--binom-test True\\\n",
    "--unmethylated-control 0.005"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#build methylpy bowtie2 genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Settings:\n",
      "  Output files: \"/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/methylpy_bt_f.*.bt2\"\n",
      "  Line rate: 6 (line is 64 bytes)\n",
      "  Lines per side: 1 (side is 64 bytes)\n",
      "  Offset rate: 4 (one in 16)\n",
      "  FTable chars: 10\n",
      "  Strings: unpacked\n",
      "  Max bucket size: default\n",
      "  Max bucket size, sqrt multiplier: default\n",
      "  Max bucket size, len divisor: 4\n",
      "  Difference-cover sample period: 1024\n",
      "  Endianness: little\n",
      "  Actual local endianness: little\n",
      "  Sanity checking: disabled\n",
      "  Assertions: disabled\n",
      "  Random seed: 0\n",
      "  Sizeofs: void*:8, int:4, long:8, size_t:8\n",
      "Input files DNA, FASTA:\n",
      "  /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/methylpy_bt_f.fasta\n",
      "Settings:\n",
      "  Output files: \"/Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/methylpy_bt_r.*.bt2\"\n",
      "  Line rate: 6 (line is 64 bytes)\n",
      "  Lines per side: 1 (side is 64 bytes)\n",
      "  Offset rate: 4 (one in 16)\n",
      "  FTable chars: 10\n",
      "  Strings: unpacked\n",
      "  Max bucket size: default\n",
      "  Max bucket size, sqrt multiplier: default\n",
      "  Max bucket size, len divisor: 4\n",
      "  Difference-cover sample period: 1024\n",
      "  Endianness: little\n",
      "  Actual local endianness: little\n",
      "  Sanity checking: disabled\n",
      "  Assertions: disabled\n",
      "  Random seed: 0\n",
      "  Sizeofs: void*:8, int:4, long:8, size_t:8\n",
      "Input files DNA, FASTA:\n",
      "  /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/methylpy_bt_r.fasta\n",
      "Reading reference sizes\n",
      "Reading reference sizes\n",
      "  Time reading reference sizes: 00:00:26\n",
      "Calculating joined length\n",
      "Writing header\n",
      "Reserving space for joined string\n",
      "Joining reference sequences\n",
      "  Time reading reference sizes: 00:00:26\n",
      "Calculating joined length\n",
      "Writing header\n",
      "Reserving space for joined string\n",
      "Joining reference sequences\n",
      "  Time to join reference sequences: 00:00:07\n",
      "bmax according to bmaxDivN setting: 171168832\n",
      "Using parameters --bmax 128376624 --dcv 1024\n",
      "  Doing ahead-of-time memory usage test\n",
      "  Passed!  Constructing with these parameters: --bmax 128376624 --dcv 1024\n",
      "Constructing suffix-array element generator\n",
      "  Time to join reference sequences: 00:00:07\n",
      "bmax according to bmaxDivN setting: 171168832\n",
      "Using parameters --bmax 128376624 --dcv 1024\n",
      "  Doing ahead-of-time memory usage test\n",
      "  Passed!  Constructing with these parameters: --bmax 128376624 --dcv 1024\n",
      "Constructing suffix-array element generator\n",
      "Building DifferenceCoverSample\n",
      "Building DifferenceCoverSample\n",
      "  Building sPrime\n",
      "  Building sPrime\n",
      "  Building sPrimeOrder\n",
      "  Building sPrimeOrder\n",
      "  V-Sorting samples\n",
      "  V-Sorting samples\n",
      "  V-Sorting samples time: 00:00:29\n",
      "  Allocating rank array\n",
      "  Ranking v-sort output\n",
      "  V-Sorting samples time: 00:00:29\n",
      "  Allocating rank array\n",
      "  Ranking v-sort output\n",
      "  Ranking v-sort output time: 00:00:07\n",
      "  Invoking Larsson-Sadakane on ranks\n",
      "  Ranking v-sort output time: 00:00:07\n",
      "  Invoking Larsson-Sadakane on ranks\n",
      "  Invoking Larsson-Sadakane on ranks time: 00:00:10\n",
      "  Sanity-checking and returning\n",
      "Building samples\n",
      "Reserving space for 12 sample suffixes\n",
      "Generating random suffixes\n",
      "QSorting 12 sample offsets, eliminating duplicates\n",
      "QSorting sample offsets, eliminating duplicates time: 00:00:00\n",
      "Multikey QSorting 12 samples\n",
      "  (Using difference cover)\n",
      "  Multikey QSorting samples time: 00:00:00\n",
      "Calculating bucket sizes\n",
      "Splitting and merging\n",
      "  Splitting and merging time: 00:00:00\n",
      "Avg bucket size: 6.84675e+08 (target: 128376623)\n",
      "Converting suffix-array elements to index image\n",
      "Allocating ftab, absorbFtab\n",
      "Entering Ebwt loop\n",
      "Getting block 1 of 1\n",
      "  No samples; assembling all-inclusive block\n",
      "  Invoking Larsson-Sadakane on ranks time: 00:00:10\n",
      "  Sanity-checking and returning\n",
      "Building samples\n",
      "Reserving space for 12 sample suffixes\n",
      "Generating random suffixes\n",
      "QSorting 12 sample offsets, eliminating duplicates\n",
      "QSorting sample offsets, eliminating duplicates time: 00:00:00\n",
      "Multikey QSorting 12 samples\n",
      "  (Using difference cover)\n",
      "  Multikey QSorting samples time: 00:00:00\n",
      "Calculating bucket sizes\n",
      "Splitting and merging\n",
      "  Splitting and merging time: 00:00:00\n",
      "Avg bucket size: 6.84675e+08 (target: 128376623)\n",
      "Converting suffix-array elements to index image\n",
      "Allocating ftab, absorbFtab\n",
      "Entering Ebwt loop\n",
      "Getting block 1 of 1\n",
      "  No samples; assembling all-inclusive block\n",
      "  Sorting block of length 684675328 for bucket 1\n",
      "  (Using difference cover)\n",
      "  Sorting block of length 684675328 for bucket 1\n",
      "  (Using difference cover)\n",
      "  Sorting block time: 00:11:45\n",
      "Returning block of 684675329 for bucket 1\n",
      "  Sorting block time: 00:14:08\n",
      "Returning block of 684675329 for bucket 1\n",
      "Exited Ebwt loop\n",
      "fchr[A]: 0\n",
      "fchr[C]: 342402452\n",
      "fchr[G]: 461631571\n",
      "fchr[T]: 461631571\n",
      "fchr[$]: 684675328\n",
      "Exiting Ebwt::buildToDisk()\n",
      "Returning from initFromVector\n",
      "Wrote 232428899 bytes to primary EBWT file: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/methylpy_bt_r.1.bt2\n",
      "Wrote 171168840 bytes to secondary EBWT file: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/methylpy_bt_r.2.bt2\n",
      "Re-opening _in1 and _in2 as input streams\n",
      "Returning from Ebwt constructor\n",
      "Headers:\n",
      "    len: 684675328\n",
      "    bwtLen: 684675329\n",
      "    sz: 171168832\n",
      "    bwtSz: 171168833\n",
      "    lineRate: 6\n",
      "    offRate: 4\n",
      "    offMask: 0xfffffff0\n",
      "    ftabChars: 10\n",
      "    eftabLen: 20\n",
      "    eftabSz: 80\n",
      "    ftabLen: 1048577\n",
      "    ftabSz: 4194308\n",
      "    offsLen: 42792209\n",
      "    offsSz: 171168836\n",
      "    lineSz: 64\n",
      "    sideSz: 64\n",
      "    sideBwtSz: 48\n",
      "    sideBwtLen: 192\n",
      "    numSides: 3566018\n",
      "    numLines: 3566018\n",
      "    ebwtTotLen: 228225152\n",
      "    ebwtTotSz: 228225152\n",
      "    color: 0\n",
      "    reverse: 0\n",
      "Total time for call to driver() for forward index: 00:15:37\n",
      "Reading reference sizes\n",
      "  Time reading reference sizes: 00:00:09\n",
      "Calculating joined length\n",
      "Writing header\n",
      "Reserving space for joined string\n",
      "Joining reference sequences\n",
      "  Time to join reference sequences: 00:00:06\n",
      "  Time to reverse reference sequence: 00:00:00\n",
      "bmax according to bmaxDivN setting: 171168832\n",
      "Using parameters --bmax 128376624 --dcv 1024\n",
      "  Doing ahead-of-time memory usage test\n",
      "  Passed!  Constructing with these parameters: --bmax 128376624 --dcv 1024\n",
      "Constructing suffix-array element generator\n",
      "Building DifferenceCoverSample\n",
      "  Building sPrime\n",
      "  Building sPrimeOrder\n",
      "  V-Sorting samples\n",
      "  V-Sorting samples time: 00:00:27\n",
      "  Allocating rank array\n",
      "  Ranking v-sort output\n",
      "  Ranking v-sort output time: 00:00:08\n",
      "  Invoking Larsson-Sadakane on ranks\n",
      "  Invoking Larsson-Sadakane on ranks time: 00:00:11\n",
      "  Sanity-checking and returning\n",
      "Building samples\n",
      "Reserving space for 12 sample suffixes\n",
      "Generating random suffixes\n",
      "QSorting 12 sample offsets, eliminating duplicates\n",
      "QSorting sample offsets, eliminating duplicates time: 00:00:00\n",
      "Multikey QSorting 12 samples\n",
      "  (Using difference cover)\n",
      "  Multikey QSorting samples time: 00:00:00\n",
      "Calculating bucket sizes\n",
      "Splitting and merging\n",
      "  Splitting and merging time: 00:00:00\n",
      "Avg bucket size: 6.84675e+08 (target: 128376623)\n",
      "Converting suffix-array elements to index image\n",
      "Allocating ftab, absorbFtab\n",
      "Entering Ebwt loop\n",
      "Getting block 1 of 1\n",
      "  No samples; assembling all-inclusive block\n",
      "  Sorting block of length 684675328 for bucket 1\n",
      "  (Using difference cover)\n",
      "Exited Ebwt loop\n",
      "fchr[A]: 0\n",
      "fchr[C]: 223134775\n",
      "fchr[G]: 223134775\n",
      "fchr[T]: 342402452\n",
      "fchr[$]: 684675328\n",
      "Exiting Ebwt::buildToDisk()\n",
      "Returning from initFromVector\n",
      "Wrote 232428899 bytes to primary EBWT file: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/methylpy_bt_f.1.bt2\n",
      "Wrote 171168840 bytes to secondary EBWT file: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/methylpy_bt_f.2.bt2\n",
      "Re-opening _in1 and _in2 as input streams\n",
      "Returning from Ebwt constructor\n",
      "Headers:\n",
      "    len: 684675328\n",
      "    bwtLen: 684675329\n",
      "    sz: 171168832\n",
      "    bwtSz: 171168833\n",
      "    lineRate: 6\n",
      "    offRate: 4\n",
      "    offMask: 0xfffffff0\n",
      "    ftabChars: 10\n",
      "    eftabLen: 20\n",
      "    eftabSz: 80\n",
      "    ftabLen: 1048577\n",
      "    ftabSz: 4194308\n",
      "    offsLen: 42792209\n",
      "    offsSz: 171168836\n",
      "    lineSz: 64\n",
      "    sideSz: 64\n",
      "    sideBwtSz: 48\n",
      "    sideBwtLen: 192\n",
      "    numSides: 3566018\n",
      "    numLines: 3566018\n",
      "    ebwtTotLen: 228225152\n",
      "    ebwtTotSz: 228225152\n",
      "    color: 0\n",
      "    reverse: 0\n",
      "Total time for call to driver() for forward index: 00:18:05\n",
      "Reading reference sizes\n",
      "  Time reading reference sizes: 00:00:09\n",
      "Calculating joined length\n",
      "Writing header\n",
      "Reserving space for joined string\n",
      "Joining reference sequences\n",
      "  Time to join reference sequences: 00:00:06\n",
      "  Time to reverse reference sequence: 00:00:01\n",
      "bmax according to bmaxDivN setting: 171168832\n",
      "Using parameters --bmax 128376624 --dcv 1024\n",
      "  Doing ahead-of-time memory usage test\n",
      "  Passed!  Constructing with these parameters: --bmax 128376624 --dcv 1024\n",
      "Constructing suffix-array element generator\n",
      "Building DifferenceCoverSample\n",
      "  Building sPrime\n",
      "  Building sPrimeOrder\n",
      "  V-Sorting samples\n",
      "  V-Sorting samples time: 00:00:31\n",
      "  Allocating rank array\n",
      "  Ranking v-sort output\n",
      "  Ranking v-sort output time: 00:00:07\n",
      "  Invoking Larsson-Sadakane on ranks\n",
      "  Invoking Larsson-Sadakane on ranks time: 00:00:10\n",
      "  Sanity-checking and returning\n",
      "Building samples\n",
      "Reserving space for 12 sample suffixes\n",
      "Generating random suffixes\n",
      "QSorting 12 sample offsets, eliminating duplicates\n",
      "QSorting sample offsets, eliminating duplicates time: 00:00:00\n",
      "Multikey QSorting 12 samples\n",
      "  (Using difference cover)\n",
      "  Multikey QSorting samples time: 00:00:00\n",
      "Calculating bucket sizes\n",
      "Splitting and merging\n",
      "  Splitting and merging time: 00:00:00\n",
      "Avg bucket size: 6.84675e+08 (target: 128376623)\n",
      "Converting suffix-array elements to index image\n",
      "Allocating ftab, absorbFtab\n",
      "Entering Ebwt loop\n",
      "Getting block 1 of 1\n",
      "  No samples; assembling all-inclusive block\n",
      "  Sorting block of length 684675328 for bucket 1\n",
      "  (Using difference cover)\n",
      "  Sorting block time: 00:11:47\n",
      "Returning block of 684675329 for bucket 1\n",
      "Exited Ebwt loop\n",
      "fchr[A]: 0\n",
      "fchr[C]: 342402452\n",
      "fchr[G]: 461631571\n",
      "fchr[T]: 461631571\n",
      "fchr[$]: 684675328\n",
      "Exiting Ebwt::buildToDisk()\n",
      "Returning from initFromVector\n",
      "Wrote 232428899 bytes to primary EBWT file: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/methylpy_bt_r.rev.1.bt2\n",
      "Wrote 171168840 bytes to secondary EBWT file: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/methylpy_bt_r.rev.2.bt2\n",
      "Re-opening _in1 and _in2 as input streams\n",
      "Returning from Ebwt constructor\n",
      "Headers:\n",
      "    len: 684675328\n",
      "    bwtLen: 684675329\n",
      "    sz: 171168832\n",
      "    bwtSz: 171168833\n",
      "    lineRate: 6\n",
      "    offRate: 4\n",
      "    offMask: 0xfffffff0\n",
      "    ftabChars: 10\n",
      "    eftabLen: 20\n",
      "    eftabSz: 80\n",
      "    ftabLen: 1048577\n",
      "    ftabSz: 4194308\n",
      "    offsLen: 42792209\n",
      "    offsSz: 171168836\n",
      "    lineSz: 64\n",
      "    sideSz: 64\n",
      "    sideBwtSz: 48\n",
      "    sideBwtLen: 192\n",
      "    numSides: 3566018\n",
      "    numLines: 3566018\n",
      "    ebwtTotLen: 228225152\n",
      "    ebwtTotSz: 228225152\n",
      "    color: 0\n",
      "    reverse: 1\n",
      "Total time for backward call to driver() for mirror index: 00:15:09\n",
      "  Sorting block time: 00:13:36\n",
      "Returning block of 684675329 for bucket 1\n",
      "Exited Ebwt loop\n",
      "fchr[A]: 0\n",
      "fchr[C]: 223134775\n",
      "fchr[G]: 223134775\n",
      "fchr[T]: 342402452\n",
      "fchr[$]: 684675328\n",
      "Exiting Ebwt::buildToDisk()\n",
      "Returning from initFromVector\n",
      "Wrote 232428899 bytes to primary EBWT file: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/methylpy_bt_f.rev.1.bt2\n",
      "Wrote 171168840 bytes to secondary EBWT file: /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/methylpy_bt_f.rev.2.bt2\n",
      "Re-opening _in1 and _in2 as input streams\n",
      "Returning from Ebwt constructor\n",
      "Headers:\n",
      "    len: 684675328\n",
      "    bwtLen: 684675329\n",
      "    sz: 171168832\n",
      "    bwtSz: 171168833\n",
      "    lineRate: 6\n",
      "    offRate: 4\n",
      "    offMask: 0xfffffff0\n",
      "    ftabChars: 10\n",
      "    eftabLen: 20\n",
      "    eftabSz: 80\n",
      "    ftabLen: 1048577\n",
      "    ftabSz: 4194308\n",
      "    offsLen: 42792209\n",
      "    offsSz: 171168836\n",
      "    lineSz: 64\n",
      "    sideSz: 64\n",
      "    sideBwtSz: 48\n",
      "    sideBwtLen: 192\n",
      "    numSides: 3566018\n",
      "    numLines: 3566018\n",
      "    ebwtTotLen: 228225152\n",
      "    ebwtTotSz: 228225152\n",
      "    color: 0\n",
      "    reverse: 1\n",
      "Total time for backward call to driver() for mirror index: 00:16:59\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Building a SMALL index\n",
      "Building a SMALL index\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "methylpy build-reference \\\n",
    "--input-files /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/Cvirginica_v300.fa \\\n",
    "--output-prefix /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/methylpy_bt \\\n",
    "--num-procs 2 \\\n",
    "--path-to-aligner /Users/Shelly/anaconda3/bin/ \\\n",
    "--aligner bowtie2\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#run methylpy with bowtie2 alignment"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Process is interrupted.\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "find /Volumes/web/seashell/bu-serine-wd/18-04-07/zr2096_*R1* \\\n",
    "| xargs basename -s _s1_R1.fastq.gz | \\\n",
    "xargs -I{} /Users/Shelly/anaconda3/bin/methylpy paired-end-pipeline \\\n",
    "--read1-files /Volumes/web/seashell/bu-serine-wd/18-04-07/{}_s1_R1.fastq.gz \\\n",
    "--read2-files /Volumes/web/seashell/bu-serine-wd/18-04-07/{}_s1_R2.fastq.gz \\\n",
    "--sample {}_Cvirgv300_pe_bt2 \\\n",
    "--path-to-output /Volumes/web/metacarcinus/Cvirginica/methylpy/ \\\n",
    "--forward-ref /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/methylpy_bt_f \\\n",
    "--reverse-ref /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/methylpy_genome/methylpy_bt_r \\\n",
    "--ref-fasta /Volumes/web/metacarcinus/Cvirginica/GENOMES/v300/Bismark_genome/Cvirginica_v300.fa \\\n",
    "--num-procs 2 \\\n",
    "--aligner bowtie2 \\\n",
    "--aligner-options \"--score-min L,0,-1.2 -u 100000 -I 60 --no-mixed --no-discordant\" \\\n",
    "--remove-clonal True \\\n",
    "--path-to-picard /Users/Shelly/anaconda3/bin/ \\\n",
    "--path-to-samtools /Users/Shelly/anaconda3/bin/ \\\n",
    "--add-snp-info True \\\n",
    "--binom-test True \\\n",
    "--unmethylated-control 0.005\n",
    "    \n",
    "    \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#run DMR finder"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "methylpy DMRfind \\\n",
    "\t--allc-files allc/allc_AD_HT.tsv.gz allc/allc_AD_IT.tsv.gz \\\n",
    "\t--samples AD_HT AD_IT \\\n",
    "\t--mc-type \"CGN\" \\\n",
    "\t--chroms 1 2 3 4 5 \\\n",
    "\t--num-procs 8 \\\n",
    "\t--output-prefix DMR_HT_IT"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
